---
layout: post
title:  "10G-1 aptamer"
date:   2024-05-05 00:00:00
author: Ke Chen, Yangyi Ren
categories: Aptamer
---
<html>
<head>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
    h1, .h1 {
    font-size: 30px;
}
  </style>
</head>
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
  .header_box {
    display: block;
    font-size: 20px;
    font-weight: bold;
    background-color: #ffffff;
    text-decoration: none;
    border-radius: 1px;
    width: 500px;
    border-width: 1px 1px 2px 1px;
    border-color: #ffffff #ffffff #ffffff #ffffff;
}
.blowheader_box{
    display: block;
      padding: 6px;
      font-size:20px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 1px;
      width:190px;
      height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
  .dot-paragraph::before {
            content: "• "; /* 点号和空格 */
            color: black; /* 设置点号颜色 */
            font-size: 20px; /* 调整点号大小 */
        }
  .dot-paragraph {
            margin: 5px 0; /* 调整带有点的段落的上下外边距 */
            line-height: 1.2; /* 调整带有点的段落的行高 */
        }
  .sequence-container {
      position: relative;
      max-width: 100%;
      white-space: normal;
      overflow-wrap: break-word;
    }
    .sequence-text {
      display: inline-block;
      white-space: nowrap;
      max-width: 100%;
      overflow: hidden;
    }
    .show-more {
      display: inline-block;
      color: #520049;
      cursor: pointer;
      font-weight: bold;
      text-decoration: underline;
    }
    .full-sequence {
      display: none;
    }
    .sequence-container.expanded .sequence-text {
      display: none;
    }
    .sequence-container.expanded .full-sequence {
      display: inline;
    }
    .sequence-container.collapsed .sequence-text {
      white-space: normal;
      display: inline-block;
      max-width: 100%;
    }
    * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
</style>
</head>
<br>
<br>

<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/9310370/" target="_blank" style="color:#520049">1997</a></h3>
    </div>
    <div class="body">
      <p>An aptamer called 10G-1, which binds specifically to the NS3 protease, was selected for the first time by in vitro genetic-selection strategy<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/9356339/" target="_blank" style="color:#520049">1997</a></h3>
    </div>
    <div class="body">
      <p>Nishikawa, S. et al. isolated aptamers that inhibited the activity of NS3 protease and helicase<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10848986/" target="_blank" style="color:#520049">2000</a></h3>
    </div>
    <div class="body">
      <p>Nishikawa, S. et al. isolated RNA aptamers that bind specifically to the NS3 protease active site in the truncated polypeptide ΔNS3<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/11118325/" target="_blank" style="color:#520049">2000</a></h3>
    </div>
    <div class="body">
      <p>The binding sites of NS3 protease domain and aptamer were analyzed by alanine scanning mutagenesis<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12655010/" target="_blank" style="color:#520049">2003</a></h3>
    </div>
    <div class="body">
      <p>Nishikawa, S. et al. constructed a G9 aptamer expression system in cultured cells, using the cytomegarovirus enhancer + chicken beta-actin globin (CAG) promoter<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/15541341/" target="_blank" style="color:#520049">2004</a></h3>
    </div>
    <div class="body">
      <p>Nishikawa, S. et al. designed a new aptamer called NEO-III-14U to inhibit both protease and helicase activity<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/22814430/" target="_blank" style="color:#520049">2012</a></h3>
    </div>
    <div class="body">
      <p>Kao, C. C. et al. showed that RNAs can bind directly to the active site cleft of the NS3 protease domain (NS3P) and inhibit proteolysis of peptide substrates. RNAs that are less apt to form intramolecular structures have a stronger inhibitory activity than RNAs with more stable base paired regions<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
    </div>
  </div>
</div>


<p class="header_box" id="description">Description</p>
<p>In 1997, Nishikawa, S. and colleagues used an in vitro genetic-selection strategy to isolate high-affinity RNA aptamers that bind to the NS3 protein, especially to its protease domain. Starting from a RNA pool that had a random sequence core of 12-18 nucleotides, aptamers that bind specifically to the NS3 protein were selected after 10 rounds of selection and amplification. A single aptamer, 10G-1, was found predominantly (71%) in the selected pool. This aptamer could bind to the NS3 protein with a binding constant of 650 nM and inhibit the proteolytic activity in vitro. By phosphate-modification-interference analysis we showed that the phosphate residues that are critical for the binding of 10G-1 to NS3 lie within the selected regions of the aptamer and that binding involves electrostatic contacts with the phosphates of regions G28-U34 and A47-A55<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>
<br>
<br>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 1997, Nishikawa, S. and colleagues used existing research to design the method required for the in vitro genetic-selection. Starting from a RNA pool that had a random sequence core of 12-18 nucleotides, aptamers that bind specifically to the NS3 protein were selected after 10 rounds of selection and amplification. A single aptamer, 10G-1, was found predominantly (71%) in the selected pool<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>10G-1 was the aptamer sequence mainly studied in the article, which had a high affinity with HCV NS3 protease. The 2D structure of the figure is based on the article by ribodraw tool to draw. The 10G-1 aptamer was named by Nishikawa, S. et al. in the article<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>5'-GGGAACUCGAUGAAGCGAAUUCUGUUGGCGAACUGUACGCAAGUACACUGGAUGACAGCCUAUCUAUCGGAUCCACG-3'</p>
<img src="/images/2D/10G-1_aptamer_2D1.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>
<br>


<p class="header_box" id="ligand-recognition">Ligand information</p> 

<p class="blowheader_box">SELEX ligand</p>
<p>Hepatitis C virus NS3 protein is a serine protease which has a trypsin-like fold. The non-structural (NS) protein NS3 is one of the NS proteins involved in replication of the HCV genome. The action of NS3 protease (NS3P), which resides in the N-terminal one-third of the NS3 protein, then yields all remaining non-structural proteins.-----From <a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF02907/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">Mass</th>
        <th onclick="sortTable(4)">Protein sequence</th>
        <th onclick="sortTable(5)">PDB ID</th>
        <th onclick="sortTable(6)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">HCV NS3 protease</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/B2Y2M9/entry#sequences" target="_blank" style="color:#520049"><b>B2Y2M9</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF02907/" target="_blank" style="color:#520049"><b>PF02907</b></a></td>
        <td name="td3">19.15 kDa</td>
        <td name="td4">
        <div class="sequence-container">
          <span class="sequence-text"></span>
          <span class="show-more" onclick="toggleSequence(event)">......</span>
          <span class="full-sequence">APITAYAQQTRGLLGCIITSLTGRDKNQVEGEVQIVSTATQTFLATCINGVCWTVYHGAGTRTIASPKGPVIQMYTNVDQDLVGWPAPQGSRSLTPCTCGSSDLYLVTRHADVIPVRRRGDSRGSLLSPRPISYLKGSSGGPLLCPAGHAVGLFRAAVCTRGVAKAVDFIPVENLETTMRS (Residues 1027-1207)</span>
        </div>
        </td>
        <td name="td5"><a href="https://www.rcsb.org/structure/3KF2" target="_blank" style="color:#520049"><b>3KF2</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/protein/ABY67662.1" target="_blank" style="color:#520049"><b>ABY67662.1</b></a></td>
      </tr>
	  </tbody>
  </table>

<p>Some isolated sequences bind to the affinity of the protein<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">10G-1 aptamer</td>
        <td name="td1">5'-GGGAACUCGAUGAAGCGAAUUCUGUUGGCGAACUGUACGCAAGUACACUGGAUGACAGCCUAUCUAUCGGAUCCACG-3'</td>
        <td name="td2">HCV NS3 protease</td>
        <td name="td3">650 nM</td>
      </tr>
      <tr>
        <td name="td0">10G-2 aptamer</td>
        <td name="td1">5'-GGGAACUCGAUGAAGCGAAUUUGUCGGGAUGUACGCAAGUACAUGUUCUCACAGGCCUAUCUAUCGGAUCCACG-3'</td>
        <td name="td2">HCV NS3 protease</td>
        <td name="td3">NA</td>
      </tr>
      <tr>
        <td name="td0">10G-8 aptamer</td>
        <td name="td1">5'-GGGAACUCGAUGAAGCGAAUUUGUCGGGAUGUACGCAAGUACUUGGUUCCUACAGGCCUAUCUAUCGGAUCCACG-3'</td>
        <td name="td2">HCV NS3 protease</td>
        <td name="td3">NA</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/10G-1_aptamer_SELEX_ligand.svg" alt="drawing" style="width:1000px;height:350px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>



<p class="blowheader_box">Similar compound(s)</p>                    
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD (Å)</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/3KF2" target="_blank" style="color:#520049"><b>3KF2</b></a>-B</td>
      <td name="td1">37.5</td>
      <td name="td2">0</td>
      <td name="td3">Polyprotein</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/2OIN" target="_blank" style="color:#520049"><b>2OIN</b></a>-B</td>
      <td name="td1">35.7</td>
      <td name="td2">0.3</td>
      <td name="td3">Polyprotein</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/2O8M" target="_blank" style="color:#520049"><b>2O8M</b></a>-A</td>
      <td name="td1">35.7</td>
      <td name="td2">0.3</td>
      <td name="td3">Protease</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/3LON" target="_blank" style="color:#520049"><b>3LON</b></a>-A</td>
      <td name="td1">35.5</td>
      <td name="td2">0.3</td>
      <td name="td3">Genome polyprotein</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/2OC0" target="_blank" style="color:#520049"><b>2OC0</b></a>-A</td>
      <td name="td1">35.5</td>
      <td name="td2">0.3</td>
      <td name="td3">Hepatitis C virus</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/2A4R" target="_blank" style="color:#520049"><b>2A4R</b></a>-A</td>
      <td name="td1">35.3</td>
      <td name="td2">0.3</td>
      <td name="td3">NS3 protease/helicase</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/2QV1" target="_blank" style="color:#520049"><b>2QV1</b></a>-B</td>
      <td name="td1">35.3</td>
      <td name="td2">0.3</td>
      <td name="td3">NS3</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/3KN2" target="_blank" style="color:#520049"><b>3KN2</b></a>-A</td>
      <td name="td1">35.3</td>
      <td name="td2">0.3</td>
      <td name="td3">HCV NS3 protease domain</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/2A4Q" target="_blank" style="color:#520049"><b>2A4Q</b></a>-A</td>
      <td name="td1">35.3</td>
      <td name="td2">0.3</td>
      <td name="td3">NS3 protease/helicase</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/3EYD" target="_blank" style="color:#520049"><b>3EYD</b></a>-A</td>
      <td name="td1">35.3</td>
      <td name="td2">0.3</td>
      <td name="td3">HCV NS3</td>
    </tr>
    </tbody>
  </table>
<br>
<br>
                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Selection of RNA aptamers that bind specifically to the NS3 protease of hepatitis C virus.</strong></font><br />
Urvil, P. T., Kakiuchi, N., Zhou, D. M., Shimotohno, K., Kumar, P. K., & Nishikawa, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/9310370/" target="_blank" style="color:#520049">European journal of biochemistry, 248(1), 130–138. (1997)</a>
<br/>

<a id="ref2"></a><font><strong>[2] Isolation of RNA aptamers specific to the NS3 protein of hepatitis C virus from a pool of completely random RNA.</strong></font><br />
Kumar, P. K., Machida, K., Urvil, P. T., Kakiuchi, N., Vishnuvardhan, D., Shimotohno, K., Taira, K., & Nishikawa, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/9356339/" target="_blank" style="color:#520049" >Virology, 237(2), 270–282. (1997)</a>
<br/>

<a id="ref3"></a><font><strong>[3] Isolation and characterization of RNA aptamers specific for the hepatitis C virus nonstructural protein 3 protease.</strong></font><br />
Fukuda, K., Vishnuvardhan, D., Sekiya, S., Hwang, J., Kakiuchi, N., Taira, K., Shimotohno, K., Kumar, P. K., & Nishikawa, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10848986/" target="_blank" style="color:#520049" >European journal of biochemistry, 267(12), 3685–3694. (2000)</a>
<br/>

<a id="ref4"></a><font><strong>[4] The RNA aptamer-binding site of hepatitis C virus NS3 protease.</strong></font><br />
Hwang, J., Fauzi, H., Fukuda, K., Sekiya, S., Kakiuchi, N., Shimotohno, K., Taira, K., Kusakabe, I., & Nishikawa, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/11118325/" target="_blank" style="color:#520049" >Biochemical and biophysical research communications, 279(2), 557–562. (2000)</a>
<br/>

<a id="ref5"></a><font><strong>[5] Inhibition of HCV NS3 protease by RNA aptamers in cells.</strong></font><br />
Nishikawa, F., Kakiuchi, N., Funaji, K., Fukuda, K., Sekiya, S., & Nishikawa, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12655010/" target="_blank" style="color:#520049" >Nucleic acids research, 31(7), 1935–1943. (2003)</a>
<br/>

<a id="ref6"></a><font><strong>[6] An RNA ligand inhibits hepatitis C virus NS3 protease and helicase activities.</strong></font><br />
Fukuda, K., Umehara, T., Sekiya, S., Kunio, K., Hasegawa, T., & Nishikawa, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/15541341/" target="_blank" style="color:#520049" >Biochemical and biophysical research communications, 325(3), 670–675. (2004)</a>
<br/>

<a id="ref7"></a><font><strong>[7] RNA binding by the NS3 protease of the hepatitis C virus.</strong></font><br />
Vaughan, R., Li, Y., Fan, B., Ranjith-Kumar, C. T., & Kao, C. C.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/22814430/" target="_blank" style="color:#520049" >Virus research, 169(1), 80–90. (2012)</a>
<br/>


<script>
    function toggleSequence(event) {
      const container = event.target.closest('.sequence-container');
      container.classList.toggle('expanded');
      const showMoreText = container.querySelector('.show-more');
      
      // 展开后按钮文本变化
      if (container.classList.contains('expanded')) {
        showMoreText.textContent = '...';  // 展开后显示 "..."
      } else {
        showMoreText.textContent = '......';  // 收起后显示 "......"
      }
    }

    // 页面加载时，限制序列文本为50个字符
    window.addEventListener('load', function() {
      const sequenceContainers = document.querySelectorAll('.sequence-container');
      sequenceContainers.forEach(container => {
        const fullSeqText = container.querySelector('.full-sequence').textContent;
        const truncatedText = fullSeqText.slice(0, 50);  // 只显示前50个字符
        container.querySelector('.sequence-text').textContent = truncatedText;
      });
    });
  </script>