---
layout: post
title:  "AML1 (RUNX1) aptamer"
date:   2024-04-03 00:00:00
author: Baowei Huang, Yuxun Sun
categories: Aptamer
---
<html>
<head>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
    h1, .h1 {
    font-size: 30px;
}
  </style>
</head>
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
  .header_box {
    display: block;
    font-size: 20px;
    background-color: #ffffff;
    text-decoration: none;
    border-radius: 1px;
    width: 500px;
    border-width: 1px 1px 2px 1px;
    border-color: #ffffff #ffffff #ffffff #ffffff;
}
.blowheader_box{
    display: block;
      padding: 6px;
      font-size:20px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 1px;
      width:190px;
	    height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
  .dot-paragraph::before {
            content: "• "; /* 点号和空格 */
            color: black; /* 设置点号颜色 */
            font-size: 20px; /* 调整点号大小 */
        }
  .dot-paragraph {
            margin: 5px 0; /* 调整带有点的段落的上下外边距 */
            line-height: 1.2; /* 调整带有点的段落的行高 */
        }
  .sequence-container {
      position: relative;
      max-width: 100%;
      white-space: normal;
      overflow-wrap: break-word;
    }
    .sequence-text {
      display: inline-block;
      white-space: nowrap;
      max-width: 100%;
      overflow: hidden;
    }
    .show-more {
      display: inline-block;
      color: #520049;
      cursor: pointer;
      font-weight: bold;
      text-decoration: underline;
    }
    .full-sequence {
      display: none;
    }
    .sequence-container.expanded .sequence-text {
      display: none;
    }
    .sequence-container.expanded .full-sequence {
      display: inline;
    }
    .sequence-container.collapsed .sequence-text {
      white-space: normal;
      display: inline-block;
      max-width: 100%;
    }
</style>
</head>
<br>
<br>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC203359/" target="_blank" style="color:#520049">2000</a></h3>
    </div>
    <div class="body">
      <p>Report the 2.6 Å resolution crystal structure of the complex between the AML1 Runt domain and CBF, which represents a paradigm for the mode of interaction of this highly conserved family of transcription factors<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12604126/" target="_blank" style="color:#520049">2003</a></h3>
    </div>
    <div class="body">
      <p>Indicate that a decrease in AML1 dosage resulting from chromosomal translocations or mutations contributes to leukemogenesis. Furthermore, dysregulated chromatin remodeling and transcriptional control appears to be a common pathway in AML1-associated leukemias that could be an important target for the development of new therapeutic agents<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2777437/" target="_blank" style="color:#520049">2009</a></h3>
    </div>
    <div class="body">
      <p>Isolated high-affinity aptamers that alter the affinity of RUNX1 for DNA and investigated their effects on DNA binding and CBF complex formation. Identified minimal short stem-loop sequences from these consensus sequences which retain binding activity<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/20140822/" target="_blank" style="color:#520049">2010</a></h3>
    </div>
    <div class="body">
      <p>In development by Antisoma plc, AS-1411 is the first oligodeoxynucleotide aptamer to reach phase I and II clinical trials for the potential treatment of cancers, including acute myelogenous leukemia (AML)<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/23997091/" target="_blank" style="color:#520049">2013</a></h3>
    </div>
    <div class="body">
      <p>The motif contains the AHþC mismatch and base triple and adopts an unusual backbone structure. Structural analysis of the aptamer motif indicated that the aptamer binds to the Runt domain by mimicking the RDE sequence and structure<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3683927/" target="_blank" style="color:#520049">2013</a></h3>
    </div>
    <div class="body">
      <p>Performed SELEX to obtain RNA aptamers that bind specifically to the AML1 protein to use as tools for better understanding AML1 and its potential utility for the diagnosis and treatment of AML1-related diseases. found that all the selected aptamers possessed a unique RNA motif and that one of these aptamers likely binds to the Runt domain in a manner similar to the DNA consensus binding sequence<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3895837/" target="_blank" style="color:#520049">2014</a></h3>
    </div>
    <div class="body">
      <p>Demonstrated a pipeline approach for developing single stranded DNA aptamer probes, phenotyping AML cells in clinical specimens, and then identifying the aptamer-recognized target protein. The developed aptamer probes and identified Siglec-5 protein may potentially be used for leukemic cell detection and therapy in our future clinical practice<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/26204224/" target="_blank" style="color:#520049">2015</a></h3>
    </div>
    <div class="body">
      <p>Developed a single-strand DNA aptamer specific for the biomarker CD117, which is highly expressed on AML cells<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/27514505/" target="_blank" style="color:#520049">2016</a></h3>
    </div>
    <div class="body">
      <p>To evaluate an engineered nanostructure to silence five important oncogenes, including BAG1, MDM2, Bcl-2, BIRC5 (survivin) and XIAP, in acute myeloid leukemia subtype 2 (AML-M2)，using nanostructure consisted of gold nanoparticles functionalized with five AOs and one anti-CD33(+)/CD34(+) aptamer (functionalized gold nanoparticle (FGN))<sup>[<a href="#ref9" style="color:#520049">9</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/27766833/" target="_blank" style="color:#520049">2016</a></h3>
    </div>
    <div class="body">
      <p>Obtained high-affinity RNA aptamers against RD under highly stringent conditions. Among the selected aptamers, S4 showed the highest binding affinity (Kd =0.044 ± 0.002 nM), whereas the Kd of the most frequently observed S1 was 0.27 ± 0.02 nM. It was previously reported that the most frequently observed aptamer showed fast association rather than high affinity<sup>[<a href="#ref10" style="color:#520049">10</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/26972431/" target="_blank" style="color:#520049">2016</a></h3>
    </div>
    <div class="body">
      <p>In order to identify novel anticancer compounds, we applied peptide aptamers targeting translationally controlled tumor protein (TCTP), an interesting target protein overexpressed in many tumors and involved in various cancer-related pathways<sup>[<a href="#ref11" style="color:#520049">11</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/27766833/" target="_blank" style="color:#520049">2016</a></h3>
    </div>
    <div class="body">
      <p>Identified high-affinity RNA aptamers that bind to RD by systematic evolution of ligands by exponential enrichment<sup>[<a href="#ref12" style="color:#520049">12</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/28845698/" target="_blank" style="color:#520049">2017</a></h3>
    </div>
    <div class="body">
      <p>Developed the first CD123 (AML tumor marker) aptamers and designed a novel CD123-aptamer-mediated targeted drug train (TDT) with effective, economical, biocompatible and high drug-loading capacity<sup>[<a href="#ref13" style="color:#520049">13</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/30519686/" target="_blank" style="color:#520049">2019</a></h3>
    </div>
    <div class="body">
      <p>Modified approach can rapidly screen reliable, stable and high binding affinity aptamers for precise cancer treatment. successfully obtained the CD33-targeting aptamer S30, which could highly recognize the C2 domain of the CD33 antigen in vitro and in vivo. Moreover, the optimized aptamer S30-T1 (i.e., core region of S30) was conjugated with doxorubicin (Dox) to synthesize S30-T1-Dox conjugates, which could specifically inhibit CD33 positive acute myeloid leukemia HL-60 cell proliferation by arresting the cell cycle at the G2 phase<sup>[<a href="#ref14" style="color:#520049">14</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/31824168/" target="_blank" style="color:#520049">2019</a></h3>
    </div>
    <div class="body">
      <p>Summarized aptamers’ preparation, chemi- cal modification and conjugation, and discussed the appli- cation of aptamers in diagnosis and treatment of leukemia through highly specifically recognizing target molecules<sup>[<a href="#ref15" style="color:#520049">15</a>]</sup></p>
    </div>
  </div>
</div>


         
<p class="header_box" id="description">Description</p>
<p>In 2000, J. Warren et al. reported the crystal structure of the complex between the AML1 Runt domain and CBF at a resolution of 2.6 Å. In 2003, Asou et al. inferred that pathways associated with AML1 could be significant targets for the development of new therapeutic agents. In 2009, L. Barton et al. isolated high-affinity aptamers that modulate the affinity of RUNX1 for DNA and examined their effects on DNA binding and CBF complex formation. In 2013, Nomura et al. used nuclear magnetic resonance to elucidate the structural basis of Runt domain recognition by the aptamer motif, while Fukunaga et al. conducted SELEX to obtain RNA aptamers that bind specifically to the AML1 protein, thereby enhancing understanding of AML1 and its potential applications in the diagnosis and treatment of AML1-related diseases. In 2016, Amano et al. obtained high-affinity RNA aptamers against the Runt domain under highly stringent conditions. In 2019, Yang et al. developed a modified approach to rapidly screen for reliable, stable, and high-binding-affinity aptamers for precise cancer treatment, successfully obtaining the CD33-targeting aptamer S30, which demonstrated high recognition of the C2 domain of the CD33 antigen both in vitro and in vivo<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref2" style="color:#520049">2</a></sup><sup>,<a href="#ref3" style="color:#520049">3</a></sup><sup>,<a href="#ref5" style="color:#520049">5</a></sup><sup>,<a href="#ref6" style="color:#520049">6</a></sup><sup>,<a href="#ref10" style="color:#520049">10</a></sup><sup>,<a href="#ref14" style="color:#520049">14</a></sup><sup>]</sup>.<br></p>
<br>
<br>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2013, Junichi Fukunaga and his colleagues performed SELEX to obtain RNA aptamers that bind specifically to the AML1 protein. These aptamers were used as tools to enhance understanding of AML1 and its potential applications in the diagnosis and treatment of AML1-related diseases. After nine rounds of selection, the 30N and 40N random RNA pools converged into eight and four independent sequences, respectively. Surface plasmon resonance (SPR) assays were employed to test the formation of the aptamer–Runt–CBFβ ternary complex<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup>.</p>
<p>Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br>
<br>    


<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>The name "Apt1-S2" appears in the article where Nomura, Y. has analyzed the structure. Here we use ribodraw to complete the figure, through the 3D structure information<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup>.</p>
<p>5'-GGACCCACCACGGCGAGGUCCA-3'</p>
<img src="/images/2D/AML1_aptamer_2D.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">3D visualisation</p>             
<p>Nomura and his colleague determined the solution structure of a 22-mer RNA (AML1 (RUNX1)-aptamer) using nuclear magnetic resonance(NMR). Although structure of the Runt domain-aptamer complex could not be determined, comparison of the aptamer structure with RDE and dsRNA suggested that the aptamer motif binds to the Runt domain by mimicking the RDE sequence and structure. The PDB ID of this structure is 2RRC<sup>[<a href="#ref5" style="color:#520049">5</a></sup><sup>]</sup>.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/AML1_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A', 
  start_residue_number:1, 
  end_residue_number:4, 
  color:{r:184,g:51,b:50}
},
{
  struct_asym_id: 'A', 
  start_residue_number:18, 
  end_residue_number:22, 
  color:{r:184,g:51,b:50}
},{
  struct_asym_id: 'A', 
  start_residue_number:5, 
  end_residue_number:7, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'A', 
  start_residue_number:15, 
  end_residue_number:15, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'A', 
  start_residue_number:13, 
  end_residue_number:13, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'A', 
  start_residue_number:17, 
  end_residue_number:17, 
  color:{r:241,g:125,b:7}
},{
  struct_asym_id: 'A', 
  start_residue_number:7, 
  end_residue_number:8, 
  color:{r:46,g:161,b:3}
},{
  struct_asym_id: 'A', 
  start_residue_number:9, 
  end_residue_number:12, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'A', 
  start_residue_number:14, 
  end_residue_number:14, 
  color:{r:46,g:161,b:3}
},
{
  struct_asym_id: 'A', 
  start_residue_number:16, 
  end_residue_number:16, 
  color:{r:46,g:161,b:3}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/2rrc.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<p>This aptamer lacks a three-dimensional structure of its complex with the protein, but its binding site is similar to that of the RDE element-binding protein. We chose the complex structure of double-stranded DNA element(RDE) and AML1 to show it. The structural PDB ID is 1HJC. AML1/RUNX1 is an essential transcription factor involved in the differentiation of hematopoietic cells. AML1 binds to the Runt-binding double-stranded DNA element (RDE) of target genes through its N- terminal Runt domain. Left: Surface representation of the binding pocket. AML1/RUNX1 (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the DNA element bound with AML1/RUNX1.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/AML1_aptamer_binding_pockets1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/AML1_aptamer_binding_pockets2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>


<p class="header_box" id="ligand-recognition">Ligand information</p>
<p class="blowheader_box">SELEX ligand</p>
<p>Fukunaga and his colleagues performed SELEX to obtain RNA aptamers against the Runt domain of AML1. Fukunaga, J. employed the surface plasmon resonance (SPR) method to calculate the dissociation constant Kd, which is determined as the ratio of the dissociation rate constant Kd_off to the association rate constant K_on, expressed as K_off/Kd_on. The Apt1-S2 aptamer is a truncated sequence of Apt1-S<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup>.</p>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">APt1</td>
        <td name="td1">5'-GGGACGCAAUGGACGACCCACCACGGCGAGGUAUCCCAUUGCCCCUAACGGCCGACAUGAGAG-3'</td>
        <td name="td3">AML1–Runt</td>
        <td name="td4">3.6 nM</td>
      </tr>
      <tr>
        <td name="td0">Apt1-S</td>
        <td name="td1">5'-GGGAUGGACGACCCACCACGGCGAGGUAUCCCAUCCCA-3'</td>
        <td name="td3">AML1–Runt</td>
        <td name="td4">0.99 nM</td>
      </tr>
      <tr>
        <td name="td0">Apt1-S2</td>
        <td name="td1">5'-GGACCCACCACGGCGAGGUCCA-3'</td>
        <td name="td3">AML1–Runt</td>
        <td name="td4">NA</td>
      </tr>
      <tr>
        <td name="td0">RDE</td>
        <td name="td1">5'-TCCCCAAACCGCAAACGAC-3'<br>3'-GTCGTTTGCGGTTTGGGGA-5'</td>
        <td name="td3">AML1–Runt</td>
        <td name="td4">9.6 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<p>The Runt domain is an evolutionary conserved protein domain. The AML1/RUNX1 gene is rearranged by the t(8;21) translocation in acute myeloid leukemia.[2] The gene is highly similar to the Drosophila melanogaster segmentation gene runt and to the mouse transcription factor PEBP2 alpha subunit gene. The region of shared similarity, known as the Runt domain, is responsible for DNA-binding and protein-protein interaction.-----From <a href="https://en.wikipedia.org/wiki/Runt_domain" target="_blank" style="color:#520049; text-decoration: underline;"><b>Wiki</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">Mass</th>
        <th onclick="sortTable(3)">Protein sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/Q03347/entry" target="_blank" style="color:#520049"><b>Q03347</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR013524/" target="_blank" style="color:#520049"><b>IPR013524</b></a></td>
        <td name="td2">48.61 kDa</td>
        <td name="td3">
        <div class="sequence-container">
          <span class="sequence-text">GGPYLQ</span>
          <span class="show-more" onclick="toggleSequence(event)">......</span>
          <span class="full-sequence">MRIPVDASTSRRFTPPSTALSPGKMSEALPLGAPDGGPALASKLRSGDRSMVEVLADHPGELVRTDSPNFLCSVLPTHWRCNKTLPIAFKVVALGDVPDGTLVTVMAGNDENYSAELRNATAAMKNQVARFNDLRFVGRSGRGKSFTLTITVFTNPPQVATYHRAIKITVDGPREPRRHRQKLDDQTKPGSLSFSERLSELEQLRRTAMRVSPHHPAPTPNPRASLNHSTAFNPQPQSQMQDARQIQPSPPWSYDQSYQYLGSITSSSVHPATPISPGRASGMTSLSAELSSRLSTAPDLTAFGDPRQFPTLPSISDPRMHYPGAFTYSPPVTSGIGIGMSAMSSASRYHTYLPPPYPGSSQAQAGPFQTGSPSYHLYYGASAGSYQFSMVGGERSPPRILPPCTNASTGAALLNPSLPSQSDVVETEGSHSNSPTNMPPARLEEAVWRPY</span>
        </div>
      </td>
        <td name="td4"><a href="https://www.rcsb.org/structure/4L0Y" target="_blank" style="color:#520049"><b>4L0Y</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?lvl=0&id=10090" target="_blank" style="color:#520049"><b>10090</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/AML1_aptamer_stru_ligand1.svg" alt="drawing" style="width:800px;height:300px;"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">4L0Y-A</td>
        <td name="td2">28</td>
        <td name="td3">0</td>
        <td name="td4">Runt-related transcription factor 1</td>
      </tr>
      <tr>
        <td name="td1">4L0Z-A</td>
        <td name="td2">24.7</td>
        <td name="td3">0.8</td>
        <td name="td4">Runt-related transcription factor 1</td>
      </tr>
      <tr>
        <td name="td1">1UUS-A</td>
        <td name="td2">8.6</td>
        <td name="td3">3</td>
        <td name="td4">Stat protein</td>
      </tr>
      <tr>
        <td name="td1">1YVL-B</td>
        <td name="td2">7.6</td>
        <td name="td3">3.2</td>
        <td name="td4">Signal transducer and activator of transcription</td>
      </tr>
      <tr>
        <td name="td1">7XDI-E</td>
        <td name="td2">7.5</td>
        <td name="td3">3.9</td>
        <td name="td4">Vp1</td>
      </tr>
      <tr>
        <td name="td1">5I5K-B</td>
        <td name="td2">7.2</td>
        <td name="td3">3.1</td>
        <td name="td4">Complement c5</td>
      </tr>
      <tr>
        <td name="td1">8OQ3-D</td>
        <td name="td2">7.2</td>
        <td name="td3">2.5</td>
        <td name="td4">Complement c3</td>
      </tr>
      <tr>
        <td name="td1">4KH9-A</td>
        <td name="td2">7.2</td>
        <td name="td3">3.2</td>
        <td name="td4">Hypothetical protein</td>
      </tr>
      <tr>
        <td name="td1">7PTP-A</td>
        <td name="td2">7.1</td>
        <td name="td3">2.7</td>
        <td name="td4">Cell surface glycoprotein</td>
      </tr>
      <tr>
        <td name="td1">5J72-A</td>
        <td name="td2">7</td>
        <td name="td3">3.4</td>
        <td name="td4">Putative n-acetylmuramoyl-l-alanine amidase,autol</td>
      </tr>
	  </tbody>
  </table>
<br>
<br>

                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Structural basis for the heterodimeric interaction between the acute leukaemia-associated transcription factors AML1 and CBFbeta.</strong></font><br />
Warren, A. J., Bravo, J., Williams, R. L., & Rabbitts, T. H<br />
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC203359/" target="_blank" style="color:#520049" >The EMBO journal, 19(12), 3004–3015. (2000)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] The role of a Runt domain transcription factor AML1/RUNX1 in leukemogenesis and its clinical implications.</strong></font><br />
Asou N.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12604126/" target="_blank" style="color:#520049">Critical reviews in oncology/hematology, 45(2), 129–150. (2003)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] Characterization of RNA aptamers that disrupt the RUNX1-CBFbeta/DNA complex.</strong></font><br />
Barton, J. L., Bunka, D. H., Knowling, S. E., Lefevre, P., Warren, A. J., Bonifer, C., & Stockley, P. G<br />
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2777437/" target="_blank" style="color:#520049">Nucleic acids research, 37(20), 6818–6830. (2009)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] AS-1411, a guanosine-rich oligonucleotide aptamer targeting nucleolin for the potential treatment of cancer, including acute myeloid leukemia.</strong></font><br />
Mongelard, F., & Bouvet, P<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/20140822/" target="_blank" style="color:#520049"> Current opinion in molecular therapeutics, 12(1), 107–114 (2010)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Solution structure of a DNA mimicking motif of an RNA aptamer against transcription factor AML1 Runt domain.</strong></font><br />
Nomura, Y., Tanaka, Y., Fukunaga, J., Fujiwara, K., Chiba, M., Iibuchi, H., Tanaka, T., Nakamura, Y., Kawai, G., Kozu, T., & Sakamoto, T<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23997091/" target="_blank" style="color:#520049">Journal of biochemistry, 154(6), 513–519 (2013)</a>
<br />

<a id="ref6"></a><font><strong>[6] The Runt domain of AML1 (RUNX1) binds a sequence-conserved RNA motif that mimics a DNA element.</strong></font><br />
Fukunaga, J., Nomura, Y., Tanaka, Y., Amano, R., Tanaka, T., Nakamura, Y., Kawai, G., Sakamoto, T., & Kozu, T<br />
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3683927/" target="_blank" style="color:#520049">RNA (New York, N.Y.), 19(7), 927–936 (2013)</a>
<br />       

<a id="ref7"></a><font><strong>[7] Developing aptamer probes for acute myelogenous leukemia detection and surface protein biomarker discovery.</strong></font><br />
Yang, M., Jiang, G., Li, W., Qiu, K., Zhang, M., Carter, C. M., Al-Quran, S. Z., & Li, Y<br />
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3895837/" target="_blank" style="color:#520049">Journal of hematology & oncology, 7, 5 (2014)</a>
<br />

<a id="ref8"></a><font><strong>[8] Oligonucleotide aptamer-drug conjugates for targeted therapy of acute myeloid leukemia.</strong></font><br />
Zhao, N., Pei, S. N., Qi, J., Zeng, Z., Iyer, S. P., Lin, P., Tung, C. H., & Zu, Y<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/26204224/" target="_blank" style="color:#520049">Biomaterials, 67, 42–51. (2015)</a>
<br />

<a id="ref9"></a><font><strong>[9] Coinhibition of overexpressed genes in acute myeloid leukemia subtype M2 by gold nanoparticles functionalized with five antisense oligonucleotides and one anti-CD33(+)/CD34(+) aptamer.</strong></font><br />
Zaimy, M. A., Jebali, A., Bazrafshan, B., Mehrtashfar, S., Shabani, S., Tavakoli, A., Hekmatimoghaddam, S. H., Sarli, A., Azizi, H., Izadi, P., Kazemi, B., Shojaei, A., Abdalaian, A., & Tavakkoly-Bazzaz, J<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/27514505/" target="_blank" style="color:#520049">Cancer gene therapy, 23(9), 315–320 (2016)</a>
<br />

<a id="ref10"></a><font><strong>[10] Kinetic and Thermodynamic Analyses of Interaction between a High-Affinity RNA Aptamer and Its Target Protein.</strong></font><br />
Amano, R., Takada, K., Tanaka, Y., Nakamura, Y., Kawai, G., Kozu, T., & Sakamoto, T<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/27766833/" target="_blank" style="color:#520049">Biochemistry, 55(45), 6221–6229 (2016)</a>
<br />

<a id="ref11"></a><font><strong>[11] Peptide aptamer identified by molecular docking targeting translationally controlled tumor protein in leukemia cells.</strong></font><br />
Kadioglu, O., & Efferth, T<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/26972431/" target="_blank" style="color:#520049">Investigational new drugs, 34(4), 515–521 (2016)</a>
<br />

<a id="ref12"></a><font><strong>[12] Kinetic and Thermodynamic Analyses of Interaction between a High-Affinity RNA Aptamer and Its Target Protein.</strong></font><br />
Amano, R., Takada, K., Tanaka, Y., Nakamura, Y., Kawai, G., Kozu, T., & Sakamoto, T<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/27766833/" target="_blank" style="color:#520049">Biochemistry, 55(45), 6221–6229 (2016)</a>
<br />

<a id="ref13"></a><font><strong>[13] Novel CD123-aptamer-originated targeted drug trains for selectively delivering cytotoxic agent to tumor cells in acute myeloid leukemia theranostics.</strong></font><br />
Wu, H., Wang, M., Dai, B., Zhang, Y., Yang, Y., Li, Q., Duan, M., Zhang, X., Wang, X., Li, A., & Zhang, L<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/28845698/" target="_blank" style="color:#520049">Drug delivery, 24(1), 1216–1229 (2017)</a>
<br />

<a id="ref14"></a><font><strong>[14] Rapid identification of specific DNA aptamers precisely targeting CD33 positive leukemia cells through a paired cell-based approach.</strong></font><br />
Yang, C., , Wang, Y., , Ge, M. H., , Fu, Y. J., , Hao, R., , Islam, K., , Huang, P., , Chen, F., , Sun, J., , Hong, F., , & Naranmandura, H.,<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/30519686/" target="_blank" style="color:#520049">Biomaterials science, 7(3), 938–950 (2019)</a>
<br />

<a id="ref15"></a><font><strong>[15] Nucleic Acid Aptamer: A Novel Potential Diagnostic and Therapeutic Tool for Leukemia.</strong></font><br />
Tan, Y., Li, Y., & Tang, F<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/31824168/" target="_blank" style="color:#520049">OncoTargets and therapy, 12, 10597–10613 (2019)</a>
<br />


<script>
    function toggleSequence(event) {
      const container = event.target.closest('.sequence-container');
      container.classList.toggle('expanded');
      const showMoreText = container.querySelector('.show-more');
      
      // 展开后按钮文本变化
      if (container.classList.contains('expanded')) {
        showMoreText.textContent = '...';  // 展开后显示 "..."
      } else {
        showMoreText.textContent = '......';  // 收起后显示 "......"
      }
    }

    // 页面加载时，限制序列文本为50个字符
    window.addEventListener('load', function() {
      const sequenceContainers = document.querySelectorAll('.sequence-container');
      sequenceContainers.forEach(container => {
        const fullSeqText = container.querySelector('.full-sequence').textContent;
        const truncatedText = fullSeqText.slice(0, 50);  // 只显示前50个字符
        container.querySelector('.sequence-text').textContent = truncatedText;
      });
    });
  </script>