---
layout: theophylline_style
title:  "AMPA receptors aptamer"
date:   2024-05-20 00:00:00
author: Bichun Wu, Jiaxin Zhao
categories: Aptamer
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>




<p class="header_box" id="description">Description</p>
<p>In 2006, Du, M., Ulrich, H., Zhao, X., Aronowski, J., & Jayaraman, V. have characterized the functional consequences of a representative aptamer from this class and show that it is a competitive antagonist of AMPA receptors and in the concentration range where it acts as an inhibitor of the AMPA receptor the RNA has no effect on the GluR6 homomeric kainate receptors, using whole-cell current recordings<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2006, Du, M., Ulrich, H., Zhao, X., Aronowski, J., & Jayaraman, V. incubated the 2'-F-RNA pool with GluR2-S1S2 protein, and the bound RNA was separated from the unbound RNA by using a nitrocellulose filter and a 96-well Minifold filtration apparatus. After washing the complexes with buffer, the protein-bound RNA was displaced with 500 μM glutamate. The collected supernatant containing the displaced RNA was phenol and chloroform-extracted, and the purified RNA was used as template for reverse-transcription (RT) PCR amplification to obtain the DNA pool for the next SELEX round. Since the RNA aptamers were being selected based on their affinity to the isolated ligand binding domain, a negative selection was performed after two initial SELEX rounds using membranes of non-transfected HEK 293 cells. The RNA molecules that did not bind to the non-transfected HEK 293 cell membranes were then used for subsequent selection rounds towards GluR2-S1S2.After eight repetitive selection rounds, the affinity of the RNA pool 8 for binding to GluR2-S1S2 was determined using a radioligand-receptor binding assay<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>The 2D structure of the figure is based on the prediction results of the RNA fold website by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>
<p>5'-ACCGAGUCCAGAAGCUUGUAGUACUCGGCGAACUACCUAACCCCCCACAAUUGUAUUAUCGCCCGCCUAGAUGGCAGUUGAAUUCUCCCUAUAGUGAGUCGUAUUAC-3'</p>
<img src="/images/2D/AMPA_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>Receptor for glutamate that functions as a ligand-gated ion channel in the central nervous system. It plays an important role in excitatory synaptic transmission. L-glutamate acts as an excitatory neurotransmitter at many synapses in the central nervous system. Binding of the excitatory neurotransmitter L-glutamate induces a conformation change, leading to the opening of the cation channel, and thereby converts the chemical signal to an electrical impulse. The receptor then desensitizes rapidly and enters a transient inactive state, characterized by the presence of bound agonist. In the presence of CACNG4 or CACNG7 or CACNG8, shows resensitization which is characterized by a delayed accumulation of current flux upon continued application of glutamate. Through complex formation with NSG1, GRIP1 and STX12 controls the intracellular fate of AMPAR and the endosomal sorting of the GRIA2 subunit toward recycling and membrane targeting (By similarity).-----from InterPro</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Alpha-Amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid (AMPA) receptors</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P42262/entry" target="_blank" style="color:#520049"><b>P42262</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/cdd/CD13727/" target="_blank" style="color:#520049"><b>CD13727</b></a></td>
        <td name="td3">43.73 kDa</td>
        <td name="td4">NSIQIGGLFPRGADQEYSAFRVGMVQFSTSEFRLTPHIDNLEVANSFAVTNAFCSQFSRGVYAIFGFYDKKSVNTITSFCGTLHVSFITPSFPTDGTHPFVIQMRPDLKGALLSLIEYYQWDKFAYLYDSDRGLSTLQAVLDSAAEKKWQVTAINVGNINNDKKDEMYRSLFQDLELKKERRVILDCERDKVNDIVDQVITIGKHVKGYHYIIANLGFTDGDLLKIQFGGANVSGFQIVDYDDSLVSKFIERWSTLEEKEYPGAHTTTIKYTSALTYDAVQVMTEAFRNLRKQRIEISRRGNAGDCLANPAVPWGQGVEIERALKQVQVEGLSGNIKFDQNGKRINYTINIMELKTNGPRKIGYWSEVDKMVVTLTELPSGNDTSGLE</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/2WJW" target="_blank" style="color:#520049"><b>2WJW</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/2891" target="_blank" style="color:#520049"><b>2891</b></a></td>
      </tr>
	  </tbody>
  </table>

<p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Aptamer #25</td>
        <td name="td1">5'-GGACGCGUGGUACCAGGCCGAUCUAUGGACGCUAUAGGCACACCGGAUACUUUAACGAUUGGCUAAGCUUCCGCGGGGAUC-3'</td>
        <td name="td2">Alpha-Amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid (AMPA) receptors</td>
        <td name="td3">37±5 nM</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/AMPA_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">7RZ4-A</td>
      <td name="td1">48.2</td>
      <td name="td2">1.3</td>
      <td name="td3">Glutamate receptor 2</td>
    </tr>
            
     <tr>
      <td name="td0">9B38-A</td>
      <td name="td1">37.8</td>
      <td name="td2">2.6</td>
      <td name="td3">Glutamate receptor ionotropic, kainate 2</td>
    </tr>
            
     <tr>
      <td name="td0">3TD9-A</td>
      <td name="td1">30.6</td>
      <td name="td2">2.9</td>
      <td name="td3">Branched chain amino acid abc transporter, peripl</td>
    </tr>
            
     <tr>
      <td name="td0">3I45-A</td>
      <td name="td1">28.7</td>
      <td name="td2">2.7</td>
      <td name="td3">Twin-arginine translocation pathway signal protei</td>
    </tr>
            
     <tr>
      <td name="td0">3I09-A</td>
      <td name="td1">28.2</td>
      <td name="td2">2.8</td>
      <td name="td3">Periplasmic branched-chain amino acid-binding pro</td>
    </tr>
            
     <tr>
      <td name="td0">4EVQ-B</td>
      <td name="td1">28.1</td>
      <td name="td2">3.0</td>
      <td name="td3">Putative abc transporter subunit, substrate-bindi</td>
    </tr>
            
     <tr>
      <td name="td0">4JB0-A</td>
      <td name="td1">27.8</td>
      <td name="td2">2.8</td>
      <td name="td3">Branched-chain amino acid transport system substr</td>
    </tr>
            
     <tr>
      <td name="td0">7S6E-B</td>
      <td name="td1">27.7</td>
      <td name="td2">2.8</td>
      <td name="td3">Urea abc transporter, periplasmic urea-binding pr</td>
    </tr>
            
     <tr>
      <td name="td0">3H5L-B</td>
      <td name="td1">27.4</td>
      <td name="td2">3.3</td>
      <td name="td3">Putative branched-chain amino acid abc</td>
    </tr>
            
     <tr>
      <td name="td0">1DP4-C</td>
      <td name="td1">27.2</td>
      <td name="td2">4.0</td>
      <td name="td3">Atrial natriuretic peptide receptor a</td>
    </tr>
    </tbody>
  </table>
                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Water soluble RNA based antagonist of AMPA receptors.</strong></font><br />
Du, M., Ulrich, H., Zhao, X., Aronowski, J., & Jayaraman, V. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/17588619/" target="_blank" style="color:#520049">Neuropharmacology, 53(2), 242–251. (2007)</a>
<br/>

<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1RAW-3D.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>
<script>
    window.addEventListener('DOMContentLoaded', function() {
      var td = document.getElementsByName('td4')[0];
      var maxLength = 100; // 设置最大长度
      var originalText = td.innerHTML;
      var displayedText = originalText.substring(0, maxLength);
      
      if (originalText.length > maxLength) {
        displayedText += '<span class="ellipsis-btn">......</span>'; // 在超出长度时添加省略号按钮
        displayedText += '<span class="hidden-content">' + originalText.substring(maxLength) + '</span>'; // 隐藏的内容
        td.innerHTML = displayedText;
        
        // 获取省略号按钮元素和隐藏的内容元素
        var ellipsisBtn = td.querySelector('.ellipsis-btn');
        var hiddenContent = td.querySelector('.hidden-content');
        
        // 添加点击事件监听器
        ellipsisBtn.addEventListener('click', function() {
          if (hiddenContent.style.display === 'none') {
            hiddenContent.style.display = 'inline'; // 展开内容
            ellipsisBtn.innerHTML = '...'; // 更新按钮文字为“...”
            td.appendChild(ellipsisBtn); // 将按钮移到内容后面
          } else {
            hiddenContent.style.display = 'none'; // 收起内容
            ellipsisBtn.innerHTML = '......'; // 更新按钮文字为省略号
            td.appendChild(ellipsisBtn); // 将按钮移到内容后面
          }
        });
      }
    });
  </script>

    </html>
