---
layout: theophylline_style
title: AMPA and kainate receptors_AB9 aptamer
date: 2024-05-22 00:00:00
author: Baowei Huang
categories: Aptamer
tags:
- Category:Mammalian
- GC:45.5
- GC:47.0
- Length:101
- Length:55
- Named:AMPA and kainate receptors_AB9
- Named:AMPA and kainate receptors_AB9s
- Type:Proteins
- Year:2017
type: Proteins
---
<html>



<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/1370372/" target="_blank" style="color:#520049">1992</a></h3>
  </div>
  <div class="body">
    <p>Mutational analysis revealed that both the positive charge and the size of the amino acid side chain located at the Q/R site control the divalent permeability of homomeric channels. Coexpression of Q/R site arginine- and glutamine-containing subunits generates cells with varying divalent permeabilities depending on the amounts of expression vectors used for cell transfection.<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8604042/" target="_blank" style="color:#520049">1996</a></h3>
  </div>
  <div class="body">
    <p>The presence of two major populations of AMPA receptor complexes: those made up of GluR1 and GluR2 and those made up of GluR2 and GluR3. Very few complexes contained both GluR1 and GluR3, whereas approximately 8% of the total AMPA receptor complexes was homomeric GluR1.<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/14622580/" target="_blank" style="color:#520049">2003</a></h3>
  </div>
  <div class="body">
    <p>AMPAR assembly and subunit stoichiometry are determined by RNA editing in the pore loop. They also demonstrate that editing at the GluR2 Q/R site regulates AMPAR assembly at the step of tetramerization.<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/17929944/" target="_blank" style="color:#520049">2007</a></h3>
  </div>
  <div class="body">
    <p>Selected a group of RNA aptamers against the recombinant GluR2Qflip AMPA receptor transiently expressed in HEK-293 (human embryonic kidney) cells using the systematic evolution of ligands by exponential enrichment (SELEX).<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/20518485/" target="_blank" style="color:#520049">2010</a></h3>
  </div>
  <div class="body">
    <p>Reported the identification of an RNA inhibitor or aptamer by an in vitro evolution approach. Using a laser-pulse photolysis technique, we further characterized the mechanism of inhibition of this aptamer on the AMPA receptor channel-opening rate process in the microsecond-to-millisecond time domain.<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/28325839/" target="_blank" style="color:#520049">2017</a></h3>
  </div>
  <div class="body">
    <p>Report the use of an in vitro evolution approach involving systematic evolution of ligands by exponential enrichment with a single AMPA receptor target (i.e. GluA1/2R) to isolate RNA aptamers that can potentially inhibit both AMPA and kainate receptors.<sup>[<a href="#ref11" style="color:#520049">11</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/34509496/" target="_blank" style="color:#520049">2021</a></h3>
  </div>
  <div class="body">
    <p>Reviewed the approach to discover RNA aptamers targeting AMPA receptors from a random sequence library (∼1014 sequences) through a process called systematic evolution of ligands by exponential enrichment (SELEX).<sup>[<a href="#ref12" style="color:#520049">12</a>]</sup></p>
  </div>
 </div>
</div>



<p class="header_box" id="description">Description</p>
<p>In 2007, Huang and his collegeague used the systematic evolution of ligands by exponential enrichment (SELEX) to selecte a group of RNA aptamers against the recombinant GluR2Qflip AMPA receptor transiently expressed in HEK-293 (human embryonic kidney) cells. In 2017, J. Jaremko and his colleagues used homologous binding and whole-cell recording assays to find an RNA aptamer most likely binds to the receptor’s regulatory site and in- hibits it noncompetitively<sup>[<a href="#ref8" style="color:#520049">8</a>,<a href="#ref12" style="color:#520049">12</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2017, To select RNA aptamers, J. Jaremko and his colleague used SELEX (31, 32), which involves multiple cycles or rounds. Each round consists of RNA-receptor binding, RNA elution, RT-PCR amplification, followed by enzymatic transcription from the DNA library to regenerate a new RNA library or pool<sup>[<a href="#ref12" style="color:#520049">12</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>The 2D structure of the figures is based on the article by ribodraw tool to draw<sup>[<a href="#ref12" style="color:#520049">12</a>]</sup>.<br></p>
<p>5'-GGGAGAAUUCAACUGCCAUCUAGGCAGAUCACGAAAAAGCGGAAUUGAGGUACCCAAGAGCUAAAAAAAGACAUCCAGUACUACAAGCUUCUGGACUCGGU-3'</p>
<img src="/images/2D/AB9_aptamer_2D.svg" alt="drawing" style="width:800px;height:400px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>



<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>The α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid receptor (also known as AMPA receptor, AMPAR, or quisqualate receptor) is an ionotropic transmembrane receptor for glutamate (iGluR) and predominantly Na+ ion channel that mediates fast synaptic transmission in the central nervous system (CNS).</p>
<p>Kainate receptors, or kainic acid receptors (KARs), are ionotropic receptors that respond to the neurotransmitter glutamate. They were first identified as a distinct receptor type through their selective activation by the agonist kainate, a drug first isolated from the algae Digenea simplex. They have been traditionally classified as a non-NMDA-type receptor, along with the AMPA receptor.-----from WIKI</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">AMPA receptor and kainate receptor</td>
        <td name="td1">
        <a href="https://www.uniprot.org/uniprotkb/P42261/entry" target="_blank" style="color:#520049"><b>P42261</b></a><br>
        <a href="https://www.uniprot.org/uniprotkb/P39086/entry" target="_blank" style="color:#520049"><b>P39086</b></a>
        </td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/cdd/CD13724/" target="_blank" style="color:#520049"><b>CD13724</b></a></td>
        <td name="td3">101.506 kDa</td>
        <td name="td4">MQHIFAFFCTGFLGAVVGANFPNNIQIGGLFPNQQSQEHAAFRFALSQLTEPPKLLPQIDIVNISDSFEMTYRFCSQFSKGVYAIFGFYERRTVNMLTSFCGALHVCFITPSFPVDTSNQFVLQLRPELQDALISIIDHYKWQKFVYIYDADRGLSVLQKVLDTAAEKNWQVTAVNILTTTEEGYRMLFQDLEKKKERLVVVDCESERLNAILGQIIKLEKNGIGYHYILANLGFMDIDLNKFKESGANVTGFQLVNYTDTIPAKIMQQWKNSDARDHTRVDWKRPKYTSALTYDGVKVMAEAFQSLRRQRIDISRRGNAGDCLANPAVPWGQGIDIQRALQQVRFEGLTGNVQFNEKGRRTNYTLHVIEMKHDGIRKIGYWNEDDKFVPAATDAQAGGDNSSVQNRTYIVTTILEDPYVMLKKNANQFEGNDRYEGYCVELAAEIAKHVGYSYRLEIVSDGKYGARDPDTKAWNGMVGELVYGRADVAVAPLTITLVREEVIDFSKPFMSLGISIMIKKPQKSKPGVFSFLDPLAYEIWMCIVFAYIGVSVVLFLVSRFSPYEWHSEEFEEGRDQTTSDQSNEFGIFNSLWFSLGAFMQQGCDISPRSLSGRIVGGVWWFFTLIIISSYTANLAAFLTVERMVSPIESAEDLAKQTEIAYGTLEAGSTKEFFRRSKIAVFEKMWTYMKSAEPSVFVRTTEEGMIRVRKSKGKYAYLLESTMNEYIEQRKPCDTMKVGGNLDSKGYGIATPKGSALRNPVNLAVLKLNEQGLLDKLKNKWWYDKGECGSGGGDSKDKTSALSLSNVAGVFYILIGGLGLAMLVALIEFCYKSRSESKRMKGFCLIPQQSINEAIRTSTLPRNSGAGASSGGSGENGRVVSHDFPKSMQSIPCMSHSSGMPLGATGL</td>
        <td name="td5">
        <a href="https://www.rcsb.org/structure/1US4" target="_blank" style="color:#520049"><b>1US4</b></a>
        </td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/nuccore/L19058.1" target="_blank" style="color:#520049"><b>L19058.1</b></a></td>
      </tr>
	  </tbody>
  </table>

  <p>J. Jaremko and his colleague prepared 32P-labeled AB9s and ran a homologous binding assay with the GluA1/2R lipid membrane. Because AB9s exhibited full inhibitory activity against GluA1/2R to determine the binding affinity of AB9s, the shorter aptamer.Using the same homologous binding assay, they carried out the binding experiment with 32P-labeled AB9 and GluA1/2R lipid membrane<sup>[<a href="#ref12" style="color:#520049">12</a>]</sup>.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">AB9</td>
        <td name="td1">GGGAGAAUUCAACUGCCAUCUAGGCAGAUCACGAAAAAGCGGAAUUGAGGUACCCAAGAGCUAAAAAAAGACAUCCAGUACUACAAGCUUCUGGACUCGGU</td>
        <td name="td2">AMPA and kainate receptors</td>
        <td name="td3">4.5 μM</td>
      </tr>
	  </tbody>
     <tbody>
      <tr>
        <td name="td0">AB9s</td>
        <td name="td1">GGGUGCCAUCUAGGCAGAUCACGAAAAAGCGAAAGCUAAAAAAAGACAUCCACCC</td>
        <td name="td2">AMPA and kainate receptors</td>
        <td name="td3">4.7 μM</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/AB9_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(1)">Named</th>
        <th onclick="sortTable(2)">CAS</th>
        <th onclick="sortTable(3)">Pubchem CID</th>
        <th onclick="sortTable(4)">Structure</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">4DDD-A</td>
      <td name="td1">36.3</td>
      <td name="td2">2.2</td>
      <td name="td3">Immunogenic protein</td>
    </tr>
            
     <tr>
      <td name="td0">3QSL-B</td>
      <td name="td1">16.1</td>
      <td name="td2">3.0</td>
      <td name="td3">Putative exported protein</td>
    </tr>
            
     <tr>
      <td name="td0">6SSY-A</td>
      <td name="td1">15.9</td>
      <td name="td2">3.0</td>
      <td name="td3">Taurine-binding periplasmic protein</td>
    </tr>
            
     <tr>
      <td name="td0">4ESW-A</td>
      <td name="td1">15.8</td>
      <td name="td2">2.9</td>
      <td name="td3">Pyrimidine biosynthesis enzyme thi13</td>
    </tr>
            
     <tr>
      <td name="td0">7S6G-A</td>
      <td name="td1">15.5</td>
      <td name="td2">3.2</td>
      <td name="td3">Phosphonate abc type transporter/ substrate bindi</td>
    </tr>
            
     <tr>
      <td name="td0">4NMY-A</td>
      <td name="td1">14.9</td>
      <td name="td2">3.2</td>
      <td name="td3">ABC-type transport system, extracellular solute-b</td>
    </tr>
            
     <tr>
      <td name="td0">8HKB-A</td>
      <td name="td1">14.8</td>
      <td name="td2">3.6</td>
      <td name="td3">Periplasmic terephthalate binding protein (tbp)</td>
    </tr>
            
     <tr>
      <td name="td0">3UN6-A</td>
      <td name="td1">14.7</td>
      <td name="td2">3.9</td>
      <td name="td3">ABC transporter substrate-binding protein</td>
    </tr>
            
     <tr>
      <td name="td0">3QK6-B</td>
      <td name="td1">14.7</td>
      <td name="td2">2.9</td>
      <td name="td3">PHND, subunit of alkylphosphonate abc transporter</td>
    </tr>
            
     <tr>
      <td name="td0">6ESK-A</td>
      <td name="td1">14.6</td>
      <td name="td2">2.9</td>
      <td name="td3">Putative periplasmic phosphite-binding-like prote</td>
    </tr>
            
     <tr>
      <td name="td0">7DM1-A</td>
      <td name="td1">14.3</td>
      <td name="td2">3.2</td>
      <td name="td3">Phosphate-binding protein psts 1</td>
    </tr>
    </tbody>
  </table>
                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Divalent ion permeability of AMPA receptor channels is dominated by the edited form of a single subunit.</strong></font><br />
Burnashev, N., Monyer, H., Seeburg, P. H., & Sakmann, B.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/1370372/" target="_blank" style="color:#520049">Neuron 8, 189–198. (1992)</a>
<br/>
            
<a id="ref2"></a><font><strong>[2] Evidence for multiple AMPA receptor complexes in hippocampal CA1/CA2 neurons.</strong></font><br />
Wenthold, R. J., Petralia, R. S., Blahos, J., II, & Niedzielski, A. S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8604042/" target="_blank" style="color:#520049">J. Neurosci. 16, 1982–1989. (1996)</a>
<br/>
            
<a id="ref3"></a><font><strong>[3] The role of RNA editing in controlling glutamate receptor channel properties.</strong></font><br />
Seeburg, P. H.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8522940/" target="_blank" style="color:#520049">J. Neurochem. 66, 1–5. (1996)</a>
<br/>
            
<a id="ref4"></a><font><strong>[4] Single-channel properties of recombinant AMPA receptors depend on RNA editing, splice variation, and subunit composition.</strong></font><br />
Swanson, G. T., Kamboj, S. K., & Cull-Candy, S. G.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8987736/" target="_blank" style="color:#520049">J. Neurosci. 17, 58–69. (1997)</a>
<br/>
            
<a id="ref5"></a><font><strong>[5] Unequal expression of allelic kainate receptor GluR7 mRNAs in human brains.</strong></font><br />
Schiffer, H. H., Swanson, G. T., Masliah, E., & Heinemann, S. F.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/11124978/" target="_blank" style="color:#520049">J. Neurosci. 20, 9025–9033. (2000)</a>
<br/>
            
<a id="ref6"></a><font><strong>[6] AMPA receptor tetramerization is mediated by Q/R editing.</strong></font><br />
Greger, I. H., Khatri, L., Kong, X., & Ziff, E. B.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/14622580/" target="_blank" style="color:#520049">Neuron. 40, 763–774. (2003)</a>
<br/>
            
<a id="ref7"></a><font><strong>[7] RNA aptamers selected against the GluR2 glutamate receptor channel.</strong></font><br />
Huang, Z., Pei, W., Jayaseelan, S., Shi, H., & Niu, L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/17929944/" target="_blank" style="color:#520049">Biochemistry, 46(44), 12648–12655. (2007)</a>
<br/>
            
<a id="ref8"></a><font><strong>[8] Potent and selective inhibition of the open-channel conformation of AMPA receptors by an RNA aptamer.</strong></font><br />
Huang, Z., Han, Y., Wang, C., & Niu, L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/20518485/" target="_blank" style="color:#520049">Biochemistry, 49(27), 5790–5798. (2010)</a>
<br/>
            
<a id="ref9"></a><font><strong>[9] Potent and selective inhibition of a single alpha-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid (AMPA) receptor subunit by an RNA aptamer.</strong></font><br />
Park, J. S., Wang, C., Han, Y., Huang, Z., & Niu, L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/21402710/" target="_blank" style="color:#520049">The Journal of biological chemistry, 286(17), 15608–15617. (2011)</a>
<br/>
            
<a id="ref10"></a><font><strong>[10] Chemically Modified, α-Amino-3-hydroxy-5-methyl-4-isoxazole (AMPA) Receptor RNA Aptamers Designed for in Vivo Use.</strong></font><br />
Huang, Z., Wen, W., Wu, A., & Niu, L. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/28872832/" target="_blank" style="color:#520049">ACS chemical neuroscience, 8(11), 2437–2445. (2017)</a>
<br/>
            
<a id="ref11"></a><font><strong>[11] Identification and characterization of RNA aptamers: A long aptamer blocks the AMPA receptor and a short aptamer blocks both AMPA and kainate receptors.</strong></font><br />
Jaremko, W. J., Huang, Z., Wen, W., Wu, A., Karl, N., & Niu, L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/28325839/" target="_blank" style="color:#520049">The Journal of biological chemistry, 292(18), 7338–7347. (2017)</a>
<br/>
            
<a id="ref12"></a><font><strong>[12] RNA aptamers for AMPA receptors.</strong></font><br />
Huang, Z., & Niu, L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/34509496/" target="_blank" style="color:#520049">Neuropharmacology, 199, 108761. (2021)</a>
<br/>





<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1RAW-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>

    </html>