---
layout: theophylline_style
title: Bacteriophage MS2 coat protein aptamer
date: 2024-03-01 00:00:00
author: Zhizhong Lu
categories: Aptamer
tags:
- Category:Other
- GC:50.0
- GC:64.0
- Length:14
- Length:32
- Named:Bacteriophage MS2 coat protein_F6 aptamer
- Named:Bacteriophage MS2 coat protein_Sequence 12 aptamer
- Type:Proteins
- Year:1992
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/4507620/" target="_blank" style="color:#520049">1972</a></h3>
    </div>
    <div class="body">
      <p>Natural RNA sequences that bind to bacteriophage coat protein<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/1469719/" target="_blank" style="color:#520049">1992</a></h3>
    </div>
    <div class="body">
      <p>The aptamer is initially acquired through the SELEX<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8440248/" target="_blank" style="color:#520049">1993</a></h3>
    </div>
    <div class="body">
      <p>The coat protein of the RNA bacteriophage MS2 binds a specific stem-loop structure in viral RNA<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/9461079/" target="_blank" style="color:#520049">1998</a></h3>
    </div>
    <div class="body">
      <p>Crystal structure of an RNA aptamer-protein complex at 2.8 Å resolution<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10606647/" target="_blank" style="color:#520049">2000</a></h3>
    </div>
    <div class="body">
      <p>RNA aptamers for the MS2 bacteriophage coat protein and the wild-type RNA operator have similar solution behaviour<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/11162119/" target="_blank" style="color:#520049">2001</a></h3>
    </div>
    <div class="body">
      <p>Probing the kinetics of formation of the bacteriophage MS2 translational operator complex: identification of a protein conformer unable to bind RNA<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup></p>
    </div>
  </div>
<div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/15496523/" target="_blank" style="color:#520049">2004</a></h3>
    </div>
    <div class="body">
      <p>The crystal structure of a high affinity RNA stem-loop complexed with the bacteriophage MS2 capsid: further challenges in the modeling of ligand-RNA interactions<sup>[<a href="#ref9" style="color:#520049">9</a>]</sup></p>
    </div>
  </div>
</div>

         
<p class="header_box" id="description">Description</p>
<p>In 1992, Craig Tuerk and Larry Gold et al. used the SELEX method to isolate the aptamer with high compatibility for the Bacteriophage R17 Coat Protein. Sequences 1-38 were isolated from the sequence pool, where sequences 7, 12, and 22 detected affinity and Bacteriophage R17 coat proteins. Afterwards, In 1998, S E Phillips and P G Stockley et al. They obtained F6 aptamer by analyzing natural wild-type RNA and truncating the mutation, and determined the complex structure of this aptamer with wild type MS2 coat protein by X-ray crystallographic methods<sup>[<a href="#ref3" style="color:#520049">3</a></sup><sup>,<a href="#ref6" style="color:#520049">6</a>]</sup>. The F6 aptamer was named by Liljas et al. in the article<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 1992, Craig Tuerk and Larry Gold et al. isolated 47 specific sequences targeting R17 coat proteins from a 32nt sequence pool after 11 rounds of selection. The affinity had increased significantly. Unlabeled RNA was denatured at 70°C for 3 minutes and then renatured at 4°C for 5 minutes prior to each selection. Within a 100 μl reaction that contained 100mM KOAc, 10 mM DTT, and 50 mM Tris at pH 7.5, R17 coat protein was permitted to equilibrate with an excess of RNA for 3 minutes at 37°C. Selections 1 to 4 were conducted at a protein concentration of 125 nm, while selections 5 to 11 were at 40 nM. All selections took place at an RNA concentration of 30 μm. As the initial selection involved 3x10<sup>14</sup> different RNA molecules, an average of 6 copies of each sequence were sampled. Protein-bound RNA was separated from unbound RNA by filtration through a nitrocellulose disk that had been pre-wetted with 50 mM Tris-OAc at pH 7.7, followed by an immediate 5 ml rinse with the same solution. The RNA molecules were recovered from the filter. A background control, devoid of protein, was also implemented in each round<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.</p>
<p>
<br>
<br> 


<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>Here we use ribodraw to complete the figure, through the 3D structure information. F6 aptamer was the aptamer sequence mainly studied in Structure article<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup>.</p>
<p>5'-CCACAGUCACUGGG-3'</p>
<img src="/images/2D/MS2_coat_aptamer_2D.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;">   
</div>

<p class="blowheader_box">3D visualisation</p>             
<p>In 1998, Stockley PG and Liljas L et al. determined the crystal structure, resolved to 2.8 Å, of an RNA aptamer bound to bacteriophage MS2 coat protein. It provided an opportunity to compare the interactions of MS2 coat protein and wild type operator with those of an aptamer, whose secondary structure differed from the wild type RNA by having a three-base loop (as opposed to a tetraloop) and an additional base pair between this loop and the sequence-specific recognition element in the stem. The PDB ID of this structure is 6MSF<sup>[<a href="#ref5" style="color:#520049" >5</a>]</sup>.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/MS2_coat_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A', 
  start_residue_number:1, 
  end_residue_number:129, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'R', 
  start_residue_number:1, 
  end_residue_number:6, 
  color:{r:46,g:161,b:3}
},
{
  struct_asym_id: 'R', 
  start_residue_number:7, 
  end_residue_number:9, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'R', 
  start_residue_number:10, 
  end_residue_number:14, 
  color:{r:46,g:161,b:3}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/MS2_coat_6msf-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>
    </body>
    </html>
    </td>
  </tr>
  </table>
  </div>


<p class="blowheader_box">Binding pocket</p>             
<p>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 6MSF at 2.8 Å resolution. MS2 coat protein (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with MS2 coat protein.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/MS2_coat_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/MS2_coat_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>



<p class="header_box" id="ligand-recognition">Ligand information</p>
  
<p class="blowheader_box">SELEX ligand</p>
<p>Craig Tuerk and Larry Gold determined the binding constant of the aptamer using nitrocellulose filtration. In the nitrocellulose filter binding assay, 100 μl reactions containing 50mM Tris-OAc at pH 7.7 were used, where 5 to 10 nCi (10 to 20 fmol) of internally labeled, gel-purified RNA, preheated for 3 minutes at 70°C, was allowed to equilibrate with variable (excess) concentrations of protein for 3 minutes at 37°C. Samples were filtered through a nitrocellulose disk that had been pre-wetted with 50 mM Tris-OAc at pH 7.7 and were immediately rinsed with 5 ml of the same solution. The filters were dried and counted with fluor in a scintillation counter. A least squares algorithm was employed to plot the percent of total RNA bound against the log of the protein concentration and to determine the dissociation constant from the generated curve. The F6 aptamer had not been tested for affinity<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.</p>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Sequence 7</td>
        <td name="td1">5’-CAGAGAUAUCACUUCUGUUCACCAUCGGGGGA-3’</td>
        <td name="td2">R17 coat protein</td>
        <td name="td3">5.9nM</td>
      </tr>
      <tr>
        <td name="td0">Sequence 12</td>
        <td name="td1">5’-AUAUAAGUAAUGGAUGCGCACCAUCGGGGCGU-3’</td>
        <td name="td2">R17 coat protein</td>
        <td name="td3">5.0nM</td>
      </tr>
      <tr>
        <td name="td0">Sequence 22</td>
        <td name="td1">5’-AUGAGAUAGAUCAUGCUCAGGAUCGCCGGG-3’</td>
        <td name="td2">R17 coat protein</td>
        <td name="td3">6.3nM</td>
      </tr>
      <tr>
        <td name="td0">F6 aptamer</td>
        <td name="td1">5'-CCACAGUCACUGGG-3'</td>
        <td name="td2">MS2 coat protein</td>
        <td name="td3">NA</td>
      </tr>
	  </tbody>
  </table>
  </div>

<p class="blowheader_box">Structure ligand</p>
<p>The Levivirus coat protein forms the bacteriophage coat that encapsidates the viral RNA. 180 copies of this protein form the virion shell. The MS2 bacteriophage coat protein controls two distinct processes: sequence-specific RNA encapsidation and repression of replicase translation-by binding to an RNA stem-loop structure of 19 nucleotides containing the initiation codon of the replicase gene. The binding of a coat protein dimer to this hairpin shuts off synthesis of the viral replicase, switching the viral replication cycle to virion assembly rather than continued replication.-----From <a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF01819/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a></p>
<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(6)">Name</th>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">Mass</th>
        <th onclick="sortTable(3)">Protein sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td6">Bacteriophage MS2 coat protein</td>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P03612/entry#names_and_taxonomy" target="_blank" style="color:#520049"><b>P03612</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF01819/" target="_blank" style="color:#520049"><b>PF01819</b></a></td>
        <td name="td2">13.84 kDa</td>
        <td name="td3">MASNFTQFVLVDNGGTGDVTVAPSNFANGVAEWISSNSRSQAYKVTCSVRQSSAQNRKYTIKVEVPKVATQTVGGVELPVAAWRSYLNMELTIPIFATNSDCELIVKAMQGLLKDGNPIPSAIAANSGIY</td>
        <td name="td4"><a href="https://www.rcsb.org/structure/1AQ3" target="_blank" style="color:#520049"><b>1AQ3</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/nuccore/V00642" target="_blank" style="color:#520049"><b>V00642</b></a></td>
      </tr>
	  </tbody>
  </table>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/MS2_coat_stru_ligand.svg" alt="drawing" style="width:800px;height:350px;"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>
  
                    
<p class="blowheader_box">Similar compound(s)</p>
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation.</p>
<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD (Å)</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/1AQ3" target="_blank" style="color:#520049"><b>1AQ3</b></a>-A</td>
        <td name="td2">21.3</td>
        <td name="td3">0</td>
        <td name="td4">Original chain</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/2VF9" target="_blank" style="color:#520049"><b>2VF9</b></a>-A</td>
        <td name="td2">14.2</td>
        <td name="td3">2.3</td>
        <td name="td4">Coat protein</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/1DWN" target="_blank" style="color:#520049"><b>1DWN</b></a>-A</td>
        <td name="td2">12.6</td>
        <td name="td3">2.7</td>
        <td name="td4">Phage coat protein</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/6YFI" target="_blank" style="color:#520049"><b>6YFI</b></a>-A</td>
        <td name="td2">12</td>
        <td name="td3">2.7</td>
        <td name="td4">Coat protein</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/2W4Y" target="_blank" style="color:#520049"><b>2W4Y</b></a>-A</td>
        <td name="td2">10.8</td>
        <td name="td3">3</td>
        <td name="td4">Caulobacter 5 virus-Like particle</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/8FEH" target="_blank" style="color:#520049"><b>8FEH</b></a>-D</td>
        <td name="td2">8.8</td>
        <td name="td3">2.2</td>
        <td name="td4">Minor capsid protein A1 fusion</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/5JZR" target="_blank" style="color:#520049"><b>5JZR</b></a>-A</td>
        <td name="td2">6.2</td>
        <td name="td3">3.8</td>
        <td name="td4">Coat protein</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/4QL0" target="_blank" style="color:#520049"><b>4QL0</b></a>-A</td>
        <td name="td2">4.3</td>
        <td name="td3">5.1</td>
        <td name="td4">Filamentous hemagglutinin transporter protein fha</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/1WZN" target="_blank" style="color:#520049"><b>1WZN</b></a>-A</td>
        <td name="td2">3.9</td>
        <td name="td3">5.3</td>
        <td name="td4">Sam-Dependent methyltransferase</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/2FWZ" target="_blank" style="color:#520049"><b>2FWZ</b></a>-A</td>
        <td name="td2">3.8</td>
        <td name="td3">4.3</td>
        <td name="td4">Hypothetical protein mtubf_01000852</td>
      </tr>
	  </tbody>
  </table>

                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><p><strong>[1] Nucleotide sequence at the binding site for coat protein on RNA of bacteriophage R17.</strong></p><br />
Bernardi, A., & Spahr, P. F.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/4507620/" target="_blank" style="color:#520049" >Proceedings of the National Academy of Sciences of the United States of America, 69(10), 3033–3037. (1972).</a>
<br />
                
<a id="ref2"></a><p><strong>[2] Enzymatic synthesis of a 21-nucleotide coat protein binding fragment of R17 ribonucleic acid.</strong></p><br />
Krug, M., de Haseth, P. L., & Uhlenbeck, O. C.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/6897195/" target="_blank" style="color:#520049">Biochemistry, 21(19), 4713–4720. (1982).</a>
<br />
                
<a id="ref3"></a><p><strong>[3] Selection of high affinity RNA ligands to the bacteriophage R17 coat protein.</strong></p><br />
Schneider, D., Tuerk, C., & Gold, L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/1469719/" target="_blank" style="color:#520049">Journal of molecular biology, 228(3), 862–869. (1992).</a>
<br />
                
<a id="ref4"></a><p><strong>[4] The RNA binding site of bacteriophage MS2 coat protein.</strong></p><br />
Peabody DS.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8440248/" target="_blank" style="color:#520049">The EMBO journal ,12(2), 595–600. (1993).</a>
<br />
                
<a id="ref5"></a><p><strong>[5] Crystal structures of MS2 coat protein mutants in complex with wild-type RNA operator fragments.</strong></p><br />
van den Worm, S. H., Stonehouse, N. J., Valegârd, K., Murray, J. B., Walton, C., Fridborg, K., Stockley, P. G., & Liljas, L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/9469847/" target="_blank" style="color:#520049">Nucleic acids research, 26(5), 1345–1351. (1998).</a>
<br />
                
<a id="ref6"></a><p><strong>[6] Crystal structure of an RNA aptamer-protein complex at 2.8 A resolution.</strong></p><br />
Convery, M. A., Rowsell, S., Stonehouse, N. J., Ellington, A. D., Hirao, I., Murray, J. B., Peabody, D. S., Phillips, S. E., & Stockley, P. G.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/9461079/" target="_blank" style="color:#520049">Nature structural biology, 5(2), 133–139. (1998).</a>
<br />
                
<a id="ref7"></a><p><strong>[7] RNA aptamers for the MS2 bacteriophage coat protein and the wild-type RNA operator have similar solution behaviour.</strong></p><br />
Parrott, A. M., Lago, H., Adams, C. J., Ashcroft, A. E., Stonehouse, N. J., & Stockley, P. G.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10606647/" target="_blank" style="color:#520049">Nucleic acids research, 28(2), 489–497. (2000).</a>
<br />
                
<a id="ref8"></a><p><strong>[8] Probing the kinetics of formation of the bacteriophage MS2 translational operator complex: identification of a protein conformer unable to bind RNA.</strong></p><br />
Lago, H., Parrott, A. M., Moss, T., Stonehouse, N. J., & Stockley, P. G.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/11162119/" target="_blank" style="color:#520049">Journal of molecular biology, 305(5), 1131–1144. (2001).</a>
<br />
                
<a id="ref9"></a><p><strong>[9] The crystal structure of a high affinity RNA stem-loop complexed with the bacteriophage MS2 capsid: further challenges in the modeling of ligand-RNA interactions..</strong></p><br />
Horn, W. T., Convery, M. A., Stonehouse, N. J., Adams, C. J., Liljas, L., Phillips, S. E., & Stockley, P. G.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/15496523/" target="_blank" style="color:#520049">RNA (New York, N.Y.), 10(11), 1776–1782. (2004).</a>
<br />
                

                       
