---
layout: theophylline_style
title: Bovine prion protein aptamer
date: 2024-03-29 00:00:00
author: Zhijie Tan, Zhaoji Qiu
categories: Aptamer
tags:
- Category:Prion
- GC:52.580000000000005
- GC:66.0
- Length:12
- Length:97
- 'Named:Bovine prion protein (bPrP)_Apt #1'
- Named:Bovine prion protein_R12 aptamer
- Type:Proteins
- Year:2007
- Year:2008
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/9343239/" target="_blank" style="color:#520049">1997</a></h3>
    </div>
    <div class="body">
      <p>The first aptamer for PrP was selected in 1997 using the SELEX method directed against recombinant Syrian hamster full-length prion protein (rPrP23–231)<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/19098441/" target="_blank" style="color:#520049">2008</a></h3>
    </div>
    <div class="body">
      <p>RNA aptamers against a bovine bPrP were obtained by means of an in vitro selection method from RNA pools containing a 55-nt randomized region<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/23180780/" target="_blank" style="color:#520049">2009</a></h3>
    </div>
    <div class="body">
      <p>High-resolution structure of an RNA aptamer (R12) against isolated domains of the bovine PrP by NMR was first reported and two lysine clusters of bPrP have been identified as binding sites for R12<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/19666720/" target="_blank" style="color:#520049">2013</a></h3>
    </div>
    <div class="body">
      <p>NMR measurements provide the first high-resolution 3D-structure of the complex formed with N-terminal PrP peptides (P1 and P16) and the R12 aptamer. What's more, R12 reduced the PrP<sub>Sc</sub> level in mouse neuronal cells persistently infected with the transmissible spongiform encephalopathy agent<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/24803670/" target="_blank" style="color:#520049">2014</a></h3>
    </div>
    <div class="body">
      <p>The hybrid method and the 3D-RISM theory are employed to demonstrate that the driving force for the binding between R12 and P16 (a PrP peptide) is a robust gain of water entropy, and the energy decrease driven by attractive interactions between R12 and P16 is compensated by the energetic dehydration effect after binding or vice-versa<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/30916478/" target="_blank" style="color:#520049">2019</a></h3>
    </div>
    <div class="body">
      <p>An anti-prion RNA aptamer, R12, inhibits the interaction of PrP with Aβto prevent Alzheimer's disease<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
</div>


         
<p class="header_box" id="description">Description</p>
<P>In 2008, Nishikawa et al. employed the SELEX method to isolate aptamers and identified four consecutive GGA triplet repeats (GGA<sup>4</sup>) in the major RNA aptamers obtained. Subsequently, in 2013, Mashima et al. utilised NMR spectroscopy to determine the first high-resolution three-dimensional structure of the complex formed by the N-terminal PrP peptides (P1 and P16) and the R12 aptamer<sup>[<a href="#ref3" style="color:#520049">3</a></sup><sup>,<a href="#ref4" style="color:#520049">4</a></sup><sup>]</sup>.<br></P>



<p class="header_box" id="SELEX">SELEX</p>
<p>In 2008, Nishikawa et al. used the SELEX method to isolate aptamers and identified four consecutive GGA triplet repeats (GGA<sup>4</sup>) in the major RNA aptamers obtained. They carried out SELEX using 97-nt RNA pool that has 55-nt randomized sequences. To increase the selection stringency, they applied the following selection pressures: (1) decreased protein concentration and reaction time, and increased washing volumes; (2) increased tRNA concentration as a non-specific competitor; and (3) increased concentration of anti-mPrP RNA aptamer16 as a specific competitor<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.</P>
<p>
<br>
<br>    



<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>R12 aptamer forms an intramolecular parallel quadruplex, which consists of G:G:G:G tetrads and G(:A):G:G(:A):G hexad planes. Two quadruplexes assemble into a dimer through intermolecular hexad–hexad stacking. Here we utilized Ribodraw to complete the figure based on the 3D structure information<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.</p>
<p>5'-GGAGGAGGAGGA-3'</p>
<img src="/images/2D/prion_protein_aptamer_2D.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>



<p class="blowheader_box">3D visualisation</p>             
<P>Mashima et al. utilised NMR spectroscopy to determine the first high-resolution three-dimensional structure of the complex formed between the N-terminal PrP peptides (P1 and P16) and the R12 aptamer. There are the similar structures between 2RQJ，2RSK and 2RU7, so only the 2RU7 with last literature is chosen<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup>.</P>
<P>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</P>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/prion_protein_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
   <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A',
  start_residue_number:1,
  end_residue_number:4,
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'A', 
  start_residue_number:5, 
  end_residue_number:5, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'A',    
  start_residue_number:6, 
  end_residue_number:7, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'A', 
  start_residue_number:8, 
  end_residue_number:9, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'A',    
  start_residue_number:10, 
  end_residue_number:10, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'A', 
  start_residue_number:11, 
  end_residue_number:11, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'A',    
  start_residue_number:12, 
  end_residue_number:12, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'C',    
  start_residue_number:25, 
  end_residue_number:36, 
  color:{r:37,g:188,b:37}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/2ru7-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
    </body>
    </html>
    </td>
  </tr>
  </table>
  </div>


<p class="blowheader_box">Binding pocket</p>             
<P>Left: A surface representation of the aptamer’s binding pocket, generated from PDB ID: 2RU7 at a resolution of 2.8 Å. The bovine prion protein is depicted with vacuum electrostatics, where blue represents positive charges and red represents negative charges. Right: Hydrogen bonds at the binding sites of the aptamer bound to the bovine prion protein.</P>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/prion_protein_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/prion_protein_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>




<p class="header_box" id="ligand-recognition">Ligand information</p> 
  
<p class="blowheader_box">SELEX ligand</p>
<P>Radioisotope labeling of RNA by in vitro transcription was carried out using α-32P-ATP.  Refolded 32P labeled aptamer (10 nM) was mixed with varying concentrations of bPrP, or its derivatives, to a total volume of 25 µl in reaction buffer [20 mM Tris-HCl (pH 7.5), 100 mM NaCl or 10 mM KCl].   After 20 min incubation, the mixture was passed through a nitrocellulose filter and washed with 500 µl of the reaction buffer.   The amount of bound RNA was measured with BAS 2500, and binding activities were calculated as the percentage of input RNA retained on the filter in the protein-RNA complex.   We determined the equilibrium dissociation constant (Kd) using GraphPad PRISM using non-linear regression curve fitting<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>. R12 aptamer is a truncated form of apt #1 aptamer.</P>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">apt #1</td>
        <td name="td1">5′-CAAUCCAUUCAUCUCUCGAAUGAGGAAGUAGCCCAAGAGGAGGAGGAGGAUGAGC-3′</td>
        <td name="td3">bovine prion protein (bPrP)</td>
        <td name="td4">82 nM</td>
      </tr>
      <tr>
        <td name="td0">apt #6</td>
        <td name="td1">5′-ACCUUCUGUUCAUCGUGACCAACCCAAUAGAUUGUGAUAAAGGAGGAGGAGGA-3′</td>
        <td name="td3">bovine prion protein (bPrP)</td>
        <td name="td4">166 nM</td>
      </tr>
      <tr>
        <td name="td0">R12 aptamer</td>
        <td name="td1">5′-GGAGGAGGAGGA-3′</td>
        <td name="td3">bovine prion protein (bPrP)</td>
        <td name="td4">NA</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<p>The major prion protein (PrP) is encoded in the human body by the PRNP gene also known as CD230 (cluster of differentiation 230).Expression of the protein is most predominant in the nervous system but occurs in many other tissues throughout the body. The protein can exist in multiple isoforms: the normal PrPC form, and the protease-resistant form designated PrPRes such as the disease-causing PrPSc (scrapie) and an isoform located in mitochondria.-----From <a href="https://en.wikipedia.org/wiki/Major_prion_protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Wiki</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">Mass</th>
        <th onclick="sortTable(3)">Protein sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P10279/entry" target="_blank" style="color:#520049"><b>P10279</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR000817/" target="_blank" style="color:#520049"><b>IPR000817</b></a></td>
        <td name="td2">28.61 KDa</td>
        <td name="td3">
        <div class="sequence-container">

          <span class="full-sequence">MVKSHIGSWILVLFVAMWSDVGLCKKRPKPGGGWNTGGSRYPGQGSPGGNRYPPQGGGGWGQPHGGGWGQPHGGGWGQPHGGGWGQPHGGGWGQPHGGGGWGQGGTHGQWNKPSKPKTNMKHVAGAAAAGAVVGGLGGYMLGSAMSRPLIHFGSDYEDRYYRENMHRYPNQVYYRPVDQYSNQNNFVHDCVNITVKEHTVTTTTKGENFTETDIKMMERVVEQMCITQYQRESQAYYQRGASVILFSSPPVILLISFLIFLIVG</span>
        </div>
      </td>
        <td name="td4"><a href="https://www.rcsb.org/structure/1DX0" target="_blank" style="color:#520049"><b>1DX0</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/gene/281427" target="_blank" style="color:#520049"><b>281427</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
<img src="/images/Structure_ligand/prion_protein_stru_ligand1.svg" alt="drawing" style="width:800px;height:300px;"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and selected the previous information with high similarity for presentation.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">1DX0-A</td>
        <td name="td2">22.5</td>
        <td name="td3">0</td>
        <td name="td4">Original chain</td>
      </tr>
      <tr>
        <td name="td1">2KUN-A</td>
        <td name="td2">12</td>
        <td name="td3">2.7</td>
        <td name="td4">major prion protein</td>
      </tr>
      <tr>
        <td name="td1">1I17-A</td>
        <td name="td2">6.1</td>
        <td name="td3">3.5</td>
        <td name="td4">prion-like protein</td>
      </tr>
      <tr>
        <td name="td1">6THH-C</td>
        <td name="td2">5.2</td>
        <td name="td3">2.8</td>
        <td name="td4">sirv2 acrid1 (gp02) anti-crispr protein</td>
      </tr>
      <tr>
        <td name="td1">7BU0-A</td>
        <td name="td2">5</td>
        <td name="td3">5</td>
        <td name="td4">uncharacterized protein</td>
      </tr>
      <tr>
        <td name="td1">2VWA-A</td>
        <td name="td2">5</td>
        <td name="td3">3.1</td>
        <td name="td4">putative uncharacterized protein pf13_0012</td>
      </tr>
      <tr>
        <td name="td1">6WB9-6</td>
        <td name="td2">4.9</td>
        <td name="td3">3.4</td>
        <td name="td4">endoplasmic reticulum membrane protein complex su</td>
      </tr>
      <tr>
        <td name="td1">2KHM-A</td>
        <td name="td2">4.8</td>
        <td name="td3">3.6</td>
        <td name="td4">fibroin-3</td>
      </tr>
      <tr>
        <td name="td1">3WFW-A</td>
        <td name="td2">4.8</td>
        <td name="td3">2.9</td>
        <td name="td4">hemoglobin-like flavoprotein fused to roadblock/l</td>
      </tr>
      <tr>
        <td name="td1">8CR1-C</td>
        <td name="td2">4.7</td>
        <td name="td3">2.5</td>
        <td name="td4">atpase asna1</td>
      </tr>
	  </tbody>
  </table>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] An intramolecular quadruplex of (GGA)4 triplet repeat DNA with a G:G:G:G tetrad and a G(:A):G(:A):G(:A):G heptad, and its dimeric interaction.</strong></font><br />
Matsugami, A., Ouhashi, K., Kanagawa, M., Liu, H., Kanagawa, S., Uesugi, S., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/11800555/" target="_blank" style="color:#520049" >Journal of Molecular Biology, 313(2):255-69. (2001)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Detection of structural changes of RNA aptamer containing GGA repeats under the ionic condition using the microchip electrophoresis.</strong></font><br />
Nishikawa, F., Murakami, K., Noda, K., Yokoyama, T., & Nishikawa, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18029754/" target="_blank" style="color:#520049">Nucleic Acids Symposium Series,(51):397-8. (2007)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] Anti-bovine Prion protein RNA aptamer containing tandem GGA repeat interacts both with recombinant bovine prion protein and its β isoform with high affinity.</strong></font><br />
Murakami, K., Nishikawa, F., Noda, K., Yokoyama, T., & Nishikawa S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/19098441/" target="_blank" style="color:#520049">Prion, 2(2):73-80. (2008)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] Structural analysis of r(GGA)4 found in RNA aptamer for bovine prion protein.</strong></font><br />
Matsugami, A., Mashima, T., Nishikawa, F., Murakami, K., & Nishikawa, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18776312/" target="_blank" style="color:#520049">Nucleic Acids Symposium Series, (52):179-80. (2008)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Unique quadruplex structure and interaction of an RNA aptamer against bovine prion protein.</strong></font><br />
Mashima, T., Matsugami, A., Nishikawa, F., Nishikawa, S., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/19666719/" target="_blank" style="color:#520049">Nucleic Acids Research, 37(18):6249-58. (2009)</a>
<br />
                
<a id="ref6"></a><font><strong>[6] Anti-prion activity of an RNA aptamer and its structural basis.</strong></font><br />
Mashima, T., Nishikawa, F., Kamatari, YO., Fujiwara, H., Saimura, M., Nagata, T., Kodaki, T., Nishikawa, S., Kuwata, K., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23180780/" target="_blank" style="color:#520049">Nucleic Acids Research, 41(2):1355-62. (2012)</a>
<br />

<a id="ref7"></a><font><strong>[7] Cross-talk between prion protein and quadruplex-forming nucleic acids: a dynamic complex formation.</strong></font><br />
Cavaliere, P., Pagano, B., Granata, V., Prigent, S., Rezaei, H., Giancola, C., & Zagari, A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23104426/" target="_blank" style="color:#520049">Nucleic Acids Research, 41(1):327-39. (2013)</a>
<br />

<a id="ref8"></a><font><strong>[8] Binding of an RNA aptamer and a partial peptide of a prion protein: crucial importance of water entropy in molecular recognition.</strong></font><br />
Hayashi, T., Oshima, H., Mashima, T., Nagata, T., Katahira, M., & Kinoshita, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/24803670/" target="_blank" style="color:#520049">Nucleic Acids Research, 42(11):6861-75. (2014)</a>
<br />

<a id="ref9"></a><font><strong>[9] The anti-prion RNA aptamer R12 disrupts the Alzheimer's disease-related complex between prion and amyloid β.</strong></font><br />
Iida, M., Mashima, T., Yamaoki, Y., So, M., Nagata, T., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/30916478/" target="_blank" style="color:#520049">FEBS Journal, 286(12):2355-2365. (2019)</a>
<br />

<a id="ref10"></a><font><strong>[10] Development and structural determination of an anti-PrPC aptamer that blocks pathological conformational conversion of prion protein.</strong></font><br />
Mashima, T., Lee, JH., Kamatari, YO., Hayashi, T., Nagata, T., Nishikawa, F., Nishikawa, S., Kinoshita, M., Kuwata, K., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/32188933/" target="_blank" style="color:#520049">Scientific Reports, 10(1):4934. (2020)</a>
<br />




<script>
    function toggleSequence(event) {
      const container = event.target.closest('.sequence-container');
      container.classList.toggle('expanded');
      const showMoreText = container.querySelector('.show-more');
      
      // 展开后按钮文本变化
      if (container.classList.contains('expanded')) {
        showMoreText.textContent = '...';  // 展开后显示 "..."
      } else {
        showMoreText.textContent = '......';  // 收起后显示 "......"
      }
    }

    // 页面加载时，限制序列文本为50个字符
    window.addEventListener('load', function() {
      const sequenceContainers = document.querySelectorAll('.sequence-container');
      sequenceContainers.forEach(container => {
        const fullSeqText = container.querySelector('.full-sequence').textContent;
        const truncatedText = fullSeqText.slice(0, 20);  // 只显示前50个字符
        container.querySelector('.sequence-text').textContent = truncatedText;
      });
    });
  </script>