---
layout: theophylline_style
title: C52 aptamer
date: 2024-05-06 00:00:00
author: Ke Chen, Yangyi Ren
categories: Aptamer
tags:
- Category:Viral
- GC:40.18
- GC:43.120000000000005
- GC:43.75
- GC:44.04
- GC:44.64
- GC:45.45
- GC:45.87
- GC:46.85
- GC:47.22
- GC:50.470000000000006
- Length:107
- Length:108
- Length:109
- Length:110
- Length:111
- Length:112
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C1
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C15
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C17
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C18
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C24
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C33
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C34
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C37
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C5
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C52
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C8
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C9
- Type:Proteins
- Year:1996
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8523524/" target="_blank" style="color:#520049">1996</a></h3>
  </div>
  <div class="body">
    <p>Aptamers were found to be functionally equivalent to the RBE when the assay system was saturated with Rev and better than the wild-type element when Rev was limiting<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8755498/" target="_blank" style="color:#520049">1996</a></h3>
  </div>
  <div class="body">
    <p>Using the SELEX method, Ellington, A. D. and colleagues obtained a series of anti-peptide aptamers that can recognize amino acid sequences<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8946856/" target="_blank" style="color:#520049">1996</a></h3>
  </div>
  <div class="body">
    <p>HIV-1 17-mer rev peptide and RNA aptamer complex formation involves adaptive binding with the alpha-helical arginine-rich basic rev peptide targeting a widened RNA major groove centred about adjacent G.A and reversed A.A mismatches<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/9713975/" target="_blank" style="color:#520049">1998</a></h3>
  </div>
  <div class="body">
    <p>Specific inhibition of viral p24 production following co-transfection of the anti-HIV Rev-binding aptamer and HIV proviral DNAs was observed<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/15610009/" target="_blank" style="color:#520049">2004</a></h3>
  </div>
  <div class="body">
    <p>More than half of the residues have increased flexibility in the Rev-RNA aptamer complex that has a higher affinity. This further suggests that the retention of conformational flexibility may be important in high-affinity ARM-RNA recognition<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/30017564/" target="_blank" style="color:#520049">2018</a></h3>
  </div>
  <div class="body">
    <p>Wingfield, P. T. et al. present the crystal structure of Rev93:scFv:RBA-14 Ternary Complex at 3.0 Å through X-ray<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
  </div>
 </div>
</div>



<p class="header_box" id="description">Description</p>
<p>In 1996, Ellington, A. D. and colleagues used the SELEX method to isolate the anti-peptide aptamers with high affinity for the HIV-1 Rev protein. HIV-1 Rev protein were selected from a random sequence RNA pool. Several of the selected RNAs could bind the free peptide more tightly than a natural RNA ligand, the Rev-binding element. In accord with the hypothesis that protein and nucleic acid binding cusps are functionally similar, interactions between aptamers and the peptide target could be disrupted by sequence substitutions. Moreover, the aptamers appeared to be able to bind peptides with different solution conformations, implying an induced fit mechanism for binding. Just as anti-peptide antibodies can sometimes recognize the corresponding epitope when presented in a protein, the anti-peptide aptamers were found to specifically bind to Rev<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 1996, Ellington, A. D. and colleagues used existing research to design the method required for the SELEX process. The RNA recognition domain of HIV-1 Rev is an ARM that spans residues 34-50. A 17-mer peptide (sRev) corresponding to this domain was used as a target for in vitro selection experiments. A RNA pool containing 71 random sequence positions was incubated with an affinity resin containing sRev<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p>
<p>


<p class="header_box" id="Structure">Structure</p>
<p>C52 was the aptamer sequence mainly studied in the article, which had a high affinity with HIV-1 Rev protein. The 2D structure of the figures is based on the prediction results of the RNA fold website by ribodraw tool to draw. The C52 aptamer was named by Ellington, A. D. et al. in the article<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p>
<p>5'-GGGAGAUACCAGCUUAUUCAAUUGCUUGGUACCGAGCUCGGAUCCACGUAGUAACGGGCCGCCAGUGUCUGGAAUUCGGGUCGUUCUUGAGAUAGUAAGUGCAAUCU-3'</p>
<img src="/images/2D/C52_aptamer_2D1.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>



<p class="header_box" id="ligand-recognition">Ligand information</p>

<p class="blowheader_box">SELEX ligand</p>
<p>REV is a viral anti-repression trans-activator protein, which appears to act post-transcriptionally to relieve negative repression of GAG and ENV production. It is a phosphoprotein whose state of phosphorylation is mediated by a specific serine kinase activity present in the nucleus. REV accumulates in the nucleoli.-----From <a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00424/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">Mass</th>
        <th onclick="sortTable(4)">Protein sequence</th>
        <th onclick="sortTable(5)">PDB ID</th>
        <th onclick="sortTable(6)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">HIV-1 Rev protein</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P04616/entry" target="_blank" style="color:#520049"><b>P04616</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00424/" target="_blank" style="color:#520049"><b>PF00424</b></a></td>
        <td name="td3">3.22 kDa</td>
        <td name="td4">
        <div class="sequence-container">
          <span class="sequence-text"></span>
          <span class="show-more" onclick="toggleSequence(event)">......</span>
          <span class="full-sequence">GAMATRQARRNRRRRWRERQRAAAAR (residues 34-50)</span>
        </div>
        </td>
        <td name="td5"><a href="https://www.rcsb.org/structure/2M1A" target="_blank" style="color:#520049"><b>2M1A</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/protein/AAA44200.1" target="_blank" style="color:#520049"><b>AAA44200.1</b></a></td>
      </tr>
	  </tbody>
  </table>

<p>Some isolated sequences bind to the affinity of the protein<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">C52</td>
        <td name="td1">5'-GGGAGAUACCAGCUUAUUCAAUUGCUUGGUACCGAGCUCGGAUCCACGUAGUAACGGGCCGCCAGUGUCUGGAAUUCGGGUCGUUCUUGAGAUAGUAAGUGCAAUCU-3'</td>
        <td name="td2">HIV-1 Rev protein</td>
        <td name="td3">19-36 nM</td>
      </tr>
	  </tbody>
     <tbody>
      <tr>
        <td name="td0">C17</td>
        <td name="td1">5'-GGGAGAUACCAGCUUAUUCAAUUGUAUUCUCCGUGGUUUAAUCAGAGUAGAGGAGCUGACUCCUUUGGUUGGACUACGUGGAGGUGCUCUUAGAUAGUAAGUGCAAUCU-3'</td>
        <td name="td2">HIV-1 Rev protein</td>
        <td name="td3">19-36 nM</td>
      </tr>
	  </tbody>
      <tbody>
      <tr>
        <td name="td0">C8</td>
        <td name="td1">5'-GGGAGAUACCAGCUUAUUCAAUUGAGCCAGUAAGUGACCCGUACUAAUACUGUUGAGUAGUAUGUAGAGGAGUGGUGAUCCUCCAAACUGCUGAGAUAGUAAGUGCAAUCU-3'</td>
        <td name="td2">HIV-1 Rev protein</td>
        <td name="td3">19-36 nM</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/C52_aptamer_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>


<p class="blowheader_box">Similar compound(s)</p>                    
<p>We used the RCSB PDB website's <a href="https://www.rcsb.org/docs/search-and-browse/advanced-search/structure-similarity-search" target="_blank" style="color:#520049; text-decoration: underline;"><b>similar structure search</b></a> to find the top 10 structures similar to HIV-1 REV PROTEIN (residues T34-R50), and calculated TM-socre values and RMSD values using the <a href="https://zhanggroup.org/TM-align/" target="_blank" style="color:#520049; text-decoration: underline;"><b>TM-align website</b></a>.</p>

<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">TM-socre</th>
        <th onclick="sortTable(2)">RMSD (Å)</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/2LD2" target="_blank" style="color:#520049"><b>2LD2</b></a></td>
      <td name="td1">0.358</td>
      <td name="td2">0.89</td>
      <td name="td3">Solution structure of the N-terminal domain of huntingtin (htt17) in presence of DPC micelles</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/1DNG" target="_blank" style="color:#520049"><b>1DNG</b></a></td>
      <td name="td1">0.442</td>
      <td name="td2">1.68</td>
      <td name="td3">NMR structure of a model hydrophilic amphipathic helical acidic peptide</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/3N95" target="_blank" style="color:#520049"><b>3N95</b></a>-E</td>
      <td name="td1">0.034</td>
      <td name="td2">1.7</td>
      <td name="td3">Crystal structure of human CRFR2 alpha extracellular domain in complex with Urocortin 2</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/1OMQ" target="_blank" style="color:#520049"><b>1OMQ</b></a></td>
      <td name="td1">0.366</td>
      <td name="td2">1.1</td>
      <td name="td3">Structure of penetratin in bicellar solution</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/7LSO" target="_blank" style="color:#520049"><b>7LSO</b></a></td>
      <td name="td1">0.44</td>
      <td name="td2">0.67</td>
      <td name="td3">L-Phenylseptin</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/2RLG" target="_blank" style="color:#520049"><b>2RLG</b></a></td>
      <td name="td1">0.34</td>
      <td name="td2">0.79</td>
      <td name="td3">NMR structure of the antimicrobial peptide RP-1 bound to SDS micelles</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/3N93" target="_blank" style="color:#520049"><b>3N93</b></a>-E</td>
      <td name="td1">0.03</td>
      <td name="td2">1.62</td>
      <td name="td3">Crystal structure of human CRFR2 alpha extracellular domain in complex with Urocortin 3</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/7LSP" target="_blank" style="color:#520049"><b>7LSP</b></a></td>
      <td name="td1">0.465</td>
      <td name="td2">0.56</td>
      <td name="td3">D-Phenylseptin - The second residue of PHE of the peptide is a D-amino acid</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/1O53" target="_blank" style="color:#520049"><b>1O53</b></a></td>
      <td name="td1">0.371</td>
      <td name="td2">0.49</td>
      <td name="td3">Solution structure of the N-terminal membrane anchor of E. coli enzyme IIA(Glucose)</td>
    </tr>
     <tr>
      <td name="td0"><a href="https://www.rcsb.org/structure/6GS9" target="_blank" style="color:#520049"><b>6GS9</b></a></td>
      <td name="td1">0.303</td>
      <td name="td2">1.36</td>
      <td name="td3">NMR structure of aurein 2.5 in SDS micelles</td>
    </tr>
    </tbody>
  </table>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] RNA aptamers selected to bind human immunodeficiency virus type 1 Rev in vitro are Rev responsive in vivo.</strong></font><br />
Symensma, T. L., Giver, L., Zapp, M., Takle, G. B., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8523524/" target="_blank" style="color:#520049">Journal of virology, 70(1), 179–187. (1996)</a>
<br/>
            
<a id="ref2"></a><font><strong>[2] Anti-peptide aptamers recognize amino acid sequence and bind a protein epitope.</strong></font><br />
Xu, W., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8755498/" target="_blank" style="color:#520049">Proceedings of the National Academy of Sciences of the United States of America, 93(15), 7475–7480. (1996)</a>
<br/>
            
<a id="ref3"></a><font><strong>[3] Deep penetration of an alpha-helix into a widened RNA major groove in the HIV-1 rev peptide-RNA aptamer complex.</strong></font><br />
Ye, X., Gorin, A., Ellington, A. D., & Patel, D. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8946856/" target="_blank" style="color:#520049">Nature structural biology, 3(12), 1026–1033. (1996)</a>
<br/>
            
<a id="ref4"></a><font><strong>[4] Receptor ligand-facilitated cationic liposome delivery of anti-HIV-1 Rev-binding aptamer and ribozyme DNAs.</strong></font><br />
Konopka, K., Düzgüneş, N., Rossi, J., & Lee, N. S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/9713975/" target="_blank" style="color:#520049">Journal of drug targeting, 5(4), 247–259. (1998)</a>
<br/>
            
<a id="ref5"></a><font><strong>[5] Retention of conformational flexibility in HIV-1 Rev-RNA complexes.</strong></font><br />
Wilkinson, T. A., Zhu, L., Hu, W., & Chen, Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/15610009/" target="_blank" style="color:#520049">Biochemistry, 43(51), 16153–16160. (2004)</a>
<br/>
            
<a id="ref6"></a><font><strong>[6] Structure of an RNA Aptamer that Can Inhibit HIV-1 by Blocking Rev-Cognate RNA (RRE) Binding and Rev-Rev Association.</strong></font><br />
Dearborn, A. D., Eren, E., Watts, N. R., Palmer, I. W., Kaufman, J. D., Steven, A. C., & Wingfield, P. T.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/30017564/" target="_blank" style="color:#520049">Structure (London, England : 1993), 26(9), 1187–1195.e4. (2018)</a>
<br/>


<script>
    function toggleSequence(event) {
      const container = event.target.closest('.sequence-container');
      container.classList.toggle('expanded');
      const showMoreText = container.querySelector('.show-more');
      
      // 展开后按钮文本变化
      if (container.classList.contains('expanded')) {
        showMoreText.textContent = '...';  // 展开后显示 "..."
      } else {
        showMoreText.textContent = '......';  // 收起后显示 "......"
      }
    }

    // 页面加载时，限制序列文本为50个字符
    window.addEventListener('load', function() {
      const sequenceContainers = document.querySelectorAll('.sequence-container');
      sequenceContainers.forEach(container => {
        const fullSeqText = container.querySelector('.full-sequence').textContent;
        const truncatedText = fullSeqText.slice(0, 20);  // 只显示前50个字符
        container.querySelector('.sequence-text').textContent = truncatedText;
      });
    });
  </script>