---
layout: theophylline_style
title: CypB aptamer
date: 2024-06-01 00:00:00
author: Zhijie Tan, Qiaozhen Liu
categories: Aptamer
tags:
- Category:Mammalian
- GC:51.2
- Length:78
- Named:Cyclophilin B (CypB)_M9-5
- Type:Proteins
- Year:2012
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="description">Description</p>
<p>In 2012, Ray et al. identified the M9-5 aptamer using an in vitro positive/negative selection strategy to identify RNA ligands (aptamers) that could detect structural differences between the secretomes of pancreatic cancer and non-cancerous cells. In 2013, Ray et al. discovered the potential M9-5 selectivity which was the post-translational modifications on cyclophilin B<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref2" style="color:#520049">2</a></sup><sup>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2012, Ray et al. identified the M9-5 aptamer used the SELEX method to isolate the aptamer after 9 rounds of selection<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>M9-5 was the aptamer sequence mainly studied in the article, which had a high affinity with CypB. The 2D structure of the figures is based on the prediction results of the RNA fold website by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>
<p>5'-GGACCUAUGCAGUAGCCAGUGUGGACUGGGCUGCCCCCCC-3'</p>
<img src="/images/2D/cypB_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>Cyclophilins exhibit peptidyl-prolyl cis-trans isomerase (PPIase) activity, accelerating protein folding by catalysing the cis-trans isomerisation of proline imidic peptide bonds in oligopeptides. They also have protein chaperone-like functions and are the major high-affinity binding proteins for the immunosuppressive drug cyclosporin A (CSA) in vertebrates.
Cyclophilins are found in all prokaryotes and eukaryotes, and have been structurally conserved throughout evolution, implying their importance in cellular function. They share a common 109 amino acid cyclophilin-like domain (CLD) and additional domains unique to each member of the family. The CLD domain contains the PPIase activity, while the unique domains are important for selection of protein substrates and subcellular compartmentalisation.This entry represents the core Î²-barrel cyclophilin-like domain.-----from Pfam</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">cyclophilin B (CypB)</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P0AFL3/entry" target="_blank" style="color:#520049"><b>P0AFL3</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR002130/" target="_blank" style="color:#520049"><b>IPR002130</b></a></td>
        <td name="td3">18.07 kDa</td>
        <td name="td4">AKGDPHVLLTTSAGNIELELDKQKAPVSVQNFVDYVNSGFYNNTTFHRVIPGFMIQGGGFTEQMQQKKPNPPIKNEADNGLRNTRGTIAMARTADKDSATSQFFINVADNAFLDHGQRDFGYAVFGKVVKGMDVADKISQVPTHDVGPYQNVPSKPVVILSATVLP</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/1J2A" target="_blank" style="color:#520049"><b>1J2A</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/protein/HCR7126556.1" target="_blank" style="color:#520049"><b>HCR7126556.1</b></a></td>
      </tr>
	  </tbody>
  </table>

<p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">M9-5</td>
        <td name="td1">5'-GGACCUAUGCAGUAGCCAGUGUGGACUGGGCUGCCCCCCC-3'</td>
        <td name="td2">CypB</td>
        <td name="td3">50 nM</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/CypB_aptamer_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">7A5P-Z</td>
      <td name="td1">24.0</td>
      <td name="td2">1.7</td>
      <td name="td3">U2 snrna</td>
    </tr>
            
     <tr>
      <td name="td0">5EX1-C</td>
      <td name="td1">22.2</td>
      <td name="td2">1.8</td>
      <td name="td3">Peptidyl-prolyl cis-trans isomerase cyclophilin T</td>
    </tr>
            
     <tr>
      <td name="td0">7QTT-V</td>
      <td name="td1">20.7</td>
      <td name="td2">1.8</td>
      <td name="td3">Splicing factor 3B subunit 3</td>
    </tr>
            
     <tr>
      <td name="td0">5JHE-A</td>
      <td name="td1">20.4</td>
      <td name="td2">1.9</td>
      <td name="td3">Peptidyl-prolyl cis-trans isomerase CYP7</td>
    </tr>
            
     <tr>
      <td name="td0">7QTT-U</td>
      <td name="td1">18.1</td>
      <td name="td2">1.9</td>
      <td name="td3">Splicing factor 3B subunit 3</td>
    </tr>
            
     <tr>
      <td name="td0">2OSE-A</td>
      <td name="td1">16.5</td>
      <td name="td2">2.8</td>
      <td name="td3">Probable peptidyl-prolyl cis-trans isomerase</td>
    </tr>
            
     <tr>
      <td name="td0">3X27-D</td>
      <td name="td1">11.9</td>
      <td name="td2">2.6</td>
      <td name="td3">Cucumopine synthase</td>
    </tr>
            
     <tr>
      <td name="td0">3KOP-F</td>
      <td name="td1">11.5</td>
      <td name="td2">3.2</td>
      <td name="td3">Uncharacterized protein</td>
    </tr>
            
     <tr>
      <td name="td0">2NNZ-A</td>
      <td name="td1">8.6</td>
      <td name="td2">2.5</td>
      <td name="td3">Hypothetical protein</td>
    </tr>
            
     <tr>
      <td name="td0">3OPF-B</td>
      <td name="td1">6.7</td>
      <td name="td2">2.7</td>
      <td name="td3">Putative uncharacterized protein ttha0988</td>
    </tr>
    </tbody>
  </table>
                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Comparing human pancreatic cell secretomes by in vitro aptamer selection identifies cyclophilin B as a candidate pancreatic cancer biomarker.</strong></font><br />
Ray, P., Rialon-Guevara, KL., Veras, E., Sullenger, BA., & White, RR.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/22484812/" target="_blank" style="color:#520049">Journal of Clinical Investgation,122(5):1734-41. (2012)</a>
<br/>

<a id="ref2"></a><font><strong>[2] Further characterization of the target of a potential aptamer biomarker for pancreatic cancer: cyclophilin B and its posttranslational modifications.</strong></font><br />
Ray, P., Sullenger, BA., & White, RR.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/24152208/" target="_blank" style="color:#520049">Nucleic Acid Therapeutics,23(6):435-42. (2013)</a>
<br/>


<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1RAW-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>
    
    </html>
