---
layout: theophylline_style
title:  "Cytohesin-2 aptamer"
date:   2024-06-05 00:00:00
author: Yuxuan Sun, Baowei Huang
categories: Aptamer
---

<div class="content-section">

<div class="content-section">

<div class="content-section">

<div class="content-section">

<div class="content-section">

<!--p style="font-size: 24px"><strong>Keywords: </strong> <br/></p>
<font ><strong>small molecule, Synthetic, dyes, PubChem CID 25863</strong></font >
<br /-->

<br>
<br>

</div>

<div class="content-section">
<h2 class="section-title" id="timeline">Timeline</h2>
<div class="timeline">
  <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/15277685/" target="_blank" style="color:#520049">2004</a></h3>
  </div>
  <div class="body">
    <p>Mirko G Theis et.al selected and applied a specific inhibitor for ARF nucleotide-binding site opener (ARNO)/cytohesin-2<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
  </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/19854646/" target="_blank" style="color:#520049">2009</a></h3>
  </div>
  <div class="body">
    <p>Mayer G et.al describe a general route starting with the de novo selection process targeting cytohesin-1 and aiming at the synthesis of caged aptamers<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
  </div>

 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/23757206/" target="_blank" style="color:#520049">2013</a></h3>
  </div>
  <div class="body">
    <p>Niebel B et.al had identified three aptamers sharing a 15 nucleotides consensus motif, which were specifically target Tiam1<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
  </div>

<font>
</div>

<div class="content-section">
<h2 class="section-title" id="description">Description</h2></font>
<font>In 2004，Theis M.G and Knorre A had used an in Vitro Selectionan to isolate RNA aptamers with high affinity for cytohesin-2. This aptamer bound to an N-terminal segment of cytohesin-2 and did not inhibit ARF-GEF function in vitro<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></font>
<br>
<br>

</div>

<div class="content-section">
<h2 class="section-title" id="SELEX">SELEX</h2>
<p>In 2004, Theis MG and colleagues used existing research to design the method required for the in vitro genetic-selection. Starting from a RNA pool that had a sequence of 40 random nucleotide positions, aptamers that bind specifically to cytohesin-2 were selected after 15 rounds of selection and amplification. The aptamer, K61, can bind specifically to cytohesin-2 were selected with high affinity<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br>
Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br>
<br>

</div>

<div class="content-section">
<h2 class="section-title" id="Structure">Structure</h2>
<font>The 2D structure of the figure is based on the article by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></font>
<font><p>5'-GGGAGAGACAAGCUUGGGUCGCGUUUUGCUAUUGAUGUUCUUUUAACCUGGCCUGUUUGGGCCUCUUGCUCUUCCUAGGAGU-3'</p></font>
<img src="/images/2D/K61_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>
<br>

<font >
</div>

<div class="content-section">
<h2 class="section-title" id="ligand-recognition">Ligand information</h2></font>  

<div class="blowheader_box">SELEX ligand</div>
<p>Cytohesin-2 is a protein that in humans is encoded by the CYTH2 gene. Cytohesin-2 (CYTH2), formerly known as Pleckstrin homology, Sec7 and coiled/coil domains 2 (PSCD2), is a member of the cytohesin family. Members of this family have identical structural organization that consists of an N-terminal coiled-coil motif, a central Sec7 domain, and a C-terminal pleckstrin homology (PH) domain.  The coiled-coil motif is involved in homodimerization, the Sec7 domain contains guanine-nucleotide exchange protein (GEP) activity, and the PH domain interacts with phospholipids and is responsible for association of CYTHs with membranes.-----From Wiki</p>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Cytohesin-2</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/Q99418/entry" target="_blank" style="color:#520049"><b>Q99418</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/panther/PTHR10663/" target="_blank" style="color:#520049"><b>PTHR10663</b></a></td>
        <td name="td3">46.546 kDa</td>
        <td name="td4">MEDGVYEPPDLTPEERMELENIRRRKQELLVEIQRLREELSEAMSEVEGLEANEGSKTLQRNRKMAMGRKKFNMDPKKGIQFLVENELLQNTPEEIARFLYKGEGLNKTAIGDYLGEREELNLAVLHAFVDLHEFTDLNLVQALRQFLWSFRLPGEAQKIDRMMEAFAQRYCLCNPGVFQSTDTCYVLSFAVIMLNTSLHNPNVRDKPGLERFVAMNRGINEGGDLPEELLRNLYDSIRNEPFKIPEDDGNDLTHTFFNPDREGWLLKLGGGRVKTWKRRWFILTDNCLYYFEYTTDKEPRGIIPLENLSIREVDDPRKPNCFELYIPNNKGQLIKACKTEADGRVVEGNHMVYRISAPTQEEKDEWIKSIQAAVSVDPFYEMLAARKKRISVKKKQEQP</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/1PBV" target="_blank" style="color:#520049"><b>1PBV</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/9266" target="_blank" style="color:#520049"><b>9266</b></a></td>
      </tr>
	  </tbody>
  </table>

  <p>The aptamer bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">K61_aptamer</td>
        <td name="td1">5'-GGGAGAGACAAGCUUGGGUCGCGUUUUGCUAUUGAUGUUCUUUUAACCUGGCCUGUUUGGGCCUCUUGCUCUUCCUAGGAGU-3'</td>
        <td name="td2">cytohesin-2</td>
        <td name="td3">115 ± 3 nM</td>
      </tr>
      <tr>
        <td name="td0">K61_aptamer</td>
        <td name="td1">5'-GGGAGAGACAAGCUUGGGUCGCGUUUUGCUAUUGAUGUUCUUUUAACCUGGCCUGUUUGGGCCUCUUGCUCUUCCUAGGAGU-3'</td>
        <td name="td2">cytohesin-1</td>
        <td name="td3">4000 nM</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/K61_aptamer_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>

<div class="blowheader_box">Similar compound</div>                    
<font><p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">6BBP-A</td>
      <td name="td1">27.9</td>
      <td name="td2">2.1</td>
      <td name="td3">Cytohesin-3, Adp-Ribosylation Factor 6</td>
    </tr>
            
     <tr>
      <td name="td0">8UCQ-A</td>
      <td name="td1">24.7</td>
      <td name="td2">1.3</td>
      <td name="td3">Sec7 Domain-Containing Protein</td>
    </tr>
            
     <tr>
      <td name="td0">7VMB-A</td>
      <td name="td1">22.3</td>
      <td name="td2">2.1</td>
      <td name="td3">Iq Motif And Sec7 Domain-Containing Protein 1</td>
    </tr>
            
     <tr>
      <td name="td0">1XSZ-A</td>
      <td name="td1">19.5</td>
      <td name="td2">2.5</td>
      <td name="td3">Guanine Nucleotide Exchange Protein</td>
    </tr>
            
     <tr>
      <td name="td0">8EZJ-A</td>
      <td name="td1">17.2</td>
      <td name="td2">2.3</td>
      <td name="td3">Arf Guanine-Nucleotide Exchange Factor 2</td>
    </tr>
            
     <tr>
      <td name="td0">5L7S-A</td>
      <td name="td1">5.2</td>
      <td name="td2">3.6</td>
      <td name="td3">Secreted Rxlr Effector Peptide Protein</td>
    </tr>
            
     <tr>
      <td name="td0">7V99-A</td>
      <td name="td1">4.4</td>
      <td name="td2">4.4</td>
      <td name="td3">Telomerase Reverse Transcriptase</td>
    </tr>
            
     <tr>
      <td name="td0">7XVK-B</td>
      <td name="td1">4.1</td>
      <td name="td2">3.9</td>
      <td name="td3">Serine/Threonine-Protein Phosphatase 2A 65 Kda Re</td>
    </tr>
            
     <tr>
      <td name="td0">8UMD-B</td>
      <td name="td1">4.1</td>
      <td name="td2">11.3</td>
      <td name="td3">Flagellar M-Ring Protein</td>
    </tr>
            
     <tr>
      <td name="td0">4J8S-A</td>
      <td name="td1">4.0</td>
      <td name="td2">4.4</td>
      <td name="td3">Ccr4-Not Transcription Complex Subunit 1</td>
    </tr>
    </tbody>
  </table>
<br>
<br>

                 

</div>

<div class="content-section">
<h2 class="section-title" id="references">References</h2>
                
<a id="ref1"></a><font><strong>[1] Discriminatory aptamer reveals serum response element transcription regulated by cytohesin-2.</strong></font><br />
Theis MG, Knorre A, Kellersch B, Moelleken J, Wieland F, Kolanus W, Famulok M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/15277685/" target="_blank" style="color:#520049">Proc Natl Acad Sci U S A. 101(31):11221-6. (2004)</a>
<br/>
            
<a id="ref2"></a><font><strong>[2] From selection to caged aptamers: identification of light-dependent ssDNA aptamers targeting cytohesin.</strong></font><br />
Mayer G, Lohberger A, Butzen S, Pofahl M, Blind M, Heckel A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/19854646/" target="_blank" style="color:#520049">Bioorg Med Chem Lett. 19(23):6561-4. (2009)</a>
<br/>
            
<a id="ref3"></a><font><strong>[3] RNA-aptamers that modulate the RhoGEF activity of Tiam1.</strong></font><br />
Niebel B, Wosnitza CI, Famulok M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23757206/" target="_blank" style="color:#520049">Bioorg Med Chem. 21(20):6239-46. (2013)</a>
<br/>

      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/pdbe-molstar-plugin.js"></script>
    

</div>

