---
layout: theophylline_style
title:  "eIF4A aptamer"
date:   2024-04-9 00:00:00
author: Zhijie Tan, Qiaozhen Liu
categories: Aptamer
---
<html>
<head>
  <style>

  </style>
</head>
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
 
.blowheader_box{
    display: block;
      padding: 6px;
      font-size:20px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 1px;
      width:190px;
      height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
  .dot-paragraph::before {
            content: "• "; /* 点号和空格 */
            color: black; /* 设置点号颜色 */
            font-size: 20px; /* 调整点号大小 */
        }
  .dot-paragraph {
            margin: 5px 0; /* 调整带有点的段落的上下外边距 */
            line-height: 1.2; /* 调整带有点的段落的行高 */
        }
        .sequence-container {
      position: relative;
      max-width: 100%;
      white-space: normal;
      overflow-wrap: break-word;
    }
    .sequence-text {
      display: inline-block;
      white-space: nowrap;
      max-width: 100%;
      overflow: hidden;
    }
    .show-more {
      display: inline-block;
      color: #520049;
      cursor: pointer;
      font-weight: bold;
      text-decoration: underline;
    }
    .full-sequence {
      display: none;
    }
    .sequence-container.expanded .sequence-text {
      display: none;
    }
    .sequence-container.expanded .full-sequence {
      display: inline;
    }
    .sequence-container.collapsed .sequence-text {
      white-space: normal;
      display: inline-block;
      max-width: 100%;
    }
</style>
</head>

<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>

         
<p class="header_box" id="description">Description</p>
<p>In 2003, Nakamura, Y. et al. generated the RNA aptamers 4Apt with high affinity for human eIF4A by SELEX, then confirming the secondary structure. Afterwards, in 2005, Nakamura, Y. et al. determined the solution structure of two essential nucleotide loops (AUCGCA and ACAUAGA) within the aptamer 4Apt using nuclear magnetic resonance (NMR) spectroscopy, and proposed that the AUCGCA loop is directly involved in the interaction with eIF4A<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref2" style="color:#520049">2</a></sup><sup>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2003, high-affinity RNA aptamers for eIF4A were generated via in vitro RNA selection by Nakamura, Y. et al. These RNAs possess different binding affinities for eIF4A, with dissociation constants ranging from 10 nM to 10 μM.. Representative RNA sequences selected from randomized RNA libraries using different selection procedures. In selection I(+) and I(−), RNAs were selected from N40 pool in the presence and absence of ATP, respectively, via affinity precipitation with Ni-NTA agarose. Selection II and III used N40 and N30 pools, respectively, and RNAs were selected in the absence of ATP via nitrocellulose membrane trapping<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>   



<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>In the previous SELEX experiment, Aptamer no.21 was islated. Aptamer no.21 showed highest binding affinity to eIF4A among the selected aptamers, the 58 nt core structure of this aptamer is required for efficient recognition of eIF4A. Furthermore, the aptamer binds only to an intact eIF4A, and not to either domain when split at the linker of the ‘dumbbell’ structure. APT58 (58 nt long), a variant of the aptamer no.21 core structure that sustained 4 nt substitutions (G3→A, C6→U, G10→A and C12→U) for NMR analysis<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p>
<p>5'-GGAGAUCGCACUCCACAUGUGAGUGAGGCCGAAACAUAGAUUCGACAGGAGGCUCACA-3'</p>
<img src="/images/2D/eIF4A_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;">   
</div>
<br>


<p class="blowheader_box">3D visualisation</p>             
<p>Nakamura, Y. et al. determined the solution structure of two essential nucleotide loops (AUCGCA and ACAUAGA) within the aptamer APT58 aptamer using NMR spectroscopy, suggesting that the C7 and C9 of the AUCGCA loop directly interact with eIF4A. Due to the lack of complex structure of 4Apt aptamer and eIF4A. In the AUCGCA loop, C7 and C9 are marked for the directly interacted bases, while ACCGCA loop forms a similar conformation to U-turn motif and that the U-turn like conformation could be important for eIF4A binding without directly interacted bases<sup>[<a href="#ref2" style="color:#520049">2</a></sup><sup>]</sup>.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/eIF4A_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'C',
  start_residue_number:1,
  end_residue_number:4,
  color:{r:184,g:51,b:50}
},
{
  struct_asym_id: 'C', 
  start_residue_number:5, 
  end_residue_number:6, 
  color:{r:133,g:39,b:181}
},
{
  struct_asym_id: 'C',    
  start_residue_number:7, 
  end_residue_number:7, 
  color:{r:241,g:125,b:7}
},{
  struct_asym_id: 'C', 
  start_residue_number:5, 
  end_residue_number:6, 
  color:{r:133,g:39,b:181}
{
  struct_asym_id: 'C', 
  start_residue_number:9, 
  end_residue_number:9, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'C', 
  start_residue_number:10, 
  end_residue_number:10, 
  color:{r:133,g:39,b:181}
},{
  struct_asym_id: 'C', 
  start_residue_number:11, 
  end_residue_number:15, 
  color:{r:184,g:51,b:50}
},
{
  struct_asym_id: 'A', 
  start_residue_number:1, 
  end_residue_number:4, 
  color:{r:207,g:62, b:125}
},{
  struct_asym_id: 'A', 
  start_residue_number:5, 
  end_residue_number:11, 
  color:{r:133,g:39,b:181}
},{
  struct_asym_id: 'A', 
  start_residue_number:12, 
  end_residue_number:16, 
  color:{r:207,g:62, b:125}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/elF4A-3D.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<p>Due to the lack of complex structure of APT58 aptamer and eIF4A. Through experiments, it is proved that C7 and C9 are related to the binding of eIF4A. Left: The aptamer in the tertiary structure, it's labeled C7, C9 in orange. Right: eIF4A (shown in vacuumm electrostatics), blue is positive charge, red is negative charge.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/eIF4A_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/eIF4A_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>


<p class="header_box" id="ligand-recognition">Ligand information</p>
<p class="blowheader_box">SELEX ligand</p>
<p> The apparent dissociation constant of recombinant eIF4A for some selected RNAs was assessed by a nitrocellulose retention assay in the absence of ATP. The formation of RNA no.21 complexes on the surface-coupled eIF4A was also monitored in the absence of ATP in real time with a BIACORE 2000 instrument based on the surface plasmon resonance (SPR) technique<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">RNA no.1</td>
        <td name="td1">5′-GGGAGACAAGAAUAAACGCUCAACAGGCGUUUAGCCUCUAAGUAACAGGGGGCCUCCCAUGAGCUUCGACAGGAGGCUCACAACAGGC-3′</td>
        <td name="td3">eIF4A</td>
        <td name="td4">3 μM</td>
      </tr>
      <tr>
        <td name="td0">RNA no.11</td>
        <td name="td1">5′-GGGAGACAAGAAUAAACGCUCAAACAUUGCAUCGACAGCUGCAAGGCUCCCGCCGUACAAACCUUCGACAGGAGGCUCACAACAGGC-3′</td>
        <td name="td3">eIF4A</td>
        <td name="td4">8 μM</td>
      </tr>
      <tr>
        <td name="td0">RNA no.20</td>
        <td name="td1">5′-GGGAGACAAGAAUAAACGCUCAAGGGGACCGCGCCCCACAUGUGAGUGAGGCCGAAACGUAGAUUCGACAGGAGGCUCACAACAGGC-3′</td>
        <td name="td3">eIF4A</td>
        <td name="td4">40 nM</td>
      </tr>
      <tr>
        <td name="td0">RNA no.21</td>
        <td name="td1">5′-GGGAGACAAGAAUAAACGCUCAAGGGGACCGCGCCCCACAUGUGAGUGAGGCCGAAACAUAGAUUCGACAGGAGGCUCACAACAGGC-3′</td>
        <td name="td3">eIF4A</td>
        <td name="td4">27 nM</td>
      </tr>
      <tr>
        <td name="td0">RNA no.30</td>
        <td name="td1">5′-GGGAGACAAGAAUAAACGCUCAAUGUGGAUGAUUUGUAUGAUCGCGCAUACAAUUCGACAGGAGGCUCACAACAGGC-3′</td>
        <td name="td3">eIF4A</td>
        <td name="td4">1 μM</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<p>ATP-dependent RNA helicase which is a subunit of the eIF4F complex involved in cap recognition and is required for mRNA binding to ribosome. In the current model of translation initiation, eIF4A unwinds RNA secondary structures in the 5'-UTR of mRNAs which is necessary to allow efficient binding of the small ribosomal subunit, and subsequent scanning for the initiator codon.-----From <a href="https://www.ebi.ac.uk/interpro/protein/UniProt/P10081/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">Mass</th>
        <th onclick="sortTable(3)">Protein sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P10081/entry" target="_blank" style="color:#520049"><b>P10081</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/protein/UniProt/P10081/" target="_blank" style="color:#520049"><b>P10081</b></a></td>
        <td name="td2">44.70kDa</td>
        <td name="td3">
        <div class="sequence-container">
          <span class="sequence-text">GGPYLQ</span>
          <span class="show-more" onclick="toggleSequence(event)">......</span>
          <span class="full-sequence">SEGITDIEESQIQTNYDKVVYKFDDMELDENLLRGVFGYGFEEPSAIQQRAIMPIIEGHDVLAQAQSGTGKTGTFSIAALQRIDTSVKAPQALMLAPTRELALQIQKVVMALAFHMDIKVHACIGGTSFVEDAEGLRDAQIVVGTPGRVFDNIQRRRFRTDKIKMFILDEADEMLSSGFKEQIYQIFTLLPPTTQVVLLSATMPNDVLEVTTKFMRNPVRILVKKDELTLEGIKQFYVNVEEEEYKYECLTDLYDSISVTQAVIFCNTRRKVEELTTKLRNDKFTVSAIYSDLPQQERDTIMKEFRSGSSRILISTDLLARGIDVQQVSLVINYDLPANKENYIHRIGRGGRFGRKGVAINFVTNEDVGAMRELEKFYSTQIEELPSDIATLLN</span>
        </div>
        </td>
        <td name="td4"><a href="https://www.rcsb.org/structure/1FUU" target="_blank" style="color:#520049"><b>1FUU</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?lvl=0&id=559292" target="_blank" style="color:#520049"><b>559292</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/eIF4A_stru_ligand.svg" alt="drawing" style="width:800px;height:300px;"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>





<p class="blowheader_box">Similar compound</p>
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and selected the previous information with high similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB).</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">1FUU-A</td>
        <td name="td2">44.2</td>
        <td name="td3">0</td>
        <td name="td4">Original chain</td>
      </tr>
      <tr>
        <td name="td1">8RC0-D</td>
        <td name="td2">27.1</td>
        <td name="td3">5</td>
        <td name="td4">CD2 antigen cytoplasmic tail-binding protein 2</td>
      </tr>
      <tr>
        <td name="td1">3I5X-A</td>
        <td name="td2">24.8</td>
        <td name="td3">9.6</td>
        <td name="td4">ATP-dependent rna helicase mss116</td>
      </tr>
      <tr>
        <td name="td1">6YVH-K</td>
        <td name="td2">24.1</td>
        <td name="td3">1.5</td>
        <td name="td4">Pre-mRNA-splicing factor cwc22 homolog</td>
      </tr>
      <tr>
        <td name="td1">7NAD-X</td>
        <td name="td2">22.9</td>
        <td name="td3">10.4</td>
        <td name="td4">25s rRNA</td>
      </tr>
      <tr>
        <td name="td1">3I32-A</td>
        <td name="td2">20.1</td>
        <td name="td3">1.9</td>
        <td name="td4">Heat resistant rna dependent atpase</td>
      </tr>
      <tr>
        <td name="td1">8VX9-B</td>
        <td name="td2">19.1</td>
        <td name="td3">6.6</td>
        <td name="td4">Hama</td>
      </tr>
      <tr>
        <td name="td1">6ZNP-A</td>
        <td name="td2">19.1</td>
        <td name="td3">8.9</td>
        <td name="td4">Uncharacterized ATP-dependent helicase ypra</td>
      </tr>
      <tr>
        <td name="td1">5V9X-A</td>
        <td name="td2">18.6</td>
        <td name="td3">9.5</td>
        <td name="td4">ATP-dependent dna helicase</td>
      </tr>
      <tr>
        <td name="td1">5DCA-A</td>
        <td name="td2">18.5</td>
        <td name="td3">7.8</td>
        <td name="td4">Pre-mrna-splicing helicase brr2</td>
      </tr>
	  </tbody>
  </table>

                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] RNA aptamers to initiation factor 4A helicase hinder cap-dependent translation by blocking ATP hydrolysis.</strong></font><br />
Oguro, A., Ohtsu, T., Svitkin, YV., Sonenberg, N., & Nakamura, Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12649492/" target="_blank" style="color:#520049" >RNA, 9(4):394-407. (2003)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] NMR structures of double loops of an RNA aptamer against mammalian initiation factor 4A.</strong></font><br />
Sakamoto, T., Oguro, A., Kawai, G., Ohtsu, T., & Nakamura, Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/15687383/" target="_blank" style="color:#520049">Nucleic Acids Research, 33(2):745-54. (2005)</a>
<br />
</html>


<script>
    function toggleSequence(event) {
      const container = event.target.closest('.sequence-container');
      container.classList.toggle('expanded');
      const showMoreText = container.querySelector('.show-more');
      
      // 展开后按钮文本变化
      if (container.classList.contains('expanded')) {
        showMoreText.textContent = '...';  // 展开后显示 "..."
      } else {
        showMoreText.textContent = '......';  // 收起后显示 "......"
      }
    }

    // 页面加载时，限制序列文本为50个字符
    window.addEventListener('load', function() {
      const sequenceContainers = document.querySelectorAll('.sequence-container');
      sequenceContainers.forEach(container => {
        const fullSeqText = container.querySelector('.full-sequence').textContent;
        const truncatedText = fullSeqText.slice(0, 20);  // 只显示前50个字符
        container.querySelector('.sequence-text').textContent = truncatedText;
      });
    });
  </script>