---
layout: theophylline_style
title: Engineered FKBP protein_s23 aptamer
date: 2025-06-08 00:00:00
author: Yuxuan Sun
categories: Aptamer
tags:
- Category:Mammalian
- GC:51.49
- GC:55.559999999999995
- GC:58.330000000000005
- Length:101
- Length:36
- Length:45
- Named:TFKBP*3R-2G (cell surface protein)_T1 s23
- Named:TFKBP*3R-2G (cell surface protein)_T2 s23
- Named:TFKBP*3R-2G (cell surface protein)_s23
- Type:Proteins
- Year:2005
type: Proteins
---


<html>
<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
</ul>
</div>

<p class="header_box" id="description">Description</p>
<p>In 2005, Verdine, G. L. et al. used SELEX to identify RNA aptamers that specifically bind to the tFKBP3R-2G complex. After seven rounds of selection, they identified two key aptamers, s23 and T1, which showed high affinity with Kd values below 50 nM. Aptamer s23 exhibited exceptional specificity for both the guanine base and the tether length in the small molecule. It also demonstrated strong binding to tFKBP3R but not to the wild-type tFKBP*, confirming its specificity for the engineered protein-small molecule interface. This study highlighted the potential of SELEX for developing RNA aptamers targeting specific composite surfaces.<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2005, Verdine, G. L. et al. designed an in vitro RNA selection process to identify aptamers that specifically bind to the TFKBP3R-2G complex. Starting with a DNA template containing a 60-nucleotide random region, they performed seven rounds of SELEX. Among the selected aptamers, s23 and T1 showed the highest affinity, binding to TFKBP3R-2G with Kd values below 50 nM. The RNA library was denatured at 94°C for 2 minutes, then cooled to room temperature in phosphate-buffered saline (PBS). The RNA pool was mixed with TFKBP*3R-2G and incubated in PBS. After incubation, the mixture was passed through a 0.45 μm nitrocellulose filter (Millipore) and washed with PBS. The bound RNAs were eluted with 7 M urea at 90°C for 5 minutes, ethanol precipitated, and reverse transcribed using AMV reverse transcriptase at 42°C for 1 hour.<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>


<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>The 2D structure of the figure is based on the article by ribodraw tool to draw. The s23 aptamer was named by Verdine, G. L. et al. in the article.<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>5'-GGGUUCGAAUGCAAGUCUGGUGAGUGUUGGGAGGGCAGUGACUUGCUUCGUAGUGUAAACUGACGUGGCGGUGUUACUUUACUGUUACGCUAGGUUACGGG-3'</p>
<img src="/images/2D/s23_aptamer_2D.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="header_box" id="ligand-recognition">Ligand information</p>
<p class="blowheader_box">SELEX ligand</p>
<p>The FKBPs, or FK506 binding proteins, constitute a family of proteins that have prolyl isomerase activity and are related to the cyclophilins in function, though not in amino acid sequence. FKBPs have been identified in many eukaryotes, ranging from yeast to humans, and function as protein folding chaperones for proteins containing proline residues. Along with cyclophilin, FKBPs belong to the immunophilin family. ------From Wiki</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">TFKBP*3R-2G</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P20081/entry" target="_blank" style="color:#520049"><b>P20081</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00254/" target="_blank" style="color:#520049"><b>PF00254</b></a></td>
        <td name="td3">12.158 kDa</td>
        <td name="td4">MSEVIEGNVKIDRISPGDGATFPKTGDLVTIHYTGTLENGQKFDSSVDRGSPFQCNIGVGQVIKGWDVGIPKLSVGEKARLTIPGPYAYGPRGFPGLIPPNSTLVFDVELLKVN</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/1YAT" target="_blank" style="color:#520049"><b>1YAT</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/855587" target="_blank" style="color:#520049"><b>855587</b></a></td>
      </tr>
	  </tbody>
  </table>

  <p>Aptamer bind to the affinity of the protein.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">s23 aptamer</td>
        <td name="td1">5'-GGGUUCGAAUGCAAGUCUGGUGAGUGUUGGGAGGGCAGUGACUUGCUUCGUAGUGUAAACUGACGUGGCGGUGUUACUUUACUGUUACGCUAGGUUACGGG-3'</td>
        <td name="td2">TFKBP*3R-2G</td>
        <td name="td3">4.3 ± 0.5 nM</td>
      </tr>
      <tr>
        <td name="td0">T1 s23 aptamer</td>
        <td name="td1">5'-CGAAUGCAAGUCUGGUGAGUGUUGGGAGGGCAGUGACUUGCUUCG-3'</td>
        <td name="td2">TFKBP*3R-2G</td>
        <td name="td3">2.8 ± 0.2  nM</td>
      </tr>
      <tr>
        <td name="td0">T2 s23 aptamer</td>
        <td name="td1">5'-GCAAGUCUGGUGAGUGUUGGGAGGGCAGUGACUUGC-3'</td>
        <td name="td2">TFKBP*3R-2G</td>
        <td name="td3">16.1 ± 3.6 nM</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/s23_aptamer_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>

<p class="blowheader_box">Similar compound</p><p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>

<p class="dot-paragraph"><b><a href="http://ekhidna2.biocenter.helsinki.fi/dali/" target="_blank" style="color:#520049; text-decoration: underline;">Dail server website</a>:</b> a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>


<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
  <thead>
    <tr>
      <th onclick="sortTable(0)">PDB</th>
      <th onclick="sortTable(1)">Z-score</th>
      <th onclick="sortTable(2)">RMSD</th>
      <th onclick="sortTable(3)">Description</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td name="td0">1 1YAT-A</td>
      <td name="td1">26.5</td>
      <td name="td2">0.0</td>
      <td name="td3">Fk506-Binding Protein</td>
    </tr>
    <tr>
      <td name="td0">2 5KLX-D</td>
      <td name="td1">20.8</td>
      <td name="td2">1.1</td>
      <td name="td3">Chimera Protein Of Ubiquitin-Like Protein Smt3</td>
    </tr>
    <tr>
      <td name="td0">3 5NJX-A</td>
      <td name="td1">20.4</td>
      <td name="td2">1.1</td>
      <td name="td3">Peptidyl-Prolyl Cis-Trans Isomerase Fkbp5</td>
    </tr>
    <tr>
      <td name="td0">4 3JYM-B</td>
      <td name="td1">19.1</td>
      <td name="td2">1.5</td>
      <td name="td3">Fk506-Binding Protein (Fkbp) From Wheat</td>
    </tr>
    <tr>
      <td name="td0">5 5MGX-G</td>
      <td name="td1">18.8</td>
      <td name="td2">1.3</td>
      <td name="td3">Yeast Hsp90 C-Terminus</td>
    </tr>
    <tr>
      <td name="td0">6 5I7P-A</td>
      <td name="td1">18.7</td>
      <td name="td2">1.5</td>
      <td name="td3">Peptidyl-Prolyl Cis-Trans Isomerase Fkbp1A, Fkbp-T</td>
    </tr>
    <tr>
      <td name="td0">7 4MSP-A</td>
      <td name="td1">18.0</td>
      <td name="td2">1.2</td>
      <td name="td3">Peptidyl-Prolyl Cis-Trans Isomerase Fkbp14</td>
    </tr>
    <tr>
      <td name="td0">8 1Q6U-A</td>
      <td name="td1">17.8</td>
      <td name="td2">1.2</td>
      <td name="td3">Fkbp-Type Peptidyl-Prolyl Cis-Trans Isomerase Fkp</td>
    </tr>
    <tr>
      <td name="td0">9 2MPH-A</td>
      <td name="td1">17.3</td>
      <td name="td2">1.6</td>
      <td name="td3">Peptidyl-Prolyl Cis-Trans Isomerase Fkbp3</td>
    </tr>
    <tr>
      <td name="td0">10 7EU3-4</td>
      <td name="td1">14.5</td>
      <td name="td2">1.6</td>
      <td name="td3">Nad(P)H-Quinone Oxidoreductase Subunit 1, Chlorop</td>
    </tr>
  </tbody>
</table>

<p class="header_box" id="references">References</p>

<a id="ref1"></a>
<font><strong>[1] In vitro selection of RNA aptamers against a composite small molecule-protein surface.</strong></font><br />
Plummer, K. A., Carothers, J. M., Yoshimura, M., Szostak, J. W., & Verdine, G. L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/16199752/" target="_blank">Nucleic Acids Res. 33(17):5602-10. (2005)</a>
<br />
    