---
layout: theophylline_style
title: GlnRs aptamer
date: 2024-04-16 00:00:00
author: Jiali Wang, Bo Fu
categories: Aptamer
tags:
- Category:Mammalian
- GC:59.0
- Length:74
- Named:Glutaminyl-tRNA synthetase_var-AGGU RNA aptamer
- Type:Proteins
- Year:1996
type: Proteins
---
<html>

<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/2459391/" target="_blank" style="color:#520049">1988</a></h3>
    </div>
    <div class="body">
      <p>Natural RNA sequences that bind to GlnRs<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/2479982/" target="_blank" style="color:#520049">1989</a></h3>
    </div>
    <div class="body">
      <p>Structure of E. coli glutaminyl-tRNA synthetase complexed with tRNA(Gln) and ATP at 2.8 A resolution<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://www.proquest.com/openview/6857e35043f04e11f53714d80cdc4dcf/1?pq-origsite=gscholar&cbl=18750&diss=y" target="_blank" style="color:#520049">1996</a></h3>
    </div>
    <div class="body">
      <p>The aptamer is initially selected through the SELEX<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10881199/" target="_blank" style="color:#520049">2000</a></h3>
    </div>
    <div class="body">
      <p>A tRNA mutant with a 30 fold increased affinity for binding to GlnRS compared to the wild-type<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/14681579/" target="_blank" style="color:#520049">2004</a></h3>
    </div>
    <div class="body">
      <p>Aptamer redesigned tRNA is nonfunctional and degraded in cells<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
</div>


         
<p class="header_box" id="description">Description</p>
<p>In 1996, Stubenrauch, M used standard in vitro selection techniques to randomly select tRNAGln molecular pools, focusing specifically on the D, T, and variable loops. This selection process was based on the ability of these tRNA molecules to bind to GlnRS. In 2000, Perona, J. J. et al. analysed the complex structure of a high-affinity tRNA aptamer bound to GlnRS using X-ray crystallography<sup>[<a href="#ref3" style="color:#520049">3</a></sup><sup>,<a href="#ref4" style="color:#520049">4</a></sup><sup>]</sup>.<br></p>



<p class="header_box" id="SELEX">SELEX</p>
<p>In 2000, Perona, J. J. et al. used Stubenrauch's tRNAGln library, which featured randomised nucleotides at positions 15–21, 47–48, and 53–61. RNAs capable of binding GlnRS were isolated through nitrocellulose filtration. After six rounds of screening, 46 sequences with a four-nucleotide replacement of the wild-type five-nucleotide variable loop (5'-CAUUC-3') were identified, 33 of which had the sequence 5'-AGGU-3'<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup>.<p>
<br>    



<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>The var-AGGU RNA aptamer, which is an enhanced variant of the initial tRNA aptamer, was selected in vitro. This selection was carried out within an in vitro selection system to enhance its interactions with GlnRS. It has been demonstrated that this optimized RNA aptamer can impede the binding of GlnRS to tRNA in vitro. Here we used RiboDraw to complete the figure, through the 3D structure information. The var-AGGU RNA aptamer was named by Perona, J. J. et al. in the article<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup>.</p>
<p>5'-UGGGGUAUCGCCAAGCGGUAAGGCACCGGAUUCUGAUUCCGGAGGUCGAGGUUCGAAUCCUCGUACCCCAGCCA-3'</p>
<img src="/images/2D/GlnRs_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;">   
</div>
<br>


<p class="blowheader_box">3D visualisation</p>             
<p>Perona, J. J. et al. have determined the crystal structure of an RNA aptamer bound to glutaminyl-tRNA synthetase at 2.7 Å resolution. The structure, however, fails to clearly reveal the relationship between the interacting bases and amino acids. The article does not provide a detailed explanation of the hydrogen bonds between the aptamer and the modified protein. The hydrogen bonds shown in our figure are predicted using PyMOL. The PDB ID of this structure is 1EXD<sup>[<a href="#ref4" style="color:#520049">4</a></sup><sup>]</sup>.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/GlnRs_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A',    
  start_residue_number:8, 
  end_residue_number:442, 
  color:{r:133,g:39,b:181}
},
{
  struct_asym_id: 'A', 
  start_residue_number:454, 
  end_residue_number:547, 
  color:{r:133, g:39,b:181}
},
{
struct_asym_id: 'B', 
  start_residue_number:902, 
  end_residue_number:907, 
  color:{r:184,g:51,b:50}
},
{
struct_asym_id: 'B', 
  start_residue_number:908, 
  end_residue_number:909, 
  color:{r:8,g:69,b:149}
},
{
struct_asym_id: 'B', 
  start_residue_number:910, 
  end_residue_number:916, 
  color:{r:241,g:125,b:7}
},
{
struct_asym_id: 'B', 
  start_residue_number:918, 
  end_residue_number:925, 
  color:{r:8,g:69,b:149}
},
{
struct_asym_id: 'B', 
  start_residue_number:926, 
  end_residue_number:943, 
  color:{r:207,g:62,b:125}
},
{
struct_asym_id: 'B', 
  start_residue_number:944, 
  end_residue_number:948, 
  color:{r:194,g:182,b:23}
},
{
struct_asym_id: 'B', 
  start_residue_number:949, 
  end_residue_number:965, 
  color:{r:24,g:152,b:140}
},
{
struct_asym_id: 'B', 
  start_residue_number:966, 
  end_residue_number:976, 
  color:{r:184,g:51,b:50}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1EXD-3D1.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<p>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 1EXD at 2.7 Å resolution. GlnRs (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with GlnRs.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/GlnRs_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/GlnRs_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>



<p class="header_box" id="ligand-recognition">Ligand information</p>
<p class="blowheader_box">SELEX ligand</p>
<p>Perona, J. J et al. determined the binding constant of the aptamer using a gel shift binding assay. Equal volumes of tRNA and GlnRS solutions were mixed and incubated for 15 minutes at room temperature. The mixtures were then loaded onto a 20% polyacrylamide gel at 4°C and run at 200 V for 5 hours. The gels were developed for 72 hours. Autoradiography and densitometry of the gels were performed using the Storm 840 phosphorimager (Molecular Dynamics). For AGGU and T1 tRNAs, the intensity of the shifted band was quantified to determine the relative amount of complex formed at each enzyme concentration. Equilibrium binding constants were determined by fitting the data to a standard hyperbolic curve using Kaleidagraph (Synergy Software)<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup>.</p >
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Wild type tRNA<sup>Gln</sup></td>
        <td name="td1">5'-UGGGGUAUCGCCAAGCGGUAAGGCACCGGAUUCUGAUUCCGGCAUUCCGAGGUUCGAAUCCUCGUACCCCAGCCA-3'</td>
        <td name="td3">GlnRs</td>
        <td name="td4">7.1 nM</td>
      </tr>
      <tr>
        <td name="td0">44-AGGU-48 (var-AGGU)</td>
        <td name="td1">5'-UGGGGUAUCGCCAAGCGGUAAGGCACCGGAUUCUGAUUCCGGAGGUCGAGGUUCGAAUCCUCGUACCCCAGCCA-3'</td>
        <td name="td3">GlnRs</td>
        <td name="td4">0.27 nM</td>
      </tr>
      <tr>
        <td name="td0">T1</td>
        <td name="td1">5'-UGGGGUAUCGCCAAAAGAAAGGCACCGGAUUCUGAUUCCGGAGGUCGAGAUUGUCGUCCUCGUACCCCAGCCA-3'</td>
        <td name="td3">GlnRs</td>
        <td name="td4">0.13 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<p>Glutaminyl-tRNA synthetase (GlnRS) cataytic core domain. These enzymes attach Gln to the appropriate tRNA. Like other class I tRNA synthetases, they aminoacylate the 2'-OH of the nucleotide at the 3' end of the tRNA. The core domain is based on the Rossman fold and is responsible for the ATP-dependent formation of the enzyme bound aminoacyl-adenylate. GlnRS contains the characteristic class I HIGH and KMSKS motifs, which are involved in ATP binding. These enzymes function as monomers. Archaea and most bacteria lack GlnRS. In these organisms, the "non-discriminating" form of GluRS aminoacylates both tRNA(Glu) and tRNA(Gln) with Glu, which is converted to Gln when appropriate by a transamidation enzyme.-----From <a href="https://www.ebi.ac.uk/interpro/entry/cdd/CD00807/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">Mass</th>
        <th onclick="sortTable(3)">Protein sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P00962/publications" target="_blank" style="color:#520049"><b>P00962</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/cdd/CD00807/" target="_blank" style="color:#520049"><b>CD00807</b></a></td>
        <td name="td2">63.48 KDa</td>
        <td name="td3">
        <div class="sequence-container">

          <span class="full-sequence">
        MSEAEARPTNFIRQIIDEDLASGKHTTVHTRFPPEPNGYLHIGHAKSICLNFGIAQDYKGQCNLRFDDTNPVKEDIEYVESIKNDVEWLGFHWSGNVRYSSDYFDQLHAYAIELINKGLAYVDELTPEQIREYRGTLTQPGKNSPYRDRSVEENLALFEKMRAGGFEEGKACLRAKIDMASPFIVMRDPVLYRIKFAEHHQTGNKWCIYPMYDFTHCISDALEGITHSLCTLEFQDNRRLYDWVLDNITIPVHPRQYEFSRLNLEYTVMSKRKLNLLVTDKHVEGWDDPRMPTISGLRRRGYTAASIREFCKRIGVTKQDNTIEMASLESCIREDLNENAPRAMAVIDPVKLVIENYQGEGEMVTMPNHPNKPEMGSRQVPFSGEIWIDRADFREEANKQYKRLVLGKEVRLRNAYVIKAERVEKDAEGNITTIFCTYDADTLSKDPADGRKVKGVIHWVSAAHALPVEIRLYDRLFSVPNPGAADDFLSVINPESLVIKQGFAEPSLKDAVAGKAFQFEREGYFCLDSRHSTAEKPVFNRTVGLRDT</span>
        </div>
      </td>
        <td name="td4"><a href="https://www.rcsb.org/structure/1EXD" target="_blank" style="color:#520049"><b>1EXD</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/nuccore/V01575" target="_blank" style="color:#520049"><b>V01575</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/GlnRs_stru_ligand.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:1000px;height:400px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
     <tr>
        <td name="td1">1EXD-A</td>
        <td name="td2">54.8</td>
        <td name="td3">0</td>
        <td name="td4">GLUTAMINE TRNA APTAMER</td>
      </tr>
      <tr>
        <td name="td1">1EUY-A</td>
        <td name="td2">51.4</td>
        <td name="td3">0.4</td>
        <td name="td4">GLUTAMINYL TRNA</td>
      </tr>
      <tr>
        <td name="td1">1O0B-A</td>
        <td name="td2">51.3</td>
        <td name="td3">0.5</td>
        <td name="td4">GLUTAMINYL TRNA</td>
      </tr>
      <tr>
        <td name="td1">1QTQ-A</td>
        <td name="td2">51.2</td>
        <td name="td3">0.4</td>
        <td name="td4">RNA (TRNA GLN II)</td>
      </tr>
      <tr>
        <td name="td1">1ZJW-A</td>
        <td name="td2">51.1</td>
        <td name="td3">0.4</td>
        <td name="td4">GLUTAMINYL-TRNA</td>
      </tr>
      <tr>
        <td name="td1">1EUQ-A</td>
        <td name="td2">50.7</td>
        <td name="td3">0.4</td>
        <td name="td4">GLUTAMINYL TRNA</td>
      </tr>
      <tr>
        <td name="td1">2RE8-A</td>
        <td name="td2">50.6</td>
        <td name="td3">0.6</td>
        <td name="td4">GLUTAMINYL-TRNA SYNTHETASE</td>
      </tr>
      <tr>
        <td name="td1">1GTR-A</td>
        <td name="td2">50.5</td>
        <td name="td3">0.5</td>
        <td name="td4">RNA (74-MER)</td>
      </tr>
      <tr>
        <td name="td1">2RD2-A</td>
        <td name="td2">50.4</td>
        <td name="td3">0.5</td>
        <td name="td4">GLUTAMINYL-TRNA SYNTHETASE</td>
      </tr>
      <tr>
        <td name="td1">1QRU-A</td>
        <td name="td2">50.3</td>
        <td name="td3">0.5</td>
        <td name="td4">TRNAGLN2</td>
      </tr>
	  </tbody>
  </table>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Overproduction and purification of Escherichia coli tRNA(2Gln) and its use in crystallization of the glutaminyl-tRNA synthetase-tRNA(Gln) complex.</strong></font><br />
Perona, J. J., Swanson, R., Steitz, T. A., & Söll, D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/2459391/" target="_blank" style="color:#520049" >Journal of molecular biology, 202(1), 121–126. (1988)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Structure of E. coli glutaminyl-tRNA synthetase complexed with tRNA(Gln) and ATP at 2.8 A resolution.</strong></font><br />
Rould, M. A., Perona, J. J., Söll, D., & Steitz, T. A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/2479982/" target="_blank" style="color:#520049">Science (New York, N.Y.), 246(4934), 1135–1142. (1989)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] In vitro selection and characterization of tRNA-like substrates of E. coli glutaminyl-tRNA synthetase.</strong></font><br />
Stubenrauch, M.<br />
<a href="https://www.proquest.com/openview/6857e35043f04e11f53714d80cdc4dcf/1?pq-origsite=gscholar&cbl=18750&diss=y" target="_blank" style="color:#520049"> Doctoral thesis, Department of Chemistry and Biochemistry, University of Colorado, Boulder, Colorado (1996)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] Tertiary core rearrangements in a tight binding transfer RNA aptamer.</strong></font><br />
Bullock, T. L., Sherlin, L. D., & Perona, J. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10881199/" target="_blank" style="color:#520049">Nature structural biology, 7(6), 497–504. (2000)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Influence of transfer RNA tertiary structure on aminoacylation efficiency by glutaminyl and cysteinyl-tRNA synthetases.</strong></font><br />
Sherlin, L. D., Bullock, T. L., Newberry, K. J., Lipman, R. S., Hou, Y. M., Beijer, B., Sproat, B. S., & Perona, J. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10860750/" target="_blank" style="color:#520049">Journal of molecular biology, 299(2), 431–446. (2000)</a>
<br />

<a id="ref6"></a><font><strong>[6] Aptamer redesigned tRNA is nonfunctional and degraded in cells.</strong></font><br />
Lee, D., & McClain, W. H.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/14681579/" target="_blank" style="color:#520049">RNA (New York, N.Y.), 10(1), 7–11. (2004)</a>
<br />       



<script>
    function toggleSequence(event) {
      const container = event.target.closest('.sequence-container');
      container.classList.toggle('expanded');
      const showMoreText = container.querySelector('.show-more');
      
      // 展开后按钮文本变化
      if (container.classList.contains('expanded')) {
        showMoreText.textContent = '...';  // 展开后显示 "..."
      } else {
        showMoreText.textContent = '......';  // 收起后显示 "......"
      }
    }

    // 页面加载时，限制序列文本为50个字符
    window.addEventListener('load', function() {
      const sequenceContainers = document.querySelectorAll('.sequence-container');
      sequenceContainers.forEach(container => {
        const fullSeqText = container.querySelector('.full-sequence').textContent;
        const truncatedText = fullSeqText.slice(0, 20);  // 只显示前50个字符
        container.querySelector('.sequence-text').textContent = truncatedText;
      });
    });
  </script>