---
layout: theophylline_style
title: GluR2 AMPA receptors_AN58 aptamer
date: 2024-05-22 00:00:00
author: Jiaxin Zhao, Bichun Wu
categories: Aptamer
tags:
- Category:Mammalian
- GC:50.0
- Length:58
- Named:GluR2 AMPA receptors_AN58
- Type:Proteins
- Year:2007
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/6320006/" target="_blank" style="color:#520049">1984</a></h3>
  </div>
  <div class="body">
    <p>The opening of the AMPA channel is thought to provide the initial membrane depolarization, thus enabling the NMDA channel to function by relieving the magnesium block<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/11848921/" target="_blank" style="color:#520049">1998</a></h3>
  </div>
  <div class="body">
    <p>AN58 is stable in that it remains fully active even after ethanol precipitation, heating for >10 min at 70 °C, and/or storage in the frozen state for more than a year<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/14567697/" target="_blank" style="color:#520049">2003</a></h3>
  </div>
  <div class="body">
    <p>At the receptor level, an AMPA receptor opens its channel in response to the binding of glutamate on the microsecond (μs) time scale, much faster than NMDA receptor channels<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/16540562/" target="_blank" style="color:#520049">2006</a></h3>
  </div>
  <div class="body">
    <p>AN58 is confirmed as a highly potent inhibitor<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/17929944/" target="_blank" style="color:#520049">2007</a></h3>
  </div>
  <div class="body">
    <p>AN58 demonstrated higher selectivity for the GluR4 AMPA receptor subunit compared to NBQX<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/28872832/" target="_blank" style="color:#520049">2017</a></h3>
  </div>
  <div class="body">
    <p>The short aptamer (69 nucleotides) FN1040s selectively inhibits the GluA1 and GluA2Qflip AMPA receptor subunits, whereas the full-length aptamer (101 nucleotides) FN1040 additionally inhibits GluK1, but not GluK2, kainate receptor, and GluN1a/2A and GluN1a/2B, the two major native NMDA receptors<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
  </div>
 </div>
</div>



<p class="header_box" id="description">Description</p>
<p>In 2007, Zhen Huang et al. used the SELEX method to isolate the aptamer with high compatibility for the Recombinant GluR2Qflip AMPA receptor transiently expressed in HEK-293 (human embryonic kidney) cells. The excessive activation of AMPA (α-amino-3-hydroxy-5-methyl-4-isoxazole propionic acid) receptors, a subtype of glutamate ion channels, has been implicated in various neurological diseases such as cerebral ischemeia and amyotrophic lateral sclerosis. Inhibitors of AMPA receptors are drug candidates for potential treatment of these diseases. One of the aptamers, AN58, is shown to competitively inhibit the receptor. The nanomolar affinity of AN58 rivals that of NBQX (6-nitro-7-sulfamoyl-benzo[f]quinoxaline-2,3-dione), one of the best competitive inhibitors<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2007, using the systematic evolution of ligands by exponential enrichment (SELEX), Zhen Huang et al. have selected a group of RNA aptamers against the recombinant GluR2Qflip AMPA receptor transiently expressed in HEK-293 (human embryonic kidney) cells<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>The 2D structure of the figure is based on the article by ribodraw tool to draw.<br></p>
<p>5'-GGGCGAAUUCAACUGCCAUCUAGGCAGUAACCAGGAGUUAGUAGGACAAGUUUCGUCC-3'</p>
<img src="/images/2D/AN58_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>



<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>Ionotropic glutamate receptors (iGluRs) are a highly conserved family of ligand-gated ion channels present in animals, plants, and bacteria, which are best characterised for their roles in synaptic communication in vertebrate nervous systems. A variant subfamily of iGluRs, the Ionotropic Receptors (IRs), consist of non-glutamate-binding chemosensory receptors first identified in Drosophila melanogaster. They function in detecting odors and tastants.-----From Pfam</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">GluR2 AMPA receptors</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P42262/entry" target="_blank" style="color:#520049"><b>P42262</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR015683/" target="_blank" style="color:#520049"><b>IPR015683</b></a></td>
        <td name="td3">98.83 kDa</td>
        <td name="td4">MQKIMHISVLLSPVLWGLIFGVSSNSIQIGGLFPRGADQEYSAFRVGMVQFSTSEFRLTPHIDNLEVANSFAVTNAFCSQFSRGVYAIFGFYDKKSVNTITSFCGTLHVSFITPSFPTDGTHPFVIQMRPDLKGALLSLIEYYQWDKFAYLYDSDRGLSTLQAVLDSAAEKKWQVTAINVGNINNDKKDEMYRSLFQDLELKKERRVILDCERDKVNDIVDQVITIGKHVKGYHYIIANLGFTDGDLLKIQFGGANVSGFQIVDYDDSLVSKFIERWSTLEEKEYPGAHTTTIKYTSALTYDAVQVMTEAFRNLRKQRIEISRRGNAGDCLANPAVPWGQGVEIERALKQVQVEGLSGNIKFDQNGKRINYTINIMELKTNGPRKIGYWSEVDKMVVTLTELPSGNDTSGLENKTVVVTTILESPYVMMKKNHEMLEGNERYEGYCVDLAAEIAKHCGFKYKLTIVGDGKYGARDADTKIWNGMVGELVYGKADIAIAPLTITLVREEVIDFSKPFMSLGISIMIKKPQKSKPGVFSFLDPLAYEIWMCIVFAYIGVSVVLFLVSRFSPYEWHTEEFEDGRETQSSESTNEFGIFNSLWFSLGAFMQQGCDISPRSLSGRIVGGVWWFFTLIIISSYTANLAAFLTVERMVSPIESAEDLSKQTEIAYGTLDSGSTKEFFRRSKIAVFDKMWTYMRSAEPSVFVRTTAEGVARVRKSKGKYAYLLESTMNEYIEQRKPCDTMKVGGNLDSKGYGIATPKGSSLRNAVNLAVLKLNEQGLLDKLKNKWWYDKGECGSGGGDSKEKTSALSLSNVAGVFYILVGGLGLAMLVALIEFCYKSRAEAKRMKVAKNAQNINPSSSQNSQNFATYKEGYNVYGIESVKI</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/2WJX" target="_blank" style="color:#520049"><b>2WJX</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/2891" target="_blank" style="color:#520049"><b>2891</b></a></td>
      </tr>
	  </tbody>
  </table>

  <p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">AN58</td>
        <td name="td1">5'-GGGCGAAUUCAACUGCCAUCUAGGCAGUAACCAGGAGUUAGUAGGACAAGUUUCGUCC-3'</td>
        <td name="td2">GluR2 AMPA receptors</td>
        <td name="td3">Kd: 0.419 ± 0.221 nM & IC50: 121 ± 7 nM</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/AN58_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">3H5V-A</td>
      <td name="td1">54.8</td>
      <td name="td2">1.0</td>
      <td name="td3">Glutamate receptor 2</td>
    </tr>
            
     <tr>
      <td name="td0">5FWY-A</td>
      <td name="td1">54.3</td>
      <td name="td2">1.3</td>
      <td name="td3">Glutamate receptor 2</td>
    </tr>
            
     <tr>
      <td name="td0">3N6V-A</td>
      <td name="td1">54.2</td>
      <td name="td2">1.1</td>
      <td name="td3">Glutamate receptor 2</td>
    </tr>
            
     <tr>
      <td name="td0">6QKZ-D</td>
      <td name="td1">54.2</td>
      <td name="td2">1.3</td>
      <td name="td3">Glua1</td>
    </tr>
            
     <tr>
      <td name="td0">3H5V-B</td>
      <td name="td1">54.2</td>
      <td name="td2">1.3</td>
      <td name="td3">Glutamate receptor 2</td>
    </tr>
            
     <tr>
      <td name="td0">3N6V-C</td>
      <td name="td1">54.2</td>
      <td name="td2">1.1</td>
      <td name="td3">Glutamate receptor 2</td>
    </tr>
            
     <tr>
      <td name="td0">3H5W-A</td>
      <td name="td1">54.0</td>
      <td name="td2">1.1</td>
      <td name="td3">Glutamate receptor 2</td>
    </tr>
            
     <tr>
      <td name="td0">3N6V-B</td>
      <td name="td1">53.9</td>
      <td name="td2">1.1</td>
      <td name="td3">Glutamate receptor 2</td>
    </tr>
            
     <tr>
      <td name="td0">4UQK-A</td>
      <td name="td1">53.9</td>
      <td name="td2">1.1</td>
      <td name="td3">Glutamate receptor 2</td>
    </tr>
            
     <tr>
      <td name="td0">4UQJ-C</td>
      <td name="td1">53.9</td>
      <td name="td2">1.1</td>
      <td name="td3">Glutamate receptor 2</td>
    </tr>
    </tbody>
  </table>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Magnesium gates glutamate-activated channels in mouse central neurones.</strong></font><br />
Nowak, L., Bregestovski, P., Ascher, P., Herbet, A., & Prochiantz, A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/6320006/" target="_blank" style="color:#520049">Nature, 307(5950), 462–465. (1984)</a>
<br/>
            
<a id="ref2"></a><font><strong>[2] Kinetics and Mechanisms for the Cleavage and Isomerization of the Phosphodiester Bonds of RNA by Brønsted Acids and Bases.</strong></font><br />
Oivanen, M., Kuusela, S., & Lönnberg, H.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/11848921/" target="_blank" style="color:#520049">Chemical reviews, 98(3), 961–990. (1998)</a>
<br/>
            
<a id="ref3"></a><font><strong>[3] Channel-opening kinetics of GluR2Q(flip) AMPA receptor: a laser-pulse photolysis study.</strong></font><br />
Li, G., Pei, W., & Niu, L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/14567697/" target="_blank" style="color:#520049">Biochemistry, 42(42), 12358–12366. (2003)</a>
<br/>
            
<a id="ref4"></a><font><strong>[4] Crystal structures of the kainate receptor GluR5 ligand binding core dimer with novel GluR5-selective antagonists.</strong></font><br />
Mayer, M. L., Ghosal, A., Dolman, N. P., & Jane, D. E. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/16540562/" target="_blank" style="color:#520049">The Journal of neuroscience : the official journal of the Society for Neuroscience, 26(11), 2852–2861. (2006)</a>
<br/>
            
<a id="ref5"></a><font><strong>[5] RNA aptamers selected against the GluR2 glutamate receptor channel.</strong></font><br />
Huang, Z., Pei, W., Jayaseelan, S., Shi, H., & Niu, L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/17929944/" target="_blank" style="color:#520049">Biochemistry, 46(44), 12648–12655. (2007)</a>
<br/>
            
<a id="ref6"></a><font><strong>[6] Chemically Modified, α-Amino-3-hydroxy-5-methyl-4-isoxazole (AMPA) Receptor RNA Aptamers Designed for in Vivo Use.</strong></font><br />
Huang, Z., Wen, W., Wu, A., & Niu, L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/28872832/" target="_blank" style="color:#520049">ACS chemical neuroscience, 8(11), 2437–2445. (2017)</a>
<br/>



<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>




    </html>
