---
layout: theophylline_style
title: T4 gp43 aptamer
date: 2024-05-10 00:00:00
author: Zhizhong Lu, Ying Ao
categories: Aptamer
tags:
- Category:Other
- GC:31.0
- GC:33.629999999999995
- GC:35.0
- GC:35.4
- Length:113
- Length:51
- Named:T4 DNA polymerase (gp43)_Major variant aptamer(truncation)
- Named:T4 DNA polymerase (gp43)_Wild type aptamer(truncation)
- Named:T4 DNA polymerase (gp43)_major variant
- Named:T4 DNA polymerase (gp43)_wild type
- Type:Proteins
- Year:1990
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="description">Description</p>
<p>In 1990, Tuerk C and Gold L. have developed a novel method for rapidly selecting preferred binding sequences from a population of random sequences. And call this method systematic evolution of ligands by exponential enrichment (SELEX). Then, RNA aptamers that bind T4 DNA polymerase were isolated by this method. Two different sequences were selected by this procedure from the calculated pool of 65,536 species. One is the wild-type sequence found in the bacteriophage mRNA; one is varied from wild type at four positions. The binding constants of these two RNA's to T4 DNA polymerase are equivalent. They are the pioneers of SELEX technology and began to use this technology to isolate aptamers early<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In vitro transcripts of variable loop sequence were mixed with purified gp43 at three different ratios of RNA to protein throughout the multiple rounds of selection on nitrocellulose filters. After the fourth round of selection and amplification, the labeled products exhibited binding to gp43 which was indistinguishable from that of the wild-type RNA. The unused RNA products from each round for one experiment and from the fourth round (of selection, amplification, and transcription) for all three experiments were gel-purified and sequenced<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>The 2D structure of the figures is based on the article by ribodraw tool to draw.<br></p>
<p>Wild type aptamer: </p>
<p>5'-UAAUAUAUCAAGAGCCUAAUAACUCGGGCUAUAAACUAAGGAAUAUCUAUG-3'<br></p>
<p>Major variant aptamer: </p>
<p>5'-UAAUAUAUCAAGAGCCUAGCAACCUGGGCUAUAAACUAAGGAAUAUCUAUG-3'</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/T4_aptamer_2D1.svg" alt="drawing" style="width:500px;height:400px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/T4_aptamer_2D2.svg" alt="drawing" style="width:500px;height:400px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>



<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>A DNA polymerase is a member of a family of enzymes that catalyze the synthesis of DNA molecules from nucleoside triphosphates, the molecular precursors of DNA. These enzymes are essential for DNA replication and usually work in groups to create two identical DNA duplexes from a single original DNA duplex. During this process, DNA polymerase "reads" the existing DNA strands to create two new strands that match the existing ones. These enzymes catalyze the chemical reaction.-----from Wiki</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">T4 DNA polymerase</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P04415/" target="_blank" style="color:#520049"><b>P04415</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR006172/" target="_blank" style="color:#520049"><b>IPR006172</b></a></td>
        <td name="td3">103.61 KDa</td>
        <td name="td4">MKEFYISIETVGNNIVERYIDENGKERTREVEYLPTMFRHCKEESKYKDIYGKNCAPQKFPSMKDARDWMKRMEDIGLEALGMNDFKLAYISDTYGSEIVYDRKFVRVANCDIEVTGDKFPDPMKAEYEIDAITHYDSIDDRFYVFDLLNSMYGSVSKWDAKLAAKLDCEGGDEVPQEILDRVIYMPFDNERDMLMEYINLWEQKRPAIFTGWNIEGFDVPYIMNRVKMILGERSMKRFSPIGRVKSKLIQNMYGSKEIYSIDGVSILDYLDLYKKFAFTNLPSFSLESVAQHETKKGKLPYDGPINKLRETNHQRYISYNIIDVESVQAIDKIRGFIDLVLSMSYYAKMPFSGVMSPIKTWDAIIFNSLKGEHKVIPQQGSHVKQSFPGAFVFEPKPIARRYIMSFDLTSLYPSIIRQVNISPETIRGQFKVHPIHEYIAGTAPKPSDEYSCSPNGWMYDKHQEGIIPKEIAKVFFQRKDWKKKMFAEEMNAEAIKKIIMKGAGSCSTKPEVERYVKFSDDFLNELSNYTESVLNSLIEECEKAATLANTNQLNRKILINSLYGALGNIHFRYYDLRNATAITIFGQVGIQWIARKINEYLNKVCGTNDEDFIAAGDTDSVYVCVDKVIEKVGLDRFKEQNDLVEFMNQFGKKKMEPMIDVAYRELCDYMNNREHLMHMDREAISCPPLGSKGVGGFWKAKKRYALNVYDMEDKRFAEPHLKIMGMETQQSSTPKAVQEALEESIRRILQEGEESVQEYYKNFEKEYRQLDYKVIAEVKTANDIAKYDDKGWPGFKCPFHIRGVLTYRRAVSGLGVAPILDGNKVMVLPLREGNPFGDKCIAWPSGTELPKEIRSDVLSWIDHSTLFQKSFVKPLAGMCESAGMDYEEKASLDFLFG</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/1NOY" target="_blank" style="color:#520049"><b>1NOY</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/1258685" target="_blank" style="color:#520049"><b>1258685</b></a></td>
      </tr>
	  </tbody>
  </table>

  <p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
      <td name="td0">Wild type aptamer</td>
      <td name="td1">5'-UAAUAUAUCAAGAGCCUAAUAACUCGGGCUAUAAACUAAGGAAUAUCUAUG-3'</td>
      <td name="td2">T4 DNA polymerase</td>
      <td name="td3">4.8 nM</td>
    </tr>
            
     <tr>
      <td name="td0">Major variant aptamer</td>
      <td name="td1">5'-UAAUAUAUCAAGAGCCUAGCAACCUGGGCUAUAAACUAAGGAAUAUCUAUG-3'</td>
      <td name="td2">T4 DNA polymerase</td>
      <td name="td3">4.8 nM</td>
    </tr>
	  </tbody>
  </table>
<img src="/images/SELEX_ligand/T4_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">2P5G-B</td>
      <td name="td1">41.6</td>
      <td name="td2">2.0</td>
      <td name="td3">Template DNA</td>
    </tr>
            
     <tr>
      <td name="td0">3CFO-A</td>
      <td name="td1">36.0</td>
      <td name="td2">1.9</td>
      <td name="td3">DNA polymerase</td>
    </tr>
            
     <tr>
      <td name="td0">6WJV-A</td>
      <td name="td1">23.7</td>
      <td name="td2">3.2</td>
      <td name="td3">DNA polymerase epsilon catalytic subunit</td>
    </tr>
            
     <tr>
      <td name="td0">8OJC-A</td>
      <td name="td1">23.6</td>
      <td name="td2">2.8</td>
      <td name="td3">DNA (47-mer)</td>
    </tr>
            
     <tr>
      <td name="td0">6S2E-A</td>
      <td name="td1">22.8</td>
      <td name="td2">3.2</td>
      <td name="td3">DNA polymerase epsilon catalytic subunit</td>
    </tr>
            
     <tr>
      <td name="td0">7OPL-A</td>
      <td name="td1">22.3</td>
      <td name="td2">3.3</td>
      <td name="td3">DNA polymerase alpha catalytic subunit</td>
    </tr>
            
     <tr>
      <td name="td0">7M5P-A</td>
      <td name="td1">21.5</td>
      <td name="td2">3.1</td>
      <td name="td3">DNA polymerase</td>
    </tr>
            
     <tr>
      <td name="td0">4FLU-A</td>
      <td name="td1">19.6</td>
      <td name="td2">3.3</td>
      <td name="td3">Pyrococcus abyssi b family DNA polymeras</td>
    </tr>
            
     <tr>
      <td name="td0">6V93-A</td>
      <td name="td1">18.8</td>
      <td name="td2">3.0</td>
      <td name="td3">DNA polymerase zeta catalytic subunit</td>
    </tr>
            
     <tr>
      <td name="td0">8HOY-A</td>
      <td name="td1">18.1</td>
      <td name="td2">3.5</td>
      <td name="td3">DNA polymerase</td>
    </tr>
    </tbody>
  </table>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Systematic evolution of ligands by exponential enrichment: RNA ligands to bacteriophage T4 DNA polymerase.</strong></font><br />
Tuerk, C., & Gold, L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/2200121/" target="_blank" style="color:#520049">Science (New York, N.Y.), 249(4968), 505–510. (1990)</a>
<br/>


<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>




    </html>
