---
layout: theophylline_style
title: GRK2 aptamer
date: 2024-04-19 00:00:00
author: Zhizhong Lu, Ying Ao
categories: Aptamer
tags:
- Category:Mammalian
- GC:50.0
- GC:57.89
- GC:58.81999999999999
- Length:18
- Length:20
- Length:28
- Length:51
- Length:57
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13.18 aptamer
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13.20 aptamer
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13.28 aptamer
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13.51
- Type:Proteins
- Year:2008
- Year:2012
type: Proteins
---
<html>



<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>

         
<p class="header_box" id="description">Description</p>
<p>In 2008, Blind, M. et al. employed the SELEX (systematic evolution of ligands by exponential enrichment) strategy to develop RNA aptamers capable of potent and selective inhibition of GRK2. This endeavour resulted in the identification of an aptamer, designated C13, which demonstrated a high affinity for binding to GRK2. Subsequently, in 2012, Tesmer, J. J. elucidated the crystallographic structures of GRK2 in complex with an RNA aptamer that potently and selectively inhibits the kinase activity. These findings have significantly contributed to the understanding of the molecular interactions between GRK2 and its RNA-based inhibitors<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref2" style="color:#520049">2</a></sup><sup>]</sup>.<br></p>



<p class="header_box" id="SELEX">SELEX</p>
<p>In 2008, Blind, M. et al. used a structurally constrained RNA library in vitro selection. They followed a different approach and designed a RNA library that has a stable stem structure 13 nucleotides (nt) in length, flanked by two restriction sites (5′-Pst1 and a 3′-HindII) and additional primer binding sites that allow reverse transcription and PCR amplification. The region comprising 20 random nucleotides resides as loop structure on top of the stem motif. After 12 selection cycles the enriched RNA library was analyzed for GRK2 binding. The enriched RNA library was cloned and the sequences of 18 monoclones were determined<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>    



<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>C13.18 and C13.28 are truncated sequences of the aptamer C13, which was originally enriched and isolated for its high affinity. The orange represents the bases that interact with the protein, while the grey indicates parts of the 3D structure that remain unresolved. Here we use ribodraw to complete the figure, through the 3D structure information<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p>
<p>C13.18 aptamer: </p>
<p>5'-CCAUACGGGAGAGAAACU-3'</p>
<p>C13.28 aptamer: </p>
<p>5'-GGCAGACCAUACGGGAGAGAAACUUGCC-3'<br></p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/GRK2_aptamer_2D1.svg" alt="drawing" style="width:500px;height:400px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/GRK2_aptamer_2D2.svg" alt="drawing" style="width:500px;height:400px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>


<p class="blowheader_box">3D visualisation</p>             
<p>Mayer, G., & Tesmer, J. J. determined the crystal structure of C13.18 aptamer bound to the GRK2 protein at 3.51 Å resolution and C13.28 aptamer bound to the GRK2 protein at 4.52 Å resolution. The PDB ID of the two aptamers are 3UZT and 3UZS<sup>[<a href="#ref2" style="color:#520049">2</a></sup><sup>]</sup>. Only the structure of 3ZUS is shown here.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/GRK2_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A', 
  start_residue_number:29, 
  end_residue_number:671, 
  color:{r:133,g:39,b:181}
},
{
  struct_asym_id: 'B', 
  start_residue_number:2, 
  end_residue_number:340, 
  color:{r:207,g:62,b:125}
},{
  struct_asym_id: 'C', 
  start_residue_number:41, 
  end_residue_number:45, 
  color:{r:184,g:51,b:50}
},
{
  struct_asym_id: 'C', 
  start_residue_number:46, 
  end_residue_number:48, 
  color:{r:46,g:161,b:3}
},
{
  struct_asym_id: 'C', 
  start_residue_number:49, 
  end_residue_number:53, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'C', 
  start_residue_number:54, 
  end_residue_number:55, 
  color:{r:46,g:161,b:3}
},
{
  struct_asym_id: 'C', 
  start_residue_number:64, 
  end_residue_number:68, 
  color:{r:184,g:51,b:50}
},
{
  struct_asym_id: 'G', 
  start_residue_number:2, 
  end_residue_number:68, 
  color:{r:207,g:62,b:125}
},{
  struct_asym_id: 'A', 
  start_residue_number:701, 
  end_residue_number:701, 
  color:{r:0,g:128,b:0}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/3uzs-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<p>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 3UZS at 4.52 Å resolution. The GRK2 protein (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with the GRK2 protein. Hydrogen bonds was predicted by Pymol.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/GRK2_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/GRK2_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>



<p class="header_box" id="ligand-recognition">Ligand information</p> 
<p class="blowheader_box">SELEX ligand</p>
<p>Used a flow cytometry-based protein-RNA interaction assay. Biotinylated GRK2 (bGRK2) was first immobilized on streptavidin beads and incubated with increasing amounts of 5'Alexa Fluor-488-labeled C13.28 (F-C13.28) or C13.18 (F-C13.18). The beads were then passed through a flow cytometer and the amount of bead-bound fluorescence was quantitated to yield a dissociation constant (Kd)<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">C13.18</td>
        <td name="td1">5'-CCAUACGGGAGAGAAACU-3'</td>
        <td name="td2">GRK2</td>
        <td name="td3">35 ± 5 nM</td>
      </tr>
      <tr>
        <td name="td0">C13.20</td>
        <td name="td1">5'-GACCAUACGGGAGAGAAACU-3'</td>
        <td name="td2">GRK2</td>
        <td name="td3">48 ± 7 nM</td>
      </tr>
      <tr>
        <td name="td0">C13.28</td>
        <td name="td1">5'-GGCAGACCAUACGGGAGAGAAACUUGCC-3'</td>
        <td name="td2">GRK2</td>
        <td name="td3">1.2 ± 0.6 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<p>GRK2, also called beta-adrenergic receptor kinase (beta-ARK) or beta-ARK1, is important in regulating several cardiac receptor responses. It plays a role in cardiac development and in hypertension. Deletion of GRK2 in mice results in embryonic lethality, caused by hypoplasia of the ventricular myocardium. GRK2 also plays important roles in the liver (as a regulator of portal blood pressure), in immune cells, and in the nervous system. Altered GRK2 expression has been reported in several disorders including major depression, schizophrenia, bipolar disorder, and Parkinsonism. GRK2 contains an N-terminal RGS homology (RH) domain, a central catalytic domain, and C-terminal pleckstrin homology (PH) domain that mediates PIP2 and G protein betagamma-subunit translocation to the membrane. GRKs phosphorylate and regulate G protein-coupled receptors (GPCRs), the largest superfamily of cell surface receptors which regulate some part of nearly all physiological functions. Phosphorylated GPCRs bind to arrestins, which prevents further G protein signaling despite the presence of activating ligand. TheGRK2 subfamily is part of a larger superfamily that includes the catalytic domains of other STKs, protein tyrosine kinases, RIO kinases, aminoglycoside phosphotransferase, choline kinase, and phosphoinositide 3-kinase.-----From <a href="https://www.ebi.ac.uk/interpro/entry/cdd/CD14223/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Interpro</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>InterPro</b></a>:</b> a database that provides functional analysis of proteins by classifying them into families and predicting the presence of domains and important sites.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">InterPro</th>
        <th onclick="sortTable(2)">Mass</th>
        <th onclick="sortTable(3)">Protein sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P21146/" target="_blank" style="color:#520049"><b>P21146</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/cdd/CD14223/" target="_blank" style="color:#520049"><b>CD14223</b></a></td>
        <td name="td2">79.66 KDa</td>
        <td name="td3">
        <div class="sequence-container">

          <span class="full-sequence">MADLEAVLADVSYLMAMEKSKATPAARASKKILLPEPSIRSVMQKYLEDRGEVTFEKIFSQKLGYLLFRDFCLKHLEEAKPLVEFYEEIKKYEKLETEEERLVCSREIFDTYIMKELLACSHPFSKSAIEHVQGHLVKKQVPPDLFQPYIEEICQNLRGDVFQKFIESDKFTRFCQWKNVELNIHLTMNDFSVHRIIGRGGFGEVYGCRKADTGKMYAMKCLDKKRIKMKQGETLALNERIMLSLVSTGDCPFIVCMSYAFHTPDKLSFILDLMNGGDLHYHLSQHGVFSEADMRFYAAEIILGLEHMHNRFVVYRDLKPANILLDEHGHVRISDLGLACDFSKKKPHASVGTHGYMAPEVLQKGVAYDSSADWFSLGCMLFKLLRGHSPFRQHKTKDKHEIDRMTLTMAVELPDSFSPELRSLLEGLLQRDVNRRLGCLGRGAQEVKESPFFRSLDWQMVFLQKYPPPLIPPRGEVNAADAFDIGSFDEEDTKGIKLLDSDQELYRNFPLTISERWQQEVAETVFDTINAETDRLEARKKTKNKQLGHEEDYALGKDCIMHGYMSKMGNPFLTQWQRRYFYLFPNRLEWRGEGEAPQSLLTMEEIQSVEETQIKERKCLLLKIRGGKQFVLQCDSDPELVQWKKELRDAYREAQQLVQRVPKMKNKPRSPVVELSKVPLIQRGSANGL</span>
        </div>
      </td>
        <td name="td4"><a href="https://www.rcsb.org/structure/1OMW" target="_blank" style="color:#520049"><b>1OMW</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/gene/282682" target="_blank" style="color:#520049"><b>282682</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/GRK2_aptamer_stru_ligand1.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">1OMW-A</td>
        <td name="td2">55.2</td>
        <td name="td3">0</td>
        <td name="td4">G-protein coupled receptor kinase 2</td>
      </tr>
      <tr>
        <td name="td1">3PSC-A</td>
        <td name="td2">51.8</td>
        <td name="td3">0.4</td>
        <td name="td4">Beta-adrenergic receptor kinase 1</td>
      </tr>
      <tr>
        <td name="td1">3PVW-A</td>
        <td name="td2">50.3</td>
        <td name="td3">0.7</td>
        <td name="td4">Beta-adrenergic receptor kinase 1</td>
      </tr>
      <tr>
        <td name="td1">7PWD-A</td>
        <td name="td2">49.5</td>
        <td name="td3">1.3</td>
        <td name="td4">Beta-adrenergic receptor kinase 1</td>
      </tr>
      <tr>
        <td name="td1">1YM7-A</td>
        <td name="td2">49.1</td>
        <td name="td3">0.8</td>
        <td name="td4">Beta-adrenergic receptor kinase 1</td>
      </tr>
      <tr>
        <td name="td1">1YM7-D</td>
        <td name="td2">49</td>
        <td name="td3">0.6</td>
        <td name="td4">Beta-adrenergic receptor kinase 1</td>
      </tr>
      <tr>
        <td name="td1">3PVU-A</td>
        <td name="td2">48.8</td>
        <td name="td3">0.5</td>
        <td name="td4">Beta-adrenergic receptor kinase 1</td>
      </tr>
      <tr>
        <td name="td1">5UKL-A</td>
        <td name="td2">48.7</td>
        <td name="td3">1.2</td>
        <td name="td4">Beta-adrenergic receptor kinase 1</td>
      </tr>
      <tr>
        <td name="td1">3CIK-A</td>
        <td name="td2">48.4</td>
        <td name="td3">1.2</td>
        <td name="td4">Beta-adrenergic receptor kinase 1</td>
      </tr>
      <tr>
        <td name="td1">6U7C-A</td>
        <td name="td2">48.3</td>
        <td name="td3">0.9</td>
        <td name="td4">Beta-adrenergic receptor kinase 1</td>
      </tr>
	  </tbody>
  </table>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] An RNA molecule that specifically inhibits G-protein-coupled receptor kinase 2 in vitro.</strong></font><br />
Mayer, G., Wulffen, B., Huber, C., Brockmann, J., Flicke, B., Neumann, L., Hafenbradl, D., Klebl, B. M., Lohse, M. J., Krasel, C., & Blind, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18230760/" target="_blank" style="color:#520049" >RNA (New York, N.Y.), 14(3), 524–534. (2008)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Molecular mechanism for inhibition of g protein-coupled receptor kinase 2 by a selective RNA aptamer.</strong></font><br />
Tesmer, V. M., Lennarz, S., Mayer, G., & Tesmer, J. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/22727813/" target="_blank" style="color:#520049">Structure (London, England : 1993), 20(8), 1300–1309. (2012)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] Crystallographic Pursuit of a Protein-RNA Aptamer Complex.</strong></font><br />
Tesmer J. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/26552823/" target="_blank" style="color:#520049"> Methods in molecular biology (Clifton, N.J.), 1380, 151–160. (2016)</a>
<br />


<script>
    function toggleSequence(event) {
      const container = event.target.closest('.sequence-container');
      container.classList.toggle('expanded');
      const showMoreText = container.querySelector('.show-more');
      
      // 展开后按钮文本变化
      if (container.classList.contains('expanded')) {
        showMoreText.textContent = '...';  // 展开后显示 "..."
      } else {
        showMoreText.textContent = '......';  // 收起后显示 "......"
      }
    }

    // 页面加载时，限制序列文本为50个字符
    window.addEventListener('load', function() {
      const sequenceContainers = document.querySelectorAll('.sequence-container');
      sequenceContainers.forEach(container => {
        const fullSeqText = container.querySelector('.full-sequence').textContent;
        const truncatedText = fullSeqText.slice(0, 20);  // 只显示前50个字符
        container.querySelector('.sequence-text').textContent = truncatedText;
      });
    });
  </script>