---
layout: theophylline_style
title: H5N1/H7N7 Hemagglutinin aptamer
date: 2024-06-02 00:00:00
author: Zhijie Tan, Qiaozhen Liu
categories: Aptamer
tags:
- Category:Viral
- GC:54.55
- GC:58.330000000000005
- Length:132
- Named:Hemagglutinin (HA) of H5N1 and H7N7_8-1
- Named:Hemagglutinin (HA) of H5N1 and H7N7_8-10
- Named:Hemagglutinin (HA) of H5N1 and H7N7_8-3
- Type:Proteins
- Year:2014
type: Proteins
---
<html>

<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="description">Description</p>
<p>In 2012, Suenaga et al. selected an aptamer (8-3) from a completely random RNA pool that binds with high affinity (∼KD 170 pM) to the hemagglutinins (HAs) derived from HPAI H5N1 (A/H5N1/Vietnam/1194/2004 and A/H5N1/Indonesia/05/2005) and H7N7 (A/H7N7/Netherlands/219/2003) influenza A viruses. Aptamer 8-3 was analyzed further to assess its ability to interfere with HA–glycan interactions using our previously established SPR-based competitive assay<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2012, Suenaga et al. identified isolated 3 specific sequences for targeting HA in a 74nt sequence poolthrough 10 rounds of selection<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>8-3 was the aptamer sequence mainly studied in the article, which had a high affinity with HA of H5N1 and H7N7. The 2D structure of the figure is based on the article by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>
<p>5'-GGAGCUCAGCCUUCACUGCGAGAUUGUUCUGACCGAGUUGCCUAGCAGGGCAACCGCUGGAACUUGAAGUCGGUAAUGCGAGCGGAAAGCCUUGGCACCACGGUCGGAUCCAC-3'</p>
<img src="/images/2D/HPAI_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>This entry represents the haemagglutinin-neuraminidase (HN) glycoprotein found in a variety of paramyxoviruses (negative-stranded RNA viruses), including Mumps virus, Human parainfluenza virus 3, and the avian pathogen Newcastle disease virus. It also includes hemagglutinin glycoproteins from Morbiliviruses, a genus belonging to the Paramyxoviridae family that includes the Measles virus. Morbiliviruses hemagglutinins have no neuraminidase activity.<br>HN is a multi-functional protein with three distinct functions: a receptor-binding (haemagglutinin) activity, a receptor-destroying (neuraminidase) activity, and a membrane fusion activity that fuses the viral envelope to the host cell membrane in order to infect the cell.-----from Pfam</p>p
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Hemagglutinins (HA)</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/Q5EP31/entry" target="_blank" style="color:#520049"><b>Q5EP31</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR000665/" target="_blank" style="color:#520049"><b>IPR000665</b></a></td>
        <td name="td3">20.86 kDa</td>
        <td name="td4">GLFGAIAGFIEGGWQGMVDGWYGYHHSNEQGSGYAADKESTQKAIDGVTNKVNSIIDKMNTQFEAVGREFNNLERRIENLNKKMEDGFLDVWTYNAELLVLMENERTLDFHDSNVKNLYDKVRLQLRDNAKELGNGCFEFYHKCDNECMESVRNGTYDYPQYSEEARLKREEISSGRLVPR</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/4N5Z" target="_blank" style="color:#520049"><b>4N5Z</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/protein/2FK0_B" target="_blank" style="color:#520049"><b>2FK0_B</b></a></td>
      </tr>
	  </tbody>
  </table>

<p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
      <td name="td0">8-3</td>
      <td name="td1">5'-GGAGCUCAGCCUUCACUGCGAGAUUGUUCUGACCGAGUUGCCUAGCAGGGCAACCGCUGGAACUUGAAGUCGGUAAUGCGAGCGGAAAGCCUUGGCACCACGGUCGGAUCCAC-3'</td>
      <td name="td2">A/H5N1/Vietnam/1194/2004</td>
      <td name="td3">0.75 nM</td>
    </tr>
            
     <tr>
      <td name="td0">8-3</td>
      <td name="td1">5'-GGAGCUCAGCCUUCACUGCGAGAUUGUUCUGACCGAGUUGCCUAGCAGGGCAACCGCUGGAACUUGAAGUCGGUAAUGCGAGCGGAAAGCCUUGGCACCACGGUCGGAUCCAC-3'</td>
      <td name="td2">A/H5N1/Indonesia/05/2005</td>
      <td name="td3">0.17 nM</td>
    </tr>
            
     <tr>
      <td name="td0">8-3</td>
      <td name="td1">5'-GGAGCUCAGCCUUCACUGCGAGAUUGUUCUGACCGAGUUGCCUAGCAGGGCAACCGCUGGAACUUGAAGUCGGUAAUGCGAGCGGAAAGCCUUGGCACCACGGUCGGAUCCAC-3'</td>
      <td name="td2">A/H7N7/Netherlands/219/2003 </td>
      <td name="td3">0.42 nM</td>
    </tr>
            
     <tr>
      <td name="td0">8-3</td>
      <td name="td1">5'-GGAGCUCAGCCUUCACUGCGAGAUUGUUCUGACCGAGUUGCCUAGCAGGGCAACCGCUGGAACUUGAAGUCGGUAAUGCGAGCGGAAAGCCUUGGCACCACGGUCGGAUCCAC-3'</td>
      <td name="td2">A/H9N2/Hongkong/1073/99</td>
      <td name="td3">260 nM</td>
    </tr>
            
     <tr>
      <td name="td0">8-3</td>
      <td name="td1">5'-GGAGCUCAGCCUUCACUGCGAGAUUGUUCUGACCGAGUUGCCUAGCAGGGCAACCGCUGGAACUUGAAGUCGGUAAUGCGAGCGGAAAGCCUUGGCACCACGGUCGGAUCCAC-3'</td>
      <td name="td2">A/H1N1/California/04/2009</td>
      <td name="td3">170 nM</td>
    </tr>
            
     <tr>
      <td name="td0">8-1</td>
      <td name="td1">5'-GGAGCUCAGCCUUCACUGCCUGUUAGAGUUUCCUAAAAGCGAACUGGCGCCCUCGUCAGCAUCUGGCAGACGAGUGGAGACGGACUAACCACAGGCACCACGGUCGGAUCCAC-3'</td>
      <td name="td2">A/H5N1/Vietnam/1194/2004</td>
      <td name="td3">2.6 μM</td>
    </tr>
            
     <tr>
      <td name="td0">8-1</td>
      <td name="td1">5'-GGAGCUCAGCCUUCACUGCCUGUUAGAGUUUCCUAAAAGCGAACUGGCGCCCUCGUCAGCAUCUGGCAGACGAGUGGAGACGGACUAACCACAGGCACCACGGUCGGAUCCAC-3'</td>
      <td name="td2">A/H5N1/Indonesia/05/2005</td>
      <td name="td3">23 nM</td>
    </tr>
            
     <tr>
      <td name="td0">8-1</td>
      <td name="td1">5'-GGAGCUCAGCCUUCACUGCCUGUUAGAGUUUCCUAAAAGCGAACUGGCGCCCUCGUCAGCAUCUGGCAGACGAGUGGAGACGGACUAACCACAGGCACCACGGUCGGAUCCAC-3'</td>
      <td name="td2">A/H7N7/Netherlands/219/2003</td>
      <td name="td3">7 nM</td>
    </tr>
            
     <tr>
      <td name="td0">8-10</td>
      <td name="td1">5'-GGAGCUCAGCCUUCACUGCGGUGACCGGAGGAAUACGCGGACGGAGAAAGGGUUGGCCUCGUGGAUGGCGGAUACCCGUCCAGGGAUCUUCUAGGCACCACGGUCGGAUCCAC-3'</td>
      <td name="td2">A/H5N1/Vietnam/1194/2004</td>
      <td name="td3">93 nM</td>
    </tr>
            
     <tr>
      <td name="td0">8-10</td>
      <td name="td1">5'-GGAGCUCAGCCUUCACUGCGGUGACCGGAGGAAUACGCGGACGGAGAAAGGGUUGGCCUCGUGGAUGGCGGAUACCCGUCCAGGGAUCUUCUAGGCACCACGGUCGGAUCCAC-3'</td>
      <td name="td2">A/H5N1/Indonesia/05/2005</td>
      <td name="td3">320 nM</td>
    </tr>
            
     <tr>
      <td name="td0">8-10</td>
      <td name="td1">5'-GGAGCUCAGCCUUCACUGCGGUGACCGGAGGAAUACGCGGACGGAGAAAGGGUUGGCCUCGUGGAUGGCGGAUACCCGUCCAGGGAUCUUCUAGGCACCACGGUCGGAUCCAC-3'</td>
      <td name="td2">A/H7N7/Netherlands/219/2003</td>
      <td name="td3">21 nM</td>
    </tr>
	</tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/HPAI_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">4F23-A</td>
      <td name="td1">11.9</td>
      <td name="td2">2.0</td>
      <td name="td3">Hemagglutinin</td>
    </tr>
            
     <tr>
      <td name="td0">1FLC-B</td>
      <td name="td1">6.7</td>
      <td name="td2">4.1</td>
      <td name="td3">Haemagglutinin-esterase-fusion glycoprotein</td>
    </tr>
            
     <tr>
      <td name="td0">5E5W-B</td>
      <td name="td1">6.5</td>
      <td name="td2">4.4</td>
      <td name="td3">Hemagglutinin-esterase</td>
    </tr>
            
     <tr>
      <td name="td0">8TH8-A</td>
      <td name="td1">5.5</td>
      <td name="td2">6.5</td>
      <td name="td3">Dynein regulatory complex protein 1/2 N-terminal</td>
    </tr>
            
     <tr>
      <td name="td0">8AFZ-B</td>
      <td name="td1">5.3</td>
      <td name="td2">7.2</td>
      <td name="td3">Sorting nexin-1</td>
    </tr>
            
     <tr>
      <td name="td0">8CR1-C</td>
      <td name="td1">5.2</td>
      <td name="td2">5.1</td>
      <td name="td3">Atpase asna1</td>
    </tr>
            
     <tr>
      <td name="td0">6VM0-A</td>
      <td name="td1">5.0</td>
      <td name="td2">3.6</td>
      <td name="td3">Glycine receptor subunit alphaz1</td>
    </tr>
            
     <tr>
      <td name="td0">5IP0-D</td>
      <td name="td1">4.9</td>
      <td name="td2">7.9</td>
      <td name="td3">Pha granule-associated protein</td>
    </tr>
            
     <tr>
      <td name="td0">8EK4-B</td>
      <td name="td1">4.9</td>
      <td name="td2">4.4</td>
      <td name="td3">Ice-binding protein tip-99A</td>
    </tr>
            
     <tr>
      <td name="td0">3FX7-B</td>
      <td name="td1">4.8</td>
      <td name="td2">4.8</td>
      <td name="td3">Putative uncharacterized protein</td>
    </tr>
    </tbody>
  </table>
                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] An aptamer that binds efficiently to the hemagglutinins of highly pathogenic avian influenza viruses (H5N1 and H7N7) and inhibits hemagglutinin-glycan interactions.</strong></font><br />
Suenaga, E., & Kumar, PK.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/24374323/" target="_blank" style="color:#520049">Acta Biomaterialia,10(3):1314-23. (2014)</a>
<br/>


<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1RAW-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>
    
    </html>
