---
layout: theophylline_style
title:  "HIV Tat aptamer"
date:   2024-04-20 00:00:00
author: Zhizhong Lu, Ying Ao
categories: Aptamer
---
<html>



<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-2443.2000.00330.x" target="_blank" style="color:#520049">2000</a></h3>
    </div>
    <div class="body">
      <p>Used an in vitro selection method and isolated a novel aptamer RNATat, a 37-mer RNA oligomer, that binds efficiently to the Tat protein of HIV-1<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-2443.2000.00331.x" target="_blank" style="color:#520049">2000</a></h3>
    </div>
    <div class="body">
      <p>Reported the use of aptamer-derived oligomers to analyze the Tat of HIV and the possible applications of such constructs in the field of biosensors<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12737819/" target="_blank" style="color:#520049">2003</a></h3>
    </div>
    <div class="body">
      <p>Determined the NMR structure of the aptamer complexed with two argininamide molecules<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/16027048/" target="_blank" style="color:#520049">2005</a></h3>
    </div>
    <div class="body">
      <p>Two biosensors have been constructed using an RNA aptamer as biorecognition element. The aptamer, specific for HIV-1 Tat protein<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/22975093/" target="_blank" style="color:#520049">2013</a></h3>
    </div>
    <div class="body">
      <p>Studied how surface charge density influences the binding of HIV-1 Tat protein using RNA aptamers on a solid surface with a diamond field-effect transistor (FET)<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/31707021/" target="_blank" style="color:#520049">2020</a></h3>
    </div>
    <div class="body">
      <p>The detection performance of antiTat aptamers for HIV-Tat protein was reported using both spectroscopic ellipsometry (SE) and surface plasmon resonance enhanced total internal reflection ellipsometry (SPReTIRE) for the first time<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/37240414/" target="_blank" style="color:#520049">2023</a></h3>
    </div>
    <div class="body">
      <p>First used in vitro NMR to examine the interaction between an RNA aptamer for HIV-1 Tat and a part of Tat containing the binding site for trans-activation response element (TAR)<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
    </div>
  </div>
</div>


         
<p class="header_box" id="description">Description</p>
<p>In 2000, Kumar, P. K. et al. used an in vitro selection approach to isolate a novel aptamer, Tat RNA aptamer, consisting of a 37-mer RNA oligomer, exhibiting high-affinity binding to the Tat protein of HIV-1. In this study, they explored various characteristics of the RNA aptamer. The RNA aptamer maintained significant binding affinity to Tat even under conditions of a substantial excess of HIV TAR, indicating its potential utility as a molecular recognition element in biosensors. In 2003, Katahira, M. et al determined the structure of the aptamer complexed with argininamide, the simplest analog of Tat, has been determined by NMR. Unique structural features responsible for the high affinity have been found: the formation of two adjacent U:A:U base triples, the resultant widening of the major groove, the formation of hydrogen bonds between a G base and argininamide, and the stabilization of the binding through stacking interaction of a guanidinium group with bases<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref3" style="color:#520049">3</a></sup><sup>]</sup>.<br></p>



<p class="header_box" id="SELEX">SELEX</p>
<p>In 2000, Kumar, P. K. et al. conducted a study involving a series of selection cycles. Initially, a binding buffer containing 5.0 mM RNA and 0.5 mM Tat protein was used to facilitate the first cycle of selection. Subsequent cycles involved varying the RNA pool concentrations and conducting competition assays with both nonspecific RNA (E. coli tRNA) and specific competitor RNA (TAR RNA). Additionally, from cycles 7 to 11, competition with another specific pool of competitor RNAs (the 12-18N pool) was introduced. The final two cycles featured a significant reduction in the concentration of Tat protein. The binding buffer consisted of 50 mM Tris-HCl (pH 7.8) and 50 mM KCl. Pre-filtering through a nitrocellulose acetate filter was performed to eliminate RNAs that selectively bound to the filter. After each cycle, the Tat-RNA complexes were collected on a filter and eluted using a solution of sodium acetate, EDTA, and urea. Reverse transcription and PCR amplification were then carried out following elution. Mutagenic PCR was employed during cycles 9 to 11. Following the 11th cycle, the PCR products were ligated into the pCRII vector. Individual clones were subsequently sequenced for analysis<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>
<br>
<br>  



<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>Here we use ribodraw to complete the figure, through the 3D structure information. RNA<sup>Tat</sup> aptamer was named by Kumar, P. K. et al. in the article<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref3" style="color:#520049">3</a></sup><sup>]</sup>.</p>
<p>5'-GGGAGCUUGAUCCCGGAAACGGUCGAUCGCUCCC-3'</p>
<img src="/images/2D/Tat_aptamer_2D.svg" alt="drawing" style="width:800px;height:300px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>



<p class="blowheader_box">3D visualisation</p>             
<p>Kumar, P. K., & Katahira, M. et al. determined  the Aptamer-Argininamide Complex structure by NMR. Unique structural features responsible for the high affinity have been found: the formation of two adjacent U:A:U base triples, the resultant widening of the major groove, the formation of hydrogen bonds between a G base and argininamide, and the stabilization of the binding through stacking interaction of a guanidinium group with bases. Structural characterization of the aptamer complexed with the RG peptide has also been carried out. Simultaneous interactions of the aptamer with two arginine residues of the RG peptide at two binding sites are strongly suggested. A combination of structural studies on the aptamer-argininamide and aptamer-RG peptide complexes provides a comprehensive explanation of the extremely high affinity of the aptamer to Tat<sup>[<a href="#ref3" style="color:#520049">3</a></sup><sup>]</sup>.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/Tat_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A', 
  start_residue_number:1, 
  end_residue_number:6, 
  color:{r:184,g:51,b:50}
},
{
  struct_asym_id: 'A', 
  start_residue_number:7, 
  end_residue_number:8, 
  color:{r:46,g:161,b:3}
},{
  struct_asym_id: 'A', 
  start_residue_number:10, 
  end_residue_number:11, 
  color:{r:46,g:161,b:3}
},
{
  struct_asym_id: 'A', 
  start_residue_number:9, 
  end_residue_number:9, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'A', 
  start_residue_number:12, 
  end_residue_number:12, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'A', 
  start_residue_number:13, 
  end_residue_number:22, 
  color:{r:8,g:69,b:149}
},{
  struct_asym_id: 'A', 
  start_residue_number:23, 
  end_residue_number:24, 
  color:{r:46,g:161,b:3}
},{
  struct_asym_id: 'A', 
  start_residue_number:28, 
  end_residue_number:28, 
  color:{r:241,g:125,b:7}
},{
  struct_asym_id: 'A', 
  start_residue_number:25, 
  end_residue_number:25, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'A', 
  start_residue_number:26, 
  end_residue_number:27, 
  color:{r:46,g:161,b:3}
},{
  struct_asym_id: 'A', 
  start_residue_number:29, 
  end_residue_number:34, 
  color:{r:184,g:51,b:50}
},{
  struct_asym_id: 'B', 
  start_residue_number:35, 
  end_residue_number:36, 
  color:{r:255,g:0,b:255}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1NBK-3D.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<p>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 1NBK. Argininamide (shown in sticks) is labeled in magenta. Right: The hydrogen bonds of binding sites of the aptamer bound with the argininamide. HIV-1 Tat peptides is a cell penetrating peptide rich in arginine.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Tat_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Tat_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>


<p class="header_box" id="ligand-recognition">Ligand information</p>
<p class="blowheader_box">SELEX ligand</p>
<p>The affinity was determined by Gel-shift binding assay. Kumar, P. K. et al performed gel-shift assays at four concentrations of RNA with varying concentrations of CQ (0.1-64 nM).The TAR-1-CQ and aptamer-CQ complexes were resolved on nondenaturing gels and estimated the amount of each complexformed. The equilibrium  dissociation  constants  were calculated by nonlinear regression to fit the saturation radiolabelled  ligand-binding isotherm. CQ and Tat-1 protein have similar binding affinity for TAR RNA, used CQ instead of Tat-1 in binding kinetics experiments  mainly to avoidproblems with possible denaturation of Tat-1 protein during expression and purification. CQ: Tat-1-derived peptids (amino acid residues 37-72)<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">RNA<sup>Tat</sup> aptamer</td>
        <td name="td1">5'-ACGAAGCUUGAUCCCGUUUGCCGGUCGAUCGCUUCGA-3'</td>
        <td name="td3">CQ(Tat protein residues 37-72)</td>
        <td name="td4">120 ± 13 pM</td>
      </tr>
      <tr>
        <td name="td0">TAR RNA</td>
        <td name="td1">5'-GGGUCUCUCUGGUUAGACCAGAUUUGAGCCUGGGAGCUCUCUGGCUAACUAGGGAACCC-3'</td>
        <td name="td3">CQ(Tat protein residues 37-72)</td>
        <td name="td4">16 ± 11 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<p>The retroviral Tat protein binds to the TAR RNA. This activates transcriptional initiation and elongation from the LTR promoter. Binding is mediated by an arginine rich region.-----From <a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00539/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(6)">Name</th>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">Mass</th>
        <th onclick="sortTable(3)">Protein sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td6">HIV-1 Tat</td>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P04608/" target="_blank" style="color:#520049"><b>P04608</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00539/" target="_blank" style="color:#520049"><b>PF00539</b></a></td>
        <td name="td2">9.8 KDa</td>
        <td name="td3">
        <div class="sequence-container">

          <span class="full-sequence">MEPVDPRLEPWKHPGSQPKTACTNCYCKKCCFHCQVCFITKALGISYGRKKRRQRRRAHQNSQTHQASLSKQPTSQPRGDPTGPKE</span>
        </div>
      </td>
        <td name="td4"><a href="https://www.rcsb.org/structure/6CYT" target="_blank" style="color:#520049"><b>6CYT</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/gene/155871" target="_blank" style="color:#520049"><b>155871</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/Tat_stru_ligand1.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">3MI9-C</td>
        <td name="td2">9.7</td>
        <td name="td3">0</td>
        <td name="td4">Cell division protein kinase 9</td>
      </tr>
      <tr>
        <td name="td1">3MIA-C</td>
        <td name="td2">8.9</td>
        <td name="td3">0.2</td>
        <td name="td4">Cell division protein kinase 9</td>
      </tr>
      <tr>
        <td name="td1">4OR5-C</td>
        <td name="td2">8.6</td>
        <td name="td3">0.4</td>
        <td name="td4">Cyclin-dependent kinase 9</td>
      </tr>
      <tr>
        <td name="td1">4OGR-D</td>
        <td name="td2">8.6</td>
        <td name="td3">0.5</td>
        <td name="td4">Cyclin-dependent kinase 9</td>
      </tr>
      <tr>
        <td name="td1">4OR5-H</td>
        <td name="td2">8.6</td>
        <td name="td3">0.3</td>
        <td name="td4">Cyclin-dependent kinase 9</td>
      </tr>
      <tr>
        <td name="td1">4OGR-H</td>
        <td name="td2">8.6</td>
        <td name="td3">0.8</td>
        <td name="td4">Cyclin-dependent kinase 9</td>
      </tr>
      <tr>
        <td name="td1">4OGR-M</td>
        <td name="td2">8.5</td>
        <td name="td3">0.5</td>
        <td name="td4">Cyclin-dependent kinase 9</td>
      </tr>
      <tr>
        <td name="td1">6CYT-D</td>
        <td name="td2">8.4</td>
        <td name="td3">0.5</td>
        <td name="td4">Cyclin-dependent kinase 9</td>
      </tr>
      <tr>
        <td name="td1">5L1Z-D</td>
        <td name="td2">8.1</td>
        <td name="td3">0.9</td>
        <td name="td4">Cyclin-dependent kinase 9</td>
      </tr>
	  </tbody>
  </table>



                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] A novel RNA motif that binds efficiently and specifically to the Tat protein of HIV and inhibits the trans‐activation by Tat of transcription in vitro and in vivo.</strong></font><br />
Yamamoto, R., Katahira, M., Nishikawa, S., Baba, T., Taira, K., & Kumar, P.K.<br />
<a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-2443.2000.00330.x" target="_blank" style="color:#520049" >Genes to Cells, 5(5), 371-388 (2000)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Molecular beacon aptamer fluoresces in the presence of Tat protein of HIV-1.</strong></font><br />
Yamamoto, R., Baba, T., & Kumar, P. K.<br />
<a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-2443.2000.00331.x" target="_blank" style="color:#520049">Genes to cells, 5(5), 389–396. (2000)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] Structural basis of the highly efficient trapping of the HIV Tat protein by an RNA aptamer.</strong></font><br />
Matsugami, A., Kobayashi, S., Ouhashi, K., Uesugi, S., Yamamoto, R., Taira, K., Nishikawa, S., Kumar, P. K., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12737819/" target="_blank" style="color:#520049"> Structure (London, England : 1993), 11(5), 533–545. (2003)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] Aptamer-based biosensors for the detection of HIV-1 Tat protein.</strong></font><br />
Tombelli, S., Minunni, M., Luzi, E., & Mascini, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/16027048/" target="_blank" style="color:#520049">Bioelectrochemistry (Amsterdam, Netherlands), 67(2), 135–141. (2005)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Effects of diamond-FET-based RNA aptamer sensing for detection of real sample of HIV-1 Tat protein.</strong></font><br />
Rahim Ruslinda, A., Tanabe, K., Ibori, S., Wang, X., & Kawarada, H.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/22975093/" target="_blank" style="color:#520049">Biosensors & bioelectronics, 40(1), 277–282. (2013)</a>
<br />

<a id="ref6"></a><font><strong>[6] Spectrophotometric ellipsometry based Tat-protein RNA-aptasensor for HIV-1 diagnosis.</strong></font><br />
Caglayan, M. O., & Üstündağ, Z.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/31707021/" target="_blank" style="color:#520049">Spectrochimica acta. Part A, Molecular and biomolecular spectroscopy, 227, 117748. (2020)</a>
<br />       

<a id="ref7"></a><font><strong>[7] Complex Formation of an RNA Aptamer with a Part of HIV-1 Tat through Induction of Base Triples in Living Human Cells Proven by In-Cell NMR.</strong></font><br />
Eladl, O., Yamaoki, Y., Kondo, K., Nagata, T., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/37240414/" target="_blank" style="color:#520049">International journal of molecular sciences, 24(10), 9069. (2023)</a>
<br /> 



<script>
    function toggleSequence(event) {
      const container = event.target.closest('.sequence-container');
      container.classList.toggle('expanded');
      const showMoreText = container.querySelector('.show-more');
      
      // 展开后按钮文本变化
      if (container.classList.contains('expanded')) {
        showMoreText.textContent = '...';  // 展开后显示 "..."
      } else {
        showMoreText.textContent = '......';  // 收起后显示 "......"
      }
    }

    // 页面加载时，限制序列文本为50个字符
    window.addEventListener('load', function() {
      const sequenceContainers = document.querySelectorAll('.sequence-container');
      sequenceContainers.forEach(container => {
        const fullSeqText = container.querySelector('.full-sequence').textContent;
        const truncatedText = fullSeqText.slice(0, 20);  // 只显示前50个字符
        container.querySelector('.sequence-text').textContent = truncatedText;
      });
    });
  </script>