---
layout: theophylline_style
title: HIV-1 Rev protein (3)
date: 2024-05-05 00:00:00
author: Ke Chen, Yangyi Ren
categories: Aptamer
tags:
- Category:Viral
- GC:56.99999999999999
- Length:35
- Named:C-terminally truncated variant of Rev (Rev93)_RBA14 aptamer
- Type:Proteins
- Year:2018
type: Proteins
---


<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/7505429/" target="_blank" style="color:#520049">1993</a></h3>
    </div>
    <div class="body">
      <p>Ellington, A. D. et al. obtained a series of aptamers with good affinity to HIV-1 Rev protein through in vitro selecting<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/7506689/" target="_blank" style="color:#520049">1993</a></h3>
    </div>
    <div class="body">
      <p>Ellington, A. D. et al. found a short helical stem and bulged nucleotides (nt) CUC ... UYGAG that have no counterpart in the wild-type (wt) element contribute to high-affinity binding<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/7664035/" target="_blank" style="color:#520049">1994</a></h3>
    </div>
    <div class="body">
      <p>The 3D model explains why RNA aptamer can bind Rev protein better<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8289245/" target="_blank" style="color:#520049">1994</a></h3>
    </div>
    <div class="body">
      <p>Tuerk, C. et al. produced small RNA ligands with high affinity for HIV-1 Rev protein by SELEX in vitro evolution method for a series of chemical modification studies<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8523524/" target="_blank" style="color:#520049">1996</a></h3>
    </div>
    <div class="body">
      <p>Aptamers were found to be functionally equivalent to the RBE when the assay system was saturated with Rev and better than the wild-type element when Rev was limiting<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8946856/" target="_blank" style="color:#520049">1996</a></h3>
    </div>
    <div class="body">
      <p>The solution structure of the 35-mer high affinity RNA aptamer binding site of HIV-1 17-mer rev peptide was determined by NMR molecular dynamics<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/9713975/" target="_blank" style="color:#520049">1998</a></h3>
    </div>
    <div class="body">
      <p>Specific inhibition of viral p24 production following co-transfection of the anti-HIV Rev-binding aptamer and HIV proviral DNAs was observed<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/15610009/" target="_blank" style="color:#520049">2004</a></h3>
    </div>
    <div class="body">
      <p>More than half of the residues have increased flexibility in the Rev-RNA aptamer complex that has a higher affinity. This further suggests that the retention of conformational flexibility may be important in high-affinity ARM-RNA recognition<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/18922466/" target="_blank" style="color:#520049">2008</a></h3>
    </div>
    <div class="body">
      <p>It was demonstrated that HIV Rev, which is limited by the limited viral genome, can also form organized RNP by assembling homologous oligomers on Rev reaction element (RRE) RNA<sup>[<a href="#ref9" style="color:#520049">9</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/23972852/" target="_blank" style="color:#520049">2013</a></h3>
    </div>
    <div class="body">
      <p>The arginine-rich RNA-binding motif of HIV-1 Rev is intrinsically disordered and folds upon RRE binding<sup>[<a href="#ref10" style="color:#520049">10</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/25486594/" target="_blank" style="color:#520049">2014</a></h3>
    </div>
    <div class="body">
      <p>Rev-RNA recognition relies on well-characterized sequence-specific contact at the IIB site and local RNA structure at the second site. The structure supports a model in which RRE utilizes the inherent plasticity of the Rev subunit interface to guide the formation of functional complexes<sup>[<a href="#ref11" style="color:#520049">11</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/30017564/" target="_blank" style="color:#520049">2018</a></h3>
    </div>
    <div class="body">
      <p>Wingfield, P. T. et al. present the crystal structure of Rev93:scFv:RBA-14 Ternary Complex at 3.0 Å through X-ray<sup>[<a href="#ref12" style="color:#520049">12</a>]</sup></p>
    </div>
  </div>
</div>


<p class="header_box" id="description">Description</p>
<p>In 1993, Lori Giver and colleagues used the SELEX method to isolate the aptamer with high affinity for the HIV-1 Rev protein. It can competitively bind Rev protein with RRE, thereby preventing RRE from interacting with Rev protein. Refer to the RBA-14 aptamer page for specific aptamer<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref12" style="color:#520049">12</a></sup><sup>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 1993, Ellington, A. D. et al. constructed a pool of random sequences. After in vitro selecting in the pool, selected RNAs were amplified via reverse transcription, polymerasevchain reaction (PCR) amplification, in vitro T7 transcription, and allowed to again compete for binding to HIV-1 Rev protein. Some aptamers with high affinity were selected after three selection cycles<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>Here we used RiboDraw to complete the figure, based the 3D structure information. The RBA-14 aptamer was named by Wingfield, P. T. et al. in the article<sup>[<a href="#ref12" style="color:#520049">12</a></sup><sup>]</sup>.</p>
<p>5'-GGCUGGACUCGUACUUCGGUACUGGAGAAACAGCC-3'</p>
<img src="/images/2D/HIV-1_Rev_aptamer_2D.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>

 

<p class="blowheader_box">3D visualisation</p>             
<p>Wingfield, P. T. et al. present the crystal structure of Rev93:scFv:RBA-14 Ternary Complex at 3.0 Å through X-ray. They used Rev:scFv as a crystallization platform for studying nucleic acid binding and made a ternary complex of Rev93, scFv and RBA-14, where the single-chain antibody functions as an assembly inhibitor, crystallization chaperone, and initial model for molecular replacement. The PDB ID of this structure is 6CF2<sup>[<a href="#ref12" style="color:#520049">12</a></sup><sup>]</sup>.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/HIV-1_Rev_aptamer_3D.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A',    
  start_residue_number:1, 
  end_residue_number:116, 
  color:{r:24,g:152,b:140}
},
{
  struct_asym_id: 'B', 
  start_residue_number:1, 
  end_residue_number:110, 
  color:{r:24,g:152,b:140}
},
{
  struct_asym_id: 'F',    
  start_residue_number:10, 
  end_residue_number:66, 
  color:{r:133,g:39,b:181}
},
{
  struct_asym_id: 'G', 
  start_residue_number:1, 
  end_residue_number:3, 
  color:{r:184,g:51,b:50}
},
{
  struct_asym_id: 'G',    
  start_residue_number:4, 
  end_residue_number:7, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'G', 
  start_residue_number:8, 
  end_residue_number:10, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'G',    
  start_residue_number:11, 
  end_residue_number:21, 
  color:{r:207,g:62,b:125}
},
{
  struct_asym_id: 'G', 
  start_residue_number:22, 
  end_residue_number:23, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'G',    
  start_residue_number:24, 
  end_residue_number:24, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'G', 
  start_residue_number:25, 
  end_residue_number:25, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'G',    
  start_residue_number:26, 
  end_residue_number:26, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'G', 
  start_residue_number:27, 
  end_residue_number:30, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'G', 
  start_residue_number:31, 
  end_residue_number:35, 
  color:{r:184,g:51,b:50}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/6CF2_3D1.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<p>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 6CF2. HIV-1 Rev93 protein (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with HIV-1 Rev93 protein. Due to the large number of interactions involved, we only show some of the more critical interactions here, and the complete content can be read in reference 12.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/HIV-1_Rev_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/HIV-1_Rev_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>



<p class="header_box" id="ligand-recognition">Ligand information</p>
  
<p class="blowheader_box">SELEX ligand</p>
<p>Wingfield, P. T. et al. determined the affinity of RBA-14 and Rev by surface plasmon resonance (SPR) method, and also compared the affinity of Rev with hairpins with Stem IIB sequences. SPR was performed on a Biacore ×100 (GE Healthcare), in 30 μL/minute HBS-EP+ buffer (10 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid, pH 7.4, 150 mM sodium chloride, 3 mM ethylenediaminetetraacetic acid, 0.05% polysorbate 20) (GE Healthcare) at 25°C<sup>[<a href="#ref12" style="color:#520049">12</a>]</sup>.</p>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">RBA-14 aptamer</td>
        <td name="td1">5'-GGCUGGACUCGUACUUCGGUACUGGAGAAACAGCC-3'</td>
        <td name="td2">HIV-1 Rev<sup>93</sup> protein</td>
        <td name="td3">5.9 nM</td>
      </tr>
      <tr>
        <td name="td0">Stem IIB aptamer</td>
        <td name="td1">5'-CCUGGGCGCAGUGUCAUUGACGCUGACGGUACAGG-3'</td>
        <td name="td2">HIV-1 Rev<sup>93</sup> protein</td>
        <td name="td3">12.9 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<p>REV is a viral anti-repression trans-activator protein, which appears to act post-transcriptionally to relieve negative repression of GAG and ENV production. It is a phosphoprotein whose state of phosphorylation is mediated by a specific serine kinase activity present in the nucleus. REV accumulates in the nucleoli.-----From <a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00424/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">Mass</th>
        <th onclick="sortTable(4)">Protein sequence</th>
        <th onclick="sortTable(5)">PDB ID</th>
        <th onclick="sortTable(6)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P04616/entry#function" target="_blank" style="color:#520049"><b>P04616</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00424/" target="_blank" style="color:#520049"><b>PF00424</b></a></td>
        <td name="td3">3.22 kDa</td>
        <td name="td4">GAMATRQARRNRRRRWRERQRAAAAR (residues T34-R50)</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/2M1A" target="_blank" style="color:#520049"><b>2M1A</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/protein/AAA44200.1" target="_blank" style="color:#520049"><b>AAA44200.1</b></a></td>
      </tr>
	  </tbody>
  </table>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/HIV-1_Rev_aptamer_stru_ligand.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>



<p class="blowheader_box">Similar compound</p>
<p>We used the RCSB PDB website's similar structure search to find the top 10 structures similar to HIV-1 REV PROTEIN (residues T34-R50), and calculated TM-socre values and RMSD values using the TM-align website.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://zhanggroup.org/TM-align/" target="_blank" style="color:#520049; text-decoration: underline;"><b>TM-align</b></a>:</b> an algorithm for sequence independent protein structure comparisons.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">TM-socre</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">2LD2</td>
        <td name="td2">0.358</td>
        <td name="td3">0.89</td>
        <td name="td4">Solution structure of the N-terminal domain of huntingtin (htt17) in presence of DPC micelles</td>
      </tr>
      <tr>
        <td name="td1">1DNG</td>
        <td name="td2">0.442</td>
        <td name="td3">1.68</td>
        <td name="td4">NMR structure of a model hydrophilic amphipathic helical acidic peptide</td>
      </tr>
      <tr>
        <td name="td1">3N95-E</td>
        <td name="td2">0.034</td>
        <td name="td3">1.7</td>
        <td name="td4">Crystal structure of human CRFR2 alpha extracellular domain in complex with Urocortin 2</td>
      </tr>
      <tr>
        <td name="td1">1OMQ</td>
        <td name="td2">0.366</td>
        <td name="td3">1.1</td>
        <td name="td4">Structure of penetratin in bicellar solution</td>
      </tr>
      <tr>
        <td name="td1">7LSO</td>
        <td name="td2">0.44</td>
        <td name="td3">0.67</td>
        <td name="td4">L-Phenylseptin</td>
      </tr>
      <tr>
        <td name="td1">2RLG</td>
        <td name="td2">0.34</td>
        <td name="td3">0.79</td>
        <td name="td4">NMR structure of the antimicrobial peptide RP-1 bound to SDS micelles</td>
      </tr>
      <tr>
        <td name="td1">3N93-E</td>
        <td name="td2">0.03</td>
        <td name="td3">1.62</td>
        <td name="td4">Crystal structure of human CRFR2 alpha extracellular domain in complex with Urocortin 3</td>
      </tr>
      <tr>
        <td name="td1">7LSP</td>
        <td name="td2">0.465</td>
        <td name="td3">0.56</td>
        <td name="td4">D-Phenylseptin - The second residue of PHE of the peptide is a D-amino acid</td>
      </tr>
      <tr>
        <td name="td1">1O53</td>
        <td name="td2">0.371</td>
        <td name="td3">0.49</td>
        <td name="td4">Solution structure of the N-terminal membrane anchor of E. coli enzyme IIA(Glucose)</td>
      </tr>
      <tr>
        <td name="td1">6GS9</td>
        <td name="td2">0.303</td>
        <td name="td3">1.36</td>
        <td name="td4">NMR structure of aurein 2.5 in SDS micelles</td>
      </tr>
	  </tbody>
  </table>
  <br>
  <br>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Selective optimization of the Rev-binding element of HIV-1.</strong></font><br />
Giver, L., Bartel, D., Zapp, M., Pawul, A., Green, M., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/7505429/" target="_blank" style="color:#520049" >Nucleic acids research, 21(23), 5509–5516. (1993)</a>
<br/>

<a id="ref2"></a><font><strong>[2] Selection and design of high-affinity RNA ligands for HIV-1 Rev.</strong></font><br />
Giver, L., Bartel, D. P., Zapp, M. L., Green, M. R., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/7506689/" target="_blank" style="color:#520049" >Gene, 137(1), 19–24. (1993)</a>
<br/>

<a id="ref3"></a><font><strong>[3] A three-dimensional model of the Rev-binding element of HIV-1 derived from analyses of aptamers.</strong></font><br />
Leclerc, F., Cedergren, R., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/7664035/" target="_blank" style="color:#520049" >Nature structural biology, 1(5), 293–300. (1994)</a>
<br/>

<a id="ref4"></a><font><strong>[4] Characterization of an in vitro-selected RNA ligand to the HIV-1 Rev protein.</strong></font><br />
Jensen, K. B., Green, L., MacDougal-Waugh, S., & Tuerk, C.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8289245/" target="_blank" style="color:#520049" >Journal of molecular biology, 235(1), 237–247. (1994)</a>
<br/>

<a id="ref5"></a><font><strong>[5] RNA aptamers selected to bind human immunodeficiency virus type 1 Rev in vitro are Rev responsive in vivo.</strong></font><br />
Symensma, T. L., Giver, L., Zapp, M., Takle, G. B., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8523524/" target="_blank" style="color:#520049" >Journal of virology, 70(1), 179–187. (1996)</a>
<br/>

<a id="ref6"></a><font><strong>[6] Deep penetration of an alpha-helix into a widened RNA major groove in the HIV-1 rev peptide-RNA aptamer complex.</strong></font><br />
Ye, X., Gorin, A., Ellington, A. D., & Patel, D. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8946856/" target="_blank" style="color:#520049" >Nature structural biology, 3(12), 1026–1033. (1996)</a>
<br/>

<a id="ref7"></a><font><strong>[7] Receptor ligand-facilitated cationic liposome delivery of anti-HIV-1 Rev-binding aptamer and ribozyme DNAs.</strong></font><br />
Konopka, K., Düzgüneş, N., Rossi, J., & Lee, N. S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/9713975/" target="_blank" style="color:#520049" >Journal of drug targeting, 5(4), 247–259. (1998)</a>
<br/>

<a id="ref8"></a><font><strong>[8] Retention of conformational flexibility in HIV-1 Rev-RNA complexes.</strong></font><br />
Wilkinson, T. A., Zhu, L., Hu, W., & Chen, Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/15610009/" target="_blank" style="color:#520049" >Biochemistry, 43(51), 16153–16160. (2004)</a>
<br/>

<a id="ref9"></a><font><strong>[9] A solution to limited genomic capacity: using adaptable binding surfaces to assemble the functional HIV Rev oligomer on RNA.</strong></font><br />
Daugherty, M. D., D'Orso, I., & Frankel, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18922466/" target="_blank" style="color:#520049" >Molecular cell, 31(6), 824–834. (2008)</a>
<br/>

<a id="ref10"></a><font><strong>[10] The arginine-rich RNA-binding motif of HIV-1 Rev is intrinsically disordered and folds upon RRE binding.</strong></font><br />
Casu, F., Duggan, B. M., & Hennig, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23972852/" target="_blank" style="color:#520049" >Biophysical journal, 105(4), 1004–1017. (2013)</a>
<br/>

<a id="ref11"></a><font><strong>[11] RNA-directed remodeling of the HIV-1 protein Rev orchestrates assembly of the Rev-Rev response element complex.</strong></font><br />
Jayaraman, B., Crosby, D. C., Homer, C., Ribeiro, I., Mavor, D., & Frankel, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/25486594/" target="_blank" style="color:#520049" >eLife, 3, e04120. (2014)</a>
<br/>

<a id="ref12"></a><font><strong>[12] Structure of an RNA Aptamer that Can Inhibit HIV-1 by Blocking Rev-Cognate RNA (RRE) Binding and Rev-Rev Association.</strong></font><br />
Dearborn, A. D., Eren, E., Watts, N. R., Palmer, I. W., Kaufman, J. D., Steven, A. C., & Wingfield, P. T.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/30017564/" target="_blank" style="color:#520049" >Structure (London, England : 1993), 26(9), 1187–1195.e4. (2018)</a>
<br/>
