---
layout: theophylline_style
title:  "HIV-1 CA aptamer"
date:   2024-05-22 00:00:00
author: Jiali Wang, Bo Fu
categories: Aptamer
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="description">Description</p>
<p>In 2022, Gruenke PR, Aneja R, Welbourn S, et al. selexed for an RNA aptamer called CA15-2. The function of this aptamer is to specifically bind to the capsid lattice structure of HIV-1 and inhibit virus replication. Research has shown that CA15-2 selectively binds to the CA lattice rather than the CA monomer or CA hexamer, indicating that it may target an interface that exists only on the CA lattice and is easily accessible. In addition, the expression of CA15-2 in virus producing cells can inhibit the replication of HIV-1, but not in target cells, indicating that it may bind to a biologically relevant and accessible site during virus production, which may not be reachable in the subsequent replication step or remain unchanged after binding to the aptamer. Importantly, CA15-2 is the first identified aptamer that can specifically recognize the HIV-1 CA lattice<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2022, Lange, M. J et al. uesd the RNA library began with a randomised region consisting of 56 nucleotides (denoted as 56N), which was estimated to encompass approximately 10<sub>15</sub> unique sequences. Through a meticulous screening process spanning 15 rounds, the CA15-2 RNA aptamer, capable of specifically binding to the CA lattice, was successfully identified. Subsequent analysis via Sanger sequencing technology revealed multiple cloned sequences, with CA15-2 being among the most prominent<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>


<p class="header_box" id="Structure">Structure</p>
<p>The 2D structure of the figure is based on the article by ribodraw tool to draw. CA15-2 aptamer aptamer binds to HIV-1 capsid (CA) protein. CA15-2 aptamer was named by Lange, M. J<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>5'-GGAAGAAGAGAAUCAUACACAAGAUCGACGUACCUCAGGGUGGUGUAUGACUGAGGUGAAGACUGUGAACCAUGGCAUGCGGGCAUAAGGUAGGUAAGUCCAUA-3'</p>
<img src="/images/2D/CA15_2_aptamer_2D1.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>

<p class="header_box" id="ligand-recognition">Ligand information</p>

<p class="blowheader_box">SELEX ligand</p>
<p>Some isolated sequences bind to the affinity of the protein<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">CA15-2</td>
        <td name="td1">5'-GGAAGAAGAGAAUCAUACACAAGAUCGACGUACCUCAGGGUGGUGUAUGACUGAGGUGAAGACUGUGAACCAUGGCAUGCGGGCAUAAGGUAGGUAAGUCCAUA-3'</td>
        <td name="td2">HIV-1 CA lattice</td>
        <td name="td3">350 ± 50 nM (using MST at a CA protein concentration of 0.1 - 1000 nM),<br>
502 ± 57 nM (using nitrocellulose filter binding assays at a CA protein concentration of 10-7500 nM)</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/HIV_1_CA_SELEX_ligand.svg" alt="drawing" style="width:1000px;height:350px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>


<p class="blowheader_box">Structure ligand</p>
<p>Coronavirus (CoV) nucleocapsid (N) proteins have 3 highly conserved domains. The N-terminal domain (NTD) (N1b), the C-terminal domain (CTD)(N2b) and the N3 region. The N1b and N2b domains from SARS CoV, infectious bronchitis virus (IBV), human CoV 229E and mouse hepatic virus (MHV) display similar topological organisations. N proteins form dimers, which are asymmetrically arranged into octamers via their N2b domains. Domains N1b and N2b are linked by another domain N2a that contains an SR-rich region in which phosphorylation of specific serine residues allows the N protein to associate with the RNA helicase DDX1 permitting template read-through, and enabling the transition from discontinuous transcription of subgenomic mRNAs (sgmRNAs) to continuous synthesis of longer sgmRNAs and genomic RNA (gRNA). It has been shown that N proteins interact with nonstructural protein 3 (NSP3) and thus are recruited to the replication-transcription complexes (RTCs).-----From <a href="https://www.uniprot.org/uniprotkb/P12493/entry" target="_blank" style="color:#520049; text-decoration: underline;"><b>Uniprot</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">Mass</th>
        <th onclick="sortTable(4)">Protein sequence</th>
        <th onclick="sortTable(5)">PDB ID</th>
        <th onclick="sortTable(6)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">HIV-1 capsid (CA)</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P12493/entry" target="_blank" style="color:#520049"><b>P12493</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR045345/" target="_blank" style="color:#520049"><b>IPR045345</b></a></td>
        <td name="td3">24.345 kDa</td>
        <td name="td4">
        <div class="sequence-container">
          <span class="sequence-text"></span>
          <span class="show-more" onclick="toggleSequence(event)">......</span>
          <span class="full-sequence">PIVQNLQGQMVHQAISPRTLNAWVKVVEEKAFSPEVIPMFSALSEGATPQDLNTMLNTVGGHQAAMQMLKETINEEAAEWDRLHPVHAGPIAPGQMREPRGSDIAGTTSTLQEQIGWMTHNPPIPVGEIYKRWIILGLNKIVRMYSPTSILDIRQGPKEPFRDYVDRFYKTLRAEQASQEVKNWMTETLLVQNANPDCKTILKALGPGATLEEMMTAC</span>
        </div>
        </td>
        <td name="td5"><a href="https://www.rcsb.org/structure/8FIU" target="_blank" style="color:#520049"><b>8FIU</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/nuccore/M19921" target="_blank" style="color:#520049"><b>M19921</b></a></td>
      </tr>
	  </tbody>
  </table>


<p class="blowheader_box">Similar compound(s)</p>                    
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD (Å)</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/8FIU" target="_blank" style="color:#520049"><b>8FIU</b></a>-A</td>
        <td name="td2">32.0</td>
        <td name="td3">0</td>
        <td name="td4">HIV-1 CAPSID</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/8FIU" target="_blank" style="color:#520049"><b>8FIU</b></a>-C</td>
        <td name="td2">30.4</td>
        <td name="td3">0.5</td>
        <td name="td4">HIV-1 CAPSID</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/7RMJ" target="_blank" style="color:#520049"><b>7RMJ</b></a>-A</td>
        <td name="td2">29.4</td>
        <td name="td3">1.3</td>
        <td name="td4">CAPSID protein P24</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/7RJ4" target="_blank" style="color:#520049"><b>7RJ4</b></a>-A</td>
        <td name="td2">29.3</td>
        <td name="td3">1.3</td>
        <td name="td4">CAPSID protein P24</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/6VKV" target="_blank" style="color:#520049"><b>6VKV</b></a>-A</td>
        <td name="td2">29.2</td>
        <td name="td3">1.3</td>
        <td name="td4">CAPSID protein P24</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/7RMJ" target="_blank" style="color:#520049"><b>7RMJ</b></a>-C</td>
        <td name="td2">29.1</td>
        <td name="td3">1.3</td>
        <td name="td4">CAPSID protein P24</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/7RJ4" target="_blank" style="color:#520049"><b>7RJ4</b></a>-C</td>
        <td name="td2">29.1</td>
        <td name="td3">1.2</td>
        <td name="td4">CAPSID protein P24</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/6VKV" target="_blank" style="color:#520049"><b>6VKV</b></a>-C</td>
        <td name="td2">29.1</td>
        <td name="td3">1.2</td>
        <td name="td4">CAPSID protein P24</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/7RAO" target="_blank" style="color:#520049"><b>7RAO</b></a>-A</td>
        <td name="td2">29.0</td>
        <td name="td3">1.4</td>
        <td name="td4">CAPSID protein P24</td>
      </tr>
      <tr>
        <td name="td1"><a href="https://www.rcsb.org/structure/7RMM" target="_blank" style="color:#520049"><b>7RMM</b></a>-A</td>
        <td name="td2">28.9</td>
        <td name="td3">0.7</td>
        <td name="td4">CAPSID protein P24</td>
      </tr>
    </tbody>
  </table>

                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Selection and identification of an RNA aptamer that specifically binds the HIV-1 capsid lattice and inhibits viral replication.</strong></font><br />
Gruenke, P. R., Aneja, R., Welbourn, S., Ukah, O. B., Sarafianos, S. G., Burke, D. H., & Lange, M. J. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/35018437/" target="_blank" style="color:#520049" >Nucleic acids research, 50(3), 1701–1717. (2022)</a>
<br/>


<script>
    function toggleSequence(event) {
      const container = event.target.closest('.sequence-container');
      container.classList.toggle('expanded');
      const showMoreText = container.querySelector('.show-more');
      
      // 展开后按钮文本变化
      if (container.classList.contains('expanded')) {
        showMoreText.textContent = '...';  // 展开后显示 "..."
      } else {
        showMoreText.textContent = '......';  // 收起后显示 "......"
      }
    }

    // 页面加载时，限制序列文本为50个字符
    window.addEventListener('load', function() {
      const sequenceContainers = document.querySelectorAll('.sequence-container');
      sequenceContainers.forEach(container => {
        const fullSeqText = container.querySelector('.full-sequence').textContent;
        const truncatedText = fullSeqText.slice(0, 20);  // 只显示前50个字符
        container.querySelector('.sequence-text').textContent = truncatedText;
      });
    });
  </script>