---
layout: theophylline_style
title: HIV-1 Tat protein_Clone 31 aptamer
date: 2024-05-06 00:00:00
author: Ke Chen, Yangyi Ren
categories: Aptamer
tags:
- Category:Viral
- GC:50.0
- Length:78
- Named:HIV-1 Tat protein_clone 31
- Type:Proteins
- Year:2000
type: Proteins
---


<html>



<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8841594/" target="_blank" style="color:#520049">1995</a></h3>
  </div>
  <div class="body">
    <p>Starting from a RNA pool that has a 120 base random sequence core, aptamers that bind specifically to the Tat protein were selected by repeating 11 rounds of selection and amplification<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10886365/" target="_blank" style="color:#520049">2000</a></h3>
  </div>
  <div class="body">
    <p>Rika Yamamoto et al. used an in vitro selection method and isolated a novel aptamer RNATat, a 37-mer RNA oligomer, that binds efficiently to the Tat protein of HIV-1<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/11132628/" target="_blank" style="color:#520049">2000</a></h3>
  </div>
  <div class="body">
    <p>Junji Kawakami et al. used Zn<sup>2+</sup> for in vitro selecting to isolate novel RNA molecules that bind to the HIV-1 Tat protein, known as zinc-dependent aptamers<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12737819/" target="_blank" style="color:#520049">2003</a></h3>
  </div>
  <div class="body">
    <p>Akimasa Matsugami et al. determined the structure of an aptamer complexed with two argininamide molecules, two adjacent U:A:U base triples were formed<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/28620664/" target="_blank" style="color:#520049">2017</a></h3>
  </div>
  <div class="body">
    <p>Yudai Yamaoki et al. proposed an RNA aptamer that acquires binding capacity against HIV-1 Tat protein via G-quadruplex formation in response to potassium ions<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/31707021/" target="_blank" style="color:#520049">2020</a></h3>
  </div>
  <div class="body">
    <p>Mustafa Oguzhan Caglayan et al. reported the HIV-Tat protein detection performance of antiTat aptamers both for the spectroscopic ellipsometry (SE) and for the surface plasmon resonance enhanced total internal reflection ellipsometry (SPReTIRE) for the first time<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/37240414/" target="_blank" style="color:#520049">2023</a></h3>
  </div>
  <div class="body">
    <p>Using in vitro nuclear magnetic resonance (NMR), Omar Eladl et al. found that two U-A∗U base triples are formed in aptamer upon binding of Tat<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
  </div>
 </div>
</div>





<p class="header_box" id="description">Description</p>
<p>In 2000, Junji Kawakami and colleagues isolated RNA aptamers that bind specifically to the HIV-1 Tat protein. It was worth mentioning that they used Zn<sup>2+</sup> to participate in in vitro selecting and obtained a series of zinc-dependent aptamers<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2000, Junji Kawakam and colleagues prepared an RNA library having a 30 nucleotide random region and applied to an in vitro RNA aptamer selection with Zn<sup>2+</sup>. After 12 rounds of selecting, many unique sequences were revealed from a library selected with Zn<sup>2+</sup> and the RNA with most abundant sequence (clone 31) bound to Tat tightly only when Zn<sup>2+</sup> existed<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>Clone 31 was the aptamer sequence mainly studied in the article, which had a high affinity with HIV-1 Tat protein. The 2D structure of the figures is based on the article by ribodraw tool to draw. These are two different prediction results in the article<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.<br></p>
<p>5'-GGGAGAAUUCCGACCAGAAGCUUUGGUUAUCAUGUUUAUGCGUACGGGCGCCCAUAUGUGCGUCUACAUGGAUCCUCA-3'</p>
 <table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/clone 31_aptamer_2D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/clone31 _aptamer_2D2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>



<p class="header_box" id="ligand-information">Ligand information</p>  
<p class="blowheader_box">SELEX ligand</p>
<font>The retroviral Tat protein binds to the TAR RNA. This activates transcriptional initiation and elongation from the LTR promoter. Binding is mediated by an arginine rich region.-----From Pfam</font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">HIV-1 Tat protein</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P04608/entry" target="_blank" style="color:#520049"><b>P04608</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00539/" target="_blank" style="color:#520049"><b>PF00539</b></a></td>
        <td name="td3">9.8 kDa</td>
        <td name="td4">MEPVDPRLEPWKHPGSQPKTACTNCYCKKCCFHCQVCFITKALGISYGRKKRRQRRRAHQNSQTHQASLSKQPTSQPRGDPTGPKE</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/6CYT" target="_blank" style="color:#520049"><b>6CYT</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/155871" target="_blank" style="color:#520049"><b>155871</b></a></td>
      </tr>
	  </tbody>
  </table>
<font>Some isolated sequences bind to the affinity of the protein.</font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">clone 31</td>
        <td name="td1">GGGAGAAUUCCGACCAGAAGCUUUGGUUAUCAUGUUUAUGCGUACGGGCGCCCAUAUGUGCGUCUACAUGGAUCCUCA</td>
        <td name="td2">HIV-1 Tat protein</td>
        <td name="td3">0.31 μM</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/clone31_aptamer_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>



<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD (Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
       <tr>
      <td name="td0">3MI9-C</td>
      <td name="td1">9.7</td>
      <td name="td2">0.0</td>
      <td name="td3">Cell division protein kinase 9</td>
    </tr>
     <tr>
      <td name="td0">3MIA-C</td>
      <td name="td1">8.9</td>
      <td name="td2">0.2</td>
      <td name="td3">Cell division protein kinase 9</td>
    </tr>
     <tr>
      <td name="td0">4OR5-C</td>
      <td name="td1">8.6</td>
      <td name="td2">0.4</td>
      <td name="td3">Cyclin-dependent kinase 9</td>
    </tr>
     <tr>
      <td name="td0">4OGR-D</td>
      <td name="td1">8.6</td>
      <td name="td2">0.5</td>
      <td name="td3">Cyclin-dependent kinase 9</td>
    </tr>
     <tr>
      <td name="td0">4OR5-H</td>
      <td name="td1">8.6</td>
      <td name="td2">0.3</td>
      <td name="td3">Cyclin-dependent kinase 9</td>
    </tr>
     <tr>
      <td name="td0">4OGR-H</td>
      <td name="td1">8.6</td>
      <td name="td2">0.8</td>
      <td name="td3">Cyclin-dependent kinase 9</td>
    </tr>
     <tr>
      <td name="td0">4OGR-M</td>
      <td name="td1">8.5</td>
      <td name="td2">0.5</td>
      <td name="td3">Cyclin-dependent kinase 9</td>
    </tr>
     <tr>
      <td name="td0">6CYT-D</td>
      <td name="td1">8.4</td>
      <td name="td2">0.5</td>
      <td name="td3">Cyclin-dependent kinase 9</td>
    </tr>
     <tr>
      <td name="td0">5L1Z-D</td>
      <td name="td1">8.1</td>
      <td name="td2">0.9</td>
      <td name="td3">Cyclin-dependent kinase 9</td>
    </tr>
     <tr>
      <td name="td0">6GS9</td>
      <td name="td1">0.303</td>
      <td name="td2">1.36</td>
      <td name="td3">NMR structure of aurein 2.5 in SDS micelles</td>
    </tr>
    </tbody>
  </table>



        
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] In vitro selection of RNA aptamers that can bind specifically to Tat protein of HIV-1.</strong></font><br />
Yamamoto, R., Toyoda, S., Viljanen, P., Machida, K., Nishikawa, S., Murakami, K., Taira, K., & Kumar, P. K.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8841594/" target="_blank" style="color:#520049">Nucleic acids symposium series, (34), 145–146. (1995)</a>
<br/>
            
<a id="ref2"></a><font><strong>[2] A novel RNA motif that binds efficiently and specifically to the Ttat protein of HIV and inhibits the trans-activation by Tat of transcription in vitro and in vivo.</strong></font><br />
Yamamoto, R., Katahira, M., Nishikawa, S., Baba, T., Taira, K., & Kumar, P. K.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10886365/" target="_blank" style="color:#520049">Genes to cells : devoted to molecular & cellular mechanisms, 5(5), 371–388. (2000)</a>
<br/>
            
<a id="ref3"></a><font><strong>[3]  In vitro selection of aptamers that act with Zn<sup>2+</sup>.</strong></font><br />
Kawakami, J., Imanaka, H., Yokota, Y., & Sugimoto, N.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/11132628/" target="_blank" style="color:#520049">Journal of inorganic biochemistry, 82(1-4), 197–206. (2000)</a>
<br/>
            
<a id="ref4"></a><font><strong>[4] Structural basis of the highly efficient trapping of the HIV Tat protein by an RNA aptamer.</strong></font><br />
Matsugami, A., Kobayashi, S., Ouhashi, K., Uesugi, S., Yamamoto, R., Taira, K., Nishikawa, S., Kumar, P. K., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12737819/" target="_blank" style="color:#520049">Structure (London, England : 1993), 11(5), 533–545. (2003)</a>
<br/>
            
<a id="ref5"></a><font><strong>[5] Development of an RNA aptamer that acquires binding capacity against HIV-1 Tat protein via G-quadruplex formation in response to potassium ions.</strong></font><br />
Yamaoki, Y., Nagata, T., Mashima, T., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/28620664/" target="_blank" style="color:#520049">Chemical communications (Cambridge, England), 53(52), 7056–7059. (2017)</a>
<br/>
            
<a id="ref6"></a><font><strong>[6] Spectrophotometric ellipsometry based Tat-protein RNA-aptasensor for HIV-1 diagnosis.</strong></font><br />
Caglayan, M. O., & Üstündağ, Z.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/31707021/" target="_blank" style="color:#520049">Spectrochimica acta. Part A, Molecular and biomolecular spectroscopy, 227, 117748. (2020)</a>
<br/>
            
<a id="ref7"></a><font><strong>[7] Complex Formation of an RNA Aptamer with a Part of HIV-1 Tat through Induction of Base Triples in Living Human Cells Proven by In-Cell NMR.</strong></font><br />
Eladl, O., Yamaoki, Y., Kondo, K., Nagata, T., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/37240414/" target="_blank" style="color:#520049">International journal of molecular sciences, 24(10), 9069. (2023)</a>
<br/>


<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1RAW-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>
    </html>
