---
layout: theophylline_style
title: HIV-1 REV peptide apatmer
date: 2024-04-08 00:00:00
author: Zhizhong Lu, Ziyu Guo
categories: Aptamer
tags:
- Category:Viral
- GC:56.99999999999999
- Length:35
- Named:HIV-1 REV peptide_35-mer HIV rev RNA aptamer
- Type:Proteins
- Year:1994
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
<div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/7505429/" target="_blank" style="color:#520049">1993</a></h3>
    </div>
    <div class="body">
      <p>RNA molecules that can bind to the Rev protein of HIV-1 have been isolated firstly<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8289245/" target="_blank" style="color:#520049">1994</a></h3>
    </div>
    <div class="body">
      <p>Jensen KB and Green L produced small RNA ligands with high affinity for HIV-1 Rev protein by SELEX in vitro evolution method for a series of chemical modification studies<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8946856/" target="_blank" style="color:#520049">1996</a></h3>
    </div>
    <div class="body">
      <p>The solution structure of the 35-mer high affinity RNA aptamer binding site of HIV-1 17-mer rev peptide was determined by NMR molecular dynamics<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10467126/" target="_blank" style="color:#520049">1999</a></h3>
    </div>
    <div class="body">
      <p>The study showed that the RNA structure determines whether the same HIV-1 Rev peptide folds into an extended conformation or an alpha-helical conformation at the time of complex formation<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10647177/" target="_blank" style="color:#520049">1999</a></h3>
    </div>
    <div class="body">
      <p>The solution structure of HTLV-1 arginine-rich Rex peptide bound to its RNA aptamer target as determined by multidimensional heteronuclear NMR was demonstrated<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div> <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/18922466/" target="_blank" style="color:#520049">2008</a></h3>
    </div>
    <div class="body">
      <p>It was demonstrated that HIV Rev, which is limited by the limited viral genome, can also form organized RNP by assembling homologous oligomers on Rev reaction element (RRE) RNA<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/23972852/" target="_blank" style="color:#520049">2013</a></h3>
    </div>
    <div class="body">
      <p>The arginine-rich RNA-binding motif of HIV-1 Rev is intrinsically disordered and folds upon RRE binding<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/25486594/" target="_blank" style="color:#520049">2014</a></h3>
    </div>
    <div class="body">
      <p>Rev-RNA recognition relies on well-characterized sequence-specific contact at the IIB site and local RNA structure at the second site. The structure supports a model in which RRE utilizes the inherent plasticity of the Rev subunit interface to guide the formation of functional complexes<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup></p>
    </div>
  </div>
</div>


         
<p class="header_box" id="description">Description</p>
<p>In 1993, A D Ellington. et al. RNA molecules that can bind to the Rev protein of HIV-1 have been isolated from random sequence nucleic acid pools based on a minimal Rev-binding element (RBE) found within the Rev Responsive Element (RRE). In 1994, Tuerk C. et al. produced small RNA ligands with high affinity for HIV-1 Rev protein by SELEX in vitro evolution method for a series of chemical modification studies. In 1996, Dinshaw J. Patel et al. combined NMR molecular dynamics method have been applied to determine the solution structure of the HIV-1 17-mer rev peptide binding to its 35-mer high affinity RNA aptamer binding site<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref2" style="color:#520049">2</a></sup><sup>,<a href="#ref3" style="color:#520049">3</a></sup><sup>]</sup>.<br></p>



<p class="header_box" id="SELEX">SELEX</p>
<p>In 1993, A D Ellington. et al. designed random sequence library to construct 76.6 pool. Sequencing selected clones PCR DNA (1 μg) was isolated following round 4 of the 76.6 pool selection and selected Rev aptamers (binding sequences; 29) from RBE pools containing limited (10-18 nt) random sequence tracts. In 1994, Tuerk C. et al. produced small RNA ligands with high affinity for HIV-1 Rev protein by SELEX in vitro evolution method for a series of chemical modification studies<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref2" style="color:#520049">2</a></sup><sup>]</sup>.</p>
<p>



<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>35-mer HIV-1 rev RNA aptamer was the aptamer sequence mainly studied in structure article. The orange represents the bases that interact with the protein. Here we use ribodraw to complete the figure, through the 3D structure information<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.</p>
<p>5'-GGCUGGACUCGUACUUCGGUACUGGAGAAACAGCC-3'</p>
<img src="/images/2D/Rev_aptamer_2D.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">3D visualisation</p>             
<p>Dinshaw J. Patel et al. determined the solution structure of the 35-mer high-affinity RNA aptamer binding site of HIV-1 17-mer rev peptide by NMR molecular dynamics. Complex formation involves adaptive binding with the alpha-helical arginine-rich basic rev peptide targeting a widened RNA major groove centred about adjacent G.A and reversed A.A mismatches. The PDB ID of this structure is 1ULL<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.<br>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/Rev_RNA_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
struct_asym_id: 'A',
start_residue_number:1,
end_residue_number:4,
color:{r:184,g:51,b:50}
},{
struct_asym_id: 'A',
start_residue_number:32,
end_residue_number:35,
color:{r:184,g:51,b:50}
},{
struct_asym_id: 'A',
start_residue_number:5,
end_residue_number:5,
color:{r:46,g:161,b:3}
},{
struct_asym_id: 'A',
start_residue_number:7,
end_residue_number:10,
color:{r:46,g:161,b:3}
},{
struct_asym_id: 'A',
start_residue_number:23,
end_residue_number:24,
color:{r:46,g:161,b:3}
},{
struct_asym_id: 'A',
start_residue_number:26,
end_residue_number:26,
color:{r:46,g:161,b:3}
},{
struct_asym_id: 'A',
start_residue_number:28,
end_residue_number:29,
color:{r:46,g:161,b:3}
},{
struct_asym_id: 'A',
start_residue_number:31,
end_residue_number:31,
color:{r:46,g:161,b:3}
},{
struct_asym_id: 'A',
start_residue_number:6,
end_residue_number:6,
color:{r:241,g:125,b:7}
},{
struct_asym_id: 'A',
start_residue_number:25,
end_residue_number:25,
color:{r:241,g:125,b:7}
},{
struct_asym_id: 'A',
start_residue_number:27,
end_residue_number:27,
color:{r:241,g:125,b:7}
},{
struct_asym_id: 'A',
start_residue_number:30,
end_residue_number:30,
color:{r:241,g:125,b:7}
},{
struct_asym_id: 'A',
start_residue_number:11,
end_residue_number:22,
color:{r:8,g:69,b:149}
},
{
struct_asym_id: 'B',
start_residue_number:1,
end_residue_number:17,
color:{r:133,g:39,b:181}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1ull.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<p>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 1ULL by NMR. HIV-1 REV peptide (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with HIV-1 REV peptide.</p>
<br><br>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Rev_RNA_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Rev_RNA_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>




<p class="header_box" id="ligand-recognition">Ligand information</p>
  
<p class="blowheader_box">SELEX ligand</p>
<p>Binding dissociation constants for RNA ligands were measured using a nitroceHulnse filter binding assay. The binding solution is flushed with 5ml of binding buffer through the pre-wet nitrocellulose filter. The RNA is eluted from the membrane and the remaining modification sites are detected. The modified RNA is also labeled on a filter and elutes to check for uniform recovery of the modified RNA. 35-mer HIV-1 rev RNA aptamer is a mutated variant of 6a aptamer<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p>
<br><br>
<div style="display: flex; justify-content: center;">
<br>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">6a aptamer</td>
        <td name="td1">5'-GGGUGCAUUGAGAAACACGUUUGUGGACUCUGUAUCU-3'</td>
        <td name="td3">HIV-1 REV peptide</td>
        <td name="td4">1 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<p>The HIV-1 Rev response element (RRE) is a highly structured, ~350 nucleotide RNA segment present in the Env coding region of unspliced and partially spliced viral mRNAs. In the presence of the HIV-1 accessory protein Rev, HIV-1 mRNAs that contain the RRE can be exported from the nucleus to the cytoplasm for downstream events such as translation and virion packaging.-----From <a href="https://en.wikipedia.org/wiki/Rev_(HIV)" target="_blank" style="color:#520049; text-decoration: underline;"><b>Wiki</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">Mass</th>
        <th onclick="sortTable(3)">Protein sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P05866/entry" target="_blank" style="color:#520049"><b>P05866</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00424/" target="_blank" style="color:#520049"><b>PF00424</b></a></td>
        <td name="td2">10.13 kDa</td>
        <td name="td3">PPPNPEGTRQARRNRRRRWRERQRQIRSIGAWILSTYLGRSAEPVPLQLPPLERLTLDCEEDCGTSGTQGVGSPQVLVESPAVLEQGTKE</td>
        <td name="td4"><a href="https://www.rcsb.org/structure/2M1A" target="_blank" style="color:#520049"><b>2M1A</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/protein/AAB12992.1" target="_blank" style="color:#520049"><b>AAB12992.1</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
<img src="/images/Structure_ligand/Rev_aptamer_stru_ligand1.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). The TM-score is a metric used to assess the similarity between two protein structures, providing a quantitative measure of their structural alignment, with higher scores indicating greater similarity. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">TM-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">1O53</td>
        <td name="td2">0.371</td>
        <td name="td3">0.49</td>
        <td name="td4">Solution structure of the N-terminal membrane anchor of E. coli enzyme IIA(Glucose)</td>
      </tr>
      <tr>
        <td name="td1">7LSP</td>
        <td name="td2">0.465</td>
        <td name="td3">0.56</td>
        <td name="td4">D-Phenylseptin - The second residue of PHE of the peptide is a D-amino acid</td>
      </tr>
      <tr>
        <td name="td1">7LSO</td>
        <td name="td2">0.44</td>
        <td name="td3">0.67</td>
        <td name="td4">L-Phenylseptin</td>
      </tr>
      <tr>
        <td name="td1">2RLG</td>
        <td name="td2">0.34</td>
        <td name="td3">0.79</td>
        <td name="td4">NMR structure of the antimicrobial peptide RP-1 bound to SDS micelles</td>
      </tr>
      <tr>
        <td name="td1">2LD2</td>
        <td name="td2">0.358</td>
        <td name="td3">0.89</td>
        <td name="td4">Solution structure of the N-terminal domain of huntingtin (htt17) in presence of DPC micelles</td>
      </tr>
      <tr>
        <td name="td1">1OMQ</td>
        <td name="td2">0.366</td>
        <td name="td3">1.1</td>
        <td name="td4">Structure of penetratin in bicellar solution</td>
      </tr>
      <tr>
        <td name="td1">6GS9</td>
        <td name="td2">0.303</td>
        <td name="td3">1.36</td>
        <td name="td4">NMR structure of aurein 2.5 in SDS micelles</td>
      </tr>
      <tr>
        <td name="td1">3N93-E</td>
        <td name="td2">0.03</td>
        <td name="td3">1.62</td>
        <td name="td4">Crystal structure of human CRFR2 alpha extracellular domain in complex with Urocortin 3</td>
      </tr>
      <tr>
        <td name="td1">1DNG</td>
        <td name="td2">0.442</td>
        <td name="td3">1.68</td>
        <td name="td4">NMR structure of a model hydrophilic amphipathic helical acidic peptide</td>
      </tr>
      <tr>
        <td name="td1">3N95-E</td>
        <td name="td2">0.034</td>
        <td name="td3">1.7</td>
        <td name="td4">Crystal structure of human CRFR2 alpha extracellular domain in complex with Urocortin 2</td>
      </tr>
	  </tbody>
  </table>

                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Selective optimization of the Rev-binding element of HIV-1.</strong></font><br />
Giver, L., Bartel, D., Zapp, M., Pawul, A., Green, M., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/7505429/" target="_blank" style="color:#520049" >Nucleic acids research, 21(23), 5509–5516. (1993)</a>
<br />

<a id="ref2"></a><font><strong>[2] Characterization of an in vitro-selected RNA ligand to the HIV-1 Rev protein.</strong></font><br />
Jensen KB, Green L, MacDougal-Waugh S, Tuerk C.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8289245/" target="_blank" style="color:#520049" >J Mol Biol, 1994, 235(1), 237-247 (1994)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] Deep penetration of an alpha-helix into a widened RNA major groove in the HIV-1 rev peptide-RNA aptamer complex.</strong></font><br />
Ye X, Gorin A, Ellington AD, Patel DJ<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8946856/" target="_blank" style="color:#520049">Nat Struct Biol, 1996, 3(12):1026-1033. (1996)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] RNA architecture dictates the conformations of a bound peptide.</strong></font><br />
Ye, X., Gorin, A., Frederick, R., Hu, W., Majumdar, A., Xu, W., McLendon, G., Ellington, A., & Patel, D. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10467126/" target="_blank" style="color:#520049"> Chem Biol, 1999;6(9), 657-669 (1999)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Anchoring an extended HTLV-1 Rex peptide within an RNA major groove containing junctional base triples.</strong></font><br />
Jiang, F., Gorin, A., Hu, W., Majumdar, A., Baskerville, S., Xu, W., Ellington, A., & Patel, D. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10647177/" target="_blank" style="color:#520049">Structure, 1999,7(12),1461-1472 (1999)</a>
<br />
                
<a id="ref6"></a><font><strong>[6] A solution to limited genomic capacity: using adaptable binding surfaces to assemble the functional HIV Rev oligomer on RNA.</strong></font><br />
Daugherty MD, D'Orso I, Frankel AD.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18922466/" target="_blank" style="color:#520049">Mol Cell, 2008, 31(6), 824-834. (2008)</a>
<br />
                
<a id="ref7"></a><font><strong>[7] The arginine-rich RNA-binding motif of HIV-1 Rev is intrinsically disordered and folds upon RRE binding.</strong></font><br />
Casu F, Duggan BM, Hennig M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23972852/" target="_blank" style="color:#520049" >Biophys J, 2013, 105(4), 1004-1017 (2013)</a>
<br />

<a id="ref8"></a><font><strong>[8] RNA-directed remodeling of the HIV-1 protein Rev orchestrates assembly of the Rev-Rev response element complex.</strong></font><br />
Jayaraman B, Crosby DC, Homer C, Ribeiro I, Mavor D, Frankel AD.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/25486594/" target="_blank" style="color:#520049" >Elife, 2014,3:e04120. (2014)</a>
<br />