---
layout: theophylline_style
title:  "HIV-1 RT aptamer"
date:   2025-04-30 00:00:00
author: Ke Chen, Yangyi Ren
categories: Aptamer
---


<html>
<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
</ul>
</div>

<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/1379730/" target="_blank" style="color:#520049">1992</a></h3>
    </div>
    <div class="body">
      <p>High-affinity ligands of the reverse transcriptase of human immunodeficiency virus type 1 (HIV-1) were isolated by the SELEX procedure from RNA populations randomized at 32 positions.<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10751399/" target="_blank" style="color:#520049">2000</a></h3>
    </div>
    <div class="body">
      <p>RNA inhibitor bound to HIV RT in a two-step process, with association rates similar to those described for model DNA/DNA and DNA/RNA substrates.<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12050367/" target="_blank" style="color:#520049">2002</a></h3>
    </div>
    <div class="body">
      <p>With the help of RNA aptamers, some of the HIV subtypes were severely inhibited (subtypes A, B, D, E, and F), while others were either moderately inhibited (subtypes C and O) or were naturally resistant to inhibition (chimeric A/D subtype).<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/25073457/" target="_blank" style="color:#520049">2014</a></h3>
    </div>
    <div class="body">
      <p>RNA aptamers interacted with HIV-1 RT in a fingers-and-thumb-closed conformation, at the bound at the nucleic acid substrate binding site.<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/31943114/" target="_blank" style="color:#520049">2020</a></h3>
    </div>
    <div class="body">
      <p>Features at the RT-aptamer interface that govern recognition specificity by a broad-spectrum antiviral aptamer, and they open new possibilities for accelerating RT maturation and interfering with viral replication.<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/32732393/" target="_blank" style="color:#520049">2020</a></h3>
    </div>
    <div class="body">
      <p>2'-fluoro-modified pyrimidines enhance affinity of RNA aptamers to HIV-1 RT.<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
</div>


<p class="header_box" id="description">Description</p>
<p>In 1992, Gold, L. et al. used the SELEX method to isolate the aptamer with high affinity for the HIV-1 RT from RNA populations randomized at 32 positions. Analysis of these ligands revealed a pseudoknot consensus with primary sequence bias at some positions. We demonstrated that at least one of the ligands inhibits cDNA synthesis by HIV reverse transcriptase but fails to inhibit other reverse transcriptases. These experiments highlight the power of SELEX to yield highly specific ligands that reduce the activity of target proteins. Such ligands may provide therapeutic reagents for viral and other diseases.<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 1992, Gold, L. et al. isolated high affinity sequences for targeting HIV-1 RT from RNA populations randomized at 32 positions through 9 rounds of selection. Nine rounds of SELEX with each population were performed as described. first completed the experiment with RNA containing the anticodon stem and loop sequence of tRNA. Nitrocellulose filter binding assays performed at the ninth round revealed that the RNA population had increased ~200 - fold in affinity to HIV - 1 RT when compared to the starting population but that the background binding to nitrocellulose filters in the absence of protein had increased from <2% of input RNA to 15% . As a consequence, during SELEX experiment 2 the initial pool of RNAs was prefiltered through nitrocellulose before the first, third, sixth, and ninth rounds of selection. Individual isolates (49 of experiment 1 and 104 of experiment 2) were cloned from these populations and sequenced. Detailed information are accessible on SELEX page.<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>


<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>Ligand 1.1 was the aptamer sequence mainly studied in the article, which had a high affinity with HIV-1 RT. The 2D structure of the figure is based on the prediction results of the RNA fold website by ribodraw tool to draw. The ligand 1.1 aptamer was named by Gold, L. et al. in the article.<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>5'-GGGAGCAUCAGACUUUUAAUCUGACAAUCAAGAAUUCCGUUUUCAGUCGGGAAAAACUGAACAAUCUAUGAAAGAAUUUUAUAUCUCUAUUGAAAC-3'</p>
<img src="/images/2D/ligand 1.1_aptamer_2D1.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="header_box" id="ligand-recognition">Ligand information</p>
<p class="blowheader_box">SELEX ligand</p>
<p>A reverse transcriptase (RT) is an enzyme used to generate complementary DNA (cDNA) from an RNA template, a process termed reverse transcription. Reverse transcriptases are used by viruses such as HIV to replicate their genomes, by retrotransposon mobile genetic elements to proliferate within the host genome, and by eukaryotic cells to extend the telomeres at the ends of their linear chromosomes.-----From Pfam</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">HIV-1 RT</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P03366/entry#names_and_taxonomy" target="_blank" style="color:#520049"><b>P03366</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00078/" target="_blank" style="color:#520049"><b>PF00078</b></a></td>
        <td name="td3">104.53 kDa</td>
        <td name="td4">Subunit P66 (Residues 600-1153): PISPIETVPVKLKPGMDGPKVKQWPLTEEKIKALVEICTEMEKEGKISKIGPENPYNTPVFAIVDFRELNKRTQDFWEVQLGIPHPAGLKKKKSVTVLDVGDAYFSVPLDEDFRKYTAFTIPGIRYQYNVLPQGWKGSPAIFQSSMTKILEPFKKQNPDIVIYQYMDDLYVGSDLEIGQHRTKIEELRQHLLRWGLTTPDKKHQKEPPFLWMGYELHPDKWTVQPIVLPEKDSWTVNDIQKLVGKLNWASQIYPGIKVRQLCKLLRGTKALTEVIPLTEEAELELAENREILKEPVHGVYYDPSKDLIAEIQKQGQGQWTYQIYQEPFKNLKTGKYARMRGAHTNDVKQLTEAVQKITTESIVIWGKTPKFKLPIQKETWETWWTEYWQATWIPEWEFVNTPPLVKLWYQLEKEPIVGAETFYVDGAANRETKLGKAGYVTNKGRQKVVPLTNTTNQKTELQAIYLALQDSGLEVNIVTDSQYALGIIQAQPDKSESELVNQIIEQLIKKEKVYLAWVPAHKGIGGNEQVDKLVSA&#10;Subunit P51 (Residues 604-1026): IETVPVKLKPGMDGPKVKQWPLTEEKIKALVEICTEMEKEGKISKIGPENPYNTPVFAIKKWRKLVDFRELNKRTQDFWEVQLGIPHPAGLKKKKSVTVLDVGDAYFSVPLDEDFRKYTAFTIPSINNETPGIRYQYNVLPQGWKGSPAIFQSSMTKILEPFKKQNPDIVIYQYMDDLYVGSDLEIGQHRTKIEELRQHLLRHPDKWTVQPIVLPEKDSWTVNDIQKLVGKLNWASQIYPGIKVRQLCKLLRGTKALTEVIPLTEEAELELAENREILKEPVHGVYYDPSKDLIAEIQKQGQGQWTYQIYQEPFKNLKTGKYARMRGAHTNDVKQLTEAVQKITTESIVIWGKTPKFKLPIQKETWETWWTEYWQATWIPEWEFVNTPPLVKLWY</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/1HMV" target="_blank" style="color:#520049"><b>1HMV</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/protein/AAA44198.1" target="_blank" style="color:#520049"><b>AAA44198.1</b></a></td>
      </tr>
	  </tbody>
  </table>

  <p>Some isolated sequences bind to the affinity of the protein.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">ligand 1.1</td>
        <td name="td1">5'-GGGAGCAUCAGACUUUUAAUCUGACAAUCAAGAAUUCCGUUUUCAGUCGGGAAAAACUGAACAAUCUAUGAAAGAAUUUUAUAUCUCUAUUGAAAC-3'</td>
        <td name="td2">HIV-1 RT</td>
        <td name="td3">5.0 nM</td>
      </tr>
      <tr>
        <td name="td0">ligand 1.3a</td>
        <td name="td1">5'-GGGAGCAUCAGACUUUUAAUCUGACAAUCAAGAAUAUCUUCCGAAGCCGAACGGGAAAACCGGCAUCUAUGAAAGAAUUUUAUAUCUCUAUUGAAAC-3'</td>
        <td name="td2">HIV-1 RT</td>
        <td name="td3"></td>
      </tr>
      <tr>
        <td name="td0">ligand 1.3b</td>
        <td name="td1">5'-GGGAGCAUCAGACUUUUAAUCUGACAAUCAAGAAUAUCUUCCGAGGCCGAACGGGAAAACCGACAUCUAUGAAAGAAUUUUAUAUCUCUAUUGAAAC-3'</td>
        <td name="td2">HIV-1 RT</td>
        <td name="td3"></td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/ligand 1.1_aptamer_SELEX_ligand1.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<img src="/images/SELEX_ligand/ligand 1.1_aptamer_SELEX_ligand1.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>


<p class="blowheader_box">Similar compound(s)</p>                    
<p>We searched the RCSB PDB for structures similar to the target protein and listed the best matches.</p><p>We used the RCSB PDB website's similar structure search to find the top 10 structures similar to HIV-1 REV PROTEIN (residues T34-R50), and calculated TM-socre values and RMSD values using the TM-align website.</p>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;">
  <thead>
    <tr>
      <th onclick="sortTable(0)">PDB</th>
      <th onclick="sortTable(1)">Z-score</th>
      <th onclick="sortTable(2)">RMSD</th>
      <th onclick="sortTable(3)">Description</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><a href="https://www.rcsb.org/structure/1HMV-A" target="_blank" style="color:#520049"><b>1HMV-A</b></a></td>
      <td>47.6</td>
      <td>0.0</td>
      <td>HIV-1 reverse transcriptase (subunit P66)</td>
    </tr>
    <tr>
      <td><a href="https://www.rcsb.org/structure/1HMV-C" target="_blank" style="color:#520049"><b>1HMV-C</b></a></td>
      <td>46.9</td>
      <td>0.1</td>
      <td>HIV-1 reverse transcriptase (subunit P66)</td>
    </tr>
    <tr>
      <td><a href="https://www.rcsb.org/structure/1HMV-E" target="_blank" style="color:#520049"><b>1HMV-E</b></a></td>
      <td>46.9</td>
      <td>0.1</td>
      <td>HIV-1 reverse transcriptase (subunit P66)</td>
    </tr>
    <tr>
      <td><a href="https://www.rcsb.org/structure/1HMV-G" target="_blank" style="color:#520049"><b>1HMV-G</b></a></td>
      <td>46.9</td>
      <td>0.1</td>
      <td>HIV-1 reverse transcriptase (subunit P66)</td>
    </tr>
    <tr>
      <td><a href="https://www.rcsb.org/structure/1HVU-G" target="_blank" style="color:#520049"><b>1HVU-G</b></a></td>
      <td>44.6</td>
      <td>1.1</td>
      <td>RNA (33 nucleotide RNA pseudoknot)</td>
    </tr>
    <tr>
      <td><a href="https://www.rcsb.org/structure/1HVU-A" target="_blank" style="color:#520049"><b>1HVU-A</b></a></td>
      <td>44.6</td>
      <td>1.1</td>
      <td>RNA (33 nucleotide RNA pseudoknot)</td>
    </tr>
    <tr>
      <td><a href="https://www.rcsb.org/structure/1HVU-J" target="_blank" style="color:#520049"><b>1HVU-J</b></a></td>
      <td>44.6</td>
      <td>1.1</td>
      <td>RNA (33 nucleotide RNA pseudoknot)</td>
    </tr>
    <tr>
      <td><a href="https://www.rcsb.org/structure/1HVU-D" target="_blank" style="color:#520049"><b>1HVU-D</b></a></td>
      <td>44.6</td>
      <td>1.1</td>
      <td>RNA (33 nucleotide RNA pseudoknot)</td>
    </tr>
    <tr>
      <td><a href="https://www.rcsb.org/structure/3ITH-A" target="_blank" style="color:#520049"><b>3ITH-A</b></a></td>
      <td>39.6</td>
      <td>1.0</td>
      <td>Reverse transcriptase/ribonuclease H</td>
    </tr>
    <tr>
      <td><a href="https://www.rcsb.org/structure/3ISN-C" target="_blank" style="color:#520049"><b>3ISN-C</b></a></td>
      <td>39.5</td>
      <td>1.2</td>
      <td>Reverse transcriptase/ribonuclease H</td>
    </tr>
  </tbody>
</table>

<p class="header_box" id="references">References</p>

<a id="ref1"></a>
<font><strong>[1] RNA pseudoknots that inhibit human immunodeficiency virus type 1 reverse transcriptase.</strong></font><br />
Tuerk, C., MacDougal, S., & Gold, L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/1379730/" target="_blank">Proceedings of the National Academy of Sciences of the United States of America, 89(15), 6988–6992. (1992)</a>
<br />
    
<a id="ref2"></a>
<font><strong>[2] HIV-1 reverse transcriptase-pseudoknot RNA aptamer interaction has a binding affinity in the low picomolar range coupled with high specificity.</strong></font><br />
Kensch, O., Connolly, B. A., Steinhoff, H. J., McGregor, A., Goody, R. S., & Restle, T.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10751399/" target="_blank">The Journal of biological chemistry, 275(24), 18271–18278. (2000)</a>
<br />
    
<a id="ref3"></a>
<font><strong>[3] Potent inhibition of human immunodeficiency virus type 1 replication by template analog reverse transcriptase inhibitors derived by SELEX (systematic evolution of ligands by exponential enrichment).</strong></font><br />
Joshi, P., & Prasad, V. R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12050367/" target="_blank">Journal of virology, 76(13), 6545–6557. (2002)</a>
<br />
    
<a id="ref4"></a>
<font><strong>[4] Insight into HIV-1 reverse transcriptase-aptamer interaction from molecular dynamics simulations.</strong></font><br />
Aeksiri, N., Songtawee, N., Gleeson, M. P., Hannongbua, S., & Choowongkomon, K.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/25073457/" target="_blank">Journal of molecular modeling, 20(8), 2380. (2014)</a>
<br />
    
<a id="ref5"></a>
<font><strong>[5] Binding interface and impact on protease cleavage for an RNA aptamer to HIV-1 reverse transcriptase.</strong></font><br />
Nguyen, P. D. M., Zheng, J., Gremminger, T. J., Qiu, L., Zhang, D., Tuske, S., Lange, M. J., Griffin, P. R., Arnold, E., Chen, S. J., Zou, X., Heng, X., & Burke, D. H.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/31943114/" target="_blank">Nucleic acids research, 48(5), 2709–2722. (2020)</a>
<br />
    
<a id="ref6"></a>
<font><strong>[6] 2'-fluoro-modified pyrimidines enhance affinity of RNA oligonucleotides to HIV-1 reverse transcriptase.</strong></font><br />
Gruenke, P. R., Alam, K. K., Singh, K., & Burke, D. H.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/32732393/" target="_blank">RNA (New York, N.Y.), 26(11), 1667–1679. (2020)</a>
<br />
    