---
layout: theophylline_style
title: HTLV-1 Rex peptide aptamer
date: 2024-04-17 00:00:00
author: Ziyu Guo, Jiali Wang
categories: Aptamer
tags:
- Category:Viral
- GC:70.0
- Length:33
- Named:HTLV-1 Rex protein_39B aptamer
- Type:Proteins
- Year:1999
type: Proteins
---
<html>



<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10233958/" target="_blank" style="color:#520049">1999</a></h3>
    </div>
    <div class="body">
      <p>The researchers identified anti-Rex aptamers by screening tightly bound RNA molecules from a pool of randomly sequenced RNA with conformational constraints<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10647177/" target="_blank" style="color:#520049">1999</a></h3>
    </div>
    <div class="body">
      <p>The solution structure of HTLV-1 arginine-rich Rex peptide bound to its RNA aptamer target was proposed and determined by multi-dimensional heteronuclear magnetic resonance spectroscopy<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12925996/" target="_blank" style="color:#520049">2003</a></h3>
    </div>
    <div class="body">
      <p>Fluorescence-based methods for evaluating the RNA affinity and specificity of HIV-1 Rev-RRE inhibitors<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/18351707/" target="_blank" style="color:#520049">2008</a></h3>
    </div>
    <div class="body">
      <p>Amino acid requirement for the high affinity binding of a selected arginine-rich peptide with the HIV Rev-response element RNA<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/26896646/" target="_blank" style="color:#520049">2016</a></h3>
    </div>
    <div class="body">
      <p>Bioavailable inhibitors of HIV-1 RNA biogenesis were identified through a Rev-based screen<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
</div>


         
<p class="header_box" id="description">Description</p>
<p>In 1999, Ellington, A. D. et al. employed RNA selection techniques to identify RNA molecules that bind tightly to REX-fusion proteins from a pool of randomly sequenced RNA with conformational constraints. These RNA molecules, termed anti-Rex aptamers, were suggested to function as RNA decoys for Rex proteins and hold potential for antiviral applications. In the same year, Patel, D. J. et al. determined the solution structure of the binding site of the Rex peptide RNA aptamer complex using a combination of nuclear magnetic resonance (NMR) and molecular dynamics methods<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref2" style="color:#520049">2</a></sup><sup>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 1999, Ellington, A. D. et al. isolated high-affinity RNA aptamers targeting the Rex protein from a pool of conformationally constrained random sequence RNAs through multiple rounds of selection. The binding affinity of the selected aptamers was significantly enhanced compared to that of the initial pool. Before each selection round, the RNA pool was denatured at 90°C for 2 minutes and then cooled to ambient temperature over 10 minutes to facilitate proper structure formation. The selection process was conducted in a 30 μl reaction containing 50 mM Tris-HCl (pH 8.0) and 50 mM KCl, with the Rex fusion protein incubated with the RNA pool. In the initial rounds, the protein concentration was higher, but it was reduced in later rounds to increase selection stringency. Specifically, selections 1 to 4 were conducted at a protein concentration of 1.8 pmol, while selections 5 to 11 were performed at lower concentrations. The RNA concentration was maintained at around 1.3 μM in the early rounds and adjusted in later rounds. The initial selection involved approximately 10<sup>13</sup>different RNA molecules, ensuring a diverse sampling of sequences<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>
<br>
<br>   


<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>Here we use ribodraw to complete the figure, through the 3D structure information. 39B aptamer was named by Ellington, A. D. in the SELEX article. 33 mer RNA aptamer was named by Patel, D. J. in the structure article<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p>
<p>5'-GGGCGCCGGUACGCAAGUACGACGGUACGCUCC-3'</p>
<img src="/images/2D/Rex16_aptamer_2D.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>



<p class="blowheader_box">3D visualisation</p>             
<p>The solution structure of the Rex peptide–RNA aptamer complex was determined by Patel, D. J. et al. using multidimensional nuclear magnetic resonance (NMR) spectroscopy. The Rex peptide, in a predominantly extended conformation, threads through a channel formed by the shallow and widened RNA major groove and a looped-out guanine. The PDB ID for this structure is 1EXY<sup>[<a href="#ref2" style="color:#520049">2</a></sup><sup>]</sup>.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/Rex16_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A', 
  start_residue_number:1, 
  end_residue_number:5, 
  color:{r:184,g:51,b:50}
},
{
  struct_asym_id: 'A', 
  start_residue_number:6, 
  end_residue_number:8, 
  color:{r:241,g:125,b:7}
},{
  struct_asym_id: 'A', 
  start_residue_number:9, 
  end_residue_number:20, 
  color:{r:24,g:152,b:140}
},
{
  struct_asym_id: 'A', 
  start_residue_number:21, 
  end_residue_number:22, 
  color:{r:194,g:182,b:23}
},
{
  struct_asym_id: 'A', 
  start_residue_number:23, 
  end_residue_number:25, 
  color:{r:241,g:125,b:7}
},{
  struct_asym_id: 'A', 
  start_residue_number:26, 
  end_residue_number:27, 
  color:{r:194,g:182,b:23}
},
{
  struct_asym_id: 'A', 
  start_residue_number:28, 
  end_residue_number:33, 
  color:{r:184,g:51,b:50}
},{
  struct_asym_id: 'B', 
  start_residue_number:101, 
  end_residue_number:116, 
  color:{r:8,g:69,b:149}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1exy.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<p>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 1EXY. HTLV-1 arginine-rich Rex peptide (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with HTLV-1 arginine-rich Rex peptide. The C7 and C23 bases in the RNA aptamer do indeed interact with amino acids in the HTLV-1 Rex peptide (primarily Arg5 and Arg7), though the text does not explicitly state whether these interactions involve hydrogen bonds. We used PYMOL predicted the hydrogen bond between C23 and Arg7.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Rex16_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Rex16_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>



<p class="header_box" id="ligand-recognition">Ligand information</p>

<p class="blowheader_box">SELEX ligand</p>
<p>A binding curve was generated by incubating increasing amounts of RNA with a constant amount of Rex protein and capturing aptamer-Rex complexes by filtration. In a complementary experiment, increasing amounts of protein were incubated with a constant amount of RNA and the formation of the aptamer-Rex complex was monitored by a gel mobility shift. The formation of the aptamer-Rex complex was again found to be concentration dependent, and the Kapp was similar. The binding ratios for pool to wild-type RNAs were determined following electrophoretic separation and quantitation with a PhosphorImager<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">39B aptamer</td>
        <td name="td1">5'-GCCUGCCGGUACGCAAGUACGACGGUACAGGC-3'</td>
        <td name="td3">HTLV-1 Rex peptide</td>
        <td name="td4">9.3 ± 1.8 (Binding ratios to wild-type XBE RNA)</td>
      </tr>
      <tr>
        <td name="td0">8-5 aptamer</td>
        <td name="td1">5'-UAGGCGACGGUACGCAAGUACUCUUGCGCCGGCCUA-3'</td>
        <td name="td3">HTLV-1 Rex peptide</td>
        <td name="td4">30 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<p>Rex escorts unspliced gag-pro-pol and singly spliced env mRNAs out of the nucleus of infected cells. These mRNAs carry a recognition sequence called Rex responsive element (RxRE or XRE) located at the 3' region of the long terminal repeat (LTR). This function is essential since most HTLV proteins are translated from unspliced or partially spliced pre-mRNAs that cannot exit the nucleus by the pathway used by fully processed cellular mRNAs. Rex itself is translated from a fully spliced mRNA that probably readily exits the nucleus. Rex's nuclear localization signal (NLS) binds directly to KPNB1/importin beta-1 without previous binding to KPNA1/importin alpha-1. KPNB1 binds to the GDP bound form of RAN (Ran-GDP) and targets Rex to the nucleus. In the nucleus, the conversion from Ran-GDP to Ran-GTP dissociates Rex from KPNB1 and allows Rex's binding to the RRE in viral pre-mRNAs. Rex multimerizes on the RRE via cooperative assembly. This multimerization is critical for its full biological activity, since it may shield the viral RNA from being spliced or down-regulated, and probably exposes Rex's nuclear export signal (NES) to the surface. Rex can then form a complex with XPO1/CRM1, RANBP3 and Ran-GTP, leading to nuclear export of the complex. Conversion from Ran-GTP to Ran-GDP mediates dissociation of the Rex/RRE/XPO1/RANBP3/RAN complex, so that Rex can return to the nucleus for a subsequent round of export (By similarity).-----From <a href="https://www.ebi.ac.uk/interpro/protein/reviewed/P0C206/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">Mass</th>
        <th onclick="sortTable(3)">Protein sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P0C206/entry" target="_blank" style="color:#520049"><b>P0C206</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/protein/reviewed/P0C206/" target="_blank" style="color:#520049"><b>P0C206</b></a></td>
        <td name="td2">2.106 Kda</td>
        <td name="td3">MPKTRRRPRRSQRKRP</td>
        <td name="td4"><a href="https://www.rcsb.org/structure/1EXY" target="_blank" style="color:#520049"><b>1EXY</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/nuccore/AF033817" target="_blank" style="color:#520049"><b>AF033817</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/Rex16_aptamer_stru_ligand1.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<p>We used the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>RCSB PDB website</b></a>'s similar structure search to find the top 10 structures similar to HIV-1 REV PROTEIN (RESIDUES 34-50), and calculated TM-socre values and RMSD values using the TM-align website.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://zhanggroup.org/TM-align/" target="_blank" style="color:#520049; text-decoration: underline;"><b>TM-socre</b></a>:</b> a metric used to assess the similarity between two protein structures, providing a quantitative measure of their structural alignment, with higher scores indicating greater similarity.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">TM-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">1EXY-B</td>
        <td name="td2">1</td>
        <td name="td3">0</td>
        <td name="td4">Solution structure of HTLV-1 peptide bound to its rna aptamer target</td>
      </tr>
      <tr>
        <td name="td1">7SVR-A</td>
        <td name="td2">0.26</td>
        <td name="td3">2.35</td>
        <td name="td4">The complex of dephosphorylated human cystic fibrosis transmembrane conductance regulator (CFTR) and Lumacaftor (VX-809)</td>
      </tr>
      <tr>
        <td name="td1">4QMG-B</td>
        <td name="td2">0.22</td>
        <td name="td3">3.35</td>
        <td name="td4">The Structure of MTDH-SND1 Complex Reveals Novel Cancer-Promoting Interactions</td>
      </tr>
      <tr>
        <td name="td1">2RNW-A</td>
        <td name="td2">0.21</td>
        <td name="td3">1.96</td>
        <td name="td4">The Structural Basis for Site-Specific Lysine-Acetylated Histone Recognition by the Bromodomains of the Human Transcriptional Co-Activators PCAf and CBP</td>
      </tr>
      <tr>
        <td name="td1">6IAW-D</td>
        <td name="td2">0.27</td>
        <td name="td3">1.53</td>
        <td name="td4">Structure of head fiber and inner core protein gp22 of native bacteriophage P68</td>
      </tr>
      <tr>
        <td name="td1">2MZ6</td>
        <td name="td2">0.16</td>
        <td name="td3">2.22</td>
        <td name="td4">NMR structure of Protegrin-3 (PG3) in the presence of DPC micelles</td>
      </tr>
      <tr>
        <td name="td1">7TT1-F</td>
        <td name="td2">0.16</td>
        <td name="td3">2.66</td>
        <td name="td4">BamABCDE bound to substrate EspP class 4</td>
      </tr>
      <tr>
        <td name="td1">4XAI-B</td>
        <td name="td2">0.24</td>
        <td name="td3">2.31</td>
        <td name="td4">Crystal Structure of red flour beetle NR2E1/TLX</td>
      </tr>
	  </tbody>
  </table>



                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Anti-Rex aptamers as mimics of the Rex-binding element.</strong></font><br />
Baskerville, S., Zapp, M., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10233958/" target="_blank" style="color:#520049" >Journal of virology, 73(6), 4962–4971. (1999)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Anchoring an extended HTLV-1 Rex peptide within an RNA major groove containing junctional base triples.</strong></font><br />
Jiang, F., Gorin, A., Hu, W., Majumdar, A., Baskerville, S., Xu, W., Ellington, A., & Patel, D. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10647177/" target="_blank" style="color:#520049">Structure (London, England : 1993), 7(12), 1461–1472. (1999)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] Fluorescence-based methods for evaluating the RNA affinity and specificity of HIV-1 Rev-RRE inhibitors.</strong></font><br />
Luedtke, N. W., & Tor, Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12925996/" target="_blank" style="color:#520049"> Biopolymers, 70(1), 103–119. (2003)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] Amino acid requirement for the high affinity binding of a selected arginine-rich peptide with the HIV Rev-response element RNA.</strong></font><br />
Sugaya, M., Nishino, N., Katoh, A., & Harada, K.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18351707/" target="_blank" style="color:#520049">Journal of peptide science : an official publication of the European Peptide Society, 14(8), 924–935.  (2008)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Bioavailable inhibitors of HIV-1 RNA biogenesis identified through a Rev-based screen.</strong></font><br />
Prado, S., Beltrán, M., Coiras, M., Bedoya, L. M., Alcamí, J., & Gallego, J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/26896646/" target="_blank" style="color:#520049">Biochemical pharmacology, 107, 14–28. (2016)</a>
<br />  
