---
layout: theophylline_style
title: L-CCL2 aptamer
date: 2024-05-05 00:00:00
author: Yuxuan Sun, Baowei Huang
categories: Aptamer
tags:
- Category:Mammalian
- GC:67.0
- Length:40
- Named:C-C motif chemokine 2 (L-CCL2)_NOX-E36 L aptamer
- Type:Proteins
- Year:2007
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/17625118/" target="_blank" style="color:#520049">2007</a></h3>
    </div>
    <div class="body">
      <p>The aptamer was isolated for the first time<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/18776253/" target="_blank" style="color:#520049">2008</a></h3>
    </div>
    <div class="body">
      <p>The physical and chemical properties of NOX-E36 RNA aptamer<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>

  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/18258851/" target="_blank" style="color:#520049">2008</a></h3>
    </div>
    <div class="body">
      <p>Ccl2 blockade with mNOX-E36–3′PEG effectively blocked macrophage recruitment into the glomerular and interstitial compartments of the kidney<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/19156777/" target="_blank" style="color:#520049">2009</a></h3>
    </div>
    <div class="body">
      <p>Ccl2 mediates the recruitment of glomerular and interstitial macrophages but this mechanism does not contribute to the progression of Alport nephropathy in Col4a3-deficient mice<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/21813474/" target="_blank" style="color:#520049">2012</a></h3>
    </div>
    <div class="body">
      <p>Pharmacological inhibition of MCP-1 with mNOX-E36 was investigated<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/24561613/" target="_blank" style="color:#520049">2014</a></h3>
    </div>
    <div class="body">
      <p>Inhibition of monocyte infiltration by targeting the chemokine CCL2 prevented fibrosis-associated angiogenesis, but did not halt fibrosis progression<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/24481979/" target="_blank" style="color:#520049">2014</a></h3>
    </div>
    <div class="body">
      <p>Pharmacological inhibition of Ly-6C(+) monocyte recruitment using the CCL2-inhibitor mNOX-E36 accelerated regression from toxic and metabolic liver fibrosis<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/25901662/" target="_blank" style="color:#520049">2015</a></h3>
    </div>
    <div class="body">
      <p>The crystal structure of the aptamer have been deterimined<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/25970072/" target="_blank" style="color:#520049">2015</a></h3>
    </div>
    <div class="body">
      <p>Spiegelmer-based inhibition of MCP-1/CCR2 with the compound mNOX-E36 resulted in moderate but statistically significant improvements in graft survival<sup>[<a href="#ref9" style="color:#520049">9</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/28186566/" target="_blank" style="color:#520049">2017</a></h3>
    </div>
    <div class="body">
      <p>Inhibition of the CCL2/CCL2 receptor axis with NOX-E36 was generally safe and well-tolerated<sup>[<a href="#ref10" style="color:#520049">10</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/28837800/" target="_blank" style="color:#520049">2017</a></h3>
    </div>
    <div class="body">
      <p>Inhibition of CCL2 in diabetic nephropathy effectively reduced proteinuria and was associated with restoration of glomerular endothelial glycocalyx size<sup>[<a href="#ref11" style="color:#520049">11</a>]</sup></p>
    </div>
  </div>
</div>


         
<p class="header_box" id="description">Description</p>
<p>In 2007, Anders, H. J. isolated the L-enantiomeric RNA oligonucleotide mNOX-E36, a so-called Spiegelmer that binds murine CCL2 with high affinity and neutralizes its action in vitro and in vivo. In 2015, Betzel C et al. chemically synthesized the NOX-E36 L aptamer, a Spiegelmer designed to bind to L-CCL2. To elucidate its structure, they employed single-wavelength anomalous diffraction (SAD) and refined the structure to a resolution of 2.05 Å. This technique allowed them to gain detailed insights into the interactions between the L aptamer and its target protein. The resulting structural data provided a comprehensive understanding of the binding interface and the conformational details of the L aptamer. This high-resolution structure was crucial for understanding the molecular basis of the L aptamer's high affinity and specificity for L-CCL2. Such detailed structural information is essential for optimizing the design of Spiegelmers as potential therapeutic agents<sup>[<a href="#ref1" style="color:#520049">1</a>,<a href="#ref8" style="color:#520049">8</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2007, Anders, H. J. et al. generated the Spiegelmer mNOX-E36 through a rigorous in vitro selection process. A DNA library with 40 internal random positions was used to facilitate the identification of high-affinity Spiegelmers. The library underwent 11 rounds of selection, which allowed for the systematic enrichment of RNA sequences with high binding affinity for murine D-CCL2. Following these selection rounds, the most prevalent and effective RNA sequence was identified. This sequence was then truncated to obtain the final candidate, mNOX-E36. The selection process was crucial in ensuring that mNOX-E36 possessed the desired properties for therapeutic applications, particularly in the context of lupus nephritis. The successful identification of mNOX-E36 highlights the potential of Spiegelmers as a novel class of therapeutic agents<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>
<br>
<br>    


<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>In 2007, Anders, H. J. et al. employed an in vitro selection process to generate the Spiegelmer mNOX-E36. mNOX-E36 is a 45-nucleotide and mouse-CCL2-specific Spiegelmer aptamer. NOX-E36 L aptamer is a 40-nucleotide and Human-CCL2-specific Spiegelmer aptamer. Here we use ribodraw to complete the figure, through the 3D structure information. The NOX-E36 L aptamer was named by Anders, H. J. et al. in the article<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>5'-GCACGUCCCUCACCGGUGCAAGUGAAGCCGAGGCUCUGCG-3'</p>
<img src="/images/2D/L-CCL2_aptamer_2D.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">3D visualisation</p>             
<p>NOX-E36 L aptamer is a Spiegelmer designed to bind to L-CCL2. To elucidate its crsytal structure, they employed single-wavelength anomalous diffraction (SAD) and refined the structure to a resolution of 2.05 Å. This technique allowed them to gain detailed insights into the interactions between the L aptamer and its target protein. The resulting structural data provided a comprehensive understanding of the binding interface and the conformational details of the L aptamer. The PDB ID of this structure is 4R8I<sup>[<a href="#ref8" style="color:#520049">8</a></sup><sup>]</sup>.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/L-CCL2_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A', 
  start_residue_number:3, 
  end_residue_number:70, 
  color:{r:133,g:39,b:181}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:1, 
  end_residue_number:3, 
  color:{r:184,g:51,b:50}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:4, 
  end_residue_number:4, 
  color:{r:24,g:152,b:140}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:5, 
  end_residue_number:5, 
  color:{r:241,g:125,b:7}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:6, 
  end_residue_number:7, 
  color:{r:24,g:152,b:140}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:8, 
  end_residue_number:8, 
  color:{r:8,g:69,b:149}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:9, 
  end_residue_number:11, 
  color:{r:241,g:125,b:7}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:12, 
  end_residue_number:13, 
  color:{r:24,g:152,b:140}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number1:4, 
  end_residue_number:16, 
  color:{r:207,g:62,b:125}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:17, 
  end_residue_number:21, 
  color:{r:24,g:152,b:140}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:22, 
  end_residue_number:22, 
  color:{r:241,g:125,b:7}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:23, 
  end_residue_number:23, 
  color:{r:24,g:152,b:140}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:24, 
  end_residue_number:24, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'B', 
  start_residue_number:25, 
  end_residue_number:25, 
  color:{r:24,g:152,b:140}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:26, 
  end_residue_number:27, 
  color:{r:241,g:125,b:7}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:28, 
  end_residue_number:30, 
  color:{r:207,g:62,b:125}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:31, 
  end_residue_number:31, 
  color:{r:24,g:152,b:140}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:32, 
  end_residue_number:33, 
  color:{r:8,g:69,b:149}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:34, 
  end_residue_number:36, 
  color:{r:24,g:152,b:140}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:37, 
  end_residue_number:39, 
  color:{r:184,g:51,b:50}
}, 
{
  struct_asym_id: 'B', 
  start_residue_number:40, 
  end_residue_number:40, 
  color:{r:24,g:152,b:140}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/4r8i-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<p>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 4R8I. L-CCL2 (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with L-CCL2.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/L-CCL2_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/L-CCL2_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>



<p class="header_box" id="ligand-recognition">Ligand information</p> 
<p class="blowheader_box">SELEX ligand</p>
<p>Betzel, C. et al. investigated the ion dependence of L aptamer binding using surface plasmon resonance (SPR), revealing a strong dependence on Ca2+, which appears to be important for proper folding of the L aptamer and will likely occupy the position of Sr2+ observed in the structure. And they also studied the effect of the mutations on L aptamer affinity and measured the L aptamer’s affinity using SPR<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup>.</p>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">L-CCL2 aptamer</td>
        <td name="td1">5'-GCACGUCCCUCACCGGUGCAAGUGAAGCCGAGGCUCUGCG-3'</td>
        <td name="td3">L-CCL2</td>
        <td name="td4">1.40 ± 0.16 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>

<p class="blowheader_box">Structure ligand</p>
<p>Acts as a ligand for C-C chemokine receptor CCR2. Signals through binding and activation of CCR2 and induces a strong chemotactic response and mobilization of intracellular calcium ions. Exhibits a chemotactic activity for monocytes and basophils but not neutrophils or eosinophils. May be involved in the recruitment of monocytes into the arterial wall during the disease process of atherosclerosis.-----From <a href="https://www.ebi.ac.uk/interpro/protein/UniProt/P13500/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">Mass</th>
        <th onclick="sortTable(3)">Protein sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P13500/entry" target="_blank" style="color:#520049"><b>P13500</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/protein/UniProt/P13500/" target="_blank" style="color:#520049"><b>P13500</b></a></td>
        <td name="td2">8.82 kDa</td>
        <td name="td3">MQPDAINAPVTCCYNFTNRKISVQRLASYRRITSSKCPKEAVIFKTIVAKEICADPKQKWVQDSMDHLDKQTQTPKT</td>
        <td name="td4"><a href="https://www.rcsb.org/structure/1DOK" target="_blank" style="color:#520049"><b>1DOK</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?lvl=0&id=9606" target="_blank" style="color:#520049"><b>9606</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/L-CCL2_aptamer_stru_ligand1.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">1DOK-A</td>
        <td name="td2">15.7</td>
        <td name="td3">0</td>
        <td name="td4">Monocyte chemoattractant protein 1</td>
      </tr>
      <tr>
        <td name="td1">7SO0-B</td>
        <td name="td2">12</td>
        <td name="td3">0.7</td>
        <td name="td4">Evasin p974</td>
      </tr>
      <tr>
        <td name="td1">2VXW-A</td>
        <td name="td2">12</td>
        <td name="td3">1.5</td>
        <td name="td4">C-C motif chemokine 5</td>
      </tr>
      <tr>
        <td name="td1">1NR4-E</td>
        <td name="td2">11.3</td>
        <td name="td3">1.5</td>
        <td name="td4">Thymus and activation-regulated chemokine</td>
      </tr>
      <tr>
        <td name="td1">5EKI-E</td>
        <td name="td2">10.4</td>
        <td name="td3">1.5</td>
        <td name="td4">C-C motif chemokine 21</td>
      </tr>
      <tr>
        <td name="td1">2JYO-A</td>
        <td name="td2">9.4</td>
        <td name="td3">2.1</td>
        <td name="td4">C-C motif chemokine 20 (small-inducible cytokine)</td>
      </tr>
      <tr>
        <td name="td1">1B2T-A</td>
        <td name="td2">8.7</td>
        <td name="td3">3.5</td>
        <td name="td4">Protein (fractalkine)</td>
      </tr>
      <tr>
        <td name="td1">2MP1-A</td>
        <td name="td2">8.3</td>
        <td name="td3">2.0</td>
        <td name="td4">C-C motif chemokine 19</td>
      </tr>
      <tr>
        <td name="td1">1O80-A</td>
        <td name="td2">7.9</td>
        <td name="td3">2.4</td>
        <td name="td4">Small inducible cytokine b10</td>
      </tr>
      <tr>
        <td name="td1">1F9P-A</td>
        <td name="td2">7.4</td>
        <td name="td3">1.6</td>
        <td name="td4">Connective tissue activating peptide-iii</td>
      </tr>
	  </tbody>
  </table>


                 
<p class="header_box" id="references">References</p>

<a id="ref1"></a><font><strong>[1] Spiegelmer inhibition of CCL2/MCP-1 ameliorates lupus nephritis in MRL-(Fas)lpr mice.</strong></font><br />
Kulkarni, O., Pawar, R. D., Purschke, W., Eulberg, D., Selve, N., Buchner, K., Ninichuk, V., Segerer, S., Vielhauer, V., Klussmann, S., & Anders, H. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/17625118/" target="_blank" style="color:#520049" >Journal of the American Society of Nephrology : JASN, 18(8), 2350–2358. (2007)</a>
<br />

<a id="ref2"></a><font><strong>[2] Physicochemical stability of NOX-E36, a 40mer L-RNA (Spiegelmer) for therapeutic applications.</strong></font><br />
Maasch, C. , Buchner, K. , Eulberg, D. , Vonhoff, S. , & Klussmann, S. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18776253/" target="_blank" style="color:#520049" >Nucleic acids symposium series, (52), 61–62. (2008)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] Late onset of Ccl2 blockade with the Spiegelmer mNOX-E36-3'PEG prevents glomerulosclerosis and improves glomerular filtration rate in db/db mice.</strong></font><br />
Ninichuk, V. , Clauss, S. , Kulkarni, O. , Schmid, H. , Segerer, S. , Radomska, E. , Eulberg, D. , Buchner, K. , Selve, N. , Klussmann, S. , & Anders, H. J<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18258851/" target="_blank" style="color:#520049">The American journal of pathology, 172(3), 628–637. (2008)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] Ccl2/Mcp-1 blockade reduces glomerular and interstitial macrophages but does not ameliorate renal pathology in collagen4A3-deficient mice with autosomal recessive Alport nephropathy.</strong></font><br />
Clauss, S. , Gross, O. , Kulkarni, O. , Avila-Ferrufino, A. , Radomska, E. , Segerer, S. , Eulberg, D. , Klussmann, S. , & Anders, H. J. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/19156777/" target="_blank" style="color:#520049">The Journal of pathology, 218(1), 40–47. (2009)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Pharmacological inhibition of the chemokine CCL2 (MCP-1) diminishes liver macrophage infiltration and steatohepatitis in chronic hepatic injury.</strong></font><br />
Baeck, C. , Wehr, A. , Karlmark, K. R. , Heymann, F. , Vucur, M. , Gassler, N. , Huss, S. , Klussmann, S. , Eulberg, D. , Luedde, T. , Trautwein, C. , & Tacke, F. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/21813474/" target="_blank" style="color:#520049">Gut,61(3), 416–426. (2012)</a>
<br />
                
<a id="ref6"></a><font><strong>[6] CCL2-dependent infiltrating macrophages promote angiogenesis in progressive liver fibrosis.</strong></font><br />
Ehling, J. , Bartneck, M. , Wei, X. , Gremse, F. , Fech, V. , Möckel, D. , Baeck, C. , Hittatiya, K. , Eulberg, D. , Luedde, T. , Kiessling, F. , Trautwein, C. , Lammers, T. , & Tacke, F.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/24561613/" target="_blank" style="color:#520049">Gut,63(12), 1960–1971. (2014)</a>
<br />

<a id="ref7"></a><font><strong>[7] Pharmacological inhibition of the chemokine C-C motif chemokine ligand 2 (monocyte chemoattractant protein 1) accelerates liver fibrosis regression by suppressing Ly-6C(+) macrophage infiltration in mice.</strong></font><br />
Baeck, C. , Wei, X. , Bartneck, M. , Fech, V. , Heymann, F. , Gassler, N. , Hittatiya, K. , Eulberg, D. , Luedde, T. , Trautwein, C. , & Tacke, F. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/24481979/" target="_blank" style="color:#520049">Hepatology (Baltimore, Md.), 59(3), 1060–1072. (2014)</a>
<br />       

<a id="ref8"></a><font><strong>[8] Crystal structure of a mirror-image L-RNA aptamer (Spiegelmer) in complex with the natural L-protein target CCL2.</strong></font><br />
Oberthür D, Achenbach J, Gabdulkhakov A, Buchner K, Maasch C, Falke S, Rehders D, Klussmann S, Betzel C.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/25901662/" target="_blank" style="color:#520049">Nat Commun, 6:6923 (2015)</a>
<br />  

<a id="ref9"></a><font><strong>[9] Spiegelmer Inhibition of MCP-1/CCR2--Potential as an Adjunct Immunosuppressive Therapy in Transplantation.</strong></font><br />
Kalnins, A. , Thomas, M. N. , Andrassy, M. , Müller, S. , Wagner, A. , Pratschke, S. , Rentsch, M. , Klussmann, S. , Kauke, T. , Angele, M. K. , Bazhin, A. V. , Fischereder, M. , Werner, J. , Guba, M. , & Andrassy, J. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/25970072/" target="_blank" style="color:#520049">Scandinavian journal of immunology, 82(2), 102–109. (2015)</a>
<br />  

<a id="ref10"></a><font><strong>[10] C-C motif-ligand 2 inhibition with emapticap pegol (NOX-E36) in type 2 diabetic patients with albuminuria.</strong></font><br />
Menne, J., Eulberg, D., Beyer, D., Baumann, M., Saudek, F., Valkusz, Z., Więcek, A., Haller, H., & Emapticap Study Group<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/28186566/" target="_blank" style="color:#520049">Nephrology, dialysis, transplantation : official publication of the European Dialysis and Transplant Association - European Renal Association,  32(2), 307–315.  (2017)</a>
<br />

<a id="ref11"></a><font><strong>[11] Systemic Monocyte Chemotactic Protein-1 Inhibition Modifies Renal Macrophages and Restores Glomerular Endothelial Glycocalyx and Barrier Function in Diabetic Nephropathy.</strong></font><br />
Boels, M. G. S. , Koudijs, A. , Avramut, M. C. , Sol, W. M. P. J. , Wang, G. , van Oeveren-Rietdijk, A. M. , van Zonneveld, A. J. , de Boer, H. C. , van der Vlag, J. , van Kooten, C. , Eulberg, D. , van den Berg, B. M. , IJpelaar, D. H. T. , & Rabelink, T. J. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/28837800/" target="_blank" style="color:#520049">The American journal of pathology, 187(11), 2430–2440. (2017)</a>
<br />  