---
layout: theophylline_style
title: Lysozyme aptamer
date: 2024-03-25 00:00:00
author: Yuxuan Sun, Baowei Huang
categories: Aptamer
tags:
- Category:Other
- GC:49.0
- GC:56.00000000000001
- Length:45
- Length:59
- Named:Hen Egg White Lysozyme_Lys1.2minE aptamer
- Named:Hen Egg White Lysozyme_Lys1.2minF aptamer
- Type:Proteins
- Year:2013
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


         
<p class="header_box" id="description">Description</p>
<p>In 2013, M. E. Girvin et al. employed the SELEX method to isolate an aptamer with high affinity for hen egg white lysozyme (HEWL). Concurrently, they examined its structure using X-ray diffraction. This aptamer has a sequence length of 59 nucleotides. M. E. Girvin et al. systematically minimized an RNA aptamer (Lys1) selected against HEWL. The resulting 59-nucleotide compact aptamer, Lys1.2minE, retains nanomolar binding affinity and the ability to inhibit lysozyme's catalytic activity. Specifically, Lys1.2minE inhibits the catalysis of large cell wall substrates but not that of small model substrates. Subsequently, M. E. Girvin et al. shortened the terminal stem of Lys1.2minE to four Watson-Crick base pairs, resulting in Lys1.2minF. They also examined its structure using X-ray diffraction. Lys1.2minF has a sequence length of 45 nucleotides. Its structure may serve as an adaptable protein-binding platform, capable of binding lysozyme with high affinity. It inhibits the cleavage of large substrates by lysozyme, and the bound aptamer has no effect on the catalysis of small substrates<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>RNA aptamers were internally labeled with [α32P]-GTP and allowed to bind to lysozyme for 1h at room temperature. The bound complex was passed through a dual filter dot blot system attached to a vacuum manifold. The nitrocellulose membrane captures the aptamer–protein complexes while unbound aptamers are trapped by the nylon filter directly beneath. Both filters are visualized by phosphor storage imaging, and the fraction protein bound was calculated  by determining the volume of radioactivity retained on the nitrocellulose divided by the total radioactivity retained by both filters. Sedimentation velocity experiments were performed on a Beckman XL-I using a standard procedure. Determining the S20, w and D20, w values of the RNA aptamers under each buffer condition specified by analytical ultracentrifugation. A standard lysozyme activity assay was used to determine the enzymatic activity<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>
<br>
<br>   



<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>In 2013, Girvin et al. utilized the SELEX method to isolate an RNA aptamer, Lys1.2minE, which demonstrated high affinity for hen egg white lysozyme. This 59-nucleotide aptamer effectively inhibited lysozyme's catalytic activity on large cell wall substrates but not on small model substrates. Subsequently, they shortened the aptamer to Lys1.2minF, a 45-nucleotide variant, which retained its high binding affinity and selective inhibition of lysozyme's activity on large substrates. Here we use ribodraw to complete the figures, through the 3D structure information. The Lys1.2minF aptamer and Lys1.2minE aptamer were named by M. E. Girvin et al. in the article<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>Lys1.2minF: 5'-GGGCGGCUAAAGAGUGCAGAGUUACUUAGUUCACUGCAGACGCCC-3'<br></p>
<p>Lys1.2minE: 5'-GGGUUCAUCAGGGCUAAAGAGUGCAGAGUUACUUAGUUCACUGCAGACUUGACGAACCC-3'</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/Lys1.2minF_2D.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/Lys1.2minE_2D.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>


<p class="blowheader_box">3D visualisation</p>             
<p>Girvin et al. resolved the 2.68 Å resolution X-ray crystal structure of the aptamer minF-lysozyme complex. The structures showed that both aptamers interact with the enzyme's active site, inhibiting its catalytic activity. Specifically, Lys1.2minE, with its 59-nucleotide sequence, binds strongly and selectively to large substrates, while Lys1.2minF, a shorter 45-nucleotide variant, retains similar binding properties but with a more compact structure. The PDB ID of this structure is 4M6D<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/Lys1.2minF_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
{
  struct_asym_id: 'F',  
  start_residue_number:1, 
  end_residue_number:129, 
  color:{r:133,g:39,b:181}
}, 
{
  struct_asym_id: 'E',  
  start_residue_number:9, 
  end_residue_number:15, 
  color:{r:184,g:51,b:50}
}, 
{
  struct_asym_id: 'E', 
  start_residue_number:45, 
  end_residue_number:50, 
  color:{r:184,g:51,b:50}
}, 
{
  struct_asym_id: 'E',
  start_residue_number:16, 
  end_residue_number:22, 
  color:{r:194,g:182,b:23}
}, 
{
  struct_asym_id: 'E', 
  start_residue_number:23, 
  end_residue_number:26, 
  color:{r:8,g:69,b:149}
}, 
{
  struct_asym_id: 'E', 
  start_residue_number:41, 
  end_residue_number:44, 
  color:{r:8,g:69,b:149}
}, 
{
  struct_asym_id: 'E', 
  start_residue_number:27, 
  end_residue_number:29, 
  color:{r:46,g:161,b:3}
}, 
{
  struct_asym_id: 'E',  
  start_residue_number:31, 
  end_residue_number:34, 
  color:{r:46,g:161,b:3}
}, 
{
  struct_asym_id: 'E', 
  start_residue_number:37, 
  end_residue_number:40, 
  color:{r:46,g:161,b:3}
}, 
{
  struct_asym_id: 'E',  
  start_residue_number:30, 
  end_residue_number:30, 
  color:{r:241,g:125,b:7}
}, 
{
  struct_asym_id: 'E',
  start_residue_number:35, 
  end_residue_number:36, 
  color:{r:241,g:125,b:7}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/4M6D-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>
<br>

<p>Girvin et al. resolved the 2.00 Å resolution X-ray crystal structure of the aptamer minE-lysozyme complex. The structures showed that both aptamers interact with the enzyme's active site, inhibiting its catalytic activity. Specifically, Lys1.2minE, with its 59-nucleotide sequence, binds strongly and selectively to large substrates, while Lys1.2minF, a shorter 45-nucleotide variant, retains similar binding properties but with a more compact structure. The PDB ID of this structure is 4M4O<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/Lys1.2minE_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">

<!-- Molstar 容器 1-->
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection2 {
            padding-top: 0px;
          }
          .controlsSection2 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox2 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer2{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection2">
            <button onclick="
              var selectSections2 = [
{
  struct_asym_id: 'A',  
  start_residue_number:1, 
  end_residue_number:129, 
  color:{r:133,g:39,b:181}
}, 
{
  struct_asym_id: 'B',  
  start_residue_number:1, 
  end_residue_number:15, 
  color:{r:184,g:51,b:50}
}, 
{
  struct_asym_id: 'B',  
  start_residue_number:45, 
  end_residue_number:59, 
  color:{r:184,g:51,b:50}
}, 
{
  struct_asym_id: 'B',  
  start_residue_number:16, 
  end_residue_number:22, 
  color:{r:194,g:182,b:23}
}, 
{
  struct_asym_id: 'B',  
  start_residue_number:23, 
  end_residue_number:26, 
  color:{r:8,g:69,b:149}
}, 
{
  struct_asym_id: 'B',  
  start_residue_number:41, 
  end_residue_number:44, 
  color:{r:8,g:69,b:149}
}, 
{
  struct_asym_id: 'B',  
  start_residue_number:27, 
  end_residue_number:29, 
  color:{r:46,g:161,b:3}
}, 
{
  struct_asym_id: 'B',  
  start_residue_number:31, 
  end_residue_number:34, 
  color:{r:46,g:161,b:3}
}, 
{
  struct_asym_id: 'B',  
  start_residue_number:37, 
  end_residue_number:40, 
  color:{r:46,g:161,b:3}
}, 
{
  struct_asym_id: 'B',  
  start_residue_number:30, 
  end_residue_number:30, 
  color:{r:241,g:125,b:7}
}, 
{
  struct_asym_id: 'B',  
  start_residue_number:35, 
  end_residue_number:36, 
  color:{r:241,g:125,b:7}
}
              ]
            viewerInstance2.visual.select({ data: selectSections2, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button style="float: left;height:25px;width: 120px;" onclick="viewerInstance2.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection2">
    <!-- Molstar container -->
      <div id="myViewer2"></div>
    </div>
    <script>
      var viewerInstance2 = new PDBeMolstarPlugin();
      var options2 = {
        customData:{
        url:'/pdbfiles/4M4O-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer2 = document.getElementById('myViewer2');
      viewerInstance2.render(viewerContainer2, options2);
  window.addEventListener('load', function() {
    var colorSelectionButton2 = document.querySelector('.controlsSection2 button');
    colorSelectionButton2.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>
<br>



<p class="blowheader_box">Binding pocket</p>             
<p>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 4M6D at 2.68 Å resolution. Lysozymes(shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with Lysozymes.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Lys1.2minF_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Lys1.2minF_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
            
<p>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 4M4O at 2 Å resolution. Lysozymes(shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with Lysozymes.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Lys1.2minE_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Lys1.2minE_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>




<p class="header_box" id="ligand-recognition">Ligand information</p> 
  
<p class="blowheader_box">SELEX ligand</p>
<p>M. E. Girvin et al. distinguished structures Lys1.2 and Lys1. 3 by trimming nucleotides predicted by each to be largely unstructured. Removing different nucleotides to test whether it can binding with lysozyme, and testing its affinity. RNA aptamers were internally labeled with [α-32P]-GTP and allowed to bind to lysozyme for 1 h at room temperature. The bound complex was passed through a dual filter dot blot system attached to a vacuum manifold. The nitrocellulose membrane captures the aptamer–protein complexes while unbound aptamers are trapped by the nylon filter directly beneath. Both filters are visualized by phosphor storage imaging, and the fraction protein bound was calculated by determining the volume of radioactivity retained on the nitrocellulose divided by the total radioactivity retained by both filters<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <!--th onclick="sortTable(0)">Sequence</th-->
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Lys1.2minF aptamer</td>
        <!--td name="td1">5'-GGGCGGCUAAAGAGUGCAGAGUUACUUAGUUCACUGCAGACGCCC-3'</td-->
        <td name="td3">HEWL(lysozyme)</td>
        <td name="td4">57 ± 3 nM</td>
      </tr>
      <tr>
        <td name="td0">Lys1.2minE aptamer</td>
        <!--td name="td1">5'-GGGUUCAUCAGGGCUAAAGAGUGCAGAGUUACUUAGUUCACUGCAGACUUGACGAACCC-3'</td-->
        <td name="td3">HEWL(lysozyme)</td>
        <td name="td4">19 ± 2 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<p>Lysozymes have primarily a bacteriolytic function; those in tissues and body fluids are associated with the monocyte-macrophage system and enhance the activity of immunoagents. Has bacteriolytic activity against M. luteus. Lysozyme C is capable of both hydrolysis and transglycosylation; it shows also a slight esterase activity. It acts rapidly on both peptide-substituted and unsubstituted peptidoglycan, and slowly on chitin oligosaccharides.-----From <a href="https://www.uniprot.org/uniprotkb/P00698/entry" target="_blank" style="color:#520049; text-decoration: underline;"><b>Uniprot</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">Mass</th>
        <th onclick="sortTable(3)">Protein sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P00698/entry" target="_blank" style="color:#520049"><b>P00698</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/panther/PTHR11407/" target="_blank" style="color:#520049"><b>PTHR11407</b></a></td>
        <td name="td2">14.31 kDa</td>
        <td name="td3">
        <div class="sequence-container">

          <span class="full-sequence">KVFGRCELAAAMKRHGLDNYRGYSLGNWVCAAKFESNFNTQATNRNTDGSTDYGILQINSRWWCNDGRTPGSRNLCNIPCSALLSSDITASVNCAKKIVSDGNGMNAWVAWRNRCKGTDVQAWIRGCRL</span>
        </div>
      </td>
        <td name="td4"><a href="https://www.rcsb.org/structure/4WM6" target="_blank" style="color:#520049"><b>4WM6</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/gene/396218" target="_blank" style="color:#520049"><b>396218</b></a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
  <tr>
    <td style="text-align:center;padding:0;">
      <img src="/images/Structure_ligand/lysozyme_stru_ligand.svg" alt="Lysozyme structure ligand" style="width:800px" class="img-responsive">
    </td>
  </tr>
</table>

<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">4WM6-A</td>
        <td name="td2">26</td>
        <td name="td3">0</td>
        <td name="td4">Hen Egg White Lysozyme</td>
      </tr>
      <tr>
        <td name="td1">1LSG-A</td>
        <td name="td2">23</td>
        <td name="td3">1</td>
        <td name="td4">Hen Egg White Lysozyme</td>
      </tr>
      <tr>
        <td name="td1">1NF5-C</td>
        <td name="td2">20.6</td>
        <td name="td3">1.4</td>
        <td name="td4">Alpha-Lactalbumin</td>
      </tr>
      <tr>
        <td name="td1">4OWD-A</td>
        <td name="td2">8.1</td>
        <td name="td3">2.7</td>
        <td name="td4">Membrane-Bound Lytic Murein Transglycosylase f</td>
      </tr>
      <tr>
        <td name="td1">4C5F-A</td>
        <td name="td2">8</td>
        <td name="td3">2.5</td>
        <td name="td4">Membrane-Bound Lytic Murein Transglycosylase c</td>
      </tr>
      <tr>
        <td name="td1">6TA9-A</td>
        <td name="td2">7.9</td>
        <td name="td3">2.5</td>
        <td name="td4">Slt Domain-Containing Protein</td>
      </tr>
      <tr>
        <td name="td1">1QTE-A</td>
        <td name="td2">7.9</td>
        <td name="td3">2.6</td>
        <td name="td4">Soluble Lytic Transglycosylase Slt70</td>
      </tr>
      <tr>
        <td name="td1">3BKH-A</td>
        <td name="td2">7.8</td>
        <td name="td3">2.8</td>
        <td name="td4">Lytic Transglycosylase</td>
      </tr>
      <tr>
        <td name="td1">5E27-A</td>
        <td name="td2">7.8</td>
        <td name="td3">2.5</td>
        <td name="td4">Resuscitation-Promoting Factor Rpfb</td>
      </tr>
      <tr>
        <td name="td1">4YIM-B</td>
        <td name="td2">7.7</td>
        <td name="td3">2.8</td>
        <td name="td4">Putative Soluble Lytic Murein Transglycosylase</td>
      </tr>
      <tr>
        <td name="td1">7LAM-A</td>
        <td name="td2">7.5</td>
        <td name="td3">3.1</td>
        <td name="td4">Lytic Transglycosylase Domain-Containing Protein</td>
      </tr>
	  </tbody>
  </table>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] An RNA aptamer possessing a novel monovalent cation-mediated fold inhibits lysozyme catalysis by inhibiting the binding of long natural substrates.</strong></font><br />
Padlan, C. S., Malashkevich, V. N., Almo, S. C., Levy, M., Brenowitz, M., & Girvin, M. E.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/24570482/" target="_blank" style="color:#520049" >RNA, 20(4), 447–461. (2013)</a>
<br />
                