---
layout: theophylline_style
title: Mitochondrion aptamer
date: 2024-05-17 00:00:00
author: Zhizhong Lu, Ying Ao
categories: Aptamer
tags:
- Category:Other
- GC:47.22
- GC:50.0
- GC:51.39
- GC:54.169999999999995
- Length:72
- Named:Mitochondria_C1
- Named:Mitochondria_C16
- Named:Mitochondria_C20
- Named:Mitochondria_C32
- Type:Cells
- Year:2002
type: Cells
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12024047/" target="_blank" style="color:#520049">2002</a></h3>
  </div>
  <div class="body">
    <p>To identify the sequences recognized by import receptors, used SELEX method selected  several aptamers<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/20348443/" target="_blank" style="color:#520049">2010</a></h3>
  </div>
  <div class="body">
    <p>Used the approach of in vitro selection, some aptamers that can be imported into yeast and human mitochondria were isolated<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/25087963/" target="_blank" style="color:#520049">2014</a></h3>
  </div>
  <div class="body">
    <p>The use of mitochondria-based  aptamer by exponential enrichment (SELEX) was explored<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/27056631/" target="_blank" style="color:#520049">2016</a></h3>
  </div>
  <div class="body">
    <p>Reported here on the development of a dual-ligand liposomal system composed of octaarginine (R8) and an RP aptamer for mitochondrial targeted<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
  </div>
 </div>
</div>



<p class="header_box" id="description">Description</p>
<p>In 2002，Bhattacharyya et al. utilized SELEX to select a diverse set of highly efficient mitochondrial import aptamers from a random sequence library. These aptamers are then referenced against the tRNA sequence database to identify potential import signals within various domains of the tRNA. In 2010, Kolesnikova, O. et al. tried SELEX to mitochondria to improve the efficacy of importable yeast and human tRNA by changing a part of sequence in natural tRNA<sup>[<a href="#ref1" style="color:#520049">1</a>,<a href="#ref2" style="color:#520049">2</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>A pool of random 16-mer sequences was incubated with isolated mitochondria in the presence of ATP. Non-imported RNA was digested with RNase, following which the mitochondria were washed. Mitochondrial RNA was then recovered and amplified using reverse transcription-PCR (RT-PCR) with specific 5′ and 3′ primers. RNA was synthesized from the amplified DNA with T7 RNA polymerase, and the import-amplification cycle was repeated. Pool 4 DNA (obtained after the fourth cycle) was digested with EcoRI and HindIII and cloned into pUC19 vector. In 2010, the SELEX method used RNA library was generated by T7 transcription on a 73-mer oligonucleotide corresponding to yeast tRNA<sup>Lys</sup><sub>CUU</sub>(tRK1) that contained 14 random nucleotides in positions that differ between imported tRK1 and tRNA<sup>Lys</sup><sub>U*UU</sub> (tRK2). The library therefore contained 2.7 × 10<sup>8</sup> possible sequences. And used two strategies to select aptamers<sup>[<a href="#ref1" style="color:#520049">1</a>,<a href="#ref2" style="color:#520049">2</a>]</sup>.
<br>


<p class="header_box" id="Structure">Structure</p>
<p>The 2D structure of the figures is based on the article by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>
<p>C1 aptamer: </p>
<p>5'-GGGACGCAGGGACUGUACCUUCUAGGCUGGGUCAUGUGACUGUAGCGG-3'<br></p>
<p>C32 aptamer: </p>
<p>5'-GGGACGCAGGGACUGUACGGAUCAGAGUCUCCUAUGUGACUGUAGCGG-3'</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/mitoch_aptamer_2D1.svg" alt="drawing" style="width:500px;height:400px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/mitoch_aptamer_2D2.svg" alt="drawing" style="width:500px;height:400px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>



<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>A mitochondrion is an organelle found in the cells of most eukaryotes, such as animals, plants and fungi. Mitochondria have a double membrane structure and use aerobic respiration to generate adenosine triphosphate (ATP), which is used throughout the cell as a source of chemical energy. They were discovered by Albert von Kölliker in 1857 in the voluntary muscles of insects. The term mitochondrion was coined by Carl Benda in 1898. The mitochondrion is popularly nicknamed the "powerhouse of the cell", a phrase coined by Philip Siekevitz in a 1957 article of the same name.-----From WiKi</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">GenBase Gene</th>
        <th onclick="sortTable(2)">GenBase protein</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Mitochondrion</td>
        <td name="td1"><a href="https://ngdc.cncb.ac.cn/genbase/search/gb/MF074188.1" target="_blank" style="color:#520049"><b>MF074188.1</b></a></td>
        <td name="td2"><a href="https://ngdc.cncb.ac.cn/genbase/search/index?accession=Mitochondrion&dataType=Protein&dataSource=" target="_blank" style="color:#520049"><b>Mitochondrion</b></a></td>
      </tr>
	  </tbody>
  </table>

  <p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
      <td name="td0">C1 aptamer</td>
      <td name="td1">5'-GGGACGCAGGGACUGUACCUUCUAGGCUGGGUCAUGUGACUGUAGCGG-3'</td>
      <td name="td2">Mitochondrion the outer membrane</td>
      <td name="td3">1.0 nM</td>
    </tr>
            
     <tr>
      <td name="td0">C16 aptamer</td>
      <td name="td1">5'-GGGACGCAGGGACUGUACGGUAAGUCCCCGGGCAUGUGACUGUAGCGG-3'</td>
      <td name="td2">Mitochondrion the outer membrane</td>
      <td name="td3">0.46 nM</td>
    </tr>
            
     <tr>
      <td name="td0">C20 aptamer</td>
      <td name="td1">5'-GGGACGCAGGGACUGUAUGGUUUUCUGGGGUUUAUGUGACUGUAGCGG-3'</td>
      <td name="td2">Mitochondrion the outer membrane</td>
      <td name="td3">1.19 nM</td>
    </tr>
            
     <tr>
      <td name="td0">C32 aptamer</td>
      <td name="td1">5'-GGGACGCAGGGACUGUACGGAUCAGAGUCUCCUAUGUGACUGUAGCGG-3'</td>
      <td name="td2">Mitochondrion the outer membrane</td>
      <td name="td3">0.1 nM</td>
    </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/mitoch_SELEX_ligand.svg" alt="drawing" style="width:1000px;height:400px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Mitochondrial RNA import in Leishmania tropica: aptamers homologous to multiple tRNA domains that interact cooperatively or antagonistically at the inner membrane.</strong></font><br />
Bhattacharyya, S. N., Chatterjee, S., & Adhya, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12024047/" target="_blank" style="color:#520049">Molecular and cellular biology, 22(12), 4372–4382. (2002)</a>
<br/>
            
<a id="ref2"></a><font><strong>[2] Selection of RNA aptamers imported into yeast and human mitochondria.</strong></font><br />
Kolesnikova, O., Kazakova, H., Comte, C., Steinberg, S., Kamenski, P., Martin, R. P., Tarassov, I., & Entelis, N.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/20348443/" target="_blank" style="color:#520049">RNA (New York, N.Y.), 16(5), 926–941. (2010)</a>
<br/>
            
<a id="ref3"></a><font><strong>[3] RNA aptamers for targeting mitochondria using a mitochondria-based SELEX method.</strong></font><br />
Tawaraya, Y., Hyodo, M., Ara, M. N., Yamada, Y., & Harashima, H.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/25087963/" target="_blank" style="color:#520049">Biological & pharmaceutical bulletin, 37(8), 1411–1415. (2014)</a>
<br/>
            
<a id="ref4"></a><font><strong>[4] A Dual-Ligand Liposomal System Composed of a Cell-Penetrating Peptide and a Mitochondrial RNA Aptamer Synergistically Facilitates Cellular Uptake and Mitochondrial Targeting.</strong></font><br />
Yamada, Y., Furukawa, R., & Harashima, H.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/27056631/" target="_blank" style="color:#520049">Journal of pharmaceutical sciences, 105(5), 1705–1713. (2016)</a>
<br/>



<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>




    </html>
