---
layout: theophylline_style
title:  "NeomycinB aptamer"
date:   2024-04-22 00:00:00
author: Linfei Li，Yuhang Luo
categories: Aptamer
---
<html>
<head>
  <style>

  </style>
</head>
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
 


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/9383458/" target="_blank" style="color:#520049">1995</a></h3>
    </div>
    <div class="body">
      <p>The aptamer was selected for the first time<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10425683/" target="_blank" style="color:#520049">1999</a></h3>
    </div>
    <div class="body">
      <p>The neomycin-B–RNA aptamer in complex structure was discovered by NMR<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10908357/" target="_blank" style="color:#520049">2000</a></h3>
    </div>
    <div class="body">
      <p>The principal mechanism for recognition and binding of neomycin B to the RNA major groove is mediated by hydrogen bonding<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/18000033/" target="_blank" style="color:#520049">2008</a></h3>
    </div>
    <div class="body">
      <p>N1 neomycin found using SELEX technology and in vitro selection<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/19217276/" target="_blank" style="color:#520049">2009</a></h3>
    </div>
    <div class="body">
      <p>SPR assay of small molecules using modified neomycin B<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/20478079/" target="_blank" style="color:#520049">2010</a></h3>
    </div>
    <div class="body">
      <p>Neomycin-B binding aptamers have been successfully selected in yeast and applied as riboswitches<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/24757168/" target="_blank" style="color:#520049">2014</a></h3>
    </div>
    <div class="body">
      <p>The pentaloop provides  neomycin-B–RNA aptamer with the ability to adapt to external influences on its structure<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup></p>
    </div>
  </div>
<div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/26661511/" target="_blank" style="color:#520049">2016</a></h3>
    </div>
    <div class="body">
      <p>Discovery of Neomycin as an Efficient Recognition Ligand for Induction Switches<sup>[<a href="#ref9" style="color:#520049">9</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/30462266/" target="_blank" style="color:#520049">2019</a></h3>
    </div>
    <div class="body">
      <p>Two-step binding mechanism of neomycin to its RNA consortium revealed by structure-guided fluorescent labeling<sup>[<a href="#ref11" style="color:#520049">11</a>]</sup></p>
    </div>
  </div>
</div>

  
<p class="header_box" id="description">Description</p>
<p>In 1995, Famulok, M. et al. and colleagues utilised in vitro selection techniques to isolate aptamers with high-affinity binding sites for neomycin B. In 1999, Patel, D. J. et al. elucidated the structure of the neomycin B-RNA aptamer complex using NMR spectroscopy. In 2000, Natarajan, K. and associates employed isothermal titration calorimetry to examine the thermodynamic parameters of a 23-nucleotide hairpin RNA aptamer complexed with the aminoglycoside neomycin B. They also evaluated the relative contributions of electrostatic interactions and hydrogen bonding to binding affinity by examining the metal-ion dependence of these binding parameters<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref2" style="color:#520049">2</a></sup><sup>,<a href="#ref3" style="color:#520049">3</a></sup><sup>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 1995, during the in vitro selection experiments, a pool of 74-nucleotide random sequences was applied to neomycin-derived agarose affinity columns. Non-specific or weakly bound RNAs were removed by washing, and the progress of the screen was assessed through neomycin enrichment. This enrichment was achieved by incrementally increasing the number of buffer washes in the first three rounds, from 5 to 30 to 50, respectively. Rounds 3 to 6 employed 50 buffer washes, resulting in a 13-fold increase in enrichment prior to the affinity wash. A significant increase in enrichment was observed in round 8, sequencing and of 21 clones from this round revealed that approximately 50% of the RNA sequences were identical. The aptamer was selected using the Capture-SELEX method<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>


<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>Here we used ribodraw to complete the figure, through the 3D structure information<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p>
<p>5'-GGACUGGGCGAGAAGUUUAGUCC-3'</p>
<img src="/images/2D/NeomycinB_aptamer_2D.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">3D visualisation</p>             
<p>Patel, D. J. and colleagues present the solution structure as determined by specific features of aminoglycoside antibiotic–RNA recognition have been probed using chemical, biochemical,spectroscopic and computational approaches on both natural RNA targets and RNA aptamers identified through in vitro selection. Specifically G9, G10, A16, G18, U19 and U21 bind to the neomycin B complex (partial hydrogen bonding predicted to be generated by pymol, see references for more details). The PDB ID of this structure is 1NEM<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/NeomycinB_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A',    
  start_residue_number:4, 
  end_residue_number:8, 
  color:{r:184,g:51,b:50}
},
{
  struct_asym_id: 'A', 
  start_residue_number:9, 
  end_residue_number:10, 
  color:{r:241,g:125,b:7}
},{
  struct_asym_id: 'A', 
  start_residue_number:11, 
  end_residue_number:12, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'A', 
  start_residue_number:13, 
  end_residue_number:15, 
  color:{r:46,g:161,b:3}
},
{
  struct_asym_id: 'A', 
  start_residue_number:16, 
  end_residue_number:16, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'A', 
  start_residue_number:17, 
  end_residue_number:17, 
  color:{r:46,g:161,b:3}
},{
   struct_asym_id: 'A', 
  start_residue_number:18, 
  end_residue_number:19, 
  color:{r:241,g:125,b:7}
},{
   struct_asym_id: 'A', 
  start_residue_number:20, 
  end_residue_number:20, 
  color:{r:8,g:69,b:149}
},{
  struct_asym_id: 'A',    
  start_residue_number:21, 
  end_residue_number:21, 
  color:{r:241,g:125,b:7}
},{
  struct_asym_id: 'A',    
  start_residue_number:22, 
  end_residue_number:26, 
  color:{r:184,g:51,b:50}
},{
  struct_asym_id: 'B',    
  start_residue_number:1, 
  end_residue_number:1, 
  color:{r:255,g:0,b:255}
},{
  struct_asym_id: 'B',    
  start_residue_number:2, 
  end_residue_number:2, 
  color:{r:255,g:0,b:255}
},{
  struct_asym_id: 'A',    
  start_residue_number:27, 
  end_residue_number:28, 
  color:{r:255,g:0,b:255}
}
       ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1nem.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<p>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 1NEM by NMR. Neomycin (shown in sticks) is labeled in magenta. Right: The hydrogen bonds of binding sites of the aptamer bound with Neomycin.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/NeomycinB_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/NeomycinB_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>




<p class="header_box" id="ligand-recognition">Ligand information</p>
<p class="blowheader_box">SELEX ligand</p>
<p>Famulok, M. and colleagues employed a range of methodologies, including isocratic elution from neomycin and equilibrium gel filtration techniques, to determine the dissociation constant of the neomycin complex both on the column and in solution. These approaches were used to provide a comprehensive assessment of the stability and affinity of neomycin<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<!--div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Neomycin B</td>
        <td name="td1">100nM</td>
      </tr>
      <tr>
        <td name="td0">paromomycin</td>
        <td name="td1">10mM</td>
      </tr>
	  </tbody>
  </table>
  </div-->
  <div style="display: flex; justify-content: center;"></div>
  <img src="/images/SELEX_ligand/NeomycinB_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef!important;display:block;margin:0 auto;border:solid 1px;border-radius:0;" class="img-responsive">
<br>

<p class="blowheader_box">Structure ligand</p>
<p>Neomycin is a broad-spectrum aminoglycoside antibiotic derived from the metabolic products of Streptomyces fradiae. It is a complex composed of three components: neomycin A, B, and C. Neomycin B, also known as framycetin, is the most active component of the complex, while neomycin C is an isomer of neomycin B, making these two stereoisomers the active constituents of neomycin. Neomycin A, or neamine, serves as a linking moiety that joins two molecules of neomycin B and C together.-----From <a href="https://go.drugbank.com/drugs/DB00994" target="_blank" style="color:#520049; text-decoration: underline;"><b>Drugbank</b></a></p>

<p class="dot-paragraph"><b>PubChem CID:</b> a unique identifier for substances in the <a href="https://pubchem.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>PubChem database</b></a>.</p>
<p class="dot-paragraph"><b><a href="https://commonchemistry.cas.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>CAS</b></a> number:</b> a global registry number for chemical substances.</p>
<p class="dot-paragraph"><b><a href="https://go.drugbank.com/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Drugbank</b></a>:</b> a comprehensive database with detailed information on drugs and drug targets.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(6)">Name</th>
        <th onclick="sortTable(0)">PubChem CID</th>
        <th onclick="sortTable(1)">Molecular Formula</th>
        <th onclick="sortTable(2)">MW</th>
        <th onclick="sortTable(3)">CAS</th>
        <th onclick="sortTable(4)">Solubility</th>
        <th onclick="sortTable(5)">Drugbank ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td6">NeomycinB</td>
        <td name="td0"><a href="https://pubchem.ncbi.nlm.nih.gov/compound/8378" target="_blank" style="color:#520049"><b>8378</b> </a></td>
        <td name="td1">C<sub>23</sub>H<sub>46</sub>N<sub>6</sub>O<sub>13</sub></td>
        <td name="td2">614.6 g/mol</td>
        <td name="td3"><a href="https://commonchemistry.cas.org/detail?cas_rn=119-04-0" target="_blank" style="color:#520049"><b>119-04-0</b></a></td>
        <td name="td4">50 mg/ml (at room temperature)</td>
        <td name="td5"><a href="https://go.drugbank.com/drugs/DB00994" target="_blank" style="color:#520049"><b>DB00994</b> </a></td>
      </tr>
	  </tbody>
  </table>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/NeomycinB_stru_ligand1.svg" alt="drawing" style="width:400px"  px="" /></td>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/NeomycinB_stru_ligand2.svg" alt="drawing" style="width:400px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


                    
<p class="blowheader_box">Similar compound(s)</p>
<p>We screened the compounds with great similarity to NeomycinB by using the <a href="https://zinc15.docking.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>ZINC database</b></a> and showed some of the compounds' structure diagrams. For some CAS numbers not available,we will supplement them with Pubchem CID.</p>

<p class="dot-paragraph"><b>ZINC ID:</b> a compound identifier used by the <a href="https://zinc15.docking.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>ZINC database</b></a>, one of the largest repositories for virtual screening of drug-like molecules.</p>
<p class="dot-paragraph"><b>PubChem CID:</b> a unique identifier for substances in the <a href="https://pubchem.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>PubChem database</b></a>.</p>
<p class="dot-paragraph"><b><a href="https://commonchemistry.cas.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>CAS</b></a> number:</b> a global registry number for chemical substances.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">ZINC ID</th>
        <th onclick="sortTable(2)">Name</th>
        <th onclick="sortTable(3)">CAS</th>
        <th onclick="sortTable(4)">Pubchem CID</th>
        <th onclick="sortTable(5)">Structure</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1"><a href="https://zinc15.docking.org/substances/ZINC71928291/" target="_blank" style="color:#520049"><b>ZINC71928291</b></a></td>
        <td name="td2">Framycetin</td>
        <td name="td3"><a href="https://commonchemistry.cas.org/detail?cas_rn=119-04-0" target="_blank" style="color:#520049"><b>119-04-0</b></a></td>
        <td name="td4"><a href="https://pubchem.ncbi.nlm.nih.gov/compound/8378" target="_blank" style="color:#520049"><b>8378</b></a></td>
        <td name="td5"><img src="/images/Similar_compound/NeomycinB_Simi_compound1.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
      <tr>
        <td name="td1"><a href="https://zinc15.docking.org/substances/ZINC85536952/" target="_blank" style="color:#520049"><b>ZINC85536952</b></a></td>
        <td name="td2">(2R,3R,4S,5R,6S)-5-amino-2-(aminomethyl)-6-[(1R,2R,3S,4R,6S)-4,6-diamino-2-[(2S,3R,4S,5R)-4-[(2R,3R,4R,5S,6S)-3-amino-6-(aminomethyl)-4,5-dihydroxyoxan-2-yl]oxy-3-hydroxy-5-(hydroxymethyl)oxolan-2-yl]oxy-3-hydroxycyclohexyl]oxyoxane-3,4-diol</td>
        <td name="td3">NA</td>
        <td name="td4"><a href="https://pubchem.ncbi.nlm.nih.gov/compound/70685100" target="_blank" style="color:#520049"><b>70685100</b></a></td>
        <td name="td5"><img src="/images/Similar_compound/NeomycinB_Simi_compound2.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
      <tr>
        <td name="td1"><a href="https://zinc15.docking.org/substances/ZINC60183170/" target="_blank" style="color:#520049"><b>ZINC60183170</b></a></td>
        <td name="td2">paromomycin</td>
        <td name="td3">NA</td>
        <td name="td4"><a href="https://pubchem.ncbi.nlm.nih.gov/compound/165580" target="_blank" style="color:#520049"><b>165580</b></a></td>
        <td name="td5"><img src="/images/Similar_compound/NeomycinB_Simi_compound3.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
      <tr>
        <td name="td1"><a href="https://zinc15.docking.org/substances/ZINC72186786/" target="_blank" style="color:#520049"><b>ZINC72186786</b></a></td>
        <td name="td2">(2R,3S,4R,5R,6R)-5-amino-2-(aminomethyl)-6-[(1R,2R,3S,4R,6S)-4,6-diamino-2-[(2S,3R,4S,5R)-4-[(2S,3R,4R,5S,6S)-3-amino-6-(aminomethyl)-4,5-dihydroxyoxan-2-yl]oxy-3-hydroxy-5-(hydroxymethyl)oxolan-2-yl]oxy-3-hydroxycyclohexyl]oxyoxane-3,4-diol</td>
        <td name="td3">NA</td>
        <td name="td4"><a href="https://pubchem.ncbi.nlm.nih.gov/compound/24871322" target="_blank" style="color:#520049"><b>24871322</b></a></td>
        <td name="td5"><img src="/images/Similar_compound/NeomycinB_Simi_compound4.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
      <tr>
        <td name="td1"><a href="https://zinc15.docking.org/substances/ZINC261494548/" target="_blank" style="color:#520049"><b>ZINC261494548</b></a></td>
        <td name="td2">(2R,3S,4R,5R,6R)-5-amino-2-(aminomethyl)-6-[(1R,2S,3S,4R,6S)-4,6-diamino-2-[(2S,3R,4S,5R)-4-[(2S,3R,4R,5S,6S)-3-amino-6-(aminomethyl)-4,5-dihydroxyoxan-2-yl]oxy-3-hydroxy-5-(hydroxymethyl)oxolan-2-yl]oxy-3-hydroxycyclohexyl]oxyoxane-3,4-diol</td>
        <td name="td3">NA</td>
        <td name="td4"><a href="https://pubchem.ncbi.nlm.nih.gov/compound/125116806" target="_blank" style="color:#520049"><b>125116806</b></a></td>
        <td name="td5"><img src="/images/Similar_compound/NeomycinB_Simi_compound5.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
      <tr>
        <td name="td1"><a href="https://zinc15.docking.org/substances/ZINC245219488/" target="_blank" style="color:#520049"><b>ZINC245219488</b></a></td>
        <td name="td2">(2R,3S,4R,5R,6S)-5-amino-2-(aminomethyl)-6-[(1S,2R,3S,4R,6S)-4,6-diamino-2-[(2S,3R,4S,5R)-4-[(2S,3R,4R,5S,6S)-3-amino-6-(aminomethyl)-4,5-dihydroxyoxan-2-yl]oxy-3-hydroxy-5-(hydroxymethyl)oxolan-2-yl]oxy-3-hydroxycyclohexyl]oxyoxane-3,4-diol</td>
        <td name="td3">NA</td>
        <td name="td4"><a href="https://pubchem.ncbi.nlm.nih.gov/compound/124762241" target="_blank" style="color:#520049"><b>124762241</b></a></td>
        <td name="td5"><img src="/images/Similar_compound/NeomycinB_Simi_compound6.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
      <tr>
        <td name="td1"><a href="https://zinc15.docking.org/substances/ZINC245219490/" target="_blank" style="color:#520049"><b>ZINC245219490</b></a></td>
        <td name="td2">(2R,3R,4R,5R,6S)-5-amino-2-(aminomethyl)-6-[(1S,2R,3S,4R,6S)-4,6-diamino-2-[(2S,3R,4S,5R)-4-[(2S,3R,4R,5S,6S)-3-amino-6-(aminomethyl)-4,5-dihydroxyoxan-2-yl]oxy-3-hydroxy-5-(hydroxymethyl)oxolan-2-yl]oxy-3-hydroxycyclohexyl]oxyoxane-3,4-diol</td>
        <td name="td3">NA</td>
        <td name="td4"><a href="https://pubchem.ncbi.nlm.nih.gov/compound/124762243" target="_blank" style="color:#520049"><b>124762243</b></a></td>
        <td name="td5"><img src="/images/Similar_compound/NeomycinB_Simi_compound7.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
      <tr>
        <td name="td1"><a href="https://zinc15.docking.org/substances/ZINC169719177/" target="_blank" style="color:#520049"><b>ZINC169719177</b></a></td>
        <td name="td2">(2R,3S,4R,5R,6R)-5-amino-2-(aminomethyl)-6-[(1S,2R,3S,4R,6S)-4,6-diamino-2-[(2S,3R,4S,5R)-4-[(2S,3R,4R,5S,6S)-3-amino-6-(aminomethyl)-4,5-dihydroxyoxan-2-yl]oxy-3-hydroxy-5-(hydroxymethyl)oxolan-2-yl]oxy-3-hydroxycyclohexyl]oxyoxane-3,4-diol</td>
        <td name="td3">NA</td>
        <td name="td4"><a href="https://pubchem.ncbi.nlm.nih.gov/compound/124605136" target="_blank" style="color:#520049"><b>124605136</b></a></td>
        <td name="td5"><img src="/images/Similar_compound/NeomycinB_Simi_compound8.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
	  </tbody>
  </table>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] A novel RNA motif for neomycin recognition.</strong></font><br />
Wallis, M. G., von Ahsen, U., Schroeder, R., & Famulok, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/9383458/" target="_blank" style="color:#520049" >Chemistry & biology, 2(8), 543–552 (1995)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Saccharide-RNA recognition in a complex formed between neomycin B and an RNA aptamer.</strong></font><br />
Jiang, L., Majumdar, A., Hu, W., Jaishree, T. J., Xu, W., & Patel, D. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10425683/" target="_blank" style="color:#520049"> Structure (London, England : 1993), 7(7), 817–827 (1999)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] Recognition of a cognate RNA aptamer by neomycin B: quantitative evaluation of hydrogen bonding and electrostatic interactions.</strong></font><br />
Cowan, J. A., Ohyama, T., Wang, D., & Natarajan, K<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10908357/" target="_blank" style="color:#520049">Nucleic acids research, 28(15), 2935–2942. (2000)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] Screening for engineered neomycin riboswitches that control translation initiation.</strong></font><br />
Weigand, J. E., Sanchez, M., Gunnesch, E. B., Zeiher, S., Schroeder, R., & Suess, B.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18000033/" target="_blank" style="color:#520049"> RNA (New York, N.Y.), 14(1), 89–97 (2008)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] SPR sensing of small molecules with modified RNA aptamers: detection of neomycin B.</strong></font><br />
de-los-Santos-Alvarez, N., Lobo-Castañón, M. J., Miranda-Ordieres, A. J., & Tuñón-Blanco, P. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/19217276/" target="_blank" style="color:#520049">Biosensors & bioelectronics, 24(8), 2547–2553. (2009)</a>
<br />
                
<a id="ref6"></a><font><strong>[6] Fluorescent nucleic acid base analogues.</strong></font><br />
Wilhelmsson L. M.  Quarterly reviews of biophysics, 43(2), 159–183.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/20478079/" target="_blank" style="color:#520049">Quarterly reviews of biophysics, 43(2), 159–183 (2010)</a>
<br />
                
<a id="ref7"></a><font><strong>[7] A strategy to enhance the binding affinity of fluorophore-aptamer pairs for RNA tagging with neomycin conjugation.</strong></font><br />
Jeon, J., Lee, K. H., & Rao, J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/22951899/" target="_blank" style="color:#520049"> Chemical communications (Cambridge, England), 48(80), 10034–10036. (2012)</a>
<br />
                
<a id="ref8"></a><font><strong>[8] An adaptable pentaloop defines a robust neomycin-B RNA aptamer with conditional ligand-bound structures.</strong></font><br />
Ilgu, M., Fulton, D. B., Yennamalli, R. M., Lamm, M. H., Sen, T. Z., & Nilsen-Hamilton, M.. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/24757168/" target="_blank" style="color:#520049">RNA (New York, N.Y.), 20(6), 815–824. (2014)</a>
<br />
                
<a id="ref9"></a><font><strong>[9] What a Difference an OH Makes: Conformational Dynamics as the Basis for the Ligand Specificity of the Neomycin-Sensing Riboswitch.</strong></font><br />
Duchardt-Ferner, E., Gottstein-Schmidtke, S. R., Weigand, J. E., Ohlenschläger, O., Wurm, J. P., Hammann, C., Suess, B., & Wöhnert, J<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/26661511/" target="_blank" style="color:#520049">Angewandte Chemie (International ed. in English), 55(4), 1527–1530. (2016)</a>
<br />
                
<a id="ref10"></a><font><strong>[10] A self-assembling RNA aptamer-based nanoparticle sensor for fluorometric detection of Neomycin B in milk.</strong></font><br />
Ling, K., Jiang, H., Zhang, L., Li, Y., Yang, L., Qiu, C., & Li, F. R.  <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/26942739/" target="_blank" style="color:#520049">Analytical and bioanalytical chemistry, 408(13), 3593–3600. (2016)</a>
<br />
                
<a id="ref11"></a><font><strong>[11] Structure guided fluorescence labeling reveals a two-step binding mechanism of neomycin to its RNA aptamer.</strong></font><br />
Gustmann, H., Segler, A. J., Gophane, D. B., Reuss, A. J., Grünewald, C., Braun, M., Weigand, J. E., Sigurdsson, S. T., & Wachtveitl, J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/30462266/" target="_blank" style="color:#520049"> Nucleic acids research, 47(1), 15–28. (2019)</a>
<br />
                
<a id="ref12"></a><font><strong>[12] Aptamer protective groups tolerate different reagents and reactions for regioselective modification of neomycin B.</strong></font><br />
Bastian, A. A., Gruszka, A., Jung, P., & Herrmann, A<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/33237073/" target="_blank" style="color:#520049">Organic & biomolecular chemistry, 18(47), 9606–9610 (2020)</a>
<br />