---
layout: theophylline_style
title: OX40 aptamer
date: 2024-05-31 00:00:00
author: Zhijie Tan, Qiaozhen Liu
categories: Aptamer
tags:
- Category:Mammalian
- GC:56.99999999999999
- GC:63.21
- GC:69.37
- Length:106
- Length:111
- Length:54
- Named:OX40_11F11
- Named:OX40_9C7
- Named:OX40_9C7 aptamer (truncation)
- Type:Proteins
- Year:2012
- Year:2013
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="description">Description</p>
<p>In 2013, Pratico et al. isolated the human OX40 (hOX40) aptamer using systematic evolution of ligands by exponential enrichment. This aptamer binds the target purified protein with high affinity [dissociation constants (Kd)<10 nM]<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2011, Pratico et al. identified isolated specific sequences for targeting HA in a 40nt sequence pool through 9 and 11 rounds of selection<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>9C7 was the aptamer sequence mainly studied in the article, which had a high affinity with OX40. The 2D structure of the figure is based on the article by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>
<p>5'-AAAAAAGAACACUUCCGAUUAGGGCCCACCCUAACGGCCGCAG-3'</p>
<img src="/images/2D/OX40_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>TNFRSF4 (also known as OX40, ACT35, CD134, IMD16, TXGP1L) activates NF-kappaB through its interaction with adaptor proteins TRAF2 and TRAF5. It also promotes the expression of apoptosis inhibitors BCL2 and BCL2lL1/BCL2-XL, and thus suppresses apoptosis. It is primarily expressed on activated CD4+ and CD8+ T cells, where it is transiently expressed and upregulated on the most recently antigen-activated T cells within inflammatory lesions. This makes it an attractive target to modulate immune responses, i.e. TNFRSF4 (OX40) blocking agents to inhibit adverse inflammation or agonists to enhance immune responses. An artificially created biologic fusion protein, OX40-immunoglobulin (OX40-Ig), prevents OX40 from reaching the T-cell receptors, thus reducing the T-cell response. Some single nucleotide polymorphisms (SNPs) of its natural ligand OX40 ligand (OX40L, CD252), which is also found on activated T cells, have been associated with systemic lupus erythematosus.-----from Pfam</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">OX40</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P43489/entry" target="_blank" style="color:#520049"><b>P43489</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/cdd/CD13406/" target="_blank" style="color:#520049"><b>CD13406</b></a></td>
        <td name="td3">17.00 kDa</td>
        <td name="td4">LHCVGDTYPSNDRCCHECRPGNGLVSRCSRSQNTVCRPCGPGFYNDVVSSKPCKPCTWCNLRSGSERKQLCTATQDTVCRCRAGTQPLDSYKPGVDCAPCPPGHFSPGDNQACKPWTNCTLAGKHTLQPASNSSDAICEDRDMDPGGSHHHHHHHH</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/8AG1" target="_blank" style="color:#520049"><b>8AG1</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/protein/7YK4_A" target="_blank" style="color:#520049"><b>7YK4_A</b></a></td>
      </tr>
	  </tbody>
  </table>

<p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">9C7</td>
        <td name="td1">5'-AAAAAAGAACACUUCCGAUUAGGGCCCACCCUAACGGCCGCAG-3'</td>
        <td name="td2">hOX40</td>
        <td name="td3">1.7 nM</td>
      </tr>
      <tr>
        <td name="td0">11F11</td>
        <td name="td1">5'-AACGGGACCACCCAUGCCAGGGGACCACCUCAGGCAGCGCCAGACGAC-3'</td>
        <td name="td2">hOX40</td>
        <td name="td3">10 nM</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/OX40_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">2HEV-R</td>
      <td name="td1">15.5</td>
      <td name="td2">2.4</td>
      <td name="td3">Tumor necrosis factor ligand superfamily member 4</td>
    </tr>
            
     <tr>
      <td name="td0">6PE9-G</td>
      <td name="td1">10.0</td>
      <td name="td2">2.7</td>
      <td name="td3">Fab heavy chain</td>
    </tr>
            
     <tr>
      <td name="td0">4YN0-A</td>
      <td name="td1">9.7</td>
      <td name="td2">2.7</td>
      <td name="td3">Tumor necrosis factor receptor superfamily member</td>
    </tr>
            
     <tr>
      <td name="td0">3ALQ-S</td>
      <td name="td1">9.5</td>
      <td name="td2">3.1</td>
      <td name="td3">Tumor necrosis factor</td>
    </tr>
            
     <tr>
      <td name="td0">1EXT-A</td>
      <td name="td1">9.3</td>
      <td name="td2">4.3</td>
      <td name="td3">Tumor necrosis factor receptor</td>
    </tr>
            
     <tr>
      <td name="td0">7KX0-D</td>
      <td name="td1">9.2</td>
      <td name="td2">2.2</td>
      <td name="td3">Cd70 antigen</td>
    </tr>
            
     <tr>
      <td name="td0">3ME4-A</td>
      <td name="td1">9.1</td>
      <td name="td2">4.7</td>
      <td name="td3">Tumor necrosis factor receptor superfamily member</td>
    </tr>
            
     <tr>
      <td name="td0">7MSG-D</td>
      <td name="td1">8.7</td>
      <td name="td2">2.4</td>
      <td name="td3">Tumor necrosis factor ligand superfamily member 1</td>
    </tr>
            
     <tr>
      <td name="td0">2UWI-A</td>
      <td name="td1">8.6</td>
      <td name="td2">4.1</td>
      <td name="td3">Crme protein</td>
    </tr>
            
     <tr>
      <td name="td0">4KGQ-D</td>
      <td name="td1">8.3</td>
      <td name="td2">3.4</td>
      <td name="td3">Tumor necrosis factor receptor superfamily member</td>
    </tr>
    </tbody>
  </table>
                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Identification and characterization of an agonistic aptamer against the T cell costimulatory receptor, OX40.</strong></font><br />
Pratico, ED., Sullenger, BA., & Nair, SK. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23113766/" target="_blank" style="color:#520049">Nucleic Acid Therapeutics, 23(1):35-43. (2013)</a>
<br/>


<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1RAW-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>

    
    </html>
