---
layout: theophylline_style
title: Pepocin aptamer
date: 2024-05-16 00:00:00
author: Zhizhong Lu, Ying Ao
categories: Aptamer
tags:
- Category:Toxin
- GC:43.0
- GC:43.75
- GC:45.0
- GC:46.339999999999996
- Length:41
- Length:42
- Length:80
- Named:Pepocin_08-09
- Named:Pepocin_8-09 aptamer
- Named:Pepocin_8-14
- Named:Pepocin_9-41U22
- Type:Proteins
- Year:2000
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="description">Description</p>
<p>In 2000, Hirao, I. et al. had isolated aptamers that target pepocin from a degenerate RNA pool through in vitro selection. These aptamers selectively bind to pepocin with dissociation constants ranging from 20 to 30 nM. Furthermore, they are capable of inhibiting the N-glycosidase activity of pepocin on rat liver 28 S rRNA<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>Selection experiments were initiated with an RNA pool containing 2 × 10<sup>13</sup> different sequences. The RNA pool that contained a region of 30 randomized nucleotides flanked by constant regions. After eight rounds of selection, the fraction of the pool which bound pepocin had increased from 0.03 to 24%. Twenty-six clones were isolated from the eighth round of selection. Twenty-three of these aptamers had a similar sequence and bound specifically to pepocin<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>The 2D structure of the figures is based on the article by ribodraw tool to draw.<br></p>
<p>5'-CAAGUCUAAUGUCGGACUCGAUAUCAAUUCACUGCAGACUUG-3'</p>
<img src="/images/2D/pepocin_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>



<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>Pepocin is a ribosome-inactivating protein (RIP). A ribosome-inactivating protein (RIP) is a protein synthesis inhibitor that acts at the eukaryotic ribosome. This protein family describes a large family of such proteins that work by acting as rRNA N-glycosylase (EC 3.2.2.22). They inactivate 60S ribosomal subunits by an N-glycosidic cleavage, which releases a specific adenine base from the sugar-phosphate backbone of 28S rRNA. RIPs exist in bacteria and plants.-----From WiKi</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Pepocin</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P80750/" target="_blank" style="color:#520049"><b>P80750</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00161/" target="_blank" style="color:#520049"><b>PF00161</b></a></td>
        <td name="td3">27.0 KDa</td>
        <td name="td4">NVRFDLSSATSSSYKTFIKNLREALPKDGKVYDIPVLLSTVMDSRRFILIDLVNYDGQSITAAIDVLNVYIVAYSTGTVSYFFQQVPAQAPKLLFKGTQQRTLPYTGNYENLQTAAKKLRENIELGLPALDSAITTLFHYNAEAAASALLVLIQTTSEAARFRYIELQIANNVGTKFKPSQTIISLENNWSALSKQIQIAKNKNGQFETPVILIDPQGNRVQITNVTSNVVTQNIQLLLNIGAT</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/3BWH" target="_blank" style="color:#520049"><b>3BWH</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?lvl=0&id=3663" target="_blank" style="color:#520049"><b>3663</b></a></td>
      </tr>
	  </tbody>
  </table>

  <p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
      <td name="td0">8-09 aptamer</td>
      <td name="td1">5'-CAAGUCUAAUGUCGGACUCGAUAUCAAUUCACUGCAGACUUG-3'</td>
      <td name="td2">pepocin</td>
      <td name="td3">21 ± 2 nM</td>
    </tr>
            
     <tr>
      <td name="td0">8-14 aptamer</td>
      <td name="td1">5'-AGUCUAGGAAGCAGUCGGACUUGUUAUCAAUUCACUGCAGACU-3'</td>
      <td name="td2">pepocin</td>
      <td name="td3">23.4 ± 2.3 nM</td>
    </tr>
            
     <tr>
      <td name="td0">9-41U22</td>
      <td name="td1">5'-GGGAGUCUGAAGUCGGACUCGUUAUCAAUUCACUUCAGACU-3'</td>
      <td name="td2">pepocin</td>
      <td name="td3">17.9 ± 2.2 nM</td>
    </tr>
	  </tbody>
  </table>
<img src="/images/SELEX_ligand/pepocin_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">3BWH-A</td>
      <td name="td1">45.6</td>
      <td name="td2">0.0</td>
      <td name="td3">Cucurmosin</td>
    </tr>
            
     <tr>
      <td name="td0">1MRJ-A</td>
      <td name="td1">40.8</td>
      <td name="td2">0.7</td>
      <td name="td3">Alpha-trichosanthin</td>
    </tr>
            
     <tr>
      <td name="td0">1MRK-A</td>
      <td name="td1">40.8</td>
      <td name="td2">0.7</td>
      <td name="td3">Alpha-trichosanthin</td>
    </tr>
            
     <tr>
      <td name="td0">1QD2-A</td>
      <td name="td1">40.6</td>
      <td name="td2">0.7</td>
      <td name="td3">Trichosanthin</td>
    </tr>
            
     <tr>
      <td name="td0">1TCS-A</td>
      <td name="td1">40.5</td>
      <td name="td2">0.7</td>
      <td name="td3">Trichosanthin</td>
    </tr>
            
     <tr>
      <td name="td0">1BRY-Y</td>
      <td name="td1">40.5</td>
      <td name="td2">0.8</td>
      <td name="td3">Bryodin i</td>
    </tr>
            
     <tr>
      <td name="td0">6Z1Y-C</td>
      <td name="td1">40.4</td>
      <td name="td2">0.7</td>
      <td name="td3">Trichobakin</td>
    </tr>
            
     <tr>
      <td name="td0">1BRY-Z</td>
      <td name="td1">40.4</td>
      <td name="td2">0.8</td>
      <td name="td3">Bryodin i</td>
    </tr>
            
     <tr>
      <td name="td0">6Z1Y-D</td>
      <td name="td1">40.3</td>
      <td name="td2">0.7</td>
      <td name="td3">Trichobakin</td>
    </tr>
            
     <tr>
      <td name="td0">1GIS-A</td>
      <td name="td1">40.2</td>
      <td name="td2">0.8</td>
      <td name="td3">Ribosome-inactivating protein alpha</td>
    </tr>
    </tbody>
  </table>





                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] In vitro selection of aptamers that bind to ribosome-inactivating toxins.</strong></font><br />
Hirao, I., Yoshinari, S., Yokoyama, S., Endo, Y., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/9586110/" target="_blank" style="color:#520049">Nucleic acids symposium series, (37), 283–284. (1997)</a>
<br/>

<a id="ref2"></a><font><strong>[2] RNA aptamers that bind to and inhibit the ribosome-inactivating protein, pepocin.</strong></font><br />
Hirao, I., Madin, K., Endo, Y., Yokoyama, S., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10671532/" target="_blank" style="color:#520049">The Journal of biological chemistry, 275(7), 4943–4948. (2000)</a>
<br/>


<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>




    </html>
