---
layout: theophylline_style
title:  "PKC aptamer"
date:   2024-05-14 00:00:00
author: Zhizhong Lu, Ying Ao
categories: Aptamer
---
<html>

<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="description">Description</p>
<p>In 1994, Conrad, R. et al. have used vitro selection technology to isolate RNA aptamers from a random sequence pool that can bind to, and specifically inhibit, protein kinase C beta II. Two of the selected RNA aptamers bind to this isozyme of protein kinase C with nanomolar affinities and inhibit activation with unprecedented selectivity. In 1996, they used a simple filter binding assay, the aptamers have been used to detect protein kinase C (PKC) introduced into a cell extract<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>The RNA pool used in these selections contained a 120-nucleotide randomized region.  RNA was eluted from filters with 400 pl of 4 M guanidine thiocyanate, precipitated, and amplified by reverse transcription, polymerase chain reaction, and T7 transcription as described previously. After rounds 3,6,9,12, and 14 of selection, radiolabeled RNA pools were assayed for their ability to be co-retained with PKC PI1 on filters. A gel shift selection was carried out in cycles 10 and 13.  After 14 cycles of selection and amplification, two of the selected RNA aptamers bind to this isozyme of protein kinase C with nanomolar affinities<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>The 2D structure of the figures is based on the article by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>
<p>PKC-6: 5'-GGGAGAAUUCCGACCAGAGGCUUACAGAGUGUGCGUAAUGGCGUUCCCAAAUUCGGGCUGGGAACCGUUCGUUCGUGUUAUGCCCGUAGAUAUGGCAAGUCGCGGAUGCUCAGUACUACACUCUUGUGGUCAGUCACAUAUGUGCGUCUACAUGGAUCCUCA-3'<br></p>
<p>PKC-10: 5'-GGGAGAAUUCCGACCAGAGGUUGUUAAGUGCGAGUGUGUUUUACUCCGAUGAUACGGGGAGCGUUAGAGUCUUAUGACCUUGUUACUCCACGUCACUGUCCAAGUCACUCCGCGUCAUAGCAGUCGGAUCCUGUACAUAUGUGCGUCUACAUGGAUCCUCA-3'</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/PKC_aptamer_2D1.svg" alt="drawing" style="width:500px;height:400px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/PKC_aptamer_2D2.svg" alt="drawing" style="width:500px;height:400px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>



<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>In cell biology, Protein kinase C, commonly abbreviated to PKC (EC 2.7.11.13), is a family of protein kinase enzymes that are involved in controlling the function of other proteins through the phosphorylation of hydroxyl groups of serine and threonine amino acid residues on these proteins, or a member of this family. PKC enzymes in turn are activated by signals such as increases in the concentration of diacylglycerol (DAG) or calcium ions (Ca2+). Hence PKC enzymes play important roles in several signal transduction cascades.-----From Wiki</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Protein kinase C beta type</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P05771/" target="_blank" style="color:#520049"><b>P05771</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR034664/" target="_blank" style="color:#520049"><b>IPR034664</b></a></td>
        <td name="td3">76.87 KDa</td>
        <td name="td4">MADPAAGPPPSEGEESTVRFARKGALRQKNVHEVKNHKFTARFFKQPTFCSHCTDFIWGFGKQGFQCQVCCFVVHKRCHEFVTFSCPGADKGPASDDPRSKHKFKIHTYSSPTFCDHCGSLLYGLIHQGMKCDTCMMNVHKRCVMNVPSLCGTDHTERRGRIYIQAHIDRDVLIVLVRDAKNLVPMDPNGLSDPYVKLKLIPDPKSESKQKTKTIKCSLNPEWNETFRFQLKESDKDRRLSVEIWDWDLTSRNDFMGSLSFGISELQKASVDGWFKLLSQEEGEYFNVPVPPEGSEANEELRQKFERAKISQGTKVPEEKTTNTVSKFDNNGNRDRMKLTDFNFLMVLGKGSFGKVMLSERKGTDELYAVKILKKDVVIQDDDVECTMVEKRVLALPGKPPFLTQLHSCFQTMDRLYFVMEYVNGGDLMYHIQQVGRFKEPHAVFYAAEIAIGLFFLQSKGIIYRDLKLDNVMLDSEGHIKIADFGMCKENIWDGVTTKTFCGTPDYIAPEIIAYQPYGKSVDWWAFGVLLYEMLAGQAPFEGEDEDELFQSIMEHNVAYPKSMSKEAVAICKGLMTKHPGKRLGCGPEGERDIKEHAFFRYIDWEKLERKEIQPPYKPKARDKRDTSNFDKEFTRQPVELTPTDKLFIMNLDQNEFAGFSYTNPEFVINV</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/2I0E" target="_blank" style="color:#520049"><b>2I0E</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/5579" target="_blank" style="color:#520049"><b>5579</b></a></td>
      </tr>
	  </tbody>
  </table>

  <p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">PKC-6 aptamer</td>
        <td name="td1">5'-GGGAGAAUUCCGACCAGAGGCUUACAGAGUGUGCGUAAUGGCGUUCCCAAAUUCGGGCUGGGAACCGUUCGUUCGUGUUAUGCCCGUAGAUAUGGCAAGUCGCGGAUGCUCAGUACUACACUCUUGUGGUCAGUCACAUAUGUGCGUCUACAUGGAUCCUCA-3'</td>
        <td name="td2">Protein kinase C βII</td>
        <td name="td3">~7 nM</td>
      </tr>
      <tr>
        <td name="td0">PKC-10 aptamer</td>
        <td name="td1">5'-GGGAGAAUUCCGACCAGAGGUUGUUAAGUGCGAGUUGUUUUACUCCGAUGAUACGGGGAGCGUUAGAGUCUUAUGACCUUGUUCUCCACGUCACUGUCCAAGUCACUCCGCGUCAUAGCAGUCGGAUCCUGUACAUAUGUGCGUCUACAUGGAUCCUCA-3'</td>
        <td name="td2">Protein kinase C βII</td>
        <td name="td3">~7 nM</td>
      </tr>
	  </tbody>
  </table>
<img src="/images/SELEX_ligand/PKC_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">3PFQ-A</td>
      <td name="td1">44.1</td>
      <td name="td2">1.1</td>
      <td name="td3">Protein kinase c beta type</td>
    </tr>
            
     <tr>
      <td name="td0">8JPB-G</td>
      <td name="td1">36.6</td>
      <td name="td2">2.3</td>
      <td name="td3">Beta-adrenergic receptor kinase 1</td>
    </tr>
            
     <tr>
      <td name="td0">5DZC-A</td>
      <td name="td1">35.6</td>
      <td name="td2">1.9</td>
      <td name="td3">Cgmp-dependent protein kinase, putative</td>
    </tr>
            
     <tr>
      <td name="td0">7APJ-A</td>
      <td name="td1">30.4</td>
      <td name="td2">1.6</td>
      <td name="td3">Rac-alpha serine/threonine-protein kinase,non-spe</td>
    </tr>
            
     <tr>
      <td name="td0">7MYJ-C</td>
      <td name="td1">29.6</td>
      <td name="td2">2.1</td>
      <td name="td3">5'-amp-activated protein kinase catalytic subunit</td>
    </tr>
            
     <tr>
      <td name="td0">4YSM-A</td>
      <td name="td1">29.5</td>
      <td name="td2">2.0</td>
      <td name="td3">Calmodulin-like domain protein kinase</td>
    </tr>
            
     <tr>
      <td name="td0">6C9J-A</td>
      <td name="td1">29.1</td>
      <td name="td2">1.8</td>
      <td name="td3">5'-amp-activated protein kinase catalytic subunit</td>
    </tr>
            
     <tr>
      <td name="td0">5U6Y-A</td>
      <td name="td1">27.6</td>
      <td name="td2">2.5</td>
      <td name="td3">Calcium/calmodulin-dependent protein kinase type</td>
    </tr>
            
     <tr>
      <td name="td0">2PML-X</td>
      <td name="td1">26.7</td>
      <td name="td2">2.5</td>
      <td name="td3">Ser/thr protein kinase</td>
    </tr>
            
     <tr>
      <td name="td0">5NCL-A</td>
      <td name="td1">26.0</td>
      <td name="td2">3.1</td>
      <td name="td3">Serine/threonine-protein kinase cbk1</td>
    </tr>
    </tbody>
  </table>



                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Isozyme-specific inhibition of protein kinase C by RNA aptamers.</strong></font><br />
Conrad, R., Keranen, L. M., Ellington, A. D., & Newton, A. C.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/7528207/" target="_blank" style="color:#520049">The Journal of biological chemistry, 269(51), 32051–32054. (1994)</a>
<br/>

<a id="ref2"></a><font><strong>[2] Detecting immobilized protein kinase C isozymes with RNA aptamers.</strong></font><br />
Conrad, R., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8937571/" target="_blank" style="color:#520049">Analytical biochemistry, 242(2), 261–265. (1996)</a>
<br/>


<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>

<script>
    window.addEventListener('DOMContentLoaded', function() {
      var td = document.getElementsByName('td4')[0];
      var maxLength = 100; // 设置最大长度
      var originalText = td.innerHTML;
      var displayedText = originalText.substring(0, maxLength);
      
      if (originalText.length > maxLength) {
        displayedText += '<span class="ellipsis-btn">......</span>'; // 在超出长度时添加省略号按钮
        displayedText += '<span class="hidden-content">' + originalText.substring(maxLength) + '</span>'; // 隐藏的内容
        td.innerHTML = displayedText;
        
        // 获取省略号按钮元素和隐藏的内容元素
        var ellipsisBtn = td.querySelector('.ellipsis-btn');
        var hiddenContent = td.querySelector('.hidden-content');
        
        // 添加点击事件监听器
        ellipsisBtn.addEventListener('click', function() {
          if (hiddenContent.style.display === 'none') {
            hiddenContent.style.display = 'inline'; // 展开内容
            ellipsisBtn.innerHTML = '...'; // 更新按钮文字为“...”
            td.appendChild(ellipsisBtn); // 将按钮移到内容后面
          } else {
            hiddenContent.style.display = 'none'; // 收起内容
            ellipsisBtn.innerHTML = '......'; // 更新按钮文字为省略号
            td.appendChild(ellipsisBtn); // 将按钮移到内容后面
          }
        });
      }
    });
  </script>


    </html>
