---
layout: theophylline_style
title:  "Prion protein 2 aptamer"
date:   2024-05-08 00:00:00
author: Qiaozhen Liu, Zhijie Tan
categories: Aptamer
---



<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<font><p class="header_box" id="description">Description</p></font>
<font>In 2006, R. Mercey and colleagues used in vitro selection to isolate RNA aptamers that bind to sheep recombinant prion protein. The minimal sequence that they found necessary for binding of RM312 to PrP presents a striking similarity with one previously described PrP aptamer of comparable affinity. In addition, we here identify the two lysine clusters contained in the N-terminal part of PrP as its main nucleic-acid binding sites<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></font>
<br>
<br>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2006, R. Mercey et al. isolated specific sequences which bind specifically to PrP23-234 (recPrP) by 4 cycles of iterative selection and amplification<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<font>The RNA aptamers bound in vitro to sheep recombinant prion protein. The 2D structure of the figure is based on the article by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></font>
<font><p>5'-AAGACGUCGGGGAUUGGCAAACCCCGUUUCCUUGAGCGACCCCCUGU-3'</p></font>
<img src="/images/2D/Prion2_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>
<br>


<font ><p class="header_box" id="ligand-recognition">Ligand information</p></font>  

<p class="blowheader_box">SELEX ligand</p>
<p>Prion protein (PrP-c) is a small glycoprotein found in high quantity in the brain of animals infected with certain degenerative neurological diseases, such as sheep scrapie and bovine spongiform encephalopathy (BSE), and the human dementias Creutzfeldt-Jacob disease (CJD) and Gerstmann-Straussler syndrome (GSS). PrP-c is encoded in the host genome and is expressed both in normal and infected cells. During infection, however, the PrP-c molecule become altered (conformationally rather than at the amino acid level) to an abnormal isoform, PrP-sc. In detergent-treated brain extracts from infected individuals, fibrils composed of polymers of PrP-sc, namely scrapie-associated fibrils or prion rods, can be evidenced by electron microscopy. The precise function of the normal PrP isoform in healthy individuals remains unknown.-----from Pfam</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Prion protein</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P04273/entry" target="_blank" style="color:#520049"><b>P04273</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR000817/" target="_blank" style="color:#520049"><b>IPR000817</b></a></td>
        <td name="td3">16.25 kDa</td>
        <td name="td4">GQGGGTHNQWNKPSKPKTNMKHMAGAAAAGAVVGGLGGYMLGSAMSRPMMHFGNDWEDRYYRENMNRYPNQVYYRPVDQYNNQNNFVHDCVNITIKQHTVTTTTKGENFTETDIKIMERVVEQMCTTQYQKESQAYYDGRRS</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/1B10" target="_blank" style="color:#520049"><b>1B10</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/nuccore/XM_005068660.4" target="_blank" style="color:#520049"><b>XM_005068660.4</b></a></td>
      </tr>
	  </tbody>
  </table>

<p>The binding affinities between aptamers and PrP weredetermined by nitrocellulose filter-binding assays.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">RM312</td>
        <td name="td1">5'-AAGACGUCGGGGAUUGGCAAACCCCGUUUCCUUGAGCGACCCCCUGU-3'</td>
        <td name="td2">The ovine PrP</td>
        <td name="td3">20 nM</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/Prion2_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>
<br>


<p class="blowheader_box">Similar compound</p>                    
<font><p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">2KUN-A</td>
      <td name="td1">10.7</td>
      <td name="td2">3.1</td>
      <td name="td3">Major prion ptotein</td>
    </tr>
            
     <tr>
      <td name="td0">1I17-A</td>
      <td name="td1">6.8</td>
      <td name="td2">3.2</td>
      <td name="td3">Prion-like ptotein</td>
    </tr>
            
     <tr>
      <td name="td0">8HM3-B</td>
      <td name="td1">4.9</td>
      <td name="td2">2.6</td>
      <td name="td3">Putative ubiquitin</td>
    </tr>
            
     <tr>
      <td name="td0">1CQX-A</td>
      <td name="td1">4.5</td>
      <td name="td2">3.7</td>
      <td name="td3">Flavohemoprotein</td>
    </tr>
            
     <tr>
      <td name="td0">8H17-A</td>
      <td name="td1">4.4</td>
      <td name="td2">4.7</td>
      <td name="td3">Tlr1989 protein</td>
    </tr>
            
     <tr>
      <td name="td0">8BF9-G</td>
      <td name="td1">4.4</td>
      <td name="td2">2.9</td>
      <td name="td3">RNA(1766)</td>
    </tr>
            
     <tr>
      <td name="td0">6THH-C</td>
      <td name="td1">4.3</td>
      <td name="td2">3.0</td>
      <td name="td3">Sirv2 acrid1 (gp02) anti-crispr protein</td>
    </tr>
            
     <tr>
      <td name="td0">5ZIQ-A</td>
      <td name="td1">4.3</td>
      <td name="td2">2.7</td>
      <td name="td3">Globin protein</td>
    </tr>
            
     <tr>
      <td name="td0">6Z5S-W</td>
      <td name="td1">4.2</td>
      <td name="td2">2.9</td>
      <td name="td3">Light harvesting complex 1 protein w</td>
    </tr>
            
     <tr>
      <td name="td0">4CEJ-B</td>
      <td name="td1">4.2</td>
      <td name="td2">3.4</td>
      <td name="td3">Atp-dependent helicase/nuclease subunit a</td>
    </tr>
    </tbody>
  </table>
<br>
<br>
                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Fast, reversible interaction of prion protein with RNA aptamers containing specific sequence patterns.</strong></font><br />
Mercey, R., Lantier, I., Maurel, M. C., Grosclaude, J., Lantier, F., & Marc, D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/16799875/" target="_blank" style="color:#520049">Archives of virology, 151(11), 2197–2214. (2006)</a>
<br/>



<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1RAW-3D.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>

    <script>
    window.addEventListener('DOMContentLoaded', function() {
      var td = document.getElementsByName('td4')[0];
      var maxLength = 100; // 设置最大长度
      var originalText = td.innerHTML;
      var displayedText = originalText.substring(0, maxLength);
      
      if (originalText.length > maxLength) {
        displayedText += '<span class="ellipsis-btn">......</span>'; // 在超出长度时添加省略号按钮
        displayedText += '<span class="hidden-content">' + originalText.substring(maxLength) + '</span>'; // 隐藏的内容
        td.innerHTML = displayedText;
        
        // 获取省略号按钮元素和隐藏的内容元素
        var ellipsisBtn = td.querySelector('.ellipsis-btn');
        var hiddenContent = td.querySelector('.hidden-content');
        
        // 添加点击事件监听器
        ellipsisBtn.addEventListener('click', function() {
          if (hiddenContent.style.display === 'none') {
            hiddenContent.style.display = 'inline'; // 展开内容
            ellipsisBtn.innerHTML = '...'; // 更新按钮文字为“...”
            td.appendChild(ellipsisBtn); // 将按钮移到内容后面
          } else {
            hiddenContent.style.display = 'none'; // 收起内容
            ellipsisBtn.innerHTML = '......'; // 更新按钮文字为省略号
            td.appendChild(ellipsisBtn); // 将按钮移到内容后面
          }
        });
      }
    });
  </script>
    </html>
