---
layout: theophylline_style
title: PSMA aptamer
date: 2024-04-16 00:00:00
author: Jiaxin Zhao,Zhizhong Lu
categories: Aptamer
tags:
- Category:Mammalian
- GC:47.0
- GC:56.99999999999999
- Length:43
- Length:65
- Named:Prostate-specific membrane antigen (PSMA)_A9 RNA aptamer
- Named:Prostate-specific membrane antigen (PSMA)_A9g aptamer
- Type:Proteins
- Year:2002
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12124337/" target="_blank" style="color:#520049">2002</a></h3>
    </div>
    <div class="body">
      <p>Lupold et al. used the extracellular part of PSMA to select an aptamer, termed A9, capable of inhibiting PSMA enzymatic activity and binding to PSMA-positive cells with low nanomolar affinity<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/16740739/" target="_blank" style="color:#520049">2006</a></h3>
    </div>
    <div class="body">
      <p>The A9 aptamer, stabilized by the incorporation of 2′-fluor-pyrimidines bases, has since been used in a variety of functional studies, including the A9-mediated delivery of therapeutic siRNA to PCa cells<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/20550178/" target="_blank" style="color:#520049">2010</a></h3>
    </div>
    <div class="body">
      <p>A9 aptamer is used in A9-targeted gold nanoparticles for PCa imaging and delivery of doxorubicin payloads<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/22004414/" target="_blank" style="color:#520049">2011</a></h3>
    </div>
    <div class="body">
      <p>A9 aptamer was modified to prepare a minimal functional part, giving rise to A9g, its truncated 43-nucleotide variant<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/25450401/" target="_blank" style="color:#520049">2014</a></h3>
    </div>
    <div class="body">
      <p>A9 aptamer is used in generation of A9-bound drug-loaded liposomes<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/32525981/" target="_blank" style="color:#520049">2020</a></h3>
    </div>
    <div class="body">
      <p>Crystal structure of human PSMA in complex with A9g, a 43-bp PSMA-specific RNA aptamer, that was determined to the 2.2 Å resolution limit<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
    </div>
  </div>
</div>


         
<p class="header_box" id="description">Description</p>
<p>In 2002, Lupold et al. used the extracellular part of PSMA to select an aptamer, termed A9. In 2006, The A9 aptamer, stabilized by the incorporation of 2′-fluor-pyrimidines bases, has since been used in a variety of functional studies, including the A9-mediated delivery of therapeutic siRNA to PCa cells. In 2014, A9 aptamer is used in generation of A9-bound drug-loaded liposomes. In 2020, Barinka C et al. published on the crystal structure of human prostate-specific membrane antigen (PSMA) in complex with A9g, a 43-bp PSMA-specific RNA aptamer. This structure was determined to a resolution limit of 2.2 Å<sup>[<a href="#ref2" style="color:#520049">2</a></sup><sup>,<a href="#ref3" style="color:#520049">3</a></sup><sup>,<a href="#ref6" style="color:#520049">6</a></sup><sup>]</sup>.<br></p>

<p class="header_box" id="SELEX">SELEX</p>
<p>In 2002, Lupold et al. used the extracellular moiety of PSMA as a target to isolate for RNA aptamer A9 that can inhibit PSMA enzyme activity and bind to PSMA-positive cells by SELEX technology. The aptamer underwent multiple rounds of screening and amplification to obtain a binding molecule with high affinity and specificity. Two specific aptamers were selected from an initial 40-mer library of approximately 6 × 10¹⁴ random-sequence RNA molecules for their ability to bind to a recombinant protein representing the extracellular 706 amino acids of prostate-specific membrane antigen (PSMA), termed xPSM. Six rounds of in vitro selection were conducted, enriching for xPSM binding as monitored by the aptamers' inhibition of xPSM's N-acetyl-alpha-linked acid dipeptidase (NAALADase) enzymatic activity. By the sixth round, 95% of the aptamer pool consisted of just two sequences. These two aptamers, designated xPSM-A9 and xPSM-A10, were cloned and found to be unique, sharing no consensus sequences<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p>
<p>
<br>
<br>    

<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>Here we utilized RiboDraw to complete the figure, based the 3D structure information. A9g aptamer was named by  Paloma H Giangrande et al. in the 2011 article<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup>.</p>
<p>5'-GGGACCGAAAAAGACCUGACUUCUAUACUAAGUCUACGUUCCC-3'</p>
<img src="/images/2D/GCP2_aptamer_2D.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>

<p class="blowheader_box">3D visualisation</p>             
<p>The complex is the crystal structure of human PSMA in complex with A9g, a 43-bp PSMA-specific RNA aptamer, that was determined to the 2.2 Å resolution limit. GCPII is a close homology of PSMA. A9g aptamer inhibits the enzyme activity of PSMA residing in the central/peripheral nervous system (also known as glutamate carboxypeptidase II, GCPII). The experimentally determined fold of the 43-nucleotide A9g aptamer in its PSMA-bound state is a relatively simple stem-loop structure that includes two stems termed S1 (7-base pairs) and S2 (4-base pairs), a 4 × 1 internal loop 1 (L1; 4 unpaired adenines A8–A11 on the 5′-side and an unpaired nucleotide A36 on the 3′-side) and a 16-nucleotide long hairpin loop 2 The PDB ID of this structure is 6RTI<sup>[<a href="#ref7" style="color:#520049">7</a></sup><sup>]</sup>.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/GCP2_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;">
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A',    
  start_residue_number:56, 
  end_residue_number:750, 
  color:{r:133,g:39,b:181}
},
{
  struct_asym_id: 'X',    
  start_residue_number:1, 
  end_residue_number:7, 
  color:{r:184,g:51,b:50}
},
{
  struct_asym_id: 'X',    
  start_residue_number:8, 
  end_residue_number:11, 
  color:{r:194,g:182,b:23}
},
{
  struct_asym_id: 'X',    
  start_residue_number:12, 
  end_residue_number:15, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'X',    
  start_residue_number:16, 
  end_residue_number:17, 
  color:{r:46,g:161,b:3}
},
{
  struct_asym_id: 'X',    
  start_residue_number:18, 
  end_residue_number:18, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'X',    
  start_residue_number:19, 
  end_residue_number:20, 
  color:{r:46,g:161,b:3}
},
{
  struct_asym_id: 'X',    
  start_residue_number:21, 
  end_residue_number:24, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'X',    
  start_residue_number:25, 
  end_residue_number:31, 
  color:{r:46,g:161,b:3}
},{
  struct_asym_id: 'X',    
  start_residue_number:32, 
  end_residue_number:35, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'X',    
  start_residue_number:36, 
  end_residue_number:36, 
  color:{r:194,g:182,b:23}
},
{
  struct_asym_id: 'X',    
  start_residue_number:37, 
  end_residue_number:43, 
  color:{r:184,g:51,b:50}
},
{
  struct_asym_id: 'X',    
  start_residue_number:5, 
  end_residue_number:6, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'X',    
  start_residue_number:8, 
  end_residue_number:8, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'X',    
  start_residue_number:12, 
  end_residue_number:12, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'X',    
  start_residue_number:39, 
  end_residue_number:39, 
  color:{r:241,g:125,b:7}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/6RTI-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<p>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 6MSF. GCPII (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with GCPII.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/GCP2_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/GCP2_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>


<p class="header_box" id="ligand-recognition">Ligand information</p>  
<p class="blowheader_box">SELEX ligand</p>
<p>For surface plasmon resonance (SPR) measurements using the BIACore 3000 device, the KD value was calculated by global fitting of six concentrations of PSMA on a constant density A9g aptamer. The data were fitted 1:1 with the mass transfer model to calculate the kinetic parameters. The binding parameter KD of A9 aptamer was determined by filtration membrane binding assay<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup>.</p>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">A9g RNA aptamer</td>
        <td name="td1">5'-GGGACCGAAAAAGACCUGACUUCUAUACUAAGUCUACGUUCCC-3'</td>
        <td name="td3">Human PSMA protein</td>
        <td name="td4">5-30 nM</td>
      </tr>
      <tr>
        <td name="td0">A9 RNA aptamer</td>
        <td name="td1">5'-GGGAGGACGAUGCGGACCGAAAAAGACCUGACUUCUAAGUCUACGUUCCCAGACGACUCGCCCGA-3'</td>
        <td name="td3">Human PSMA protein</td>
        <td name="td4">110 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<p>Glutamate carboxypeptidase II, also known as N-acetyl-L-aspartyl-L-glutamate peptidase I (NAALADase I), NAAG peptidase, or prostate-specific membrane antigen (PSMA) is an enzyme that in humans is encoded by the FOLH1 (folate hydrolase 1) gene. Human GCPII contains 750 amino acids and weighs approximately 84 kDa.-----From <a href="https://en.wikipedia.org/wiki/Glutamate_carboxypeptidase_II" target="_blank" style="color:#520049; text-decoration: underline;"><b>Wiki</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">Mass</th>
        <th onclick="sortTable(3)">Protein sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/Q04609/entry" target="_blank" style="color:#520049"><b>Q04609</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/cdd/CD08022/" target="_blank" style="color:#520049"><b>CD08022</b></a></td>
        <td name="td2">84.33 KDa</td>
        <td name="td3">
        <div class="sequence-container">

          <span class="full-sequence">MWNLLHETDSAVATARRPRWLCAGALVLAGGFFLLGFLFGWFIKSSNEATNITPKHNMKAFLDELKAENIKKFLYNFTQIPHLAGTEQNFQLAKQIQSQWKEFGLDSVELAHYDVLLSYPNKTHPNYISIINEDGNEIFNTSLFEPPPPGYENVSDIVPPFSAFSPQGMPEGDLVYVNYARTEDFFKLERDMKINCSGKIVIARYGKVFRGNKVKNAQLAGAKGVILYSDPADYFAPGVKSYPDGWNLPGGGVQRGNILNLNGAGDPLTPGYPANEYAYRRGIAEAVGLPSIPVHPIGYYDAQKLLEKMGGSAPPDSSWRGSLKVPYNVGPGFTGNFSTQKVKMHIHSTNEVTRIYNVIGTLRGAVEPDRYVILGGHRDSWVFGGIDPQSGAAVVHEIVRSFGTLKKEGWRPRRTILFASWDAEEFGLLGSTEWAEENSRLLQERGVAYINADSSIEGNYTLRVDCTPLMYSLVHNLTKELKSPDEGFEGKSLYESWTKKSPSPEFSGMPRISKLGSGNDFEVFFQRLGIASGRARYTKNWETNKFSGYPLYHSVYETYELVEKFYDPMFKYHLTVAQVRGGMVFELANSIVLPFDCRDYAVVLRKYADKIYSISMKHPQEMKTYSVSFDSLFSAVKNFTEIASKFSERLQDFDKSNPIVLRMMNDQLMFLERAFIDPLGLPDRPFYRHVIYAPSSHNKYAGESFPGIYDALFDIESKVDPSKAWGEVKRQIYVAAFTVQAAAETLSEVA</span>
        </div>
      </td>
        <td name="td4"><a href="https://www.rcsb.org/structure/2OOT" target="_blank" style="color:#520049"><b>2OOT</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/protein/AAA60209.1" target="_blank" style="color:#520049"><b>AAA60209.1</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/GCP2_aptamer_stru_ligand1.svg" alt="drawing" style="width:800px;height:300px;"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and selected the previous information with high similarity for presentation.The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB).</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">6RTI-A</td>
        <td name="td2">64.5</td>
        <td name="td3">0</td>
        <td name="td4">Glutamate carboxypeptidase 2</td>
      </tr>
      <tr>
        <td name="td1">3BI0-A</td>
        <td name="td2">64.7</td>
        <td name="td3">0.5</td>
        <td name="td4">Glutamate carboxypeptidase 2</td>
      </tr>
      <tr>
        <td name="td1">3FF3-A</td>
        <td name="td2">60.9</td>
        <td name="td3">0.8</td>
        <td name="td4">Glutamate carboxypeptidase iii</td>
      </tr>
      <tr>
        <td name="td1">4TWE-B</td>
        <td name="td2">53.7</td>
        <td name="td3">1.8</td>
        <td name="td4">N-acetylated-alpha-linked acidic dipeptidase-like</td>
      </tr>
      <tr>
        <td name="td1">6WRX-B</td>
        <td name="td2">43.8</td>
        <td name="td3">2.5</td>
        <td name="td4">Transferrin receptor protein 1</td>
      </tr>
      <tr>
        <td name="td1">1DE4-I</td>
        <td name="td2">42.9</td>
        <td name="td3">2.5</td>
        <td name="td4">Hemochromatosis protein</td>
      </tr>
      <tr>
        <td name="td1">3S9L-A</td>
        <td name="td2">42.5</td>
        <td name="td3">2.5</td>
        <td name="td4">Transferrin receptor protein 1</td>
      </tr>
      <tr>
        <td name="td1">3KAS-A</td>
        <td name="td2">40.7</td>
        <td name="td3">2.7</td>
        <td name="td4">Transferrin receptor protein 1</td>
      </tr>
      <tr>
        <td name="td1">1CX8-A</td>
        <td name="td2">40.7</td>
        <td name="td3">2.7</td>
        <td name="td4">Transferrin receptor protein</td>
      </tr>
      <tr>
        <td name="td1">2NSU-A</td>
        <td name="td2">40.6</td>
        <td name="td3">2.7</td>
        <td name="td4">Transferrin receptor protein 1</td>
      </tr>
	  </tbody>
  </table>

                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Three small ribooligonucleotides with specific arginine sites.</strong></font><br />
Connell, G. J., Illangesekare, M., & Yarus, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8504070/" target="_blank" style="color:#520049" >Biochemistry, 32(21), 5497–5502. (1993)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Identification and characterization of nuclease-stabilized RNA molecules that bind human prostate cancer cells via the prostate-specific membrane antigen.</strong></font><br />
Lupold, S. E., Hicke, B. J., Lin, Y., & Coffey, D. S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12124337/" target="_blank" style="color:#520049">Cancer research, 62(14), 4029–4033. (2002)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] Aptamer mediated siRNA delivery.</strong></font><br />
Chu, T. C., Twu, K. Y., Ellington, A. D., & Levy, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/16740739/" target="_blank" style="color:#520049"> Nucleic acids research, 34(10), e73. (2006)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] A drug-loaded aptamer-gold nanoparticle bioconjugate for combined CT imaging and therapy of prostate cancer.</strong></font><br />
Kim, D., Jeong, Y. Y., & Jon, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/20550178/" target="_blank" style="color:#520049">ACS nano, 4(7), 3689–3696. (2010)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Rational truncation of an RNA aptamer to prostate-specific membrane antigen using computational structural modeling.</strong></font><br />
Rockey, W. M., Hernandez, F. J., Huang, S. Y., Cao, S., Howell, C. A., Thomas, G. S., Liu, X. Y., Lapteva, N., Spencer, D. M., McNamara, J. O., Zou, X., Chen, S. J., & Giangrande, P. H.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/22004414/" target="_blank" style="color:#520049">Nucleic acid therapeutics, 21(5), 299–314. (2011)</a>
<br />

<a id="ref6"></a><font><strong>[6] RNA aptamer-conjugated liposome as an efficient anticancer drug delivery vehicle targeting cancer cells in vivo.</strong></font><br />
Baek, S. E., Lee, K. H., Park, Y. S., Oh, D. K., Oh, S., Kim, K. S., & Kim, D. E.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/25450401/" target="_blank" style="color:#520049">Journal of controlled release : official journal of the Controlled Release Society, 196, 234–242. (2014)</a>
<br />       

<a id="ref7"></a><font><strong>[7] Structural basis of prostate-specific membrane antigen recognition by the A9g RNA aptamer.</strong></font><br />
Ptacek, J., Zhang, D., Qiu, L., Kruspe, S., Motlova, L., Kolenko, P., Novakova, Z., Shubham, S., Havlinova, B., Baranova, P., Chen, S. J., Zou, X., Giangrande, P., & Barinka, C.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/32525981/" target="_blank" style="color:#520049">Nucleic acids research, 48(19), 11130–11145. (2020)</a>
<br />


<script>
    function toggleSequence(event) {
      const container = event.target.closest('.sequence-container');
      container.classList.toggle('expanded');
      const showMoreText = container.querySelector('.show-more');
      
      // 展开后按钮文本变化
      if (container.classList.contains('expanded')) {
        showMoreText.textContent = '...';  // 展开后显示 "..."
      } else {
        showMoreText.textContent = '......';  // 收起后显示 "......"
      }
    }

    // 页面加载时，限制序列文本为50个字符
    window.addEventListener('load', function() {
      const sequenceContainers = document.querySelectorAll('.sequence-container');
      sequenceContainers.forEach(container => {
        const fullSeqText = container.querySelector('.full-sequence').textContent;
        const truncatedText = fullSeqText.slice(0, 20);  // 只显示前50个字符
        container.querySelector('.sequence-text').textContent = truncatedText;
      });
    });
  </script>