---
layout: theophylline_style
title: pUC19-aptamer
date: 2024-05-20 00:00:00
author: Bichun Wu, Jiaxin Zhao
categories: Aptamer
tags:
- Category:Mammalian
- GC:56.79
- Length:81
- Named:β-catenin_pUC19
- Type:Proteins
- Year:2006
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="description">Description</p>
<p>In 2006, Lee, H. K., & Jeong, S. selected a high-affinity RNA aptamer that associated with beta-catenin in vivo. Nuclear localized aptamer inhibited beta-catenin-dependent transcription of cyclin D1 and c-myc in colon cancer cells; thus, cells stably expressing the aptamer exhibited cell cycle arrest and reduced tumor forming potential. Most significantly, the aptamer prevented the alternative splicing induced by stabilized beta-catenin<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2006, Lee, H. K., & Jeong, S. used an RNA library of random 50-nucleotide sequences (1 × 1014 different molecules, IDT) as starting material for the SELEX procedure . In each cycle, RNA was preincubated with glutathione S-transferase (GST) protein in binding buffer [25 mmol/L HEPES (pH 7.5), 150 mmol/L NaCl, 1 mmol/L MgCl2, 2 mmol/L DTT, and 40 units RNase inhibitor]. GST-bound RNA was discarded, and RNA bound to GST-Arm 1-12 was selected with glutathione-Sepharose beads. After four and eight cycles of selection, the selected sequences were cloned into pUC19 (pUC19-Aptamer) and sequenced<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>The 2D structure of the figure is based on the prediction results of the RNA fold website by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>
<p>5'-GGACGCGUGGUACCAGGCCGAUCUAUGGACGCUAUAGGCACACCGGAUACUUUAACGAUUGGCUAAGCUUCCGCGGGGAUC-3'</p>
<img src="/images/2D/pUC19_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>Three different forms of catenin (designated alpha, beta and gamma) comprise the cytoplasmic domain of the cadherin cell-cell adhesion complex. Beta-catenin forms a cadherin/beta-catenin/alphaE-catenin complex that can tether the tripartite adhesion complex and regulate actin dynamics.-----from InterPro</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">β-catenin</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/F1QGH7/entry" target="_blank" style="color:#520049"><b>F1QGH7</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR013284/" target="_blank" style="color:#520049"><b>IPR013284</b></a></td>
        <td name="td3">57.64 kDa</td>
        <td name="td4">AVVNLINYQDDAELATRAIPELTKLLNDEDQVVVNKAAVMVHQLSKKEASRHAIMRSPQMVSAIVRTMQNTNDVETARCTAGTLHNLSHHREGLLAIFKSGGIPALVKMLGSPVDSVLFYAITTLHNLLLHQEGAKMAVRLAGGLQKMVALLNKTNVKFLAITTDCLQILAYGNQESKLIILASGGPQALVNIMRTYTYEKLLWTTSRVLKVLSVCSSNKPAIVEAGGMQALGLHLTDPSQRLVQNCLWTLRNLSDAATKQEGMEGLLGTLVQLLGSDDINVVTCAAGILSNLTCNNYKNKMMVCQVGGIEALVRTVLRAGDREDITEPAICALRHLTSRHQEAEMAQNAVRLHYGLPVVVKLLHPPSHWPLIKATVGLIRNLALCPANHAPLREQGAIPRLVQLLVRAHQDTQRRTSMGGTQQQFVEGVRMEEIVEGCTGALHILARDVHNRIVIRGLNTIPLFVQLLYSPIENIQRVAAGVLCELAQDKEAAEAIEAEGATAPLTELLHSRNEGVATYAAAVLFRMS</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/1JDH" target="_blank" style="color:#520049"><b>1JDH</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/30265" target="_blank" style="color:#520049"><b>30265</b></a></td>
      </tr>
	  </tbody>
  </table>

<p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">pUC19-aptamer</td>
        <td name="td1">5'-GGACGCGUGGUACCAGGCCGAUCUAUGGACGCUAUAGGCACACCGGAUACUUUAACGAUUGGCUAAGCUUCCGCGGGGAUC-3'</td>
        <td name="td2">β-catenin</td>
        <td name="td3">5 nM</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/pUC19_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">4R0Z-A</td>
      <td name="td1">37.5</td>
      <td name="td2">4.5</td>
      <td name="td3">Protein humpback-2</td>
    </tr>
            
     <tr>
      <td name="td0">6KBN-C</td>
      <td name="td1">32.6</td>
      <td name="td2">5.9</td>
      <td name="td3">Vacuolar protein 8</td>
    </tr>
            
     <tr>
      <td name="td0">7VPT-A</td>
      <td name="td1">32.4</td>
      <td name="td2">5.2</td>
      <td name="td3"> C. glabrata importin alpha arm domain - upc2 nls</td>
    </tr>
            
     <tr>
      <td name="td0">3L6X-A</td>
      <td name="td1">30.7</td>
      <td name="td2">3.9</td>
      <td name="td3">Catenin delta-1</td>
    </tr>
            
     <tr>
      <td name="td0">7JTK-X</td>
      <td name="td1">28.8</td>
      <td name="td2">6.4</td>
      <td name="td3">Flagellar radial spoke protein 1</td>
    </tr>
            
     <tr>
      <td name="td0">6SA8-A</td>
      <td name="td1">27.9</td>
      <td name="td2">20.2</td>
      <td name="td3">Ring-like darpin-armadillo fusion h83_d01</td>
    </tr>
            
     <tr>
      <td name="td0">3NMZ-A</td>
      <td name="td1">26.0</td>
      <td name="td2">7.2</td>
      <td name="td3">Apc variant protein</td>
    </tr>
            
     <tr>
      <td name="td0">5XGC-A</td>
      <td name="td1">24.9</td>
      <td name="td2">12.3</td>
      <td name="td3">Rap1 gtpase-gdp dissociation stimulator 1</td>
    </tr>
            
     <tr>
      <td name="td0">6HB3-B</td>
      <td name="td1">21.1</td>
      <td name="td2">3.7</td>
      <td name="td3">Protein hgh1</td>
    </tr>
            
     <tr>
      <td name="td0">7Y8R-L</td>
      <td name="td1">21.0</td>
      <td name="td2">10.6</td>
      <td name="td3">Histone h3</td>
    </tr>
    </tbody>
  </table>
                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Modulation of oncogenic transcription and alternative splicing by beta-catenin and an RNA aptamer in colon cancer cells.</strong></font><br />
Lee, H. K., & Jeong, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/17079480/" target="_blank" style="color:#520049">Cancer research, 66(21), 10560–10566. (2006)</a>
<br/>


<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1RAW-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>

    
    </html>
