---
layout: theophylline_style
title: R5 helix peptide aptamer
date: 2024-05-16 00:00:00
author: Zhizhong Lu, Ying Ao
categories: Aptamer
tags:
- Category:Other
- GC:55.13
- GC:58.97
- Length:78
- Named:R5 helix peptide_B-2
- Named:R5 helix peptide_B-6
- Type:Proteins
- Year:2000
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="description">Description</p>
<p>In 2000, Fukusho, S. et al. performed an in vitro selection of RNA aptamer on a 27 MHz quartz-crystal microbalance (QCM) for a simple R5 helix peptide  as a model of N peptide from bacteriophade lambda. In 2014，In vitro selection of RNA (30 random-nucleotide region) was carried out with N-model-peptide immobilized on a 27 MHz quartz-crystal microbalance (QCM), and the RNA recognition specificity of the peptide was studied<sup>[<a href="#ref1" style="color:#520049">1</a>,<a href="#ref3" style="color:#520049">3</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>The R5-peptide was immobilized on Au electrode of a 27 MHz QCM plate through Au-S interaction having a long polyethylenoxide spacer. The R5-peptide-immobilized QCM was soaked into the buffer solution and the frequency decrease (mass increase) responding to the addition of random ssRNA was monitored in the aqueous solution. After monitoring selection process as frequency decrease (mass increase), the QCM was washed and the selected RNA was recovered. The ssRNA was reversed transcribed to DNA and amplified with PCR reaction. Then, the dsDNA was transcribed and used as the ssRNA for next selection. Selection processes were repeated 1-7 cycles. Random region sequences for selected clones after the seventh selection cycle with N-model-peptide<sup>[<a href="#ref1" style="color:#520049">1</a>,<a href="#ref3" style="color:#520049">3</a>]</sup>.
<br>


<p class="header_box" id="Structure">Structure</p>
<p>The 2D structure of the figures is based on the article by ribodraw tool to draw<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.<br></p>
<p>B-2 RNA aptamer: </p>
<p>5'-GGGAAACUGGAUGGAAUGGGCUCGAUGAAAAUCGACCGUGCGCUGAAAAGCACGCGAGGUCCUGCUGUAAGUGUGCCA-3'<br></p>
<p>B-6 RNA aptamer: </p>
<p>5'-GGGAAACUGGAUGGAAUGGGCUCGCAAGGCCCCCUGAGCUGCACAAGUUCAUGGCGAGGUCCUGCUGUAAGUGUGCCA-3'</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/R5_aptamer_2D1.svg" alt="drawing" style="width:500px;height:400px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/R5_aptamer_2D2.svg" alt="drawing" style="width:500px;height:400px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>



<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>A simple R5 helix peptide as a model of N peptide from bacteriophade λ. Enterobacteria phage λ (lambda phage, coliphage λ, officially Escherichia virus Lambda) is a bacterial virus, or bacteriophage, that infects the bacterial species Escherichia coli (E. coli). It was discovered by Esther Lederberg in 1950. The virus particle consists of a head and a tail that can have tail fibers. The whole particle consists of 12–14 different proteins with more than 1000 protein molecules total and one DNA molecule located in the phage head. All characterized lambdoid phages possess an N protein-mediated transcription antitermination mechanism, with the exception of phage HK022.-----From WiKi</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Bacteriophage λ N-Peptide</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P03045/" target="_blank" style="color:#520049"><b>P03045</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR020952/" target="_blank" style="color:#520049"><b>IPR020952</b></a></td>
        <td name="td3">12.3 KDa</td>
        <td name="td4">MDAQTRRRERRAEKQAQWKAANPLLVGVSAKPVNRPILSLNRKPKSRVESALNPIDLTVLAEYHKQIESNLQRIERKNQRTWYSKPGERGITCSGRQKIKGKSIPLI</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/5LM7" target="_blank" style="color:#520049"><b>5LM7</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/2703540" target="_blank" style="color:#520049"><b>2703540</b></a></td>
      </tr>
	  </tbody>
  </table>

  <p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
      <td name="td0">B-2 RNA aptamer</td>
      <td name="td1">5'-GGGAAACUGGAUGGAAUGGGCUCGAUGAAAAUCGACCGUGCGCUGAAAAGCACGCGAGGUCCUGCUGUAAGUGUGCCA-3'</td>
      <td name="td2">N-model-peptide</td>
      <td name="td3">24 ± 2 nM</td>
    </tr>
            
     <tr>
      <td name="td0">B-6 RNA aptamer</td>
      <td name="td1">5'-GGGAAACUGGAUGGAAUGGGCUCGCAAGGCCCCCUGAGCUGCACAAGUUCAUGGCGAGGUCCUGCUGUAAGUGUGCCA-3'</td>
      <td name="td2">N-model-peptide</td>
      <td name="td3">11 ± 0.1 nM</td>
    </tr>
            
     <tr>
      <td name="td0">B-7 RNA aptamer</td>
      <td name="td1">5'-GGGAAACUGGAUGGAAUGGGCUCGCCAAAGUUGAAAAGCAGGAUACACGGCACUGCGAGGUCCUGCUGUAAGUGUGCCA-3'</td>
      <td name="td2">N-model-peptide</td>
      <td name="td3">5.3 ± 0.5 nM</td>
    </tr>
	  </tbody>
  </table>
<img src="/images/SELEX_ligand/R5_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">5LM7-N</td>
      <td name="td1">7.2</td>
      <td name="td2">0.0</td>
      <td name="td3">Transcription termination/antitermination protein</td>
    </tr>
            
     <tr>
      <td name="td0">6GOV-N</td>
      <td name="td1">4.4</td>
      <td name="td2">2.5</td>
      <td name="td3">Transcription termination/antitermination protein</td>
    </tr>
            
     <tr>
      <td name="td0">5LM7-F</td>
      <td name="td1">4.3</td>
      <td name="td2">2.6</td>
      <td name="td3">Transcription termination/antitermination protein</td>
    </tr>
            
     <tr>
      <td name="td0">6EXN-D</td>
      <td name="td1">2.1</td>
      <td name="td2">7.2</td>
      <td name="td3">U2 snRNA</td>
    </tr>
            
     <tr>
      <td name="td0">6CP3-Z</td>
      <td name="td1">2.1</td>
      <td name="td2">15.1</td>
      <td name="td3">ATP synthase subunit 9, mitochondrial</td>
    </tr>
            
     <tr>
      <td name="td0">4AUR-A</td>
      <td name="td1">2.0</td>
      <td name="td2">39.6</td>
      <td name="td3">Leoa</td>
    </tr>
    </tbody>
  </table>



               
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] In vitro selection and analysis of RNA aptamer recognize arginine-rich motif (ARM) model peptide on a QCM.</strong></font><br />
Fukusho, S., Furusawa, H., & Okahata, Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12903331/" target="_blank" style="color:#520049">Nucleic acids symposium series, (44), 187–188. (2000)</a>
<br/>
            
<a id="ref2"></a><font><strong>[2] In vitro selection and evaluation of RNA aptamers that recognize arginine-rich-motif model peptide on a quartz-crystal microbalance.</strong></font><br />
Fukusho, S., Furusawa, H., & Okahata, Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12120324/" target="_blank" style="color:#520049">Chemical communications (Cambridge, England), (1), 88–89. (2002)</a>
<br/>
            
<a id="ref3"></a><font><strong>[3] Arginine arrangement of bacteriophage λ N-peptide plays a role as a core motif in GNRA tetraloop RNA binding.</strong></font><br />
Furusawa, H., Fukusho, S., & Okahata, Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/24623705/" target="_blank" style="color:#520049">Chembiochem : a European journal of chemical biology, 15(6), 865–871. (2014)</a>
<br/>


<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>




    </html>
