---
layout: theophylline_style
title: Restriction endonucleases (REases) KpnI-aptamer
date: 2024-05-31 00:00:00
author: Yangyi Ren, Chen Ke
categories: Aptamer
tags:
- Category:Mammalian
- GC:37.31
- GC:41.18
- GC:44.78
- GC:46.27
- Length:67
- Length:68
- Named:Restriction endonucleases (REases) KpnI_Anti-KpnI RNA Aptamer 20
- Named:Restriction endonucleases (REases) KpnI_Anti-KpnI RNA Aptamer 24
- Named:Restriction endonucleases (REases) KpnI_Anti-KpnI RNA Aptamer 29
- Named:Restriction endonucleases (REases) KpnI_Anti-KpnI RNA Aptamer 30
- Type:Proteins
- Year:2015
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="description">Description</p>
<p>In 2015, Estefanía Mondragón and colleagues used SELEX (Systematic Evolution of Ligands by Exponential enrichment) technique to isolate high affinity RNA aptamers that bind to the Reases KpnI protein<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2015, Estefanía Mondragón and L. James Maher, III isolated 10 specific sequences for targeting Restriction endonucleases (REases) in 20 rounds of selection<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>The 2D structure of the figure is based on the article by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>
<p>5'-GGGAUAUCCUUCAUGAUUCGGAAAAGCGUAACCGCGGCCAAUCAUGCAUGAAGCGUUCCAUAUUUUU-3'</p>
<img src="/images/2D/REases_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>The restriction endonucleases (REases) KpnI is an orthodox Type IIP enzyme, which binds to DNA in the absence of metal ions and cleaves the DNA sequence 5′-GGTAC^C-3′ in the presence of Mg2+ as shown generating 3′ four base overhangs.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Restriction endonucleases (REases) KpnI</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P04062/entry" target="_blank" style="color:#520049"><b>P04062</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF02055" target="_blank" style="color:#520049"><b>PF02055</b></a></td>
        <td name="td3">59.716 kDa</td>
        <td name="td4">MEFSSPSREECPKPLSRVSIMAGSLTGLLLLQAVSWASGARPCIPKSFGYSSVVCVCNATYCDSFDPPTFPALGTFSRYESTRSGRRMELSMGPIQANHTGTGLLLTLQPEQKFQKVKGFGGAMTDAAALNILALSPPAQNLLLKSYFSEEGIGYNIIRVPMASCDFSIRTYTYADTPDDFQLHNFSLPEEDTKLKIPLIHRALQLAQRPVSLLASPWTSPTWLKTNGAVNGKGSLKGQPGDIYHQTWARYFVKFLDAYAEHKLQFWAVTAENEPSAGLLSGYPFQCLGFTPEHQRDFIARDLGPTLANSTHHNVRLLMLDDQRLLLPHWAKVVLTDPEAAKYVHGIAVHWYLDFLAPAKATLGETHRLFPNTMLFASEACVGSKFWEQSVRLGSWDRGMQYSHSIITNLLYHVVGWTDWNLALNPEGGPNWVRNFVDSPIIVDITKDTFYKQPMFYHLGHFSKFIPEGSQRVGLVASQKNDLDAVALMHPDGSAVVVVLNRSSKDVPLTIKDPAVGFLETISPGYSIHTYLWRRQ</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/1OGS" target="_blank" style="color:#520049"><b>1OGS</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/2629" target="_blank" style="color:#520049"><b>2629</b></a></td>
      </tr>
	  </tbody>
  </table>

<p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
      <td name="td0">anti-KpnI RNA Aptamer 20</td>
      <td name="td1">5'-GGGAUAUCCUUCAUGAUUCGGAAAAGCGUAACCGCGGCCAAUCAUGCAUGAAGCGUUCCAUAUUUUU-3'</td>
      <td name="td2">REases KpnI</td>
      <td name="td3">11.6 ± 2.3 nM</td>
    </tr>
     <tr>
      <td name="td0">anti-KpnI RNA Aptamer 24</td>
      <td name="td1">5'-GGGAUAUCCUUCAUGGUGUGAUUAAACGUAAUCUCAAUCGAUCAUGCAUGAAGCGUUCCAUAUUUUU-3'</td>
      <td name="td2">REases KpnI</td>
      <td name="td3">28.7 ± 5.2 nM</td>
    </tr>
     <tr>
      <td name="td0">anti-KpnI RNA Aptamer 29</td>
      <td name="td1">5'-GGGAUAUCCCUACAUGAUAAUCGGGUGAGUAAGGCCUAAACAUCAUGCAUGAAGCGUUCCAUAUUUUU-3'</td>
      <td name="td2">REases KpnI</td>
      <td name="td3">419 ± 46.6 nM</td>
    </tr>
     <tr>
      <td name="td0">anti-KpnI RNA Aptamer 30</td>
      <td name="td1">5'-GGGAUAUCCUUCAUGAUUCGGAAAAGCGUAACCGCGGCCGAUCAUGCAUGAAGCGUUCCAUAUUUUU-3'</td>
      <td name="td2">REases KpnI</td>
      <td name="td3">18.6 ± 4.5 nM</td>
    </tr>
   </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/REases_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">2V3E-A</td>
      <td name="td1">66.2</td>
      <td name="td2">0.7</td>
      <td name="td3">Glucosylceramidase</td>
    </tr>
            
     <tr>
      <td name="td0">6KRN-A</td>
      <td name="td1">37.0</td>
      <td name="td2">2.1</td>
      <td name="td3">Mating factor alpha,gh30 xylanase b</td>
    </tr>
            
     <tr>
      <td name="td0">4QAW-A</td>
      <td name="td1">33.0</td>
      <td name="td2">2.1</td>
      <td name="td3">Xyn30d</td>
    </tr>
            
     <tr>
      <td name="td0">3CLW-A</td>
      <td name="td1">31.4</td>
      <td name="td2">2.6</td>
      <td name="td3">Conserved exported protein</td>
    </tr>
            
     <tr>
      <td name="td0">5N8K-A</td>
      <td name="td1">27.9</td>
      <td name="td2">2.4</td>
      <td name="td3">Galactocerebrosidase</td>
    </tr>
            
     <tr>
      <td name="td0">5JVK-A</td>
      <td name="td1">23.4</td>
      <td name="td2">3.6</td>
      <td name="td3">Uncharacterized protein</td>
    </tr>
            
     <tr>
      <td name="td0">1R8L-B</td>
      <td name="td1">22.0</td>
      <td name="td2">3.2</td>
      <td name="td3">Endo-beta-1,4-galactanase</td>
    </tr>
            
     <tr>
      <td name="td0">5BX9-A</td>
      <td name="td1">21.9</td>
      <td name="td2">3.8</td>
      <td name="td3">Pslg</td>
    </tr>
            
     <tr>
      <td name="td0">2VRK-A</td>
      <td name="td1">21.7</td>
      <td name="td2">2.8</td>
      <td name="td3">Alpha-l-arabinofuranosidase</td>
    </tr>
            
     <tr>
      <td name="td0">8I0A-A</td>
      <td name="td1">21.0</td>
      <td name="td2">2.9</td>
      <td name="td3">Alpha-l-arabinofuranosidase</td>
    </tr>
    </tbody>
  </table>
                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1]  RNA aptamer inhibitors of a restriction endonuclease.</strong></font><br />
Mondragón, E., & Maher III, L. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/26184872/" target="_blank" style="color:#520049">Nucleic acids research, 43(15), 7544-7555. (2015)</a>
<br/>


<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1RAW-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>
    
    </html>
