---
layout: theophylline_style
title: RF-1-aptamer
date: 2024-06-02 00:00:00
author: Zhijie Tan, Qiaozhen Liu
categories: Aptamer
tags:
- Category:Bacteria
- GC:56.10000000000001
- Length:82
- Named:Escherichia coli release factor 1_Class II-1
- Type:Proteins
- Year:2007
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="description">Description</p>
<p>In 2007, Sando et al. selected the class II-1 aptamer against gel-immobilized Escherichia coli release factor 1 (RF-1) responsible for translation termination at amber (UAG) stop codon using a pool of 84-nt RNAs containing a randomized sequence of 50 nt<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2007, Suenaga et al. identified isolated specific sequences for targeting HA in a 84nt sequence pool through 11 rounds of selection<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>Class II-1 was the aptamer sequence mainly studied in the article, which had a high affinity with RF 1. The 2D structure of the figure is based on the article by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>
<p>5'-GGACCGAGAAGUUACCCUGUAAUCUUAGGAUGAAUCGCAUGCUCUAGCGACCUUUUCGGCUUCGGCGUACGCACAUCGCAGCAAC-3'</p>
<img src="/images/2D/RF1_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>This entry represents peptide chain release factor 1 (PrfA, RF-1), and excludes the related peptide chain release factor 2 (PrfB, RF-2). RF-1 helps recognise and terminate translation at UAA and UAG stop codons. This entry also includes chloroplast release factor APG3, which is an orthologue of E. coli RF1 and is essential for chloroplast development. APG3 could complement E. coli RF1 mutants, but not RF2 mutants, indicating that AP3 may function as RF1 in E. coli, showing specificity of stop codon recognition.-----from Pfam</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">release factor 1 (RF-1)</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/Q9X183/entry" target="_blank" style="color:#520049"><b>Q9X183</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR004373/" target="_blank" style="color:#520049"><b>IPR004373</b></a></td>
        <td name="td3">119.15 kDa</td>
        <td name="td4">MKEKKKEIEKLLARPDLTPEQMKNYGMEYAKIEEIENITNRIKETQEFIELLREEGENELEIEKYEKELDQLYQELLFLLSPEASDKAIVEIRPGTGGEEAALFARDLFRMYTRYAERKGWNLEVAEIHETDLGGIREVVFFVKGKNAYGILKYESGVHRVQRVPVTESGGRIHTSTATVAVLPEIEEKDIEIRPEDLKIETFRASGHGGQYVNKTESAVRITHLPTGIVVSCQNERSQYQNKQTALRILRARLYQLQKEQKEREISQKRKSQIGTGERSEKIRTYNFPQNRVTDHRINYTSYRLQEILDGDLDEIISKLIEHDIENNLEEVLGIGASVEEK</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/1RQ0" target="_blank" style="color:#520049"><b>1RQ0</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_004081559.1" target="_blank" style="color:#520049"><b>WP_004081559.1</b></a></td>
      </tr>
	  </tbody>
  </table>

<p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">class II-1 aptamer</td>
        <td name="td1">5'-GGACCGAGAAGUUACCCUGUAAUCUUAGGAUGAAUCGCAUGCUCUAGCGACCUUUUCGGCUUCGGCGUACGCACAUCGCAGCAAC-3'</td>
        <td name="td2">RF 1</td>
        <td name="td3">30 nM</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/RF1_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">1GQE-A</td>
      <td name="td1">25.7</td>
      <td name="td2">4.8</td>
      <td name="td3">Release factor 2</td>
    </tr>
            
     <tr>
      <td name="td0">5U4J-V</td>
      <td name="td1">14.1</td>
      <td name="td2">2.7</td>
      <td name="td3">16S rrna</td>
    </tr>
            
     <tr>
      <td name="td0">4NEY-B</td>
      <td name="td1">5.6</td>
      <td name="td2">4.4</td>
      <td name="td3">Engineered protein or277</td>
    </tr>
            
     <tr>
      <td name="td0">2RSM-A</td>
      <td name="td1">5.5</td>
      <td name="td2">5.5</td>
      <td name="td3">Probable peptide chain release factor c12orf65 HO</td>
    </tr>
            
     <tr>
      <td name="td0">4KYZ-A</td>
      <td name="td1">5.2</td>
      <td name="td2">8.6</td>
      <td name="td3">Designed protein or327</td>
    </tr>
            
     <tr>
      <td name="td0">3LAY-J</td>
      <td name="td1">4.9</td>
      <td name="td2">5.0</td>
      <td name="td3">Zinc resistance-associated protein</td>
    </tr>
            
     <tr>
      <td name="td0">4UER-J</td>
      <td name="td1">4.8</td>
      <td name="td2">3.4</td>
      <td name="td3">Eif1a</td>
    </tr>
            
     <tr>
      <td name="td0">6D3G-C</td>
      <td name="td1">4.6</td>
      <td name="td2">13.4</td>
      <td name="td3">Beta-lactamase</td>
    </tr>
            
     <tr>
      <td name="td0">6ELU-A</td>
      <td name="td1">4.6</td>
      <td name="td2">3.3</td>
      <td name="td3">Serum resistance associated VSG protein</td>
    </tr>
            
     <tr>
      <td name="td0">1U8S-B</td>
      <td name="td1">4.5</td>
      <td name="td2">4.6</td>
      <td name="td3">Glycine cleavage system transcriptional</td>
    </tr>
    </tbody>
  </table>
                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] In vitro selection of RNA aptamer against Escherichia coli release factor 1.</strong></font><br />
Suenaga, E., & Kumar, PK. Sando, S., Ogawa, A., Nishi, T., Hayami, M., & Aoyama, Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/17188871" target="_blank" style="color:#520049">Bioorganic & Medicinal Chemistry Letters,17(5):1216-20. (2007)</a>
<br/>


<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1RAW-3D.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>

    <script>
    window.addEventListener('DOMContentLoaded', function() {
      var td = document.getElementsByName('td4')[0];
      var maxLength = 100; // 设置最大长度
      var originalText = td.innerHTML;
      var displayedText = originalText.substring(0, maxLength);
      
      if (originalText.length > maxLength) {
        displayedText += '<span class="ellipsis-btn">......</span>'; // 在超出长度时添加省略号按钮
        displayedText += '<span class="hidden-content">' + originalText.substring(maxLength) + '</span>'; // 隐藏的内容
        td.innerHTML = displayedText;
        
        // 获取省略号按钮元素和隐藏的内容元素
        var ellipsisBtn = td.querySelector('.ellipsis-btn');
        var hiddenContent = td.querySelector('.hidden-content');
        
        // 添加点击事件监听器
        ellipsisBtn.addEventListener('click', function() {
          if (hiddenContent.style.display === 'none') {
            hiddenContent.style.display = 'inline'; // 展开内容
            ellipsisBtn.innerHTML = '...'; // 更新按钮文字为“...”
            td.appendChild(ellipsisBtn); // 将按钮移到内容后面
          } else {
            hiddenContent.style.display = 'none'; // 收起内容
            ellipsisBtn.innerHTML = '......'; // 更新按钮文字为省略号
            td.appendChild(ellipsisBtn); // 将按钮移到内容后面
          }
        });
      }
    });
  </script>
    </html>
