---
layout: theophylline_style
title: Ribosomal protein S8 aptamer
date: 2024-04-10 00:00:00
author: Qiaozhen Liu, Zhaoji Qiu
categories: Aptamer
tags:
- Category:Bacteria
- GC:57.99999999999999
- Length:38
- Named:Ribosomal protein S8_RNA-2 aptamer
- Type:Proteins
- Year:1997
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/9056763/" target="_blank" style="color:#520049">1997</a></h3>
    </div>
    <div class="body">
      <p>Performed in vitro iterative selection of RNA aptamers that bind S8<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/25140011/" target="_blank" style="color:#520049">2014</a></h3>
    </div>
    <div class="body">
      <p>Structure analysis of an RNA aptamer against ribosomal protein S8 from Bacillus anthracis<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/29413905/" target="_blank" style="color:#520049">2018</a></h3>
    </div>
    <div class="body">
      <p>Present data indicate that S8 recognizes the aptamer through an induced-fit rather than a population-shift mechanism<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
</div>


         
<p class="header_box" id="description">Description</p>
<p>In 1997, Moine Herve et al. conducted in vitro iterative selection of RNA aptamers capable of binding to S8. These aptamers revealed features in the S8 binding site that had been concealed during previous characterizations by the high base conservation throughout evolution. In 2014, Milya Davlieva et al. investigated the structural adaptation of an RNA aptamer binding to bacterial ribosomal protein S8<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref2" style="color:#520049">2</a></sup><sup>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2014, Milya Davlieva et al. identified RNA sequences that lacked the conserved features of helix 21 but could bind the S8 protein with high affinity and specificity. The starting library had molecules with 16 randomized nucleotides in an RNA hairpin stem. After 10 rounds of selection, the RNA pool was cloned and 40 inserts sequenced. Alignment showed native-like and non-natural binding sites<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p>
<p>
<br>
<br>   



<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>The structure of aptamer RNA-2 is clearly defined, having a global fold of a hairpin that is terminated on one end by the UUCG tetraloop. The RNA-2 aptamer was named by Milya Davlieva et al. in the article. The RNA-2 aptamer was the main aptamer sequence studied in the Structure article. Here, we utilized Ribodraw to complete the figure based on the 3D structure information<<sup>[<a href="#ref2" style="color:#520049">2</a></sup><sup>,<a href="#ref5" style="color:#520049">5</a></sup><sup>]</sup>.</p>
<p>5'-GGGAUGCUCAGUGAUCCUUCGGGAUAUCAGGGCAUCCC-3'</p>
<img src="/images/2D/ribosomal_protein_S8_aptamer_2D.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">3D visualisation</p>             
<p>Milya Davlieva et al. sovled the crystal structure, at 2.69 Å resolution, of an RNA aptamer bound to Bacillus ribosomal S8 protein has been determined. In the free state, the internal loop of the ribosomal protein S8 aptamer hairpin stem encompasses G-A, U-U and A-A mismatches, adopting an overall helical A-form geometry. For the purpose of binding Ribosomal protein S8, the internal loop undergoes a significant rearrangement of its secondary structure, giving rise to the formation of a base triple and a base quartet. The PDB ID of this structure is 4PDB<sup>[<a href="#ref2" style="color:#520049">2</a></sup><sup>]</sup>.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/ribosomal_protein_S8_aptamer_3D1.svg" alt="drawing" style="width:500px;height:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">

      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px !important;
            height: 500px !important;
            position:relative;
            min-width: 500px;
            min-height: 500px;
          }
          
          /* 确保Molstar插件不受外部样式影响 */
          .msp-plugin {
            width: 500px !important;
            height: 500px !important;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
{
  struct_asym_id: 'A', 
  start_residue_number:27, 
  end_residue_number:155, 
  color:{r:133,g:39,b:181}
},
{
  struct_asym_id: 'I', 
  start_residue_number:1, 
  end_residue_number:9, 
  color:{r:184,g:51,b:50}
},
{
  struct_asym_id: 'I', 
  start_residue_number:10, 
  end_residue_number:14, 
  color:{r:46,g:161,b:3}
},
{
  struct_asym_id: 'I', 
  start_residue_number:15, 
  end_residue_number:15, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'I', 
  start_residue_number:16, 
  end_residue_number:17, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'I', 
  start_residue_number:18, 
  end_residue_number:23, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'I', 
  start_residue_number:24, 
  end_residue_number:27, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'I', 
  start_residue_number:28, 
  end_residue_number:29, 
  color:{r:46,g:161,b:3}
},
{
  struct_asym_id: 'I', 
  start_residue_number:30, 
  end_residue_number:39, 
  color:{r:184,g:51,b:50}
},
{
  struct_asym_id: 'I', 
  start_residue_number:1, 
  end_residue_number:9, 
  color:{r:184,g:51,b:50}
}
]
viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/4pdb-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<p>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 4PDB at 2.69 Å resolution. ribosomal S8 protein (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with ribosomal S8 protein.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/ribosomal_protein_S8_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/ribosomal_protein_S8_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>



<p class="header_box" id="ligand-recognition">Ligand information</p>
<p class="blowheader_box">SELEX ligand</p>
<p>Milya Davlieva et al. investigated the affinity of the RNA aptamer via EMSA (electrophoretic mobility shift assay) and ITC (isothermal titration calorimetry) experiments<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">RNA-2 aptamer</td>
        <td name="td1">5'-GGGAUGCUCAGUGAUCCUUCGGGAUAUCAGGGCAUCCC-3'</td>
        <td name="td3">Bacillus ribosomal protein S8</td>
        <td name="td4">110 ± 30 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<p>40S ribosomal protein S8 is a protein that in humans is encoded by the RPS8 gene. Ribosomes, the organelles that catalyze protein synthesis, consist of a small 40S subunit and a large 60S subunit. Together these subunits are composed of 4 RNA species and approximately 80 structurally distinct proteins. This gene encodes a ribosomal protein that is a component of the 40S subunit. The protein belongs to the S8E family of ribosomal proteins.-----From <a href="https://en.wikipedia.org/wiki/40S_ribosomal_protein_S8" target="_blank" style="color:#520049; text-decoration: underline;"><b>Wiki</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">Mass</th>
        <th onclick="sortTable(3)">Protein sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/Q81VR6/entry" target="_blank" style="color:#520049"><b>Q81VR6</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR035987/" target="_blank" style="color:#520049"><b>IPR035987</b></a></td>
        <td name="td2">29.24 kDa</td>
        <td name="td3">
         <div class="sequence-container">

          <span class="full-sequence">VMTDPIADMLTAIRNANMVRHEKLEVPASKIKREIAEILKREGFIRDYEYIEDNKQGILRIFLKYGPNERVITGLKRISKPGLRVYVKAHEVPRVLNGLGIAILSTSQGVLTDKEARQKGTGGEIIAYVIVMTDPIADMLTAIRNANMVRHEKLEVPASKIKREIAEILKREGFIRDYEYIEDNKQGILRIFLKYGPNERVITGLKRISKPGLRVYVKAHEVPRVLNGLGIAILSTSQGVLTDKEARQKGTGGEIIAYVI</span>
        </div>
      </td>
        <td name="td4"><a href="https://www.rcsb.org/structure/1SEI" target="_blank" style="color:#520049"><b>1SEI</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/gene/947802" target="_blank" style="color:#520049"><b>947802</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/ribosomal_protein_S8_stru_ligand1.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and selected the previous information with high similarity for presentation.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">1SEI-A</td>
        <td name="td2">28</td>
        <td name="td3">0</td>
        <td name="td4">Original chain</td>
      </tr>
      <tr>
        <td name="td1">3RF2-A</td>
        <td name="td2">18.5</td>
        <td name="td3">1.9</td>
        <td name="td4">30s ribosomal protein s8</td>
      </tr>
      <tr>
        <td name="td1">8D8J-H</td>
        <td name="td2">18.2</td>
        <td name="td3">1.8</td>
        <td name="td4">probable s-adenosyl-l-methioni</td>
      </tr>
      <tr>
        <td name="td1">7QIX-T</td>
        <td name="td2">17.2</td>
        <td name="td3">2.2</td>
        <td name="td4">18s rrna body</td>
      </tr>
      <tr>
        <td name="td1">1I6U-A</td>
        <td name="td2">17.1</td>
        <td name="td3">2.2</td>
        <td name="td4">16s rrna fragment</td>
      </tr>
      <tr>
        <td name="td1">6ZUO-W</td>
        <td name="td2">16.6</td>
        <td name="td3">2.4</td>
        <td name="td4">pre-18s ribosomal rna</td>
      </tr>
      <tr>
        <td name="td1">7PWF-W</td>
        <td name="td2">16.4</td>
        <td name="td3">2.3</td>
        <td name="td4">ribosomal protein s29a</td>
      </tr>
      <tr>
        <td name="td1">6WDR-W</td>
        <td name="td2">15.8</td>
        <td name="td3">2.4</td>
        <td name="td4">40s ribosomal protein a0-a</td>
      </tr>
      <tr>
        <td name="td1">7PKU-h</td>
        <td name="td2">13</td>
        <td name="td3">2.9</td>
        <td name="td4">ms35</td>
      </tr>
      <tr>
        <td name="td1">4OCH-A</td>
        <td name="td2">6.8</td>
        <td name="td3">2.4</td>
        <td name="td4">endonuclease muts2</td>
      </tr>
	  </tbody>
  </table>



                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] The RNA binding site of S8 ribosomal protein of Escherichia coli: Selex and hydroxyl radical probing studies.</strong></font><br />
Moine, H., Cachia, C., Westhof, E., Ehresmann, B., & Ehresmann, C.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/9056763/" target="_blank" style="color:#520049" >RNA (New York, N.Y.), 3(3), 255–268. (1997)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Structure analysis of free and bound states of an RNA aptamer against ribosomal protein S8 from Bacillus anthracis.</strong></font><br />
Davlieva, M., Donarski, J., Wang, J., Shamoo, Y., & Nikonowicz, E. P.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/25140011/" target="_blank" style="color:#520049">Nucleic acids research, 42(16), 10795–10808. (2014)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] The intrinsic flexibility of the aptamer targeting the ribosomal protein S8 is a key factor for the molecular recognition.</strong></font><br />
Autiero, I., Ruvo, M., Improta, R., & Vitagliano, L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/29413905/" target="_blank" style="color:#520049"> Biochimica et Biophysica Acta (BBA)-General Subjects, 1862(4), 1006-1016. (2018)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] Modelling aptamers with nucleic acid mimics (NAM): From sequence to three-dimensional docking.</strong></font><br />
Oliveira, R., Pinho, E., Sousa, A. L., Dias, Ó., Azevedo, N. F., & Almeida, C.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/35320268/" target="_blank" style="color:#520049">PloS one, 17(3), e0264701. (2022)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Investigating RNA-protein recognition mechanisms through supervised molecular dynamics (SuMD) simulations.</strong></font><br />
Pavan, M., Bassani, D., Sturlese, M., & Moro, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/36458023/" target="_blank" style="color:#520049">NAR genomics and bioinformatics, 4(4), lqac088. (2022)</a>
<br />     




<script>
    function toggleSequence(event) {
      const container = event.target.closest('.sequence-container');
      container.classList.toggle('expanded');
      const showMoreText = container.querySelector('.show-more');
      
      // 展开后按钮文本变化
      if (container.classList.contains('expanded')) {
        showMoreText.textContent = '...';  // 展开后显示 "..."
      } else {
        showMoreText.textContent = '......';  // 收起后显示 "......"
      }
    }

    // 页面加载时，限制序列文本为50个字符
    window.addEventListener('load', function() {
      const sequenceContainers = document.querySelectorAll('.sequence-container');
      sequenceContainers.forEach(container => {
        const fullSeqText = container.querySelector('.full-sequence').textContent;
        const truncatedText = fullSeqText.slice(0, 20);  // 只显示前50个字符
        container.querySelector('.sequence-text').textContent = truncatedText;
      });
    });
  </script>
