---
layout: theophylline_style
title:  "Spinach RNA aptamer, Aptamer 24-2-min(Spinach)"
date:   2024-04-07 00:00:00
author: Jiaxin Zhao, Ke Chen
categories: Aptamer
---
<html>



<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
<div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/21798953/" target="_blank" style="color:#520049">2011</a></h3>
    </div>
    <div class="body">
      <p>The first fluorogenic RNA aptamer ‘Spinach’ which was capable of turning on the fluorescence were isolated upon the binding of small-molecule fluorophores<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/23488798/" target="_blank" style="color:#520049">2013</a></h3>
    </div>
    <div class="body">
      <p>Spinach RNA was engineered as a genetically-encoded tag to the cellular metabolites and second messenger cyclic di-AMP (cdiA)<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/24952597/" target="_blank" style="color:#520049">2014</a></h3>
    </div>
    <div class="body">
      <p>Piccirilli, J. A. et al. used antibody-assisted crystallography to determine the structures of Spinach RNA both with and without bound fluorophore at resolutions of 2.2 Å and 2.4 Å respectively<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/25026079/" target="_blank" style="color:#520049">2014</a></h3>
    </div>
    <div class="body">
      <p>Ferré-D'Amaré, A. R. et al. solved the cocrystal structure of Spinach RNA bound to its cognate exogenous chromophore<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/25964329/" target="_blank" style="color:#520049">2015</a></h3>
    </div>
    <div class="body">
      <p>Jaffrey, S. R. et al. showed that Spinach RNA can be swapped into various riboswitches, allowing metabolite binding to induce Spinach fluorescence directly<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/29309709/" target="_blank" style="color:#520049">2018</a></h3>
    </div>
    <div class="body">
      <p>Piccirilli, J. A. et al. replaced the P2 stem loop of Spinach RNA with the pentaloop and crystallized the aptamer as a complex with Fab BL3–6<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
</div>


         
<p class="header_box" id="description">Description</p>
<p>In 2011, Jaffrey, S. R. et al. isolated the first fluorogenic RNA aptamer ‘Spinach’ upon the binding of small-molecule fluorophores, which was capable of turning on the fluorescence. In 2018, Piccirilli, J. A. et al. replaced the P2 stem loop of Spinach RNA with the pentaloop and crystallized the aptamer as a complex with Fab BL3–6. Fab BL3-6S97N is an affinity-matured variant of the Fab BL3-6 antibody fragment, featuring a key S97N mutation in CDR L3. This mutation introduces asparagine, forming additional hydrogen bonds with RNA nucleobase A40, enhancing binding affinity by ~20-fold compared to the parent Fab. The crystal structure of Fab BL3-6S97N bound to the GAAACAC RNA motif shows N97’s carboxamide group within hydrogen-bonding distance of A40, strengthening interactions<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref6" style="color:#520049">6</a></sup><sup>]</sup>.<br></p>



<p class="header_box" id="SELEX">SELEX</p>
<p>In 2011, Jaffrey, S. R. et al. identified RNA sequences that bind and activate the fluorescence of GFP fluorophores beginning with 3,5-dimethoxy-4-hydroxybenzylidene imidazolinone (DMHBI). And then they performed SELEX with a library containing ~5 × 10<sup>13</sup> RNA molecules and selected RNAs for their ability to bind DMHBI-agarose. After five rounds of selection, the pool of RNAs weakly activated DMHBI fluorescence, with further increases in fluorescence up to round 10<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>
<br>
<br>   



<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>In 2011, the first fluorogenic RNA aptamer ‘Spinach’ which was capable of turning on the fluorescence were isolated upon the binding of small-molecule fluorophores. Here we utilized RiboDraw to complete the figure, based the 3D structure information<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>5'-GACGCGACCGAAAUGGUGAAGGACGGGUCCAGUGCGAAACACGCACUGUUGAGUAGAGUGUGAGCUCCGUAACUGGUCGCGUC-3'</p>
<img src="/images/2D/spinach_aptamer_2D.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">3D visualisation</p>             
<p>In 2018, Piccirilli, J.A. et al. determined the crystal structure of Spinach RNA aptamer in complex with Fab BL3-6S97N, at 1.64 Å resolution. The PDB ID of this structure is 6B14<sup>[<a href="#ref6" style="color:#520049">6</a></sup><sup>]</sup>.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/spinach_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'H',    
  start_residue_number:4, 
  end_residue_number:215, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'R',    
  start_residue_number:2, 
  end_residue_number:21, 
  color:{r:184,g:51,b:50}
},
{
  struct_asym_id: 'R',    
  start_residue_number:22, 
  end_residue_number:23, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'R',    
  start_residue_number:24, 
  end_residue_number:25, 
  color:{r:194,g:182,b:23}
},
{
  struct_asym_id: 'R',    
  start_residue_number:26, 
  end_residue_number:27, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'R',    
  start_residue_number:28, 
  end_residue_number:28, 
  color:{r:194,g:182,b:23}
},
{
  struct_asym_id: 'R',    
  start_residue_number:29, 
  end_residue_number:37, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'R',    
  start_residue_number:38, 
  end_residue_number:42, 
  color:{r:46,g:161,b:3}
},
{
  struct_asym_id: 'R',    
  start_residue_number:43, 
  end_residue_number:53, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'R',    
  start_residue_number:54, 
  end_residue_number:54, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'R',    
  start_residue_number:55, 
  end_residue_number:56, 
  color:{r:194,g:182,b:23}
},
{
  struct_asym_id: 'R',    
  start_residue_number:57, 
  end_residue_number:57, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'R',    
  start_residue_number:58, 
  end_residue_number:58, 
  color:{r:194,g:182,b:23}
},
{
  struct_asym_id: 'R',    
  start_residue_number:59, 
  end_residue_number:59, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'R',    
  start_residue_number:60, 
  end_residue_number:60, 
  color:{r:194,g:182,b:23}
},
{
  struct_asym_id: 'R',    
  start_residue_number:61, 
  end_residue_number:61, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'R',    
  start_residue_number:62, 
  end_residue_number:63, 
  color:{r:194,g:182,b:23}
},
{
  struct_asym_id: 'R',    
  start_residue_number:64, 
  end_residue_number:84, 
  color:{r:184,g:51,b:50}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/6B14-3D.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<p>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 6B14. Fab BL3-6S97N (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with Fab BL3-6S97N.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/spinach_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/spinach_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>



<p class="header_box" id="ligand-recognition">Ligand information</p> 
<p class="blowheader_box">SELEX ligand</p>
<p>Antibody fragments such as Fabs possess properties that can enhance protein and RNA crystallization and therefore can facilitate macromolecular structure determination. Piccirilli, J. A, et al. mainly studied the crystal structure of the complex of Spinach RNA with Fab BL3-6S97N. According to the article, the affinity of Spinach RNA with Fab BL3-6S97N complex is 25 ± 6 nM. In addition, the crystal structure of the Spinach RNA complex with DFHBI was investigated with an affinity of 300 ± 68 nM. The binding constants of selected RNA clones and related mutants were determined by nitrocellulose filter binding assay as reported previously<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup>.</p>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Spinach aptamer</td>
        <td name="td1">5'-GACGCGACCGAAAUGGUGAAGGACGGGUCCAGUGCGAAACACGCACUGUUGAGUAGAGUGUGAGCUCCGUAACUGGUCGCGUC-3'</td>
        <td name="td3">Fab BL3-6S97N</td>
        <td name="td4">25 ± 6 nM</td>
      </tr>
      <tr>
        <td name="td0">Spinach aptamer</td>
        <td name="td1">5'-GACGCGACCGAAAUGGUGAAGGACGGGUCCAGUGCGAAACACGCACUGUUGAGUAGAGUGUGAGCUCCGUAACUGGUCGCGUC-3'</td>
        <td name="td3">DFHBI</td>
        <td name="td4">300 ± 68 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>

<p class="blowheader_box">Structure ligand</p>
<p>Fab (Antigen-binding fragment), also called antigen-binding fragment, is a region in the antibody structure that can bind to antigen. It consists of a complete light chain (variable region and constant region) and a partial heavy chain structure (variable region and a constant region fragment). The light chain and heavy chain are connected by a disulfide bond, which is small in size and has a molecular weight of 47-48 kDa.-----From <a href="https://pubmed.ncbi.nlm.nih.gov/29309709/" target="_blank" style="color:#520049; text-decoration: underline;"><b>article</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">Mass</th>
        <th onclick="sortTable(3)">Protein sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniref/UniRef90_UPI001C68730C" target="_blank" style="color:#520049"><b>UniRef90_UPI001C68730C</b></a></td>
        <td name="td1">NA</td>
        <td name="td2">Heavy chain-24.74 kDa<br>Light chain-23.37 kDa</td>
        <td name="td3">
        <div class="sequence-container">
          <span class="sequence-text">GGPYLQ</span>
          <span class="show-more" onclick="toggleSequence(event)">......</span>
          <span class="full-sequence">Heavy chain-EVQLVESGGGLVQPGGSLRLSCAASGFYISYSSIHWVRQAPGKGLEWVASISPYSGSTYYADSVKGRFTISADTSKNTAYLQMNSLRAEDTAVYYCARQGYRRRSGRGFDYWGQGTLVTVSSASTKGPSVFPLAPSSKSTSGGTAALGCLVKDYFPEPVTVSWNSGALTSGVHTFPAVLQSSGLYSLSSVVTVPSSSLGTQTYICNVNHKPSNTKVDKKVEPKSC<br>Light chain-SDIQMTQSPSSLSASVGDRVTITCRASQSVSSAVAWYQQKPGKAPKLLIYSASSLYSGVPSRFSGSRSGTDFTLTISSLQPEDFATYYCQQSYSFPSTFGQGTKVEIKRTVAAPSVFIFPPSDEQLKSGTASVVCLLNNFYPREAKVQWKVDNALQSGNSQESVTEQDSKDSTYSLSSTLTLSKADYEKHKVYACEVTHQGLSSPVTKSFNRGEC</span>
        </div>
        </td>
        <td name="td4"><a href="https://www.rcsb.org/structure/6B3K" target="_blank" style="color:#520049"><b>6B3K</b></a></td>
        <td name="td5">NA</td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/Spinach_aptamer_stru_ligand.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and selected the previous information with high similarity for presentation.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">6B14-H</td>
        <td name="td2">34.1</td>
        <td name="td3">0</td>
        <td name="td4">Original chain</td>
      </tr>
      <tr>
        <td name="td1">6RUL-A</td>
        <td name="td2">22.5</td>
        <td name="td3">1.8</td>
        <td name="td4">GFP-lama-f98 a gfp enhancer nanobody with cpdhfr</td>
      </tr>
      <tr>
        <td name="td1">8IQS-H</td>
        <td name="td2">22.4</td>
        <td name="td3">4</td>
        <td name="td4">M11 vl-sarah</td>
      </tr>
      <tr>
        <td name="td1">4HJJ-H</td>
        <td name="td2">22</td>
        <td name="td3">1.4</td>
        <td name="td4">Interleukin-18</td>
      </tr>
      <tr>
        <td name="td1">7JZ1-H</td>
        <td name="td2">21.1</td>
        <td name="td3">5.3</td>
        <td name="td4">Mgc34 heavy chain</td>
      </tr>
      <tr>
        <td name="td1">6XUX-A</td>
        <td name="td2">20.3</td>
        <td name="td3">1.3</td>
        <td name="td4">Nanobody,glucosidase ygjk,glucosidase ygjk,nanobo</td>
      </tr>
      <tr>
        <td name="td1">7CEA-A</td>
        <td name="td2">20.2</td>
        <td name="td3">4</td>
        <td name="td4">Huts-4 vh(s112c)-sarah</td>
      </tr>
      <tr>
        <td name="td1">7RGA-E</td>
        <td name="td2">19.5</td>
        <td name="td3">1.4</td>
        <td name="td4">Nano clostridial antibody mimetic protein 3 vhh</td>
      </tr>
      <tr>
        <td name="td1">4A6Y-A</td>
        <td name="td2">19.3</td>
        <td name="td3">3</td>
        <td name="td4">Antibody bbe6.12h3 light chain</td>
      </tr>
      <tr>
        <td name="td1">7KHF-A</td>
        <td name="td2">18.8</td>
        <td name="td3">14.9</td>
        <td name="td4">Mdb1 fab heavy chain</td>
      </tr>
	  </tbody>
  </table>



                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] RNA mimics of green fluorescent protein.</strong></font><br />
Paige, J. S., Wu, K. Y., & Jaffrey, S. R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/21798953/" target="_blank" style="color:#520049" >Science (New York, N.Y.), 333(6042), 642–646. (2011)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] RNA-based fluorescent biosensors for live cell imaging of second messengers cyclic di-GMP and cyclic AMP-GMP.</strong></font><br />
Kellenberger, C. A., Wilson, S. C., Sales-Lee, J., & Hammond, M. C.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23488798/" target="_blank" style="color:#520049">Journal of the American Chemical Society, 135(13), 4906–4909. (2013)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] A G-quadruplex-containing RNA activates fluorescence in a GFP-like fluorophore.</strong></font><br />
Huang, H., Suslov, N. B., Li, N. S., Shelke, S. A., Evans, M. E., Koldobskaya, Y., Rice, P. A., & Piccirilli, J. A. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/24952597/" target="_blank" style="color:#520049">Nature chemical biology, 10(8), 686–691. (2014)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] Structural basis for activity of highly efficient RNA mimics of green fluorescent protein.</strong></font><br />
Warner, K. D., Chen, M. C., Song, W., Strack, R. L., Thorn, A., Jaffrey, S. R., & Ferré-D'Amaré, A. R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/25964329/" target="_blank" style="color:#520049">Nature structural & molecular biology, 21(8), 658–663. (2014)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Imaging metabolite dynamics in living cells using a Spinach-based riboswitch.</strong></font><br />
You, M., Litke, J. L., & Jaffrey, S. R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/25964329/" target="_blank" style="color:#520049">Proceedings of the National Academy of Sciences of the United States of America, 112(21), E2756–E2765. (2015)</a>
<br />

<a id="ref6"></a><font><strong>[6] Affinity maturation of a portable Fab-RNA module for chaperone-assisted RNA crystallography.</strong></font><br />
Koirala, D., Shelke, S. A., Dupont, M., Ruiz, S., DasGupta, S., Bailey, L. J., Benner, S. A., & Piccirilli, J. A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/29309709/" target="_blank" style="color:#520049">Nucleic acids research, 46(5), 2624–2635. (2018)</a>
<br />       


<script>
    function toggleSequence(event) {
      const container = event.target.closest('.sequence-container');
      container.classList.toggle('expanded');
      const showMoreText = container.querySelector('.show-more');
      
      // 展开后按钮文本变化
      if (container.classList.contains('expanded')) {
        showMoreText.textContent = '...';  // 展开后显示 "..."
      } else {
        showMoreText.textContent = '......';  // 收起后显示 "......"
      }
    }

    // 页面加载时，限制序列文本为50个字符
    window.addEventListener('load', function() {
      const sequenceContainers = document.querySelectorAll('.sequence-container');
      sequenceContainers.forEach(container => {
        const fullSeqText = container.querySelector('.full-sequence').textContent;
        const truncatedText = fullSeqText.slice(0, 20);  // 只显示前50个字符
        container.querySelector('.sequence-text').textContent = truncatedText;
      });
    });
  </script>