---
layout: theophylline_style
title: Subtilisin aptamer
date: 2024-05-08 00:00:00
author: Qiaozhen Liu, Zhijie Tan
categories: Aptamer
tags:
- Category:Bacteria
- GC:51.0
- Length:51
- Named:Subtilisin_RNA-1
- Type:Proteins
- Year:1999
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="description">Description</p>
<p>In 1999, Hiroshi Takeno and colleagues used in vitro selection to isolate RNA aptamers that bind to protease subtilisin. The RNA aptamer displayed specific inhibition toward the subtilisin activity and almost no inhibitory activity toward trypsin and chymotrypsin, although these enzymes are serine proteases similar to subtilisin<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 1999, Hiroshi Takeno et al. isolated sequences that bind to subtilisin by eight rounds of selection<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>The RNA aptamers were directly bound to subtilin. The 2D structure of the figure is based on the article by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>
<p>5'-GCCACUCGCUCAACACGGUAAGUAGAGACCUAGUGGUACAUAAAGGA-3'</p>
<img src="/images/2D/Subtilisin_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>Subtilisin is a serine endopeptidase in peptidase family S8, subfamily S8A. Characterized peptidases in this entry include subtilisin Carlsberg (S08.001), TK-subtilisin (S08.129), subtilisin PB92 (S08.003) and subtilisin BPN' (S08.034). TK-subtilisin is secreted from the hyperthermophilic archaeon Thermococcus kodakaraensis as an inactive precursor. TK-subtilisin is matured from pro-TK-subtilisin upon autoprocessing and degradation of the propeptide. Unlike other subtilisins, the folding of the unprocessed form of pro-TK-subtilisin is induced by Ca2+binding which is almost completed prior to autoprocessing. Ca2+is required for activity, unlike the bacterial subtilisins. The propeptide is not required for folding of the mature domain, unlike the bacterial subtilases, because of the stability produced from Ca2+binding. Subtilisin BPN'/Novo is extremely similar in structure to subtilisin Carlsberg though it has a 30% difference in amino acid sequence. The substrate binding regions are also similar and two possible Ca2+binding sites have been identified.-----from Pfam</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Subtilisin</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P00782/entry" target="_blank" style="color:#520049"><b>P00782</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR034202/" target="_blank" style="color:#520049"><b>IPR034202</b></a></td>
        <td name="td3">26.37 kDa</td>
        <td name="td4">GPYGVSQIKAPALHSQGYTGSNVKVAVIDSGIDSSHPALKVAGGASFVPSETNPFQDNNSHGTHVAGTVLAVAPSASLYAVKVLGADGSGQYSWIINGIEWAIANNMDVINMSLGGPSGSAALKAAVDKAVASGVVVVAAAGNEGTSGSSSTVGYPGKYPSVIAVGAVDSSNQRASFSSVGPELDVMAPGVSIWSTLPGNKYGAKSGTCMASPHVAGAAALILSKHPNWTNTQVRSSLENTTTKLGDSFYYGKGLINVEAAAQ</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/1GNS" target="_blank" style="color:#520049"><b>1GNS</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/9528228" target="_blank" style="color:#520049"><b>9528228</b></a></td>
      </tr>
	  </tbody>
  </table>

<p>To analyze subtilisin inhibition by RNA-1, subtilisin reaction using the chromogenic substrate was measured in the presence or absence of RNA-1.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">RNA-1</td>
        <td name="td1">5'-GCCACUCGCUCAACACGGUAAGUAGAGACCUAGUGGUACAUAAAGGA-3'</td>
        <td name="td2">Subtilisin</td>
        <td name="td3">2.5 μM</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/Subtilisin_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">8JMW-A</td>
      <td name="td1">41.0</td>
      <td name="td2">1.5</td>
      <td name="td3">S8 family serine peptidase</td>
    </tr>
            
     <tr>
      <td name="td0">3AFG-B</td>
      <td name="td1">38.8</td>
      <td name="td2">1.6</td>
      <td name="td3">Subtilisin-like serine peptidase</td>
    </tr>
            
     <tr>
      <td name="td0">1V6C-B</td>
      <td name="td1">38.5</td>
      <td name="td2">1.6</td>
      <td name="td3">Alkaline serine peptidase</td>
    </tr>
            
     <tr>
      <td name="td0">4KG7-A</td>
      <td name="td1">36.0</td>
      <td name="td2">1.8</td>
      <td name="td3">Peptidase S8 and S53, subtilisin, kexin, sedolisi</td>
    </tr>
            
     <tr>
      <td name="td0">7EDD-A</td>
      <td name="td1">35.6</td>
      <td name="td2">1.8</td>
      <td name="td3">C5a peptidase</td>
    </tr>
            
     <tr>
      <td name="td0">1R6V-A</td>
      <td name="td1">35.0</td>
      <td name="td2">2.2</td>
      <td name="td3">Subtilisin-like serine peptidase</td>
    </tr>
            
     <tr>
      <td name="td0">1R64-A</td>
      <td name="td1">34.4</td>
      <td name="td2">2.0</td>
      <td name="td3">Kexin</td>
    </tr>
            
     <tr>
      <td name="td0">3QFH-D</td>
      <td name="td1">33.4</td>
      <td name="td2">1.9</td>
      <td name="td3">Epidermin leader peptide processing serine protea</td>
    </tr>
            
     <tr>
      <td name="td0">8POl-B</td>
      <td name="td1">33.3</td>
      <td name="td2">1.9</td>
      <td name="td3">Subtilisin-like peptidase 1</td>
    </tr>
            
     <tr>
      <td name="td0">3I6S-A</td>
      <td name="td1">33.2</td>
      <td name="td2">1.8</td>
      <td name="td3">Subtilisin-like peptidase</td>
    </tr>
    </tbody>
  </table>
                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Selection of an RNA molecule that specifically inhibits the protease activity of subtilisin.</strong></font><br />
Takeno, H., Yamamoto, S., Tanaka, T., Sakano, Y., & Kikuchi, Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10348914/" target="_blank" style="color:#520049">Journal of biochemistry, 125(6), 1115–1119. (1999)</a>
<br/>



<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1RAW-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>

    
    </html>
