---
layout: theophylline_style
title: TetR aptamer
date: 2024-04-10 00:00:00
author: Qiaozhen Liu,Zhaoji Qiu
categories: Aptamer
tags:
- Category:Bacteria
- GC:57.99999999999999
- Length:43
- Named:Transcription repressor TetR_TetR-RNA aptamer
- Type:Proteins
- Year:2009
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/19246008/" target="_blank" style="color:#520049">2009</a></h3>
    </div>
    <div class="body">
      <p>The TetR-binding RNA aptamer was identified by a combination of in vitro selection for TetR binding and an in vivo screening for aptamer activity via a transcription reporter assay<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/22021209/" target="_blank" style="color:#520049">2011</a></h3>
    </div>
    <div class="body">
      <p>The mechanistic basis for RNA aptamer-based induction of TetR was analysed<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/29036355/" target="_blank" style="color:#520049">2017</a></h3>
    </div>
    <div class="body">
      <p>The TetR aptamer was applied to control miRNA biogenesis in human cells<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/31504742/" target="_blank" style="color:#520049">2019</a></h3>
    </div>
    <div class="body">
      <p>TetR-binding aptamer for the control of translation and pre-mRNA splicing<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/32052019/" target="_blank" style="color:#520049">2020</a></h3>
    </div>
    <div class="body">
      <p>The crystal structure of TetR-RNA aptamer complex was determined<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
</div>


         
<p class="header_box" id="description">Description</p>
<p>In 2009, Anke Hunsicker and colleagues identified an RNA aptamer by means of in vitro screening for TetR binding and in vivo screening for TetR inducible binding, which induces tet-controlled gene expression in E. coli. In 2020, Florian C. Grau and co-workers presented the crystal structure of the TetR-RNA aptamer complex at a resolution of 2.7 Å<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref5" style="color:#520049">5</a></sup><sup>]</sup>.<br></p>



<p class="header_box" id="SELEX">SELEX</p>
<p>In 2009, Anke Hunsicker et al. isolated sequences that bind TetR through 12 rounds of selection. They used a two-step approach to identify RNA aptamers that induce the transcriptional regulator TetR in vivo. First, an automated in vitro selection protocol was employed to isolate RNA aptamers that bind TetR. After 12 rounds of selection, a significant enrichment of TetR-binding RNA was detected and its specificity of binding was verified by a filter-retention assay<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p>
<p>
<br>
<br>  



<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>The TetR-binding aptamer is capable of inducing TetR-controlled gene expression. TetR-binding aptamers were isolated via in vitro selection along with an additional screening step, which was crucial for identifying aptamers that are active within the cell. Mutational analyses defined the minimal active sequence and highlighted the aptamer bases involved in induction. Here we utilized Ribodraw to complete the figure based on the 3D structure information. TetR-RNA aptamer was named by Anke Hunsicker et al. in the article<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref5" style="color:#520049">5</a></sup><sup>]</sup>.</p>
<p>5'-GGCCGGAGAAUGUUAUGGCGCGAAAGCGCAGAGAAAACCGGUC-3'</p>
<img src="/images/2D/TetR_aptamer_2D.svg" alt="drawing" style="width:800px;height:350px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">3D visualisation</p>             
<p>Florian C. Grau1 et al. sovled the crystal structure, at 2.7 Å resolution, of an RNA aptamer bound to the transcription repressor TetR has been determined. The fold of the RNA aptamer bears no resemblance to regular B-DNA, and neither does the thermodynamic characterization of the complex formation reaction. Nevertheless, the functional aptamer-binding epitope of TetR is fully contained within its DNA-binding epitope. The PDB ID of this structure is 6SY4<sup>[<a href="#ref5" style="color:#520049">5</a></sup><sup>]</sup>.</p>
<p>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/TetR_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A', 
  start_residue_number:3, 
  end_residue_number:202, 
  color:{r:133,g:39,b:181}
},
{
  struct_asym_id: 'B', 
  start_residue_number:3, 
  end_residue_number:202, 
  color:{r:133,g:39,b:181}
},
{
  struct_asym_id: 'C', 
  start_residue_number:4, 
  end_residue_number:7, 
  color:{r:181,g:51,b:50}
},
{
  struct_asym_id: 'C', 
  start_residue_number:8, 
  end_residue_number:8, 
  color:{r:243,g:125,b:7}
},
{
  struct_asym_id: 'C', 
  start_residue_number:9, 
  end_residue_number:11, 
  color:{r:181,g:51,b:50}
},
{
  struct_asym_id: 'C', 
  start_residue_number:12, 
  end_residue_number:12, 
  color:{r:243,g:125,b:7}
},
{
  struct_asym_id: 'C', 
  start_residue_number:13, 
  end_residue_number:14, 
  color:{r:46,g:161,b:3}
},
{
  struct_asym_id: 'C', 
  start_residue_number:15, 
  end_residue_number:17, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'C', 
  start_residue_number:18, 
  end_residue_number:29, 
  color:{r:8,g:69,b:149}
},
{
  struct_asym_id: 'C', 
  start_residue_number:30, 
  end_residue_number:31, 
  color:{r:241,g:125,b:7}
},
{
  struct_asym_id: 'C', 
  start_residue_number:32, 
  end_residue_number:32, 
  color:{r:46,g:161,b:3}
},{
  struct_asym_id: 'C', 
  start_residue_number:33, 
  end_residue_number:33, 
  color:{r:241,g:125,b:7}
},{
  struct_asym_id: 'C', 
  start_residue_number:34, 
  end_residue_number:41, 
  color:{r:184,g:51,b:50}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/6sy4-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<p>Left: A surface representation of the binding pocket of the aptamer originating from PDB ID: 6SY4 at a resolution of 2.7 Å is presented. In the case of TetR (displayed with vacuum electrostatics), blue is indicative of a positive charge, whereas red represents a negative charge. Right: The hydrogen bonds at the binding sites where the aptamer binds to TetR are shown.</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/TetR_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/TetR_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>



<p class="header_box" id="ligand-recognition">Ligand information</p>
<p class="blowheader_box">SELEX ligand</p>
<p>Florian C. Grau et al. investigated the affinity of RNA aptamer K1 for TetR using EMSA (electrophoretic mobility shift assay) and ITC (isothermal titration calorimetry) experiments. EMSA gave a K₍D₎ of 80 nM; ITC recorded a K₍D₎ of 5.6 nM and confirmed a 1-to-1 stoichiometry. ITC also showed the complex formation is enthalpy-driven (ΔH = –156.0 kJ mol⁻¹) with large entropy reduction (TΔS = –108.8 kJ mol⁻¹)<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup>.</p>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">TetR binding aptamer</td>
        <td name="td1">5'-GGCCGGAGAAUGUUAUGGCGCGAAAGCGCAGAGAAAACCGGUC-3'</td>
        <td name="td2">TetR</td>
        <td name="td3">80 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<p>TetR family regulators have been implicated in the transcriptional regulation of multidrug efflux pumps, pathways for the biosynthesis of antibiotics, responses to osmotic stress and toxic chemicals, control of catabolic pathways, differentiation processes, and pathogenicity. The TetR proteins identified within over 115 genera of bacteria and archaea possess a common helix-turn-helix (HTH) structure within their DNA-binding domain.Nevertheless, TetR proteins are capable of functioning in diverse ways. They can directly bind to a target operator to exert their effect. For instance, TetR binds to the Tet(A) gene to repress it in the absence of tetracycline. Alternatively, they can be involved in complex regulatory cascades where the TetR protein may either be modulated by another regulator or can trigger the cellular response.This entry relates to the tetracycline transcriptional repressor TetR, which binds to the Tet(A) gene to suppress its expression when tetracycline is absent.-----From <a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR003012/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a></p>

<p class="dot-paragraph"><b>UniProt ID:</b> uniquely identifies protein sequences in the <a href="https://www.uniprot.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>UniProt database</b></a>, a resource for protein information.</p>
<p class="dot-paragraph"><b><a href="https://www.ebi.ac.uk/interpro/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Pfam</b></a>:</b> a widely recognised database of protein families and domains.</p>
<p class="dot-paragraph"><b><a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank" style="color:#520049; text-decoration: underline;"><b>GenBank</b></a>:</b> maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank" style="color:#520049; text-decoration: underline;"><b>NCBI</b></a>(National Center for Biotechnology Information), is a database of nucleotide sequences from various organisms, vital for genetic and molecular biology research.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Mass" target="_blank" style="color:#520049; text-decoration: underline;"><b>Mass</b></a>:</b> an intrinsic property of a body.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">Mass</th>
        <th onclick="sortTable(3)">Protein sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P04483/entry" target="_blank" style="color:#520049"><b>P04483</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR003012/" target="_blank" style="color:#520049"><b>IPR003012</b></a></td>
        <td name="td2">46.51 kDa</td>
        <td name="td3">
        <div class="sequence-container">

          <span class="full-sequence">SRLNRESVIDAALELLNETGIDGLTTRKLAQKLGIEQPTLYWHVKNKRALLDALAVEILARHHDYSLPAAGESWQSFLRNNAMSFRRALLRYRDGAKVHLGTRPDEKQYDTVETQLRFMTENGFSLRDGLYAISAVSHFTLGAVLEQQEHTAALTDRPAAPDENLPPLLREALQIMDSDDGEQAFLHGLESLIRGFEVQLTALLQIVSRLNRESVIDAALELLNETGIDGLTTRKLAQKLGIEQPTLYWHVKNKRALLDALAVEILARHHDYSLPAAGESWQSFLRNNAMSFRRALLRYRDGAKVHLGTRPDEKQYDTVETQLRFMTENGFSLRDGLYAISAVSHFTLGAVLEQQEHTAALTDRPAAPDENLPPLLREALQIMDSDDGEQAFLHGLESLIRGFEVQLTALLQIV</span>
        </div>
      </td>
        <td name="td4"><a href="https://www.rcsb.org/structure/4V2F" target="_blank" style="color:#520049"><b>4V2F</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/gene/912848" target="_blank" style="color:#520049"><b>912848</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/TetR_stru_ligand1.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<p>We used the <a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a> to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation.</p>

<p class="dot-paragraph"><a href="http://ekhidna2.biocenter.helsinki.fi/dali/#:~:text=The%20Dali%20server%20is%20a%20network%20service%20for%20comparing%20protein" target="_blank" style="color:#520049; text-decoration: underline;"><b>Dail server website</b></a>: a network service for comparing protein structures in 3D. Dali compares them against those in the <a href="https://www.rcsb.org/" target="_blank" style="color:#520049; text-decoration: underline;"><b>Protein Data Bank</b></a> (PDB).</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Standard_score" target="_blank" style="color:#520049; text-decoration: underline;"><b>Z-score</b></a>:</b> a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious.</p>
<p class="dot-paragraph"><b><a href="https://en.wikipedia.org/wiki/Root_mean_square_deviation" target="_blank" style="color:#520049; text-decoration: underline;"><b>RMSD</b></a>:</b> (Root Mean Square Deviation) is used to measure the degree to which atoms deviate from the alignment position.</p>
<p class="dot-paragraph"><b>PDB:</b> PDB ID+ chain name.</p>

<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">6YS4-A</td>
        <td name="td2">28.6</td>
        <td name="td3">0</td>
        <td name="td4">Original chain</td>
      </tr>
      <tr>
        <td name="td1">3ZQG-A</td>
        <td name="td2">25</td>
        <td name="td3">1.1</td>
        <td name="td4">Tracycline repressor protein class b from trans</td>
      </tr>
      <tr>
        <td name="td1">3BQY-A</td>
        <td name="td2">19.6</td>
        <td name="td3">2</td>
        <td name="td4">Putative tetr family transcriptional regulator</td>
      </tr>
      <tr>
        <td name="td1">1Z0X-A</td>
        <td name="td2">18.1</td>
        <td name="td3">2.3</td>
        <td name="td4">Transcriptional regulator,tetr family</td>
      </tr>
      <tr>
        <td name="td1">2HXO-A</td>
        <td name="td2">15</td>
        <td name="td3">3.1</td>
        <td name="td4">Putative tetr-family transcriptional regulator</td>
      </tr>
      <tr>
        <td name="td1">2Y31-A</td>
        <td name="td2">15</td>
        <td name="td3">3.1</td>
        <td name="td4">Putative repressor simreg2</td>
      </tr>
      <tr>
        <td name="td1">2HXI-B</td>
        <td name="td2">14.9</td>
        <td name="td3">3.3</td>
        <td name="td4">Putative transcriptional regulator</td>
      </tr>
      <tr>
        <td name="td1">1ZK8-A</td>
        <td name="td2">14.6</td>
        <td name="td3">2.8</td>
        <td name="td4">Transcriptional regulator, tetr family</td>
      </tr>
      <tr>
        <td name="td1">2NP5-A</td>
        <td name="td2">13.1</td>
        <td name="td3">2.9</td>
        <td name="td4">Transcriptional regulator</td>
      </tr>
      <tr>
        <td name="td1">2ZCX-A</td>
        <td name="td2">12.9</td>
        <td name="td3">3.5</td>
        <td name="td4">Tetr-family transcriptional regulator</td>
      </tr>
	  </tbody>
  </table>
  <br>
  <br>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] An RNA aptamer that induces transcription.</strong></font><br />
Hunsicker, A., Steber, M., Mayer, G., Meitert, J., Klotzsche, M., Blind, M., Hillen, W., Berens, C., & Suess, B.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/19246008/" target="_blank" style="color:#520049" >Chemistry & biology, 16(2), 173–180. (2009)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Mechanistic basis for RNA aptamer-based induction of TetR.</strong></font><br />
Steber, M., Arora, A., Hofmann, J., Brutschy, B., & Suess, B.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/22021209/" target="_blank" style="color:#520049">Chembiochem : a European journal of chemical biology, 12(17), 2608–2614. (2011)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] Design and implementation of a synthetic pre-miR switch for controlling miRNA biogenesis in mammals.</strong></font><br />
Atanasov, J., Groher, F., Weigand, J. E., & Suess, B.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/29036355/" target="_blank" style="color:#520049"> Nucleic acids research, 45(22), e181. (2017)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] Robust gene expression control in human cells with a novel universal TetR aptamer splicing module.</strong></font><br />
Mol, A. A., Groher, F., Schreiber, B., Rühmkorff, C., & Suess, B.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/31504742/" target="_blank" style="color:#520049">Nucleic acids research, 47(20), e132. (2019)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] The complex formed between a synthetic RNA aptamer and the transcription repressor TetR is a structural and functional twin of the operator DNA-TetR regulator complex.</strong></font><br />
Grau, F. C., Jaeger, J., Groher, F., Suess, B., & Muller, Y. A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/32052019/" target="_blank" style="color:#520049">Nucleic acids research, 48(6), 3366–3378. (2020)</a>
<br />

<a id="ref6"></a><font><strong>[6] Inducible nuclear import by TetR aptamer-controlled 3' splice site selection.</strong></font><br />
Mol, A. A., Vogel, M., & Suess, B.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/33148600/" target="_blank" style="color:#520049">RNA (New York, N.Y.), 27(2), 234–241. (2021)</a>
<br />       


<script>
    function toggleSequence(event) {
      const container = event.target.closest('.sequence-container');
      container.classList.toggle('expanded');
      const showMoreText = container.querySelector('.show-more');
      
      // 展开后按钮文本变化
      if (container.classList.contains('expanded')) {
        showMoreText.textContent = '...';  // 展开后显示 "..."
      } else {
        showMoreText.textContent = '......';  // 收起后显示 "......"
      }
    }

    // 页面加载时，限制序列文本为50个字符
    window.addEventListener('load', function() {
      const sequenceContainers = document.querySelectorAll('.sequence-container');
      sequenceContainers.forEach(container => {
        const fullSeqText = container.querySelector('.full-sequence').textContent;
        const truncatedText = fullSeqText.slice(0, 20);  // 只显示前50个字符
        container.querySelector('.sequence-text').textContent = truncatedText;
      });
    });
  </script>