---
layout: theophylline_style
title: Trypanosome VSG aptamer
date: 2024-05-15 00:00:00
author: Zhizhong Lu, Ying Ao
categories: Aptamer
tags:
- Category:Parasite
- GC:63.74999999999999
- Length:80
- Named:Trypanosome variant surface glycoprotein (VSG)_2-16
- Type:Proteins
- Year:1999
type: Proteins
---
<html>

<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10198434/" target="_blank" style="color:#520049">1999</a></h3>
  </div>
  <div class="body">
    <p>Using a combinatorial selection method in conjunction with live trypanosomes as the binding target,  the RNA ligands (aptamers) for constant surface components can be isolated<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/11557345/" target="_blank" style="color:#520049">2001</a></h3>
  </div>
  <div class="body">
    <p>Analysed the fate of the aptamers upon binding to the flagellar pocket<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12582125/" target="_blank" style="color:#520049">2003</a></h3>
  </div>
  <div class="body">
    <p>Aptamers tethered to an antigenic side group are capable of directing antibodies to the surface of the parasite in vitro<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/16925510/" target="_blank" style="color:#520049">2006</a></h3>
  </div>
  <div class="body">
    <p>Reported the selection of a 2'-NH(2)-modified RNA aptamer that binds to live trypanosomes with an affinity of 70 +/- 15 nM<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/18220540/" target="_blank" style="color:#520049">2008</a></h3>
  </div>
  <div class="body">
    <p>Reported the optimization of the 2-16 RNA for its applications in vivo<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/23017685/" target="_blank" style="color:#520049">2013</a></h3>
  </div>
  <div class="body">
    <p>Demonstrate the capabilities of the hybrid functional material carbon nanotubes/aptamer for the creation of a new generation of nuclease-resistant aptasensors used African Trypanosomes aptamer<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
  </div>
 </div>
</div>



<p class="header_box" id="description">Description</p>
<p>In 1999, Homann, M., & Göringer, H. U. described the selection of three classes of RNA aptamers that crosslink to a single 42 kDa protein located within the flagellar pocket of the parasite. The RNAs associate rapidly and with high affinity. In 2003, Göringer, H. U. had employed a systematic method known as SELEX to identify compact RNA molecules, termed aptamers, which exhibit a strong binding affinity for VSGs, reaching subnanomolar levels. These RNA molecules possess the ability to distinguish between various VSG types and can adhere to the exterior of active trypanosome cells. Furthermore, when these aptamers are attached to a specific antigenic component, they can facilitate the targeting of antibodies to the parasite's surface in laboratory settings<sup>[<a href="#ref1" style="color:#520049">1</a>,<a href="#ref3" style="color:#520049">3</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In the process of selection, they initiated with a vast pool comprising 10^16 RNA molecules, which was calculated to encompass around 2 × 10^15 distinct sequences. This collection featured a segment of 40 nucleotides with variability, surrounded by primer binding sites that were 21 and 24 nucleotides in length respectively. These primer binding sites, in conjunction, helped to confine the molecular weight of the chosen aptamers to a range of approximately 25 to 27 kilodaltons. In both experiments, 13 rounds of selective binding and subsequent amplification were performed. Maximal enrichment of trypanosome-interacting RNAs was achieved in both selections in round 12 at which time ∼18% of the input RNAs was associated with the parasite cells. DNA templates from RNA pools of round 12 were cloned and the sequences of 53 clones from each selection experiment were determined<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<font>The 2D structure of the figures is based on the article by ribodraw tool to draw.</font>
<p>5'-CGUCCAUCGGGUGGCCCGUGUCUGAGCGGGGACGGGGACUUGAGCGCCGCUGUCCGACUGAAUU-3'</p>
<img src="/images/2D/2-16_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>



<p class="header_box" id="ligand-recognition">Ligand information</p>
<p class="blowheader_box">SELEX ligand</p>
<p>Variant surface glycoprotein (VSG) is a ~60kDa protein which densely packs the cell surface of protozoan parasites belonging to the genus Trypanosoma. This genus is notable for their cell surface proteins. They were first isolated from Trypanosoma brucei in 1975 by George Cross. VSG allows the trypanosomatid parasites to evade the mammalian host's immune system by extensive antigenic variation. They form a 12–15 nm surface coat. VSG dimers make up ~90% of all cell surface protein and ~10% of total cell protein.-----From WiKi</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Trypanosome variant surface glycoprotein (VSG)</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P26332/" target="_blank" style="color:#520049"><b>P26332</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00913/" target="_blank" style="color:#520049"><b>PF00913</b></a></td>
        <td name="td3">51.04 KDa</td>
        <td name="td4">MPSNQEARLFLAVLVLAQVLPILVDSAAEKGFKQAFWQPLCQVSEELDDQPKGALFTLQAAASKIQKMRDAALRASIYAEINHGTNRAKAAVIVANHYAMKADSGLEALKQTLSSQEVTATATASYLKGRIDEYLNLLLQTKESGTSGCMMDTSGTNTVTKAGGTIGGVPCKLQLSPIQPKRPAATYLGKAGYVGLTRQADAANNFHDNDAECRLASGHNTNGLGKSGQLSAAVTMAAGYVTVANSQTAVTVQALDALQEASGAAHQPWIDAWKAKKALTGAETAEFRNETAGIAGKTGVTKLVEEALLKKKDSEASEIQTELKKYFSGHENEQWTAIEKLISEQPVAQNLVGDNQPTKLGELEGNAKLTTILAYYRMETAGKFEVLTQKHKPAESQQQAAETEGSCNKKDQNECKSPCKWHNDAENKKCTLDKEEAKKVADETAKDGKTGNTNTTGSSNSFVISKTPLWLAVLLF</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/1VSG" target="_blank" style="color:#520049"><b>1VSG</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?lvl=0&id=5702" target="_blank" style="color:#520049"><b>5702</b></a></td>
      </tr>
	  </tbody>
  </table>

  <p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">2-16 aptamer</td>
        <td name="td1">5'-CGUCCAUCGGGUGGCCCGUGUCUGAGCGGGGACGGGGACUUGAGCGCCGCUGUCCGACUGAAUU-3'</td>
        <td name="td2">Trypanosoma brucei</td>
        <td name="td3">60 ± 17 nM</td>
      </tr>
	  </tbody>
  </table>
<img src="/images/SELEX_ligand/2-16_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">5LY9-A</td>
      <td name="td1">33.9</td>
      <td name="td2">2.2</td>
      <td name="td3">Variant surface glycoprotein mitat 1.1</td>
    </tr>
            
     <tr>
      <td name="td0">2VSG-A</td>
      <td name="td1">21.9</td>
      <td name="td2">3.2</td>
      <td name="td3">Variant surface glycoprotein iltat 1.24</td>
    </tr>
            
     <tr>
      <td name="td0">6SOY-A</td>
      <td name="td1">16.0</td>
      <td name="td2">3.2</td>
      <td name="td3">Esag6, subunit of heterodimeric transferrin recep</td>
    </tr>
            
     <tr>
      <td name="td0">8B3E-A</td>
      <td name="td1">12.3</td>
      <td name="td2">4.0</td>
      <td name="td3">Variant surface glycoprotein 397</td>
    </tr>
            
     <tr>
      <td name="td0">6Z7B-A</td>
      <td name="td1">12.1</td>
      <td name="td2">4.1</td>
      <td name="td3">Variant surface glycoprotein sur</td>
    </tr>
            
     <tr>
      <td name="td0">8OK7-A</td>
      <td name="td1">12.1</td>
      <td name="td2">3.8</td>
      <td name="td3">Variant surface glycoprotein 558</td>
    </tr>
            
     <tr>
      <td name="td0">6Z8H-A</td>
      <td name="td1">11.5</td>
      <td name="td2">4.6</td>
      <td name="td3">Variant surface glycoprotein mitat 1.13</td>
    </tr>
            
     <tr>
      <td name="td0">8OVB-C</td>
      <td name="td1">8.1</td>
      <td name="td2">3.7</td>
      <td name="td3">Complement c3f fragment</td>
    </tr>
            
     <tr>
      <td name="td0">2Y44-A</td>
      <td name="td1">8.1</td>
      <td name="td2">2.7</td>
      <td name="td3">Glutamic acid/alanine-rich protein</td>
    </tr>
            
     <tr>
      <td name="td0">6XZ6-A</td>
      <td name="td1">7.8</td>
      <td name="td2">3.6</td>
      <td name="td3">Garp domain-containing protein</td>
    </tr>
    </tbody>
  </table>





                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Combinatorial selection of high affinity RNA ligands to live African trypanosomes.</strong></font><br />
Homann, M., & Göringer, H. U.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10198434/" target="_blank" style="color:#520049">Nucleic acids research, 27(9), 2006–2014. (1999)</a>
<br/>
            
<a id="ref2"></a><font><strong>[2] Uptake and intracellular transport of RNA aptamers in African trypanosomes suggest therapeutic "piggy-back" approach.</strong></font><br />
Homann, M., & Göringer, H. U.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/11557345/" target="_blank" style="color:#520049">Bioorganic & medicinal chemistry, 9(10), 2571–2580. (2001)</a>
<br/>
            
<a id="ref3"></a><font><strong>[3] Targeting the variable surface of African trypanosomes with variant surface glycoprotein-specific, serum-stable RNA aptamers.</strong></font><br />
Lorger, M., Engstler, M., Homann, M., & Göringer, H. U.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12582125/" target="_blank" style="color:#520049">Eukaryotic cell, 2(1), 84–94. (2003)</a>
<br/>
            
<a id="ref4"></a><font><strong>[4] Serum-stable RNA aptamers to an invariant surface domain of live African trypanosomes.</strong></font><br />
Homann, M., Lorger, M., Engstler, M., Zacharias, M., & Göringer, H. U.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/16925510/" target="_blank" style="color:#520049">Combinatorial chemistry & high throughput screening, 9(7), 491–499. (2006)</a>
<br/>
            
<a id="ref5"></a><font><strong>[5] RNA aptamers as potential pharmaceuticals against infections with African trypanosomes.</strong></font><br />
Göringer, H. U., Homann, M., Zacharias, M., & Adler, A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/16594626/" target="_blank" style="color:#520049">Handbook of experimental pharmacology, (173), 375–393. (2006)</a>
<br/>
            
<a id="ref6"></a><font><strong>[6] Post-SELEX chemical optimization of a trypanosome-specific RNA aptamer.</strong></font><br />
Adler, A., Forster, N., Homann, M., & Göringer, H. U.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18220540/" target="_blank" style="color:#520049">Combinatorial chemistry & high throughput screening, 11(1), 16–23. (2008)</a>
<br/>
            
<a id="ref7"></a><font><strong>[7] Ultrasensitive and real-time detection of proteins in blood using a potentiometric carbon-nanotube aptasensor.</strong></font><br />
Zelada-Guillén, G. A., Tweed-Kent, A., Niemann, M., Göringer, H. U., Riu, J., & Rius, F. X.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23017685/" target="_blank" style="color:#520049">Biosensors & bioelectronics, 41, 366–371. (2013)</a>
<br/>



<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar.css">
      <script src="https://cdn.jsdelivr.net/npm/pdbe-molstar@3.3.0/build/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>




    </html>
