---
layout: theophylline_style
title: ZAP aptamer
date: 2024-05-29 00:00:00
author: Zhijie Tan, Qiaozhen Liu
categories: Aptamer
tags:
- Category:Mammalian
- GC:63.89
- GC:64.0
- Length:72
- Length:75
- Named:Zinc-finger antiviral protein (ZAP)_21-E
- Named:Zinc-finger antiviral protein (ZAP)_21-K
- Type:Proteins
- Year:2010
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="description">Description</p>
<p>In 2010, Huang et al. used the SELEX method to isolate ZAP-binding RNAaptamers. After 21 rounds of selection, ZAP-binding aptamers were isolated. Sequence analysis revealed that they are G-rich RNAs with predicted stem-loop structures containing conserved “GGGUGG” and “GAGGG” motifs in the loop region<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2010, Huang et al. used the SELEX method to isolate the aptamer with 40 nucleotides of random sequence, after 21 rounds of selection. Sequence analysis revealed that they are G-rich RNAs with predicted stem-loop structures containing conserved “GGGUGG” and “GAGGG” motifs in the loop region<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>21E was the aptamer sequence mainly studied in the article, which had a high affinity with ZAP. The 2D structure of the figure is based on the article by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>
<p>5'-GGGAACAGUCCGAGCCUGCGUGCGAGGAGGUGGAGAGGGUGGGUGUGCGUGGCGUGGGGUGAAUUCGUCAUA-3'</p>
<img src="/images/2D/ZAP_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>



<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>Zinc-finger antiviral protein (ZAP) is a host factor that specifically inhibits the replication of certain viruses, such as HIV-1, by targeting viral mRNA for degradation. N-terminal domain of ZAP is the major functional domain which contains four zinc-finger motifs. This entry represents the third zinc finger type CCCH.-----from Pfam</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Zinc-finger antiviral protein (ZAP)</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/Q8K3Y6/entry" target="_blank" style="color:#520049"><b>Q8K3Y6</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR040954/" target="_blank" style="color:#520049"><b>IPR040954</b></a></td>
        <td name="td3">26.21 kDa</td>
        <td name="td4">GPLGMADPGVCCFITKILCAHGGRMTLEELLGEIRLPEAQLYELLETAGPDRFVLLETGGQAGITRSVVATTRARVCRRKYCQRPCDSLHLCKLNLLGRCHYAQSQRNLCKYSHDVLSEQNFQILKNHELSGLNQEELACLLVQSDPFFLPEICKSYKGEGRKQTCGQPQPCERLHICEHFTRGNCSYLNCLRSHNLMDRKVLTIMREHGLSPDVVQNIQDICNNKHAR</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/3U9G" target="_blank" style="color:#520049"><b>3U9G</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/protein/ADD71159.1" target="_blank" style="color:#520049"><b>ADD71159.1</b></a></td>
      </tr>
	  </tbody>
  </table>

<p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">21E</td>
        <td name="td1">5'-UGCGUGCGAGGAGGUGGAGAGGGUGGGUGUGCGUGGCGUG-3'</td>
        <td name="td2">Zinc-finger antiviral protein (ZAP)</td>
        <td name="td3">NA</td>
      </tr>
      <tr>
        <td name="td0">21K</td>
        <td name="td1">5'-GGGAACAGUCCGAGCCAAGCGGUAGCGUCAGGGGGUGGAGGGAGGAGGCCGCGUGGUGUGGGUGAAUUCGUCAUA-3'</td>
        <td name="td2">Zinc-finger antiviral protein (ZAP)</td>
        <td name="td3">NA</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/ZAP_aptamer_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">3U9G-A</td>
      <td name="td1">37.2</td>
      <td name="td2">0.0</td>
      <td name="td3">Zinc finger CCCH-type antiviral Protein 1</td>
    </tr>
            
     <tr>
      <td name="td0">2ZKZ-C</td>
      <td name="td1">5.5</td>
      <td name="td2">2.2</td>
      <td name="td3">Transcriptional repressor pagr</td>
    </tr>
            
     <tr>
      <td name="td0">2RDP-A</td>
      <td name="td1">5.5</td>
      <td name="td2">2.7</td>
      <td name="td3">Putative transcriptional regulator marr</td>
    </tr>
            
     <tr>
      <td name="td0">7EL3-A</td>
      <td name="td1">5.4</td>
      <td name="td2">2.8</td>
      <td name="td3">Homoprotocatechuate degradation operon regulator</td>
    </tr>
            
     <tr>
      <td name="td0">7WZE-A</td>
      <td name="td1">5.3</td>
      <td name="td2">3.1</td>
      <td name="td3">Uncharacterized HTH-type transcriptional regulato</td>
    </tr>
            
     <tr>
      <td name="td0">3S2W-A</td>
      <td name="td1">5.2</td>
      <td name="td2">2.9</td>
      <td name="td3">Transcriptional regulator, marr family</td>
    </tr>
            
     <tr>
      <td name="td0">2WTE-A</td>
      <td name="td1">5.2</td>
      <td name="td2">2.8</td>
      <td name="td3">CSA3</td>
    </tr>
            
     <tr>
      <td name="td0">3BPX-A</td>
      <td name="td1">5.0</td>
      <td name="td2">3.0</td>
      <td name="td3">Transcriptional regulator, marr family</td>
    </tr>
            
     <tr>
      <td name="td0">3VB2-B</td>
      <td name="td1">5.0</td>
      <td name="td2">2.5</td>
      <td name="td3">Multiple antibiotic resistance protein marr</td>
    </tr>
            
     <tr>
      <td name="td0">2NYX-B</td>
      <td name="td1">4.9</td>
      <td name="td2">2.9</td>
      <td name="td3">Probable transcriptional regulatory protein, RV14</td>
    </tr>
    </tbody>
  </table>
                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Analyses of SELEX-derived ZAP-binding RNA aptamers suggest that the binding specificity is determined by both structure and sequence of the RNA.</strong></font><br />
Huang, Z., Wang, X., & Gao, G.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/21203916/" target="_blank" style="color:#520049">Protein Cell, 1(8), 752-9. (2010)</a>
<br/>
